# LHD1, an Allele of DTH8/Ghd8, Controls Late Heading Date in Common Wild Rice (Oryza rufipogon)F

Xiaodong Dai, Younian Ding, Lubin Tan, Yongcai Fu, Fengxia Liu, Zuofeng Zhu, Xianyou Sun, Xuewen Sun, Ping Gu, Hongwei Cai and Chuanqing Sunâˆ—

State Key Laboratory of Plant Physiology and Biochemistry, National Evaluation Centre for Agricultural Wild Plants (Rice), Laboratory of Crop Heterosis and Utilization of the Ministry of Education, Beijing Key Laboratory of Crop Genetic Improvement, Department of Plant Genetics and Breeding, China Agricultural University, Beijing 100193, China
Corresponding author Tel/Fax: +86 10 6273 1811; E-mail: suncq@cau.edu.cn
F Articles can be viewed online without a subscription.
Available online on 11 September 2012 at http://www.jipb.net and http://www.wileyonlinelibrary.com/journal/jipb doi: 10.1111/j.1744-7909.2012.01166.x

# Abstract

Flowering at suitable time is very important for plants to adapt to complicated environments and produce their seeds successfully for reproduction. In rice (Oryza rufipogon Griff.) photoperiod regulation is one of the important factors for controlling heading date. Common wild rice, the ancestor of cultivated rice, exhibits a late heading date and a more sensitive photoperiodic response than cultivated rice. Here, through map-based cloning, we identified a major quantitative trait loci (QTL) LHD1 (Late Heading Date 1), an allele of DTH8/Ghd8, which controls the late heading date of wild rice and encodes a putative HAP3/NF-YB/CBF-A subunit of the CCAAT-box-binding transcription factor. Sequence analysis revealed that several variants in the coding region of LHD1 were correlated with a late heading date, and a further complementary study successfully rescued the phenotype. These results suggest that a functional site for LHD1 could be among those variants present in the coding region. We also found that LHD1 could down-regulate the expression of several floral transition activators such as Ehd1, Hd3a and RFT1 under long-day conditions, but not under short-day conditions. This indicates that LHD1 may delay flowering by repressing the expression of Ehd1, Hd3a and RFT1 under long-day conditions.

Keywords: Rice; LHD1; heading date; CCAAT-box-binding; transcription factor.

Dai X, Ding Y, Tan L, Fu Y, Liu F, Zhu Z, Sun X, Sun X, Gu P, Cai H, Sun C (2012) LHD1, an allele of DTH8/Ghd8, controls late heading date in common wild rice (Oryza rufipogon). J. Integr. Plant Biol. 54(10), 790â€“799.

# Introduction

The phase transition to flowering is very important in the life of flowering plants. Flowering at the proper time allows plant species to optimize their adaptation to the environment, and successfully produce their progeny (BÂ¨aurle and Dean 2006; Kobayashi and Weigel 2007). The time when plants begin flowering is regulated by a combination of endogenous and exogenous signals, and is controlled by a complex genetic network (Izawa et al. 2003; Turck et al. 2008).

Rice (Oryza rufipogon Griff.) is a short-day plant which will flower earlier under short-day conditions. The photoperiod pathway plays an important role in regulating the heading date in rice. A few critical genes are involved in photoperiodic flowering control (Yano et al. 2000; Hayama et al. 2002; Kojima et al. 2002; Doi et al. 2004; Luo et al. 2005; Xue et al. 2008; Wei et al. 2010; Yan et al. 2011). OsGI (an ortholog of GI) is an inhibitor of flowering under long-day conditions, and its expression is controlled by circadian rhythm (Hayama et al. 2002). Hd1 (Heading date 1, an ortholog of CO) has two opposite biological influences on flowering: it can promote flowering under short-day conditions, and delay flowering under long-day conditions (Yano et al. 2000). Like FT, Hd3a (Heading date 3a, an ortholog of _F T ) is known to be a strong floral promoter, and its expression is regulated by Hd1 (Kojima et al. 2002). Under long-day conditions, the transcriptional activity of Hd1is post-transcriptionally regulated by phytochromes in the presence of light, and floral transition is repressed in these circumstances. In contrast, under short-day conditions, the highest accumulation of expression of Hd1 will occur in the darkness, and then Hd3a is expressed to promote flowering (Izawa et al. 2002; Hayama and Coupland 2004; Izawa 2007). The expression of Hd1 is positively correlated with that of OsGI, but Hd3a mRNA levels will decrease when the expression of OsGI increases (Hayama et al. 2003). The OsGI-Hd1- Hd3a pathway is conserved in rice and Arabidopsis. Other photoperiod genes are also involved in the regulation of rice flowering. Ehd1 (Early heading date 1), which encodes a Btype response regulator, works as the main floral regulator and promotes flowering under both long-day and short-day conditions (Doi et al. 2004). Ghd7, which encodes a CCT domain protein, is an important heading date gene. Ghd7can delay flowering by suppressing the expression of Ehd1 and Hd3a under long-day conditions (Xue et al. 2008). DTH8/Ghd8, a major quantitative trait loci (QTL) with pleiotropic effects, can delay flowering and increase plant height and grain number per panicle in rice under long-day conditions (Wei et al. 2010; Yan et al. 2011).

Common wild rice (Oryza rufipogon Griff.), the ancestor species of cultivated rice, exhibits a late heading date and a more sensitive photoperiodic response than does cultivated rice. A major QTL on chromosome 8 controlling heading date was previously identified in the backcross population derived from Yuanjiang common wild rice (O. rufipogon, YJCWR, collected from Yunnan Province), and an elite indica cultivar Teqing (Tan et al. 2004). In this study, we used map-based cloning to isolate this heading date QTL, referred to as LHD1 (Late Heading Date 1). The LHD1 gene was found to encode a CCAAT-box-binding transcription factor, which can inhibit flowering under long-day conditions by suppressing the expression of several downstream genes for heading date in rice.

# Results

# Identification of a wild rice introgression line with late flowering

A set of introgression lines covering the whole genome of Yuanjiang common wild rice (YJCWR) was developed from crosses using a common wild rice from Yuanjiang County, Yunnan Province, China as a donor, and the Chinese indica commercial cultivar Teqing as a recipient (Tan et al. 2007). In this population, an introgression line YIL79 exhibited late flowering 750.2 Â±  1.6 Â c ) under long-day conditions ( 14 Â h light 70 ğš‘ dark), while the recipient Teqing line showed a normal flowering time ( 115.6 Â± 3.0 Â d ). Under short-day conditions 10 ğ”¥ light/14 h dark), both lines showed a similar flowering time ( 109.6 Â± 2.2 d for YIL79, 105.3 Â± 2.8 d for Teqing) (Figure 1).


Figure 1. The heading date between YIL79 and Teqing.

(A) Phenotypes of YIL79 (left) and Teqing (right). Plants were grown 115 d under long-day conditions (14 h light/10 h dark). Teqing plants flowered, whereas YIL79 plants were still at the vegetable stage. Red arrow, flowering panicle in Teqing.

(B) Days to flowering in YIL79, Teqing/YIL79 and Teqing in different photoperiodic conditions. LD, long-day conditions (14 h light/10 h dark); SD, short-day conditions (10 h light/14 h dark).

To analyze the genetic basis for the late flowering time of wild rice, we constructed an ğ–¥_2 population derived from crosses between YIL79 and Teqing. All of the ğ–¥_â†‘ progeny exhibited late flowering ( 145Â±2.8Â d) under long-day conditions, which was similar to that of the introgression line YIL79. In one ğ–¥_2 population with 1000 plants, the ratio of plants exhibiting late flowering (759) to plants exhibiting normal flowering (241) was 3:1. These results indicate that this late heading trait is controlled by a single dominant gene, referred to as Late Heading Date 1 (LHD1), and the O. rufipogon-derived allele could delay heading under long-day conditions in plants with a Teqing background.

# Map-based cloning of Late Heading Date 1 (LHD1)

To map the LHD1 gene, we first used 117 simple-sequencerepeat (SSR) markers with polymorphisms between YJCWR and Teqing to detect the introgressed chromosomal fragment of YJCWR in YIL79. 4 chromosomal segments from wild rice were detected on chromosomes 5, 8, 9 and 10 (Figure 2A).


Figure 2. Map-based cloning of the LHD1 gene in rice.

(A) Graphical genotype of introgression line YIL79. Gray bar represents homozygous chromosomal fragment of Teqing, dark bar represents homozygous chromosomal fragment of YJCWR.
(B) Genetic map of the LHD1 gene in rice; the gene was mapped between RM38 and RM126.
(C) Fine-mapping of the LHD1 gene.
(D) A Yuanjiang common wild rice (YJCWR) bacterial artificial chromosome (BAC) clone was identified to cover the fine-mapping region. Arrow, the construct of pLHD1.
(E) Phenotypes of ZH17 (Zhonghua17 harboring an empty plasmid) (left) and Transgenic-positive ğ–³_2 plant (right).
(F) Days to flowering of three independent ğ–³_2 lines homozygous for the LHD1 and ZH17 under LD, long-day (14 h light/10 h dark) and SD, short-day (10 h light/14 h dark) conditions.


Figure 2. Continued

The SSRs on chromosome 8 were shown to have a high linkage with the phenotype. Genetic linkage analysis of 200 ğ–¥_2 individuals showed that the LHD1 gene is located between SSR markers RM38 and RM126 on the short arm of chromosome 8 (Figure 2B).

To fine-map the gene, 1  900 ğ–¥_2 plants were used, and LHD1 was mapped between two insertion-deletion (InDel) markers M4560 and M5164136 (Figure 2C). 6 new markers in this region were then developed, and the location of LHD1 was further narrowed down to within a 25-ğ—„ ğ–» genomic region between the M516470 and M516495 markers (Figure 2C). There are three putative genes in this region: two transferase family proteins (LOC_Os08g07720, LOC_Os08g07730) and a CCAAT-box-binding transcription factor (LOC_Os08g07740). The CCAAT-box-binding transcription factor had been reported to play an important role in flowering in plants. Therefore, LOC_Os08g07740 was selected as the candidate gene.

To confirm whether or not the CCAAT-box-binding transcription factor (LOC_Os08g07740) is in fact the target gene, we first identified a bacterial artificial chromosome (BAC) clone (YJ09140504) from a genomic BAC library of YJCWR. We then introduced the construct (pLHD1) containing the entire coding sequence (CDS) of the CCAAT-box-binding transcription factor of YJCWR and the 1461-bp promoter region into japonica cultivar Zhonghua 17 (Figure 2D). All 21 independently-generated transgenic lines showed a late flowering time (120.5Â±1.5Â d) ,while the control plants showed a normal flowering time ?08Â±1.6Â d ) (Figure 2E, F). These results suggest that ğ–« ğ–® ğ–¢_- Os8g07740 corresponding to LHD1 controls the late flowering time in wild rice.

# Functional analysis of LHD1 in rice

LHD1 has a 903-bp coding sequence without introns, and encodes a protein of 300 amino acids. The CCAAT-box domain was located from the position of 57 to 152. Comparing the coding sequence between YIL79 and Teqing, we identified 7 single nucleotide polymorphisms (SNPs) and a 6-bp InDel, and these DNA changes can result in a 5 amino acid substitution and a 2-glycine InDel (Figure 3A).

Expression analysis with RT-PCR showed that LHD1 transcripts were abundant in leaf blades, but were less abundant in roots, culms, nodes and leaf sheaths, and were undetectable in panicles (Figure 3B). To further confirm these findings, we did transformation with a construct which contained a fusion GUS reporter gene driven by the1539-bp promoter of the LHD1 gene. The expression pattern in the transgenic plants by GUS staining showed that expression could only be detected in leaf blades, which is consistent with the RT- PCR results (data not shown).

To identify the subcellular localization of the LHD1 protein, a fusion protein of LHD1-GFP under the control of the cauliflower mosaic virus (CaMV) 35S promoter was made and introduced into onion epidermal cells. Transiently-expressed LHD1-GFP demonstrated that LHD1 protein accumulated in the nucleus (Figure 3C), suggesting that LHD1 could be a transcription factor.

# LHD1 delayed flowering by down-regulating the expression of Ehd1, Hd3a and RFT1

LHD1 is a key gene controlling heading date, which has been confirmed by a series of genetic evidence. To elaborate the role of LHD1in the rice flowering pathway, we examined the expression levels of LHD1 and other heading date genes (OsGI, OsMADS50, OsMADS51, Hd1, Ehd1, Hd3a, RFT1) in both YIL79 and Teqing under long-day and short-day conditions, Figure 3. Functional analysis of the LHD1 gene in rice.



(A) Comparison of predicted amino acids between YIL79 and Teqing. Red bar, CCAAT-box domain.
(B) Expression analysis of various tissues using semi-quantitative RT-PCR.
(Câ€“H) Subcellular localization of the LHD1 protein.
(C) and (F) Bright-field images of onion epidermal cells.
(D) Nuclear localization of the Lhd1-GFP fusion protein.
(E) The merged image of (C) and (D).
(G) Onion epidermal cells bombarded with the construct having GFP alone as the control.
(H) The merged image of (F) and (G).

respectively. We found that no difference in the LHD1 expression levels could be detected between YIL79 and Teqing under both long-day and short-day conditions, respectively, suggesting that late flowering did not result from the LHD1 transcript. For OsGI, OsMADS50, OsMADS51, Hd1, no difference could be detected under long-day and short-day conditions between YIL79 and Teqing. The expression levels of Ehd1, Hd3a and RFT1 were less in YIL79 than those in Teqing under long-day conditions, but not under short-day conditions, indicating that LHD1 may suppress the expressions of these floral transition activators (Figure 4).

# Sequence analysis of LHD1 in cultivated rice

To determine the functional site of LHD1, we carried out a large sequence analysis focused on the CDS of LHD1 across 73 rice accessions, including one accession of wild rice, 30 indica cultivars, and 42 japonica cultivars (Table S1). We found that the sequences of 72 cultivars could be divided into 7 types.

The cultivars with Type 5 contained a SNP at site 323, which can result in a gene frameshift. The cultivars containing Type 7 had a 19-bp deletion and one SNP deletion at sites 93 and 96, and these deletions can also cause frameshifts (Figure 5). Although the functional site remained unknown in the rest of the rice cultivars, the SNP (C/G) at position +475 , which causes a single substitution from Proline (P) to Alanine (A) residue, turned to be the best functional candidate site for these cultivars (Figure 5).

# Discussion

Asian rice was domesticated from common wild rice thousands of years ago, and the genetic diversity of modern cultivated rice was greatly decreased and many valuable genes have been lost. Common wild rice can be used as an important genetic pool, and can provide valuable genes and widen the genetic diversity for modern cultivated rice. In order to employ the


Figure 4. Analysis on transcript of 8 genes for flowering time in rice.

Expression of LHD1, OsGI, OsMADS50, OsMADS51, Hd1, Ehd1, Hd3a, RFT1 in YIL79 (black pillars) and Teqing (white pillars) under long-day ( â†‘4Â h light/10 h dark) and short-day ( â†‘0 ğ— light/14 h dark) conditions. These experiments were repeated at three times.

Figure 5. Large sequence analysis of the LHDs gene in rice.



The relationship between different types of LHD1 and heading date under long-day conditions. Sequence variations are represented by different colors. The number of cultivars and the mean value of HD (heading date) under long-day conditions with each type of sequence are shown in the column at the right.

valuable genes of common wild rice, advanced backcross and subsequent molecular marker-associated selection have been shown to be an efficient strategy. The introgressed library from the donor genome of common wild rice can not only eliminate the influence of unfavorable genes for gene mapping, but can also divide complicated traits into several single Mendelian factors, and these genes can then be cloned through mapbased cloning, oneâ€“by-one. Previous research has provided sufficient and vivid proof to support this viewpoint. For example, Tan et al. (2008) isolated a key gene, PROG1, controlling plant architecture, Yu et al. (2007) demonstrated that tiller angle is controlled by a major quantitative trait locus, TAC1, Lin et al. (2007) elucidated that seed shattering is controlled by a single dominant gene, SHA1, encoding a member of the trihelix family of plant-specific transcription factors, and Xue et al. (2008) identified the quantitative trait locus Ghd7, which plays a crucial role in increasing productivity and adaptability of rice.

LHD1, a strong floral transition inhibitor, can delay rice flowering under long-day conditions. In this study, we found that there was no significant difference of expression between YIL79 and Teqing under long-day and short-day conditions, so we speculated that the reason for the change of heading date was not caused by LHD1 expression, but by nucleotide changes of the coding sequence. In the coding region, there is considerable variation between YIL79 and Teqing, including 7 SNPs and a 6-bp InDel (Figure 5). Among these variants, no variant except the ğ–¢/ğ–¦ substitution present in the position of 475-bp after the start codon of LHD1 could be the functional site (Figure 5), but further research is needed to prove this.

LHD1 may work as a negative regulator for flowering. In this study, we found that the transcripts for several flowering promoting factors, such as Ehd1, Hd3a and RFT1, were fewer in YIL79 than in Teqing under long-day conditions, and the expression levels were the same under short-day conditions (Figure 4). These observations indicated that expression of these floral transition activators was suppressed in YIL79 under long-day condition, which resulted in YIL79 exhibiting later flowering than Teqing under long-day conditions. Previous research has shown that Ehd1 acts upstream of Hd3a and RFT1 and promotes their expression, so LHD1 may have two ways to regulate their expression: directly regulate Hd3a and RFT1, or indirectly govern the two genes through Ehd1 (Doi et al. 2004), then activate the flowering in rice (Figure 6). The expressions levels of other genes (OsGI, OsMADS50, OsMADS51, Hd1) in YIL79 and Teqing showed that they might function upstream of LHD1 or parallel to LHD1. In addition, CONSTANS (CO) and the CCAAT-box-binding complex share the same important functional domain, and CO might replace AtHAP2 in the HAP complex to form a trimeric CO/AtHAP3/AtHAP5 complex in Arabidopsis. Over-expression of AtHAP2 or AtHAP3 could delay flowering by impairing formation of a CO/AtHAP3/AtHAP5 complex, leading to decreased expression of FT (Wenkel et al. 2006). Furthermore, Hap3b, a putative CCAAT-binding transcription factor gene, promoted flowering under long-day conditions, but not under short-day conditions (Cai et al. 2007). It conferred influence on flowering under long-day condition, which was similar to LHD1. Hap3b promoted flowering by interacting with CO or COL proteins. As a homologue of CO, Hd1 delayed flowering under long-day conditions and promoted flowering under short-day conditions. The nature of interaction with the CCAAT-Box-Binding factor is still unknown in rice. In this study, we examined expression of Hd1 in YIL79 and Teqing, and found no difference between YIL79 and Teqing under long-day and short-day conditions, respectively. Thus, we propose that Hd1 might function upstream of LHD1, similar to Hap3b.


Figure 6. Schematic representation of the genetic pathway controlling flowering time in which LHD1 was involved under long-day conditions.

Recently, a major QTL DTH8/Ghd8, another allele of LHD1, which can delay flowering (making it later by about 10âˆ¼12 d), and can increase plant height and grain number per panicle under long-day conditions, was cloned by map-based cloning using a cultivated rice population (Wei et al. 2010; Yan et al. 2011). Two mutations of DTH8/Ghd8 cause a frameshift mutation and premature termination of translation that result in the loss of protein function. However, LHD1 only delays the heading date (about 34 days) under long-day conditions, but does not increase plant height and grain number per panicle (Table S2). There are 7 SNPs and a 6-bp that result in 5 amino acid substitution and a 2-glycine InDel in the lhd1 coding sequence of Teqing. These results indicate that there are different genetic effects of LHD1 and DTH8/Ghd8 in wild and cultivated rice. Consecutive selection for LHD1 occurred in rice domestication in order to adapt for the change of photoperiodic response.

# Materials and Methods

# Plant materials and growth conditions

Yuanjiang common wild rice (YJCWR, Oryza rufipogon Griff.), a common wild rice accession collected as rhizomes from Yunnan Province, China, and Teqing, an elite indica cultivar (O. sativa L.), were used as donor and recurrrent parents in the process of construction of a set of introgression lines, respectively. YIL79 is one introgression line displaying a late heading date. ğ–¥_â†‘ plants were derived from a cross between YIL79 and Teqing. The population (ğ–¥_2) used for high-resolution mapping was self-pollinated progenies of ğ–¥_â†‘ .

YIL79, Teqing, and the map population were grown in Beijing (day length > 14 ğ”¥_ğ”¬ . Transgenic plants and the material used for analyzing the expression of LHD1 and other heading date genes were grown under long-day conditions ( ^14Â h light/ â†­ dark) and short-day conditions ( 10Â ğ— light/14 h dark), respectively.

# DNA extraction and genotype analysis

Fresh leaves were collected at the seedling stage and then ground in liquid nitrogen. DNA was then extracted by the standard cetyltrimethylammonium bromide (CTAB) method (Rogers and Bendich 1988). A total volume of 20Â L reaction mixture was composed of 1 ğ—‡ ğ—€/ ğ–« template DNA, 2L of 10Ã— buffer (0.1 ğ—† ğ—ˆ ğ—…/ğ–« Tris-HCl (ğ—‰ ğ–§ 9.0) , 0.5 mol/L KCl, 7.5 ğ—† ğ—† ğ—ˆ ğ—…/ğ–« ğ–¬ ğ—€ ğ–¢ ğ—…_2 ,0.1% Triton Ã—-100_. ), 2 mol of each primer, 2.5 ğ—† ğ—† ğ—ˆ ğ—…/ğ–« each of deoxynucleotide triphosphates (dNTP) and 1 unit of Taq DNA polymerase (Promega). Amplification was carried out on the program for the initial denaturing step with 94Â ^âˆ˜C for 5 min, followed by 35 cycles for 1 min at 94Â ^âˆ˜C , 1 min at 55 ^âˆ˜C ,2 min at 72 ^âˆ˜C , with a final extension at 72 ^âˆ˜C for 10 ğ—† ğ—‚ ğ—‡ .

# Fine-mapping of LHD1

We used the ğ–¥_2 population, derived from the backcross between Teqing and YIL79, to map the LHD1 gene. Firstly, we select 200 ğ–¥_2 individuals to do preliminary genetic mapping of the LHD1 locus, and LHD1 was then mapped between SSR markers RM38 and RM126. An additional 1 900 ğ–¥_2 plants were used to fine-map the LHD1 gene. Two insertion-deletion (InDel) markers M4560 and M5164136 flanking LHD1 identified nine and two recombinant plants, respectively. Finally, six new polymorphic markers in this region were developed, and LHD1 was subsequently narrowed down to within a 25-kb genomic region between the M516470 and M516495 markers. The molecular markers used for fine-mapping LHD1 are shown in Table S3.

# Vector construction and transformation

For complementation tests, a positive bacterial artificial chromosome (BAC) clone including LHD1 was isolated from the genomic BAC library of YJCWR. A 3688-bp fragment including a 1461-bp upstream sequence, the putative coding region, and a 1324-bp downstream sequence digested by HindIII was inserted into the binary vector pCAMBIA1300. The construct p35S::LHD1-GFP containing LHD1 fused with GFP was driven by an CaMV 35S promoter. The construct p35S::LHD1- GFP was transformed into onion epidermal cells by particle bombardment, and other plasmid constructs were transferred into japonica cultivar Zhonghua 17 via particle bombardment transformation.

# Semi-quantitative RT-PCR and quantitative RT-PCR analysis of gene expression

We extracted the total RNA from leaves using an RNeasy Plant Mini Kit (Qiagene). First-stand cDNA synthesis was done from 2Â g of total RNA, using the BcaBEST RNA PCR Kit (TaKaRa). We carried out Semi-quantitative RT-PCR to amplify the LHD1 transcripts with the primers sqLhd1F and sqLhd1R. Quantitative RT-PCR was done using the ABI Prism 7900 Sequence Detection System (Applied Biosystems). cDNA corresponding to 5 ng of total RNA was used as the template and was amplified with primers qLhd1F and qLhd1R using the SYBR Green Master Mix (Applied Biosystems Incorporation). We normalized the levels of LHD1 transcripts by endogenous 18S rRNA transcripts amplified with primers 18SF and 18SR. Each set of experiments was repeated three times, and the relative quantification method (ddCt) was used to evaluate quantitative variation. The quantitative real-time PCR procedure was conducted at 94Â ^âˆ˜C for 3 min, followed by 40 cycles at 94Â ^âˆ˜C for ^30 s ,58 ^âˆ˜C for ^30 âˆ˜ , and 72 Î©^âˆ˜C for _30 ğšœ .

# cknowledgements

This work was supported by the Special Fund for Agro-scientific Research in the Public Interest (201003021), the Project of Conservation and Utilization of Agricultural Wild Plants of the Ministry of Agriculture of China, and the National HighTech Research and Development (863) Program of China (2012AA101103).

Received 10 Jul. 2012 Accepted 3 Sept. 2012
