# Natural variations in grain length 10 (GL10) regulate rice grain size

Article¬†¬†in¬†¬†Journal of Genetics and Genomics ¬∑ February 2022
DOI: 10.1016/j.jgg.2022.01.008

# READS 236

# 20 authors, including:

Fangping Li South China Agricultural University 23 PUBLICATIONS¬†¬†¬†286 CITATIONS

Zhe Ji
University of Oxford
7 PUBLICATIONS¬†¬†¬†133 CITATIONS SEE PROFILE

SEE PROFILE

Bu Suhong
South China Agricultural University
33 PUBLICATIONS¬†¬†¬†296 CITATIONS SEE PROFILE
Guiquan Zhang
South China Agricultural University
104 PUBLICATIONS¬†¬†¬†1,909 CITATIONS SEE PROFILE

Original research

# Natural variations in grain length 10 (GL10) regulate rice grain size

Penglin Zhan a, b, Shuaipeng Ma b, Zhili Xiao b, Fangping Li a, b, Xin Wei b, Shaojun Lin b, Xiaoling Wang b, Zhe Ji c, Yu Fu a, b, Jiahao Pan a, b, Mi Zhou b, Yue Liu b, Zengyuan Chang b, Lu Li b, Suhong Bu b, Zupei Liu a, b, Haitao Zhu a, b, Guifu Liu a, b, Guiquan Zhang a, b, Shaokui Wang a, b,

aGuangdong Laboratory for Lingnan Modern Agriculture, Guangzhou, Guangdong 510642, China
b State Key Laboratory for Conservation and Utilization of Subtropical Agro-Bioresources, Guangdong Provincial Key Laboratory of Plant Molecular
Breeding, South China Agricultural University, Guangzhou, Guangdong 510642, China
c Department of Plant Sciences, University of Oxford, Oxford OX1 3RB, UK

# A R T I C L E I N F O

# A B S T R A C T

Article history:
Received 29 October 2021
Received in revised form
23 January 2022
Accepted 24 January 2022
Available online 10 February 2022
Keywords:
Rice
GL10
OsMADS56
Grain length
SSSL
GA

Grain size is an important determinant of grain weight and yield in rice. Although several genes related to grain size have been identified, natural variations in these genes that affect grain size are poorly characterized. Here, we describe the grain length QTL GL10, encoding MADS56, which positively regulates grain length and grain weight. A natural allelic variation of NIL-gl10, containing an ‚àº1.0¬≠ùóÑ ùñª deletion in the first exon that abolishes its transcription, results in shorter grain length, lower grain weight and delayed flowering in gl10 plants. The knockout of _G L 70 in the HJX74 background leads to grain phenotypes similar to that of NIL-gl10, while overexpression of _G L 70 results in increased grain length and weight and earlier heading date. G L 70 regulates grain length by promoting greater longitudinal cell growth in the grain glume. Additionally, GL10 participates in the regulation of gibberellic acid (GA) signaling pathway genes in young panicle tissues. Analysis of _G L 70 haplotypes shows obvious divergence between the japonica and indica lineages. Our findings reveal an allelic variation of G L 7O that may explain differences in grain length among modern cultivars and could be used to breed rice varieties with optimized grain shape.

Copyright  2022, Institute of Genetics and Developmental Biology, Chinese Academy of Sciences, and Genetics Society of China. Published by Elsevier Limited and Science Press. All rights reserved.

# Introduction

Rice (Oryza sativa L.) is consumed as a staple crop by at least half of the world‚Äôs population. Grain length and width determine rice grain size and therefore affect yield and appearance, thus representing two of the most agronomically important traits in rice breeding (Li and Li, 2016). Many quantitative trait loci (QTLs) attributable to natural variations have been identified and functionally characterized for their role in controlling grain size in rice, including GRAIN SIZE 3 (GS3), GRAIN WIDTH 2 (GW2), GS2, THOUSAND-GRAIN WEIGHT 3 (TGW3), TGW6, GW8, GW7/GL7, GRAIN LENGTH AND WIDTH7 (GLW7), GS5, GW5, GL6, GW6, TGW2, GL4, GL3.1, and GS9 (Fan et al., 2006; Song et al., 2007; Mao et al., 2010; Li et al., 2011; Qi et al., 2012; Wang et al., 2012, 2015, 2019; Ishimaru et al., 2013; Hu et al., 2015; Si et al., 2016; Liu et al., 2017; Wu et al., 2017; Hu et al., 2018b; Zhao et al., 2018; Ruan et al., 2020; Shi et al., 2020). Most of these genes were found participating in regulatory signaling pathways, such as the ubiquitin proteasome pathway, G-protein signaling, phytohormone signaling, and transcriptional regulation (Zuo and Li, 2014; Li et al., 2019). For example, GW2, which encodes a predicted RING-type protein with E3 ubiquitin ligase activity, promotes cell proliferation and larger size of spikelet hull through the ubiquitin-proteasome pathway (Song et al., 2007). GS3 is a major gene regulating grain length and grain weight through G-protein signaling (Fan et al., 2006; Mao et al., 2010). TGW6 encodes a novel protein with indole-3-acetic acid (IAA)-glucose hydrolase activity that positively regulates grain yield by acting on the source organs (Ishimaru et al., 2013). TGW3 protein interacts with AUXIN RESPONSE FACTOR 4 (OsARF4) to regulate grain length and weight (Hu et al., 2018b). GL3.1/qGL3 controls grain length and yield through interactions with OsGSK3 that modulate brassinosteroid (BR) signaling (Qi et al., 2012; Gao et al., 2019). GW5 is a positive regulator of BR signaling and improves grain yield in rice (Liu et al., 2017). The protein encoded by GS9 interacts with OsOFP14 and OsOFP8 in the regulation of grain morphology and serves a regulatory target of OsGSK2 kinase (Zhao et al., 2018). GW6 encodes a GA-regulated GAST family protein that positively influences grain width (Shi et al., 2020). GW8 encodes the OsSPL16 transcription factor, which controls grain width via transcriptional regulation of GW7 (Wang et al., 2012, Wang et al., 2015). Similarly, GLW7, GS2, and GL6 are all transcriptional regulators that participate in determining grain size (Hu et al., 2015; Si et al., 2016; Wang et al., 2019). Other natural variation-based QTLs related to grain size still require characterization.

Among the gene families that participate in grain development, MADS-box family genes contribute to essential regulatory activities in controlling plant flower architecture, flowering induction, and vegetative development (Dai et al., 2016). Several MADS-box genes that regulate flower morphology also help control seed size. Notably, OsMADS1, encoding MADS-box factor 1, is associated with grain length regulation, and was domesticated through forward genetics crosses and breeding selection (Yu et al., 2018). OsMADS1 acts as a key downstream effector of G-protein bg dimers involved in grain size regulation (Liu et al., 2018). OsMADS34 and OsMADS55 are two downstream target genes of OsMADS1 that control spikelet meristem and organ differentiation (Dai et al., 2016; Khanday et al., 2016). OsMADS6 performs essential functions in endosperm nutrient accumulation, and consequently influences starch filling and seed shape in rice (Zhang et al., 2010). OsMADS34 participates in the determination of grain size and panicle shape in rice. The osmads34-t mutant exhibits smaller grain formation on its secondary branches of panicles as well as a lower seed-setting rate (Zhang et al., 2016). OsMADS29 participates in embryonic development and starch filling in endosperm cells, revealed by the smaller grain size, and abnormal embryo and endosperm cells in plants with RNA interference (RNAi)- based suppression of this gene (Nayar et al., 2013). These findings collectively support the role of MADS family genes in regulating flower morphology and seed development.

Gibberellin acids (GAs) are a class of hormones that function as primary signal molecules in a diverse array of developmental processes and signal pathways. In particular, GAs are well established to be involved in regulation of seed germination, stem elongation, leaf expansion, trichome development, pollen maturation, and the induction of flowering in rice (Daviere and Achard, 2013). Some examples of the numerous genes that participate in GA biosynthesis and signal transduction related to rice seed development, flowering, plant defense and plant development include OsCPS1 (Otomo et al., 2004), OsKS1 (Margis et al., 2005), GA20ox1 (Oikawa et al., 2004), GA20ox2 (Ye et al., 2015), GA20ox3 (Sakata et al., 2014), GA3ox1 (Itoh et al., 2001), GA3ox2 (Tong et al., 2014), SLR1 (Hirano et al., 2012), AP2 (Yaish et al., 2010), WRKY71 (Xie et al., 2006), OsEUI1 (Zhang et al., 2008), OsGRF1 (Luo et al., 2005) and GW6 (Shi et al., 2020). Furthermore, the QTL GNP1 encodes GA20ox1, which increases grain number and yield by promoting cytokinin activity and influencing GA1 and GA3 content in rice (Wu et al., 2016). OsGASR9 is a positive regulator of the GA response pathway, which modulates grain size and yield in rice (Li et al., 2019). In addition, GW6, which is in the GAST family and regulates grain width and weight, functions in the GA response pathway and participates in regulation of GA biosynthesis (Shi et al., 2020).

Here, we report the identification and characterization of GL10, a MADS-box transcription factor that regulates grain size in rice. gl10, an allelic variant of G L 7O , was characterized containing a 1019 bp deletion that results in the loss of its transcriptional regulatory activity. Consequently, gl10-bearing plants exhibit shorter cells in glumes, which ultimately reduces grain length. The study expands the scope of our understanding regarding the role of genetic factors and their accompanying mechanisms in determining grain shape in rice, and thus provides a resource for targeted breeding of specific varieties.

# Results

# Identification of qGL10 regulating grain length and grain weight

We have developed a single segment substitution line (SSSL) library over the past 30 years using Huajingxian74 (HJX74, an elite indica rice cultivated in south China) as a recipient parent, with each SSSL carrying only one substitution segment when detected by SSR markers (Zhang 2019, 2021). By screening for shorter grain length, we identified an SSSL carrying a 994.8‚ÄìùóÑ ùñª segment donated from chromosome 10 of the Lemont variety (japonica in the United States), referred to as SSSL-gl10. We then backcrossed SSSL-gl10 with HJX74 as a recurrent parent to obtain a ùñ° ùñ¢_5ùñ•_2 generation. We identified a near-isogenic line (NIL) containing the Lemont allele within the HJX74 background that had a 4ÃÉ50 kb substitution segment ranging from marker M15 to marker PSM170 donated by Lemont in this population, which we designated NIL-gl10. The HJX74 parental line was correspondingly designated as NIL-GL10. Phenotypic comparisons of NIL-GL10 and NIL-gl10 revealed significant differences in grain length, grain width, ratio of grain length to grain width, 1000-grain weight, and heading date (Figs.1Ae1E, S1A, S1B). Other quantitative traits, including plant height, panicle length, effective tiller number, and number of grains per plant were not significantly different between the two lines (Fig. S1CeS1F). We also noted that, within the ùñ° ùñ¢_5ùñ•_2 population, grain length differed significantly among the three genotypes of progeny, with the heterozygote phenotype intermediate between that of the two homozygotes (Fig. S2). These results indicate that a semi-dominant G L 7O allele from NIL-gl10 was likely responsible for the reduced grain length and grain weight.

# Map-based cloning of GL10

Initially, we mapped the GL10 locus to an interval between marker M15 and marker NP-11 using a population of 1200 plants derived from crossing NIL-GL10 and NIL-gl10 (Fig. 1F). Then, we mapped the G L 7O to a 18.5¬†ùóÑ ùñª region between marker NC9 and marker NC14 in the ùñ° ùñ¢_5ùñ•_3 generation, which included a total of 4000 plants (Fig. 1G). This region contained one predicted ORF, Os10g0536100, encoding a MADS-box family protein as described in the rice annotation project (https://rapdb.dna.affrc.go.jp/). We then generated a GL10-GFP fusion reporter, which localized to the nucleus detected using NIL-GL10 protoplasts (Fig. S3A). RT-qPCR analysis revealed that GL10 was strongly transcribed in root, stem, leaf, and panicle tissues, with 1e3 cm-long young panicles showing higher GL10 transcription levels than panicles of other lengths (Fig. S3B). Comparison of the GL10 genomic sequence from NILGL10 and NIL-gl10 yielded 84 variations between the two lines (Table. S1). Specifically, one 1019-bp deletion was identified in the 5^' untranslated region ^ UTR) and the first exon region, and a total of 6 variations were located in the third, fourth, and fifth exons of the g I10 allele (Fig. 1H). Interestingly, this 1019-bp deletion resulted in the absence of detectable gl10 expression in several tissues, as determined by RT-qPCR assays, whereas GL10 was abundantly transcribed in all tissues (Figs. 1I and S3B). These results suggest that this deletion could abolish the function of GL10, thus affecting grain length in rice.

# GL10 positively regulates grain length, grain weight and heading date

To verify the function of G L 7O as a grain length regulator, we developed a series of mutant G L 7O transgenic plants through CRISPR/Cas9-based genome editing of NIL-GL10 (HJX74). The two resulting knockout lines (KO-GL10) harbored a frameshift mutation that caused premature termination of translation (Fig. S4A and S4B). These KO-GL10 lines exhibited significantly decreased grain length, grain width, and 1000-grain weight compared to those of the HJX74 wild-type line (Fig. 2A, 2C, 2D, 2E). In addition, heading date differed significantly between KO-GL10 and HJX74 (Fig. S4C), although we found no detectable differences in plant height, effective tiller numbers, or grain number (Fig. S4DeS4F). We also generated a genetic complementation construct for GL10 overexpression in the NIL-gl10 background via transformation with a p35S-GL10 vector (Fig. S5A and S5B). The GL10 overexpression lines exhibited significantly increased grain length and grain weight compared to that of NIL-gl10 (Fig. 2B, 2F, 2G, 2H). Moreover, OE-GL10 grain phenotypes were distinct from that of KO-GL10 plants, with grain length and heading date positively correlating with G L 7O expression level. Notably, the heading stage of OE-GL10 was earlier than that of NIL-gl10 (Fig. S5C). In addition, we found no significant differences between OE-GL10 and NIL-gl10 in grain width, plant height, effective tiller numbers, and grain number per plant (Fig. S5DeS5F). These results indicate that GL10 was a positive regulator of grain length, grain weight and heading date.


Fig. 1. Map-based cloning of GL10. A: The morphology of mature grains from NIL-GL10 and NIL-gl10 plants. BeE: Comparisons of grain length (B), grain width (C), ratio of grain length to width (D) and 1000-grain weight (E) (n‚â•100) . Data are shown as means ¬±‚â§D . F: G L 7O was preliminary mapped between markers PSM344 and PSM710 on chromosome 10 using 1200 ùñ° ùñ¢_5ùñ•_2 individuals. G: GL10 was narrowed down to an 18.5-kb genomic DNA region between markers NC9 and NC14 using 4000 ùñ° ùñ¢_5ùñ•_3 individuals. The numbers below the bar in panel indicate the number of recombinant lines between every two adjacent markers. The genotype and grain length of the recombinant lines (L1eL4) and the NILs are also shown. The black and white bars represent chromosomal segments homozygous for NIL-gl10 and NIL-GL10 (HJX74) alleles, respectively. H: The structure and allelic variation of the candidate gene Os10g0536100 (GL10), the only predicted open reading frame in the 18.5¬≠ k b delimited region. I: Semi-quantitative analysis of G L 70 in NIL-GL10 and NIL-gl10 plants. SD, standard deviation; YP, young panicles. Scale bar, 1 cm (A).

# GL10 regulates grain length by promoting cell elongation

Grain size in rice is primarily restricted by the size of the spikelet hulls, which is determined by cell proliferation and cell expansion (Shi et al., 2020). We therefore conducted scanning electron microscopy (SEM) assays to quantify the epidermal cell size of GL10 and g IŒ∑ O spikelet hulls in mature grains. The epidermal cell length of the spikelet hulls was significantly longer in NIL-GL10 plants than in NILg IŒ∑ O plants, although the numbers of cells in the spikelet hulls were similar (Fig. 3Ae3D). In addition, the SEM assays showed that HJX74 plants exhibited greater longitudinal grain length than plants from either KO-GL10 line (Fig. S6A, S6C, S6D), while the grain length of the ùñ™ ùñÆ-G L 10 plants was similar to that of NIL-gl10 plants. We also observed that the epidermal cell sizes in spikelet hulls of the OEGL10 plants were significantly greater than in those of NIL-gl10 plants (Fig. S6B, S6E, S6F). These results indicate that GL10 controls grain length by positively influencing the length of epidermal cells in the spikelet hull.


Fig. 2. Confirmation of the role of G L 7O in promoting grain length in rice. A and B: Mature grains of CRISPR/Cas9-knockout GL10 lines (KO-GL10) (A) and GL10 overexpression transgenic lines (ùñÆ ùñ§-ùñ¶L 10) (B). CeE: Comparisons of grain length (C), grain width ( D) and 1000-grain width (E) (n‚â•100) of HJX74 carrying CRISPR/Cas9-edited GL10 (NIL-GL10). Duncan‚Äôs multiple range test, a=0.01 . FeH: Comparisons of grain length (F), grain width (G) and 1000-grain width (H) (n‚â•100) of G L Œ∑0 overexpression lines in NIL-gl10 background. Duncan‚Äôs multiple range test, Œ±=0.01 . Students t test, ^** ,P<0.01 . Scale bar, 5 ùóÜ ùóÜ (ùñ†) .


Fig. 3. Histological comparisons of the spikelet hulls between NIL- .G L 70 and NIL-gl10 plants. A and B: Scanning electron micrographs of the outer surfaces of the mature glumes of NILGL10 (A) and NIL-gl10 (B). C: Number of epidermal cells in the longitudinal direction in the red boxes in (A) and (B). D: Longitudinal length of the epidermal cells in NIL-GL10 and NIL-gl10 plants (n=5) . Data are shown as means ¬±‚â§D , Student‚Äôs t-test. Scale bar, 100¬†m (A and B).

# GL10 is involved in the GA signaling pathway

To explore the regulatory mechanism by which G L 7O affects grain length, we carried out transcriptome sequencing (RNA-seq) analysis of young panicles from NIL- .G L 7O and NIL-gl10 plants to determine which genes are regulated by GL10 as well as their corresponding regulatory networks. The results revealed 554 up-regulated and 447 down-regulated differentially expressed genes (DEGs) (Fig. S7A). KEGG enrichment analysis showed that DEGs were enriched in pathways related to plant hormone signal transduction, such as the GA, ethylene, ABA and IAA pathways (Figs. S7B and S8). To determine whether G L 7O participated in GA biosynthesis and signal transduction, we examined the expression levels of genes known to participate in GA biosynthesis and signal transduction in young panicles of NIL-GL10 and NIL-gl10 plants. The results showed that GA biosynthetic genes, such as OsCPS1, OsKS1, GA20ox1, G A200√ó2 , GA20ox3, GA3ox1, and G A30x2 , were significantly downregulated in NIL-gl10 plants. The only exception was SLR1, which was not differentially expressed (Fig. 4A). Similarly, GA signal transduction genes, such as AP2, WRKY71, OsEUI1, OsGRF1, and GW6, were also significantly down-regulated in NIL-gl10 plants (Fig. 4B).

We also examined seedling sensitivity to exogenous ùñ¶ ùñ†_3 and paclobutrazol (PBZ) in both NIL-GL10 and NIL-gl10 plants by measuring seedling height and sheath length of the second leaf. The seedling height and second leaf sheath length of the NIL-GL10 plants were significantly greater under treatment with high concentrations of ùñ¶ ùñ†_3 (Fig. 4Ce4E) than in the control plants which not be treated with ùñ¶ ùñ†_3 , whereas seedling height and second leaf sheath length followed a decreasing trend with increasing concentrations of PBZ. Notably, seedling heights were significantly different between NILG L 7O and NIL-gl10 plants treated with 0 or 1 mol PBZ (Fig. 4Fe4H). Further comparison of seedling height and sheath length between ùñ™ ùñÆ-G L 10 and NIL- .G L 7O plants treated with ùñ¶ ùñ†_3 and PBZ showed that NIL- .G L 7O plants were taller and had longer sheaths than the KOGL10 plants under both treatment conditions (Fig. S9AeS9F). These results imply that G L 7O is likely to be involved in GA biosynthesis and signaling transduction related to the regulation of grain size.

# Natural variations in the GL10 gene

To investigate the natural variation in the GL10 gene, 68 Oryza accessions with abundant genetic diversity were selected according to sequenced Oryza genome information (Stein et al., 2018; Zhao et al., 2018; Xie et al., 2021). We detected 14G L10 haplotypes by CDS alignment among the 68 Oryza accessions and categorized the structural variations as either SV type 1 or SV type 2 (Fig. 5A; Table S2). The haplotype of most indica and rufipogon wild rice at the GL10 locus belongs to Hap5 of the SV type 1 group, whereas most japonica and some indica landraces were distributed among Hap7, Hap11 and Hap13, which belong to SV type 2 (Fig. 5; Table S3). Additionally, the 1019-bp deletion of the gl10 in the first exon, leading to a complete change in the transcript, results in categorization of this variation as SV type 2 (Fig. 5B). To identify this allelic SV, we designed an InDel marker specific to this 1019-bp deletion for practical genotyping of different rice lines. In particular, the shorter DNA bands were detected in japonica by agarose gel electrophoresis assays due to the deletion, whereas the indica rice and AA genome of wild oryza species produced larger DNA fragments (Fig. S10A). In tests of this InDel marker, we successfully screened out the SSSL-gl10IRAT261 line from our SSSL library, which taken IRAT261 (a japonica from Nigeria with the g IŒ∑ O allele) as donor parent and also exhibited shorter grain length than the NIL-GL10 plants (Fig. S10B). Taken together, these results suggest that the G L 7O structural variation resulted in obvious differences between japonica and indica, and some G L 7O variants from indica landrace varieties had been introgressed into the japonica genome.


Fig. 4. GL10 is involved in gibberellin (GA) signaling regulation. A: Relative expression of GA biosynthesis genes in young panicles of NIL- .G L 70 and NIL-gl10 plants. B: Relative expression of GA signaling transport genes in young panicles of NIL-GL10 and NIL-gl10 plants (n=3) . CeE: Seedling height and second leaf sheath length in ùñ¶ ùñ†_3 treated seedlings (n=20) . FeH: Seedling height and second leaf sheath length in paclobutrazol (PBZ) treated seedlings n = 20) ). Data are shown as means ¬± ùöÇ ùô≥ . Student‚Äôs t-test; , P<0.05 ; , P<0.01 .Scale bar, 2 cm (C and F).

# Discussion

In this work, we reported that the qGL10 locus on the long arm of chromosome 10 was a contributing factor in quantitative traits related to seed morphology in rice. This locus was consistent with the previously reported qTGW10.2 locus (Zhu et al., 2019). The candidate gene of q G L 1O was identified as OsMADS56, a homolog of OsMADS50, which highly similar to OsMADS1, OsMADS5, OsMADS14, OsMADS15, and OsMADS18 in a phylogenetic analysis of OsMADS family genes (Fig. S11). MADS-box transcription factors participate in plant growth and development mainly by influencing flowering time, floral organ morphogenetic development and fruit development (Ryu et al., 2009; Khanday et al., 2013). For example, OsMADS1, OsMADS6, OsMADS29 and OsMADS34 play important roles in glume and seed development (Zhang et al., 2010, 2016; Nayar et al., 2013; Dai et al., 2016). In contrast, OsMADS7 and OsMADS8 are members of the SEP-like family of genes, and knockout of these genes results in late flowering; homeotic changes of lodicules, stamens and carpels into palea/lemma-like organs; and loss of floral determinacy (Cui et al., 2010). OsMADS56 and its closely related homolog OsMADS50 regulate flowering in rice (Ryu et al., 2009). Our results indicate that overexpression of OsMADS56 causes early heading and increased grain yield (Figs. S5C and S12), whereas heading date is delayed and grain yield is decreased in osmads56 mutant plants (Figs. S4C and S12). Notably, however, all changes in heading date in different lines were accompanied by variations in grain length and grain weight. These findings indicate that the GL10 regulates grain size and heading date in rice simultaneously.


Fig. 5. Haplotypes and structural variation in GL10. A: Haplotypes distribution of GL10. The different colors and their areas in the pie chart represent the distribution of rice varieties/ species. B: Structural variation in the GL10 genomic sequence.

Phytohormones have various functions in seed growth, flowering and plant development (Zuo and Li, 2014). In previous reports, BR and auxin have previously been shown to participate in the control of seed development and plant growth (Li et al., 2019); however, relatively little is known about the role of GA in plant seed development. The GA-regulated GAST family protein, GW6, positively regulates grain width and weight (Shi et al., 2020), while AP1, AG, PI, AP3 and SEP3 encode ABC-type MADS domain proteins involving in hormone synthesis and signaling regulation (Dornelas et al., 2011). In particular, GA directly activates AP3, P I and AG during flower development (Yu et al., 2004). Here, we analyzed GA biosynthetic and signal transduction gene expression in young panicles of NIL-GL10 and NIL-gl10 plants, and found that these genes were significantly downregulated in NIL-gl10 plants (Fig. 4A and 4B). Moreover, the seedling height and sheaf length were both significantly greater in NIL-GL10 plants under exposure to high concentrations of ùñ¶ ùñ†_3 (Fig. 4Ce4E), indicating that G L 7O is involved in the regulation of GA signaling pathway genes, especially during development of young panicles in rice.

Rice grain size is an important factor for determining grain yield and quality. Breeders have made considerable improvements in rice cultivation and created an abundance of varieties with different grain lengths (Yu et al., 2018). The process of domestication is influenced by the geographical distribution of a gene in a given crop, the occurrence of mutations, the regional preferences of people growing and consuming the crop and the effects of the gene itself (Ishimaru et al., 2013; Wu et al., 2017). We aligned 70 homologous GL10 CDS sequences, which led to the detection of 14G L10 haplotypes. Most japonica rice varieties belong to Hap7 in the SV type 2 group, while most indica rice varieties are in Hap5, although some indica landraces belong to Hap7 and Hap11 (Fig. 5; Table S3). These biases in the distribution of GL10 haplotypes revealed that GL10 was probably involved as an important site in a major divergence or domestication event of rice subspecies. Considering that this trait controls the characteristics of rice grains and, consequently, the evolutionary history of rice as a domesticated crop, SVs of G L 7O may have participated in the emergence of the japonica rice grain type. However, the effects of differences in G L 7O haplotypes attributable to SVs and other small variations, such as SNPs and InDels, on grain size requires further investigation and experimental confirmation.

In this study, we report the GL10 gene, which is a regulator of rice grain length and find that the shorter grains characteristic of the gl10 allele are due to decreased cell expansion. GL10 encodes OsMADS56, which regulates the transcription of genes involved in GA signaling. We found that GL10 functions as a positive regulator of grain length, grain weight, and heading date in rice. These findings expand the scope of our understanding of the genetic mechanisms and regulatory networks governing grain size and provide a potential genetic resource for breeding rice with specific grain characteristics.

# Materials and methods

# Plant materials and growth condition

SSSL-gl10 was derived from the crossing and backcrossing of rice variety Lemont (japonica) and Huajingxian74 (HJX74; an elite xian (indica) variety from south China). The Lemont was used as a donor parent and HJX74 was used as a receptor parent to obtain a ùñ° ùñ¢_4ùñ•_2 generation by multiple backcrossing and self-crossing. NIL-GL10 and NIL-gl10 lines in the ùñ° ùñ¢_5ùñ•_2 generation were acquired using maker assisted selection (MAS). All rice plants were cultivated in a paddy field at South China Agriculture University, Guangzhou, China (23^‚àò07^' N, 1 13^‚àò 15^' E). Standard cultivation practices were used, and diseases and insect pests were controlled in rice cultivation.

# Fine mapping of GL10

For G L 7O mapping, both genotype and phenotype were used to determine the mapping region in ùñ° ùñ¢_5ùñ•_2 generations. Then, MAS was performed on each substitution segment plants. A series of InDel markers were produced using the HJX74 and Nipponbare genome sequences (http://192.168.87.153/, https://rapdb.dna.affrc.go.jp). With these, mapping of the G L 7O locus was refined to an 18.5¬≠ k b region between marker NC9 and marker NC14 using 4000 ùñ° ùñ¢_5ùñ•_3 lines. The candidate gene was amplified from genomic DNA for sequencing and the gene ORF was also amplified from RNA extracted from 2 to 3 cm young panicles. The expression of the candidate gene in different tissues in NIL- .G L 7O plants was quantified by RT-qPCR. All primers used in this study are listed in Table S4.

# RNA extraction and RT-qPCR analysis

Total RNA was extracted from the NIL-GL10 and NIL-gl10 tissues using TRIZOL reagent (Invitrogen) following the manufacturer‚Äôs instruction. The cDNA was produced using 5√ó All-In-One RT MasterMix (with an AccuRT Genomic DNA Removal Kit). RT-qPCR was performed using 1√ó SYBR Green Master Mix and an ABI 7500 realtime PCR system. The Rice ACTIN gene was used as the internal control, and each RT-qPCR program was performed three times. At least three biological replicates were performed for each analysis. All the primers used in RT-qPCR analysis are listed in Table S4.

# Vector construction and plant genetic transformation

A CRISPR/Cas9 gene editing construct of GL10 was designed as previously described (Hu et al., 2018a). To construct an overexpression vector, the full ORF of GL10 was amplified from the NILGL10 cDNA and cloned into a pXQ35S vector. All the construct vectors were introduced into Agrobacterium tumefaciens strain EHA105. The cas9 and overexpression vectors were transferred into HJX74 and NIL-gl10 by Agrobacterium-mediated transformation, respectively. Primers flanking the target sites in CRISPR/Cas9 gene editing of G L 7O were used for PCR amplification, and sequencing the PCR products to identify mutant plants. All agronomic traits were observed in the ùñ≥_2 generation of transgenic plants. The primers used for vector constructions are listed in Table S4.

# Subcellular localization and phylogenetic analysis

To generate the vector pOX:eGFP-GL10, the full-length coding region of ùñ¶ùñ´ Œ†^|ùñß|√ó74 was cloned into pOX:eGFP vector. The plasmids were transformed using PEG-mediated transformation methods into rice protoplasts, and fluorescence was observed under a laser confocal scanning microscope (ZEISS LSCM 7DUO 780   7 Live). Phylogenetic analysis of MADS family genes in rice was conducted using MAGA6 based on the neighbor-joining (NJ) model; bootstrap values were estimated with 1000 replicates. The primers used for subcellular localization assay are listed in Table S4.

# Histological analysis

Naturally dried mature grain samples were sputter coated with gold and subsequently observed using a scanning electron microscope (ZEISS EVO MA 15). Image J was used to measure cell length and numbers.

# Exogenous ùô∂ ùô∞_3 and PBZ treatment of NIL-GL10 and NIL-gl10 seedlings

Germinated seeds of NIL-GL10 and NIL-gl10 were grown on 96- well plates in distilled, deionized ùñß_2ùñÆ with different concentrations of ùô∂ ùô∞_3 and PBZ under 16-h light/8-h dark condition. Ten-day-old seedlings were collected for further analysis and photographed. Seedling height and second leaf sheath length were measured using Image J.

# Agronomic traits analysis

Agronomic traits were evaluated in different period of plant growth periods. Heading date was calculated at the time when the first panicle sprouted. Grain length, grain weight, 1000-grain weight, panicle length

and number of grains per plant were investigated after the rice harvest at the maturity stage. Grain size was measured using Microtek ScanWizard EZ scanner V-2.140 and Wan Shen grain analyzer.

# Neutrality test and haplotype network

Sixty-eight homologous GL10 sequences were selected from sequenced and resequenced Oryza accessions (15 wild accessions and 55 cultivars) (Stein et al., 2018; Zhao et al., 2018; Xie et al., 2021) to analyze the GL10 ORF and mine for other natural variations in the GL10 gene (Table S3). Multiple sequences of GL10 genomic DNA were aligned using Mafft. Nucleotide diversity was calculated and Tajima‚Äôs D test was performed using DnaSP version 6.12.03 (Rozas et al., 2017). Haplotype frequency data were processed with DnaSP version 6.12.03 and median-joining (MJ) networks were generated using PopART with some modifications (each continuous InDel was considered as a site) (Leigh and Bryant, 2015).

# Statistical analysis

All the data was analyzed with IBM SPSS Statistics 20. The comparison of two groups were performed with two tailed Student‚Äôs t-test, and Duncan‚Äôs multiple-range tests were performed for multiple mean comparisons.

# CRediT authorship contribution statement

Penglin Zhan: Conceptualization, Data curation, Methodology, Investigation, Validation, Writing - Original draft. Shuaipeng Ma, Zhili Xiao, Fangping Li, Xin Wei, Shaojun Lin and Xiaoling Wang: Data curation, Investigation, Validation. Zhe Ji: Writing - Original draft. Yu Fu, Jiahao Pan, Mi Zhou, Yue Liu, Zengyuan Chang and Lu Li: Investigation, Validation. Suhong Bu, Zupei Liu, Haitao Zhu and Guifu Liu: Writing - Review     Editing. Guiquan Zhang: Conceptualization, Project administration, Resources, Supervision. Shaokui Wang: Conceptualization, Project administration, Resources, Supervision, Writing - Original draft, Writing - Review     Editing.

# Conflict of interest

The authors declare that they have no conflicts of interests.

# Acknowledgments

This work was supported by the National Natural Science Foundation of China (32072040, 31622041) and the Key Projects of Basic Research and Applied Basic Research of Guangdong Province (2019B030302006).

# Supplementary data

Supplementary data to this article can be found online at https:// doi.org/10.1016/j.jgg.2022.01.008.
