# Regulation of OsSPL14 by OsmiR156 defines ideal plant architecture in rice

Yongqing Jiao1,4, Yonghong 𝐖𝐚𝐧𝐠^1,4 , Dawei 𝐗𝐮𝐞^2-4 , Jing Wang1, Meixian 𝐘𝐚𝐧^2 , Guifu Liu1, Guojun Dong2, Dali Zeng2, Zefu Lu^1 , Xudong Z𝐡𝐮^2 , Qian Qian2     Jiayang Li^1

Increasing crop yield is a major challenge for modern agriculture. The development of new plant types, which is known as ideal plant architecture (IPA), has been proposed as a means to enhance rice yield potential over that of existing high-yield varieties . Here, we report the cloning and characterization of a semidominant quantitative trait locus, IPA1 (Ideal Plant Architecture 1), which profoundly changes rice plant architecture and substantially enhances rice grain yield. The IPA1 quantitative trait locus encodes OsSPL14 (SOUAMOSA PROMOTER BINDING PROTEIN-LIKE 14) and is regulated by microRNA (miRNA) OsmiR156 in vivo. We demonstrate that a point mutation in OsSPL14 perturbs OsmiR156-directed regulation of OsSPL14, generating an ‘ideal’ rice plant with a reduced tiller number, increased lodging resistance and enhanced grain yield. Our study suggests that OsSPL14 may help improve rice grain yield by facilitating the breeding of new elite rice varieties.

Rice plant architecture is crucial for grain yield and is determined by plant height, tiller number and angle, and panicle morphology. The important characteristics of the ideal plant architecture (IPA) include low tiller numbers with few unproductive tillers, more grains per panicle than the currently cultivated varieties, and thick and sturdy stems1,2. However, the molecular mechanisms for generating the IPA and increasing yield potential remain to be elucidated, and genes applicable for improving rice plant architecture are very limited. Different rice varieties have distinct plant architecture and therefore distinct yield potential. In comparison to currently cultivated rice varieties, including the indica variety Taichung Native 1 (TN1), the japonica line Shaoniejing (SNJ) has plant architecture that exemplifies the concept of IPA, including plant height, tiller number and panicle morphology (Fig. 1a). Backcrossing SNJ with TN1 or Hui7 (a japonica line) shows that plant height, tiller number and panicle morphology do not segregate in the backcross progenies, implying that the differences in plant architecture between TN1 or Hui7 and SNJ may be controlled by a single locus. We therefore designated the locus that determines the SNJ architecture as IPA1. Genetic analysis indicated that IPA1 was semidominant, given that the phenotype of heterozygous plants (O s S P L I4^IPA1/ipal) ) was ­ intermediate between those of the homozygous plants OsSPL14^IPA1/IPA1 and OsSPL14ipa1/ipa1 (Supplementary Fig. 1).

For the convenience of scoring the phenotype, we chose tiller number as the trait to use in mapping the IPA1 locus. Using 110 BC_2F_2 plant lines generated from the backcross between SNJ and TN1, we detected the largest-effect quantitative trait locus (QTL), which explained 29.9% of tiller number variance, at q T n& . qTn8 was mapped to the long arm of chromosome 8 between markers RM149 and RM1345 (Fig. 16,𝐜 ), which is most likely the same locus as that of a previously reported QTL that defines rice tiller number . To clone the gene underlying the IPA1 locus, 5,500 BC_2F_2 plants that had similar tiller numbers to TN1 were identified and genotyped with newly developed molecular markers (Supplementary Table 1).

We narrowed the candidate region containing the IPA1 locus to ∼78 kb between markers M4 and M5 (Fig. 1d), which contains 12 predicted genes or ORFs (Fig. 1e and Supplementary Table 2; see URLs). Sequencing of the 12 genes in SNJ showed only one point mutation in the third exon of OsSPL14 (LOC_Os08g39890; RAP ID Os08g0509600) compared to the Nipponbare variety of rice. This nucleotide substitution leads to an amino acid change from leucine to isoleucine in SNJ plants (Fig. 1f and Supplementary Fig. 2). In addition, Ri22, a japonica line showing similar plant architecture to SNJ, was also found to harbor an identical mutation in OsSPL14, whereas no mutation at this locus could be detected in the rice varieties TN1, 93-11 or Zhonghua11. We found that the expression level of OsSPL14 was affected by the point mutation (Supplementary Fig. 3).

To determine whether OsSPL14 underlies the IPA1 QTL, we ­performed a confirmation test by generating transgenic plants expressing different levels of OsSPL14 in the Nipponbare and Ri22 lines, the japonica varieties suitable for gene transformation. We introduced the plasmid carrying OsSPL14 (designated g O s S P L I4 ), which contained a 7.2-kb genomic DNA fragment, into Nipponbare (see Online Methods). The gOsSPL14 transgenic lines had reduced tillers, stronger culms and increased panicle branches and grain yield (Supplementary Fig. 4). In contrast, the OsSPL14 RNA interference (RNAi) transgenic plants in the Ri22 background generated more tillers and showed a marked reduction in plant height, diameter of culms, panicle branches and grain number (Supplementary Fig. 5). Therefore, we concluded that OsSPL14 is the gene responsible for ideal plant architecture.



SPL genes, which share a highly conserved DNA-binding domain (the SBP domain), represent a family of plant-specific transcription factors that are involved in the regulation of flowering time, phase change, leaf initiation and other developmental processes in higher plants5–18. In the rice genome, there are 19 S P L genes and OsSPL14 (also known as IPA1 and WFP) is most similar to Arabidopsis SPL9 (ref. 8) (Supplementary Fig. 6), which has been suggested to be involved in regulating plastochron length and leaf size15,18. OsSPL14 is localized to the nucleus (Fig. 1g), consistent with a role as a trans­cription factor. RNA in situ hybridization revealed that OsSPL14 was ­predominantly Figure 1  Map-based cloning of the IPA1 QTL and subcellular localization of the OsSPL14-GFP fusion protein. (a) The plant architecture of SNJ and TN1. Scale bars, 10 𝖼 𝗆 . (b) Location of IPA1 on rice chromosome 8. (c) Coarse linkage map of IPA1. (d) High-resolution linkage map of IPA1. The number of recombinants between the molecular marker and IPA1 is indicated. (e) Annotation of the candidate region surrounding IPA1 on BAC AP006049. Arrows indicate the putative genes predicted in the Rice Genome Annotation Database. (f) OsSPL14 structure and the mutation site in SNJ. The white boxes represent the 5^' and 3^' untranslated regions, the black boxes represent the coding sequences and lines between boxes represent introns. The red asterisk indicates the OsmiR156 target site. (g) Subcellular localization of OsSPL14-GFP in rice root cells. Scale bars, 10 m .

expressed in the shoot apex at both the vegetative (Fig. 2a) and re­productive (Fig. 2b) stages. It was also highly expressed in the promordia of primary and secondary branches (Fig. 2c).


Figure 2  Expression pattern of OsSPL14 and confirmation of OsmiR156- directed regulation on OsSPL14. (a–c) OsSPL14 expression patterns revealed by mRNA in situ hybridization. Scale bars, 200 𝗆 . (d) OsmiR156 cleavage sites in O s S P L4 𝗆 𝖱 𝖭 𝖠 𝗌 determined by RNA ligase-mediated 5^' RACE. The vertical lines represent the nucleotides that base-pair with OsmiR156 and dots show the mismatched nucleotide. The numbers above the sequence indicate the location of the nucleotide in the O s S P L14 coding sequence. The positions corresponding to the 5^' ends of the cleaved OsSPL14 mRNAs determined by 5^' RACE and the frequency of 5^' RACE clones corresponding to each site are indicated by arrows. (e) IPA1 transcript levels in various organs, including shoot apexes of seedlings (SA), culms (C), leaves (L), leaf sheaths (LS), panicles after heading (P) and seedling roots (R). Values in e are means and s.d. of three independent experiments. (f) Expression pattern of OsmiR156. OsmiR156 transcript levels were determined by RNA blot analysis in various organs.

Previous studies suggested that 11 O s S P L genes were putative targets of OsmiR156 (ref. 8), a member of an miRNA family consisting of 19–25-base-pair noncoding single-stranded regulatory RNAs19. Bioinformatic analysis indicated that OsSPL14 has the OsmiR156 complementary site in the coding region (Supplementary Fig. 2). To determine whether OsSPL14 could be regulated by OsmiR156 in vivo, we mapped the OsmiR156-directed cleavage sites of OsSPL14 by using RNA ligase–mediated rapid amplification of cDNA ends (RLM-RACE). Sequencing of 14 randomly chosen clones showed that 13 clones had 5^' ends of the cleaved fragments in the middle region of the OsmiR156 target site, indicating that OsSPL14 can be precisely cleaved by OsmiR156 in vivo (Fig. 2d).

Expression patterns of OsSPL14 and OsmiR156 in various rice organs revealed by real-time PCR and miRNA gel blot analyses showed that OsSPL14 was highly expressed in the culm and shoot apex, which is complementary with the expression pattern of OsmiR156 in vivo (Fig. 2e,f). Consistently, overexpression of OsmiR156 resulted in a substantial decrease in OsSPL14 transcripts (Supplementary Fig. 7a), whereas the interruption of OsmiR156 (overexpression of MIM156)20 led to a marked increase in OsSPL14 transcripts (Supplementary Fig. 7b). These results indicated that OsSPL14 was regulated by the OsmiR156-directed cleavage in vivo.

MicroRNAs play important roles in modulating their target genes mainly through transcript cleavage in plants and mainly through translation repression in animals19,21,22. Recent studies also showed that the translation inhibition of targets by miRNA was widespread in higher plants . To determine the molecular mechanism by which OsSPL14 is regulated by OsmiR156 in vivo, we sequenced the RT-PCR products encompassing the cleavage site using the heterozygous (O s S P L I4^IPA1/ipal) plants and found that the intact mRNAs existed mainly as the O s S P L I4^ipa1 form (Fig. 3a). To confirm this, we further sequenced 37 clones from the RT-PCR products and found that 33 clones had the OsSPL14ipa1 form. These results suggest that the point mutation of OsSPL14 in SNJ perturbed the cleavage of the OsSPL14 transcripts by OsmiR156.


Figure 3  Effects of the point mutation in O s S P L4"P a1 on the OsmiR156-directed regulation of OsSPL14. (a) Chromatogram of intact OsSPL14IPA1/ipa1 mRNAs recovered from RT-PCR with primers flanking the cleavage site. Arrows indicate the different nucleotides in O s S P L4^|𝖯 𝖠1 and OsSPL14ipa1 cDNAs. (b) Phenotypes of O s S P L4^|𝖯 𝖠| -GFP, OsSPL14ipa1-GFP and O s S P L4^|𝖯 𝖠17𝗆_-G F P transgenic plants. Scale bar, 10 𝖼 𝗆 . The blue letter indicates the mutation site in the OsSPL14ipa1 mRNA. The asterisks and red letters indicate the mutation sites and mutant nucleotides introduced into the OsmiR156 target site in the O s S P L4^|𝖯 𝖠17𝗆_- G F P transgene, which interrupts the OsmiR156 target site without changing the amino acid residues. (c) Protein levels of IPA1 in Nipponbare (NP), 𝚐O s S P L 𝚕4^𝚕𝙿 𝙰1.2 ,𝚐O s S P L 𝚕4^𝚕𝙿 𝙰 𝚕.3 and 𝚐O s S P L l4^𝚒 𝚙 𝚊 𝚕.1 transgenic plants. Above, protein blot with the antibody to IPA1; below, Ponceau S staining showing equal loading of proteins. (d) Transcript levels of IPA1 revealed by real-time PCR in Nipponbare (NP), 𝚐O s S P L14^𝚕𝙿 𝙰1.2 ,𝚐O s S P L14^𝚕𝙿 𝙰1.3 and 𝚐O s S P L4^𝗂 𝗉 𝖺 𝗅.1 transgenic plants. Values are means and s.d. of three independent experiments.

We generated transgenic plants carrying the O s S P L I4^IPA7m_· -GFP transgene, which contained seven mismatches to OsmiR156 but did not introduce any amino acid changes (Fig. 3b). The phenotype of O s S P L I4^IPA7m -GFP transgenic plants was very similar to those of OsSPL14ipa1-GFP plants, and both of them had more severe phenotypes than the O s S P L I4^IPA1 -GFP transgenic plants (Fig. 3b), ­indicating that perturbing the OsmiR156 cleavage of OsSPL14 even without ­changing the resulting amino acid sequence leads to alteration of the plant architecture in SNJ. To investigate whether the point mutation perturbed the miRNA-directed translation repression of OsSPL14, we measured OsSPL14 protein levels in transgenic plants with the plasmids OsSPL14ipa1 (gO s S P L I4^ipa1) and O s S P L I4^IPA1 (gOsSP L I4^IPA1) . We found that gO s S P L I4^ipa1.1 transgenic plants had a significantly increased protein level as compared to both gO s S P L I4^IPA1.2 and gO s S P L I4^IPA1.3 plants, although the mRNA level in gO s S P L I4^ipa1.1 plants was lower than that in gOs S P L I4^IPA1 -3 (Fig. 3c,d), suggesting that the point mutation may also perturb the OsmiR156-directed translation repression of OsSPL14. Therefore, the point mutation in the OsmiR156-complementary site of OsSPL14 perturbs the OsmiR156-directed transcriptional cleavage and translation repression in rice.

To evaluate the application potential of OsSPL14 for optimizing rice plant architecture and eventually improving grain yields, we generated a near-isogenic line (NIL) in the Hui7 genetic background that Figure 4  Phenotypic characterization of NIL OsSPL14ipa1 plants. (a) Gross morphologies of NIL OsSPL14IPA1 and NIL OsSPL14ipa1 plants at maturity. Scale bar, 10 cm. (b) Panicles of NIL O s S P L4^𝖨 𝖯 𝖠1 and NIL OsSPL14ipa1. Scale bar, 10 𝖼 𝗆 . (c) Culms of NIL O s S P L4^|𝖯 𝖠1 and NIL OsSPL14ipa1. Scale bar, 5 cm. (d) Crosssections of culms of NIL OsSPL14IPA1 and NIL OsSPL14ipa1 plants. The middle and right panels are magnifications of indicated square and circle regions in the left panel, respectively. BV, big vascular bundles; SV, small vascular bundles; SC, sclerenchyma cells. Bars, 100 m .(e) Comparison of the maximum bending force on third internodes between NIL O s S P L4^|𝖯 𝖠 𝗅 and NIL OsSPL14ipa1. (f) Comparison of primary branch number per main panicle between NIL O s S P L4^𝖨 𝖯 𝖠1 and NIL OsSPL14ipa1. (g) Comparison of secondary branch number per main panicle between NIL OsSPL14IPA1 and NIL O s S P L40^∘ . (h) Comparison of grain number per main panicle between NIL O s S P L4^𝖨 𝖯 𝖠1 and NIL OsSPL14ipa1.



(j) Comparison of the grain yield per main panicle between NIL Values in e–j are means ±≤.4 . (e–h,j, n=12 plants; i, represent significance difference determined by the Student’s t-test at P<0.01 .

Table 1  Yield test in a paddy between XS11 and NIL XS11 OsSPL14ipa1 plants


Data are from plants in randomized complete block design with three replications under ­natural condition in Hangzhou, China, in 2008. The planting density was 19.98 c m×23.31 c m, with one plant per hill. The area per plot was 6.67 m^2 . Values are means ±≤𝖽 .

contains the O s S P L I4^ipa1 allele. The resulting NIL OsSPL14ipa1 plant showed an increase in accumulation of the OsSPL14 transcript and protein in comparison with NIL O s S P L I4^IPA1 plants (Supplementary Fig. 8). Compared with the NIL O s S P L I4^IPA1 plant, the NIL OsSPL14ipa1 plant is taller (Fig. 4a and Supplementary Fig. 9a) and has fewer tillers (Fig. 4a and Supplementary Fig. 9b), denser panicles (Fig. 4b) and stronger culms (Fig. 4c and Supplementary Fig. 9c). In-depth characterization of these traits revealed that the reduced tiller number in NIL OsSPL14ipa1 plants resulted from the prolonged plastochron (Supplementary Fig. 10a). The cross-section of the culm internodes showed that the NIL OsSPL14ipa1 plant had more vascular bundles and sclerenchyma cells than the NIL OsSPL14IPA1 plant (Fig. 4d and Supplementary Fig. 10b). Consistently, the culm mechanical strength of the NIL OsSPL14ipa1 plant was significantly increased in comparison with NIL O s S P L I4^IPA1 plants (Fig. 4e). More importantly, statistical analysis indicated that the NIL OsSPL14ipa1 panicle produced more primary and secondary branches (Fig. 4f,g) and more grains (Fig. 4h). Thousand-grain weight increased from 27.2 g in NIL O s S P L I4^IPA1 to 30.2 g in NIL OsSPL14ipa1 (Fig. 4i). Theoretically, the grain yield of NIL O s S P L I4^ipa1 panicles showed an increase of 6.5 g per main panicle compared to 4.6 g per main panicle in NIL O s S P L I4^IPA1 (Fig. 4j). Introduction of the O s S P L I4^ipa1 alleleinto Xiushui 11 (XS11), a japonica rice variety cultivated in South China, increased the grain yield by ∼10% in the test plot (Table 1). These results indicate that OsSPL14 is a pleiotropic gene that appears to confer an ideal rice plant architecture with few unproductive tillers, enhanced grain yield per panicle and an elevated lodging resistance. Taken together, our findings show that OsSPL14 is an important semidominant regulator of rice plant architecture. OsSPL14 may help facilitate the genetic engineering and molecular breeding of elite rice varieties with ideal plant architecture to reach much higher yield potential.

URLs. http://rice.plantbiology.msu.edu/cgi-bin/gbrowse/rice/.

# Methods

Methods and any associated references are available in the online ­version of the paper at http://www.nature.com/naturegenetics/.

Accession codes. IPA1 is deposited in GenBank under the following number: GU136674.

Note: Supplementary information is available on the Nature Genetics website.

# Acknowledgments

We thank K. Chong (Institute of Botany, Chinese Academy of Sciences) for providing the pTCK303 vector. This work was supported by grants from Ministry of Agriculture of the People’s Republic of China (2008ZX08009), Ministry of Science and Technology (2005CB1208) and National Natural Science Foundation of China (30710103903).

# AUTHOR CONTRIBUTIONS

Y.J. and Y.W. designed the research, performed experiments, analyzed data and wrote the paper. D.X. performed experiments and analyzed data. J.W., M.Y., G.L., G.D., D.Z., Z.L and X.Z. performed the experiments. Q.Q. designed the research and analyzed the data. J.L. supervised the project, designed research, analyzed data and wrote the paper.

# COMPETING FINANCIAL INTERESTS

The authors declare no competing financial interests.

Published online at http://www.nature.com/naturegenetics/. Reprints and permissions information is available online at http://npg.nature.com/ reprintsandpermissions/.

1.	 Khush, G.S. Breaking the yield frontier of rice. GeoJournal 35, 329–332 (1995).
2.	 Virk, P.S., Khush, G.S.     Peng, S. Breeding to enhance yield potential of rice at IRRI: the ideotype approach. Int. Rice Res. Notes 29, S1–S9 (2004).
3.	 Miyamoto, N. et al. Quantitative trait loci for phyllochron and tillering in rice. Theor. Appl. Genet. 109, 700–706 (2004).
4.	 Fukuta, Y. et al. Identification of low tiller gene in rice two varieties, Aikawa 1 and Shuho in rice (Oryza sativa L.). Plant and Animal Genome XII Abstract, 167 (2004).
5.	 Klein, J., Saedler, H.     Huijser, P. A new family of DNA binding proteins includes putative transcriptional regulators of the Antirrhinum majus floral meristem identity gene SQUAMOSA. Mol. Gen. Genet. 250, 7–16 (1996).
6.	 Cardon, G. et al. Molecular characterisation of the Arabidopsis SBP-box genes. Gene 237, 91–104 (1999).
7.	 Gandikota, M. et al. The miRNA156/157 recognition element in the 3^' UTR of the Arabidopsis SBP box gene SPL3 prevents early flowering by translational inhibition in seedlings. Plant J. 49, 683–693 (2007).
8.	 Xie, K., Wu, C.     Xiong, L. Genomic organization, differential expression, and interaction  of  SQUAMOSA  promoter-binding-like  transcription  factors  and microRNA156 in rice. Plant Physiol. 142, 280–293 (2006).
9.	 Unte, U.S. et al. SPL8, an SBP-box gene that affects pollen sac development in Arabidopsis. Plant Cell 15, 1009–1019 (2003).
10.	Stone, J.M., Liang, X., Nekl, E.R.     Stiers, J.J. Arabidopsis AtSPL14, a plantspecific SBP-domain transcription factor, participates in plant development and sensitivity to fumonisin B1. Plant J. 41, 744–754 (2005).
11.	Manning, K. et al. A naturally occurring epigenetic mutation in a gene encoding an SBP-box transcription factor inhibits tomato fruit ripening. Nat. Genet. 38, 948–952 (2006).
12.	Wu, G.     Poethig, R.S. Temporal regulation of shoot development in Arabidopsis thaliana by miR156 and its target SPL3. Development 133, 3539–3547 (2006).
13.	Zhang, Y., Schwarz, S., Saedler, H.     Huijser, P. SPL8, a local regulator in a subset of gibberellin-mediated developmental processes in Arabidopsis. Plant Mol. Biol. 63, 429–439 (2007).
14.	Lee, J., Park, J.J., Kim, S.L., Yim, J.     An, G. Mutations in the rice liguleless gene result in a complete loss of the auricle, ligule, and laminar joint. Plant Mol. Biol. 65, 487–499 (2007).
15.	Wang, J.W., Schwab, R., Czech, B., Mica, E.     Weigel, D. Dual effects of miR156- targeted SPL genes and CYP78A5/KLUH on plastochron length and organ size in Arabidopsis thaliana. Plant Cell 20, 1231–1243 (2008).
16.	Wu, G. et al. The sequential action of miR156 and miR172 regulates developmental timing in Arabidopsis. Cell 138, 750–759 (2009).
17.	Wang, J.W., Czech, B.     Weigel, D. miR156-regulated SPL transcription factors define an endogenous flowering pathway in Arabidopsis thaliana. Cell 138, 738–749 (2009).
18.	Schwarz, S., Grande, A.V., Bujdoso, N., Saedler, H.     Huijser, P. The microRNA regulated SBP-box genes SPL9 and SPL15 control shoot maturation in Arabidopsis. Plant Mol. Biol. 67, 183–195 (2008).
19.	Stefani, G.     Slack, F.J. Small non-coding RNAs in animal development. Nat. Rev. Mol. Cell Biol. 9, 219–230 (2008).
20.	Franco-Zorrilla, J.M. et al. Target mimicry provides a new mechanism for regulation of microRNA activity. Nat. Genet. 39, 1033–1037 (2007).
21.	Mallory, A.C.     Vaucheret, H. Functions of microRNAs and related small RNAs in plants. Nat. Genet. 38Suppl, S31–S36 (2006).
22.	Jones-Rhoades, M.W., Bartel, D.P.     Bartel, B. MicroRNAs and their regulatory roles in plants. Annu. Rev. Plant Biol. 57, 19–53 (2006).
23.	Brodersen, P. et al. Widespread translational inhibition by plant miRNAs and siRNAs. Science 320, 1185–1190 (2008).

# ONLINE METHODS

Plant materials. The japonica background SNJ and Ri22 carry the OsSPL14ipa1 allele and have similar plant architecture. The japonica line Hui7 and the indica variety TN1 have different plant architecture from SNJ and Ri22. Plants from the Hui7×SN] F_1 generation were backcrossed four times with Hui7 to generate the near-isogenic line Hui7 (NIL OsSPL14ipa1). To breed an elite rice variety harboring the O s S P L I4^ipal allele, XS11×SN] F_1 plants were backcrossed three times with XS11 to generate the BC_3F_1 seeds. From the BC_3F_2 generation, the NIL XS11 OsSPL14ipa1 was developed for the yield test in this study.

Constructs for genetic transformation. To construct the gOs S P L I4^IPA1 transformation plasmid, a 7.2-kb DNA fragment containing a 1,117-bp upstream sequence, the entire O s S P L I4^IPA1 and a 2,326-bp downstream region was amplified from the Nipponbare genomic DNA using primers gIPA11F and gIPA11R (Supplementary Table 1), then digested with KpnI and XbaI and ligated to the binary vector pCAMBIA1300 for gene transformation. To construct the OsSPL14 RNAi plasmid, two DNA fragments (610 base pairs long) were amplified from the full-length cDNA clone AK107191 using two pairs of primers RNAi1F and RNAi1R, and RNAi2F and RNAi2R (Supplementary Table 1). The BamHI-KpnI– and SacISpeI–digested fragments were cloned into the binary vector pTCK303^24 .To construct the O̅s S P L I4^IPA1 -GFP plasmid, the PCR products containing the O s S P L14^IPA1 promoter region and the entire OsSPL14 coding sequence were amplified using primers gIPA11R and IPA1GFP2R (Supplementary Table 1) from the mixture of OsSPL14 promoter–containing PCR products amplified with primers gIPA11R and IPA1GFP1R (Supplementary Table 1) and the OsSPL14 coding sequence–containing PCR products with primers IPA1OE1F and IPA1GFP2R (Supplementary Table 1). The KpnI-XbaI– digested fragments were ligated into the binary vector AHLG. The mutations in OsSPL14ipa1-GFP and O̅s S P L14^I P A17m. -GFP were introduced by primers gipa11F, gipa11R and 7mIPA1GFP1F, 7mIPA1GFP1R (Supplementary Table 1). To construct the miRNA156OE plasmid, the PCR fragments amplified from the full length cDNA clone AK110797 using primers 156OE1F and 156OE1R (Supplementary Table 1) were subcloned into the BamHISacI–digested binary vector pTCK303. The MIM156 construct was generated according to previously reported method25, and the PCR fragment was subcloned into the BamHI-SacI–digested pTCK303 vector.

5^' modified rapid amplification of cDNA ends. 5^' modified RACE was performed according to the method reported previously . Briefly, total RNA was isolated from 2-week-old NIL OsSPL14IPA1 seedlings using the TRIzol Reagent (Invitrogen). The gene specific primers of OsSPL14 for the first and second PCR products are IPA1-1570R and IPA1RT1R (Supplementary Table 1), respectively. The second PCR products were gel purified and subcloned into the pGEM-T Easy Vector (Promega) for sequencing.

Antibody preparation. A DNA fragment encoding the OsSPL14IPA1 amino acid residues 1–96 amplified with the primers IPA1OE1F and IPA1P1R (Supplementary Table 1) was subcloned into pET28a. The recombinant OsSPL14IPA1 protein was expressed in BL21 cells, purified by Chelating Sepharose Fast Flow (GE Healthcare) according to the supplier’s instructions and used to raise polyclonal antibodies in rabbits. The resulting OsSPL14IPA1 antibodies were purified through an IgG affinity chromatography column before use. The specificity of the antibody was determined by protein blot (Supplementary Fig. 11).

Determination of transcript levels of OsSPL14 by real-time PCR and OsmiR156 with RNA blot. Total RNAs were prepared using a TRIzol kit according to the user’s manual (Invitrogen). Total RNAs from various organs were split into two portions, one for the detection of OsSPL14 by real-time PCR and one for OsmiR156 by RNA blot. For determination of OsSPL14 transcripts, one microgram of total RNA was treated with DNase I and used for cDNA synthesis with a reverse transcription kit (Promega). Real-time PCR experiments were performed using gene-specific primers in a total volume of 10 l with 1 l of the reverse transcriptase reactions, 1 M of the gene-specific primers IPA1rtime1F and IPA1rtime1R (Supplementary Table 1) and 5 L SYBR Green Master mix (Applied Biosystems) on an ABI 7900 real-time PCR machine (Applied Biosystems) according to the manufacturer’s instructions. The rice Ubiquitin gene (Os03g0234200) was used as the internal control. The relative OsSPL14 transcript levels in various organs were compared with that in the root, after normalization with Ubiquitin transcript and averaged from three independent replicates. For the detection of OsmiR156, 40 g of total RNAs from various organs were resolved in 15% polyacrylamide-urea gels, electro-transferred to Zeta-Probe membranes (Bio-Rad) and hybridized to an antisense oligonucleotide probe (Supplementary Table 1).

Determination of OsSPL14 transcripts and OsSPL14IPA1 protein abundance. Two-week-old seedlings were ground into fine powder in liquid nitrogen and divided into two parts, one for detecting of O̅s S P L I4^IPA1 transcripts and the other for measuring OsSPL14IPA1 protein abundance. For detecting O s S P L I4^IPA1 transcripts by RNA blot, 20 g of total RNAs were separated on a 1.2% denaturing gel, transferred to nylon filters (Hybond- ·N^+ , Amersham) and hybridized to the OsSPL14-specific probe amplified from the full-length cDNA clone AK107191 using the primers RNAi1F and RNAi1R (Supplementary Table 1). For measuring the OsSPL14IPA1 protein abundance, total proteins were extracted using the extraction buffer ( 500 mM Tris-HCl, pH 7.5, 150 mM NaCl, 0.1% NP-40 detergent, 4 M urea, 1 mM PMSF inhibitor) and quantified by the Bio-Rad protein assay. Protein blot was performed with the SuperSignal West chemiluminescence kit according to the manufacturer’s protocol (Pierce Chemical) using the OsSPL14IPA1 antibody.

Measurement of physical properties. The maximum bending force was mea­ sured using a digital force/length tester (5848 microtester, Istron; http://www. instron.com/). The third internodes of NIL OsSPL14IPA1 and NIL OsSPL14ipa1 plants were used for the measurement.

RNA in situ hybridization and subcellular localization of OsSPL14. RNA in situ hybridization was performed as described previously27 with minor modification. Briefly, the 610-bp region of OsSPL14 was subcloned into the pGEM-T Easy (Promega) vector and used as the template to generate sense and antisense RNA probes. Digoxigenin-labeled RNA probes were prepared using a DIG Northern Starter Kit (cat. no. 2039672, Roche) according to the manufacturer’s instruction. Slides were observed under bright field through a microscope (Leica DMR) and photographed with a Micro Color chargecoupled device camera (Apogee Instruments).

To examine the subcellular localization of OsSPL14^IPA1 in rice cells, the PCR fragment amplified from the full length cDNA clone AK107191 using the primers IPA1GFPACF and IPA1GFP2R (Supplementary Table 1) was ligated into the binary vector AHLG under the control of the rice ACTIN promoter. The recombinant plasmid was introduced into Nipponbare plants by Agrobacterium tumefaciens–mediated transformation. Transgenic seeds of the T_1 progeny were screened on agar plates containing 50 mg/1 hygromycin. Fluorescence analysis was carried out on roots 5 d post germination. GFP signals were examined and photographed under a confocal microscope at an excitation wavelength of 488 nm (FluoView 1000; Olympus).

24.	Wang, Z. et al. A practical vector for efficient knockdown of gene expression in rice (Oryza sativa L.). Plant Mol. Biol. Rep. 22, 409–417 (2004).
25.	Franco-Zorrilla, J.M. et al. Target mimicry provides a new mechanism for regulation of microRNA activity. Nat. Genet. 39, 1033–1037 (2007).
26.	Llave, C., Xie, Z., Kasschau, K.D.     Carrington, J.C. Cleavage of Scarecrow-like mRNA targets directed by a class of Arabidopsis miRNA. Science 297, 2053–2056 (2002).
27.	Li, P. et al. LAZY1 controls rice shoot gravitropism through regulating polar auxin transport. Cell Res. 17, 402–410 (2007).