# Overexpression of RICE FLOWERING LOCUS T 1 (RFT1) Induces Extremely Early Flowering in Rice

Richa Pasriga1,2, Jinmi Yoon1,2, Lae-Hyeon Cho1, and Gynheung An1,

1Graduate School of Biotechnology and Crop Biotech Institute, Kyung Hee University, Yongin 17104, Korea, 2These authors contributed equally to this work.
Correspondence: genean@khu.ac.kr
https://doi.org/10.14348/molcells.2019.0009
www.molcells.org

RICE FLOWERING LOCUS T 1 (RFT1) is a major florigen that functions to induce reproductive development in the shoot apical meristem (SAM). To further our study of RFT1, we overexpressed the gene and examined the expression patterns of major regulatory genes during floral transition and inflorescence  development.  Overexpression  induced  extremely early flowering in the transgenics, and a majority of those calli directly formed spikelets with a few spikelets, thus bypassing normal vegetative development. FRUITFULL (FUL)- clade genes OsMADS14, OsMADS15, and OsMADS18 were highly  induced  in  the RFT1-expressing  meristems. OsMADS34 was also induced in the meristems. This indicated that RFT1 promotes the expression of major regulatory genes that are important for inflorescence development. RFT1 overexpression also induced SEPALLATA (SEP)-clade genes OsMADS1, OsMADS5, and OsMADS7 in the greening calli before floral transition occurred. This suggested their possible roles at the early reproductive stages. We found it interesting that expression of OsFD1 as well as OsFD2 and OsFD3 was strongly increased in the RFT1-expressing calli and spikelets. At a low frequency, those calli produced plants with a few leaves that generated a panicle with a small number of spikelets. In the transgenic leaves, the FULclade genes and OsMADS34 were induced, but SEP-clade gene expression was not increased. This indicated that OsMADS14, OsMADS15, OsMADS18, and OsMADS34 act immediately downstream of RFT1.

Keywords: florigen, flowering, FRUITFULL, OsFD1, RFT1, SEPALLATA

# INTRODUCTION

Flowering plants undergo a transition between the vegetative and reproductive stage. In rice (Oryza sativa), once flowering is triggered, the shoot apical meristem (SAM) is converted into the rachies meristem (RM), which produces the first bract primordium at the opposite side of the flag leaf. This RM generates the primary branch meristem (PBM) in the axils of those newly developed bracts. During primary branch elongation, the PBM produces a secondary branch meristem (SBM). Both PBM and SBM are eventually converted to a terminal spikelet meristem, which then forms a single floral meristem (FM) that then successively produces floral organs (Itoh et al., 2005).

Reproductive development is triggered by the accumulation of florigens in the leaf phloem (Komiya et al., 2008; Tamaki et al., 2007). In rice, Heading date 3a (Hd3a) and RICE FLOWERING LOCUS T1 (RFT1) are florigen genes that dominate under short-day and long-day conditions, respectively (Komiya et al., 2008; Tamaki et al., 2015). They are expressed preferentially in phloem cells when flowering signal is induced developmentally or environmentally (Cho et al., 2017). The 200-bp region between ‚Äì245 bp and ‚Äì45 bp from the transcription initiation site of Hd3a is responsible for this phloem-specific expression (Pasriga et al., 2018).

Expression of the rice florigen genes is promoted by a type-B responsive regulatory element Ehd1 after the formation of a homodimer (Cho et al., 2016). Homodimerization of the protein is inhibited by cytokinin-inducible OsRR1.

Transcription of Ehd1 is suppressed by several upstream regulatory elements, such as Ghd7, Hd1, COL4, AP2, and OsLFL1, but is induced by positive elements, including OsID1, Ehd4, OsMADS51, and Hd1 (Cho et al., 2017; Lee and An, 2015; Tsuji et al., 2011).

Hd3a and RFT1 proteins are transferred to the SAM, where they activate downstream target genes to initiate reproductive processes. Hd3a protein interacts with 14-3-3 protein of the Gf14 family, which mediates the interaction between Hd3a and OsFD1, thus forming a ternary complex that is targeted to the nucleus (Taoka et al., 2011; Tsuji et al., 2013). This ‚Äòflorigen activation complex (FAC)‚Äô stimulates expression of OsMADS14, OsMADS15, and OsMADS18, all FRUITFULL (FUL)-clade MADS box genes, as well as expression of OsMADS34, a SEPALLATA (SEP)-clade gene (Kobayashi et al., 2012; Taoka et al., 2011). The complex binds with the OsMADS15 promoter to induce expression of the target gene (Tsuji et al., 2011). Expression of OsMADS14 and OsMADS15 is decreased in the SAM of Hd3a RNAi or RFT1 RNAi plants, which indicates that they are downstream genes of the florigens (Komiya et al., 2008, 2009; Tsuji et al., 2011).

Florigens also function outside of the SAM. For example, Hd3a protein is accumulated in axillary meristems and forms an FAC to promote branching (Tsuji et al., 2015). This florigen induces outgrowth by lateral buds through a mechanism independent of strigolactone. Both Hd3a and RFT1 also form transcriptional activation or repression complexes in rice leaves, where the proteins interact with Gf14c and OsFD1 (Brambilla et al., 2017). Transient induction of Hd3a and RFT1 expression induces transcript levels of OsMADS14 and OsMADS15 within 16 h (Brambilla et al., 2017). In addition, overexpression of OsFD1 increases transcript levels of OsMADS14 and OsMADS15 as well as those of Ehd1, Hd3a, and RFT1 in the leaves (Brambilla et al., 2017).

Suppression of all three FUL-clade genes ‚Äì OsMADS14, OsMADS15, and OsMADS18 ‚Äì slightly delays the reproductive transition. However, further depletion of OsMADS34 from triple knockdown plants significantly delays that transition, which means that all four of those genes coordinately act in the SAM to induce reproductive development (Kobayashi et al., 2012). OsMADS34 is induced in the SAM upon transition  to  reproductive  development  and  also  while glumes are initiated during spikelet development (Kobayashi et al., 2012). This observation is consistent with genetic information that mutations in the gene causes reduced primary branches and altered glume morphology (Gao et al., 2010; Kobayashi et al., 2012). OsMADS34 interacts with OsMADS14 and OsMADS15, which suggests a possibility that they form a ternary complex to induce downstream genes (Kobayashi et al., 2012). OsMADS34 also functions together with OsMADS1 and OsMADS5 in determining spikelet identity (Wu et al., 2018).

Mis-expression of Hd3a markedly accelerates the reproductive transition and produces terminal tissues at the tips of transgenic rice plants (Izawa et al., 2002; Kojima et al., 2002). The terminal tissues often contain a floret containing several stamen-like and a pistil-like organ. In severe cases, spikelets develop directly from calli (Kobayashi et al., 2012).

When Hd3a is expressed in osmasd34 mutant plants, the transgenic plants produced elongated stems, which implies a transition to reproductive development (Kobayashi et al., 2012). However, they do not develop inflorescence meristems and, instead, repeatedly generate vegetative shoots that result in a bushy appearance. This suggests that OsMADS34 is needed for initiation of inflorescence development.

In this study we observed that overexpression of RFT1 resulted in the direct formation of spikelets from most of the transgenic calli. Transcript analyses at different developmental stages indicated that expression of OsMADS14, OsMADS15, OsMADS18, and OsMADS34 in those calli was strongly induced during the reproductive phase. In addition, OsMADS1, OsMADS5, and OsMADS7 were transiently induced in greening calli before that phase.

# MATERIALS AND METHODS

# Plant growth and sampling

Rice plants (var. japonica; ‚ÄòNipponbare‚Äô) were grown in a paddy field at Yongin, Korea. Shoot apexes containing vegetative SAMs were sampled at two stages: V1, approximately 70 days after sowing (DAS); and V2, at 737ÃÉ6 DAS. The transition stage V/R occurred at approximately 79 DAS. Shoot apical regions containing reproductive SAMs were isolated at four stages, as described by Tamaki et al. (2015). The R1 stage was at approximately 82 DAS, when the first bract was formed; R2, at approximately 85 DAS, when PBM development was initiated; R3, at approximately 88 DAS, when PBMs were elongating; and R4, at approximately 91 DAS, when SBM development was initiated. We also sampled inflorescences at three stages, as described by Itoh et al. (2005). Stage In6 indicated the time at which the inflorescences were 1.5 ùóÜ ùóÜ long while spikelet development was beginning; In7, inflorescences approximately 20 ùóÜ ùóÜ long, and floral organs starting to form; and In8, panicles approximately 200 ùóÜ ùóÜ long and reproductive organs now mature.

# Vector construction and rice transformation

For generation of the RFT1-overexpression construct, the 537-bp full length cDNA was amplified by PCR using mRNA prepared from japonica rice (‚ÄòDongjin‚Äô) and a pair of primers (CGCAAGCTTATGGCCGGCAGCGGCAGGGA and GCACTA GTCTAGGGGTAGACCCTCCTGC, where underlined sequences are HindIII and SpeI enzyme sites, respectively). The PCR fragment was cloned into the pGA3426 binary vector under the control of the maize ubiquitin 1 promoter using the restriction enzyme sites (Kim et al., 2009). After checking its quality by DNA-sequencing, we transformed the construct into Agrobacterium tumefaciens LBA4404. Transgenic rice plants were generated by the stable transformation method, as previously reported (An et al., 1988). The putative transgenics were confirmed via selection on hygromycin and transferred to shoot induction media containing 40 mg L^-1 hygromycin, 2.5 ùóÜ ùóÄ ùñ´^-1 kinetin, and 0.1 m g L^-1 NAA.

# RNA isolation and quantitative RT-PCR analyses

Total RNA was isolated from the samples of various tissue types using RNAiso Plus (TaKaRa, Japan; http://www.takarabio. com), and was quantified by a Nano Nanodrop ND-2000 Spectrophotometer (Thermo Scientific, USA, http://www. nanodrop.com), as described previously (Cho et al., 2018). The cDNA was prepared with 10 ng of the oligo (dT)18 primer, 2.5 mM deoxy ribonucleotide triphosphates, and Moloney murine leukemia virus reverse transcriptase. Quantitative real-time RT-PCR (qRT-PCR) was performed with the synthesized cDNAs as templates and the gene-specific primers (Supplementary Table S1), using SYBR Premix Ex TaqÔÉ§ II (TaKaRa) and the Rotor-Gene 6000 instrument system (Corbett Research, Sydney, Australia; http://www.corbettlifescience.com). Rice ubiquitin 1 (Ubi1) served as an internal control, and at least three biological replicates were analyzed. We used the data only when the melting curve showed a single sharp peak.

# Histochemical analysis

Samples of various tissues were fixed in formalin‚Äìacetic acid‚Äì alcohol (FAA) solution after vacuum-infiltration, as described previously (Yoon et al., 2014; 2017). After incubation overnight at 4^‚àò , the samples were dehydrated through an ethanol series (50, 70, 90, and 100% ). They were treated with a tert-butyl alcohol series and paraffin was infiltrated. The fixed tissues were sectioned to 10¬≠m thickness with a microtome (model 2165; Leica Microsystems, http://www. leica-microsystems.com/). After attachment to coated slides, the samples were rehydrated with 100% Histo choice for clearing, using an ethanol series (100, 70, 50, and 30% ) and distilled water. They were then stained with toluidine and observed under a BX61 microscope (Olympus, http://www. olympus-global.-com/en/).

# Statistical analyses

The P values were generated by ANOVA with the Tukey HSD test using the test groups R program (Cohen and Cohen, 2008).

# RESULTS

# Overexpression of RFT1 induces extremely early flowering

Overexpression of Hd3a prompts extremely early flowering (Izawa et al., 2002; Kobayashi et al., 2012; Kojima et al., 2002). However, the roles of RFT1 when over-expressed have not been as thoroughly investigated. To expand on our knowledge about the functions of that gene, we generated transgenic calli expressing full-length RFT1 cDNA under the control of the maize ubiquitin 1 promoter. Among the 30 independently transformed calli, three regenerated leaves and plantlets (Figs. 1A-1C), but did not proceed to mature vegetative stages. Instead, they produced small panicles with only a few spikelets (Figs. 1D-1F). Results from qRT-PCR analyses showed that RFT1 was expressed in all three transgenic plants, indicating that the early flowering phenotypes were due to its overexpression (Fig. 1G).

We assayed leaf samples to study regulatory genes that are affected by overexpression of RFT1. Two FUL-clade MADS box genes, OsMADS14 and OsMADS15, were examined because they are considered downstream genes of RFT1 in the SAM (Komiya et al., 2009). Our data indicated that expression levels of both MADS box genes were significantly higher in the transgenic leaves when compared with transcript levels detected in the leaves of control plants that had been transformed with the empty vector (Figs. 1H and 1I). This observation supported our proposal that they are immediate target genes of RFT1. Expression of OsMADS18, which belongs to the same clade with OsMADS14 and OsMADS15, was also enhanced in the transgenic leaves (Fig. 1J).

We also studied SEP clade genes because some members are associated with the development of inflorescence meristems (Gao et al., 2010; Kobayashi et al., 2012). Transcript levels of OsMADS34 were much higher in the RFT1- expressing leaves than in the control (Fig. 1K). As a key regulatory gene, OsMADS34 is preferentially expressed in inflorescence meristems and controls the development of inflorescences and spikelets (Kobayashi et al., 2012). In contrast, the transcript level of another SEP clade gene, OsMADS1, was decreased in the transgenic leaves (Fig. 1L). Expression levels of the other SEP clade genes ‚Äì OsMADS5, OsMADS7, and OsMADS8 ‚Äì were extremely low in the control leaves, and overexpression of RFT1 did not significantly alter their expression (Figs. 1M-1O). These findings indicated that O S^- MADS34 is the only SEP member that is stimulated in the leaves by RFT1. Finally, expression of OsMADS50, which is homologous to the SUPPRESSOR OF OVEREXPRESSION OF CO 1 (SOC1) in Arabidopsis (Lee et al., 2004), was suppressed in RFT1-overexpressing leaves (Fig. 1P).

Florigen proteins induce downstream genes together with OsFD1 (Kobayashi et al., 2012). Analysis of three FD-like genes revealed that expression of OsFD1 did not differ between RFT1-expressing leaves and WT leaves (Fig. 1Q). However, transcript levels of OsFD2 and OsFD3 were much higher in those transgenic leaves (Figs. 1R and 1S).

Because transient activation of Hd3a and RFT1 expression in leaves suppresses transcript levels of endogenous Hd3a, RFT1, and Ehd1 (Brambilla et al., 2017), we measured their transcript levels to determine whether similar suppression occurred in transgenic leaves that over-express RFT1. Our assay results showed that Ehd1 expression remained at a very low level in those leaves (Fig. 1T). Whereas endogenous RFT1 expression was also unaltered (Fig. 1U), transcript levels of Hd3a were reduced in the RFT1 leaves (Fig. 1V).

# RFT1 expression directly induces spikelet development from regenerating calli

All of the remaining 27 calli directly developed into spikelets without plant regeneration (Figs. 2C and 2D). Seven transgenics were selected to analyze expression for the introduced RFT1. Here, RFT1 was expressed at high levels in all of the transformed calli, whereas calli derived from the vector alone as a control did not express the gene (Fig. 2A).

Approximately one month after transfer to shoot induction medium, RFT1-overexpressing calli developed leaf-like organs (Fig. 2C) that were significantly different from the leaves developed from WT calli (Fig. 2B). Those leaf-like organs were shorter than normal but the number produced from each callus was higher than that of leaves developing from the regenerated WT plants. The leaf-like organs were stacked together to form spikelet-like structures. After rapid generation of multiple leaf-like organs, spikelets appeared at the tops of the shoots (Fig. 2D). Occasionally, multiple carpels developed that were often fused to each other (Figs. 2D1, 2D6 and 2D7). Although stamens also formed, they usually numbered fewer than 6 and were morphologically abnormal (Fig. 2D).


Fig. 1. Expression analyses of regulatory genes controlling reproductive development in transgenic plants expressing RFT1. (A-C) Three independent plants regenerated from RFT1-expressing calli. LB, leaf blades; LS, leaf sheath; S, spikelet. (D-F) Spikelets from three independent plants. (G) Transcript levels of RFT1 in transgenic plants #8, #9, and #10, compared with control plant transformed with empty vector. Transcript levels of O S M A D S14 (ùñß) , OsMADS15 (I), OsMADS18 (J), OsMADS34 (K), OsMADS1 (L), OsMADS5 (M), OsMADS7 (N), OsMADS8 (O), OsMADS50 (P), OsFD1 (Q), O S F D2 (R), OsFD3 (S), Ehd1 (T), endogenous RFT1 (U), and Hd3a (V) in leaves. Expression levels are relative to OsUbi1. Error bars indicate standard deviation for 6 biological replicates. Scale bar =2 ùóÜ ùóÜ .‚ãÜ and ‚ãÜ‚ãÜ indicate significant difference at ùóâ<0.05 and p<0.01 , respectively.


Fig. 2. Phenotypes of RFT1-overexpressing transgenics. (A) Transcript levels of RFT1 in 7 independent transgenics (#1 through #7) compared with control plant transformed with empty vector. Expression levels are relative to OsUbi1. (B) Regenerating transgenic plant with empty vector. Scale bar =5 ùóÜ ùóÜ . (C) Transgenic plants #1 (C1), #2 (C2), #3 (C3), #4 (C4), #5 (C5), #6 (C6), and #7 (C7). Bars =5 ùóÜ ùóÜ .(D) Close-up of spikelets from #1 (D1), #2 (D2), #3 (D3), #4 (D4), #5 (D5), #6 (D6), and #7 (D7). Bar Œ†_ùóÇ=2Œ†_ùóÜ ùóÜ . (E) Regenerating leaves from control plant transformed with empty vector. Red bar indicates location of cross-cut. (F and G) Cross section of leaves from control plant. Asterisks indicate vascular bundles. (H) Spikelet from RFT1-overexpressing transgenic at Stage 3. Red bar indicates location of cross-cut. (I and J) Cross section of spikelet from RFT1-overexpressing transgenic. Asterisks indicate vascular bundles. L, lemma; P, palea. Scale bars =2 ùóÜ ùóÜ (E, H) or 100 m (F, G, I, and J).

To investigate the nature of those leaf-like organs, we fixed them in paraffin and observed their cross cut sections under the microscope (Fig. 2H). The samples possessed three or five vascular bundles that resembled palea or lemma, respectively (Figs. 2I and 2J). In contrast, leaves from WT calli carried more than five vascular bundles, similar to leaves from normal plants (Figs. 2E-2G). This finding indicates that the multiple palea/lemma-like organs arose directly from the RFT1-transgenic calli, bypassing the usual vegetative development and branching.


Fig. 3. Expression analyses of regulatory genes during spikelet development in RFT1- overexpressing transgenics compared with WT. (A) Stage 1. RFT1-overexpressing transgenic callus with greening tissues approximately 1 month after shoot induction. (B) Stage 2. Transgenic callus with multiple leaf-like organs, approximately 6 d after Stage 1. (C) Stage 3. Floret inside spikelet from transgenic, approximately 5 d after Stage 2. (D-F) Control plants transformed with empty vector, visualized at stages equivalent to ST1, ST2, and ST3 of RFT1-expressing transgenics. Scale bar: 2 ùóÜ ùóÜ . Transcript levels of OsMADS14 (G), OsMADS15 (H), OsMADS18 (I), OsMADS34 (J), OsMADS1 (K), OsMADS5 (L), OsMADS7 (M), OsMADS8 (N), OsMADS50 (O), OsFD1 (P), OsFD2 (Q), OsFD3 (R), RFL (S), OsMADS2 (T), OsMADS3 (U), and OsMADS4 (V). Expression levels are relative to OsUbi1. Error bars indicate standard deviation for 6 biological replicates.

# Expression patterns of regulatory genes during spikelet development from calli

Molecular events during spikelet development from the calli were monitored using RNA extracted from tissues at three different stages: Stage 1, greening undifferentiated tissues at approximately one month after shoot induction (Fig. 3A);

Stage 2, meristem tissues covered by the palea/lemma-like organs at approximately 6 d after Stage 1 (Fig. 3B); and Stage 3, florets with carpels and stamens at approximately 5 d after Stage 2 (Fig. 3C). We collected WT samples at similar stages even though they had differentiated to leaves rather than spikelets (Figs. 3D-3F).


Fig. 4. Expression analyses of regulatory genes in inner and outer organs of spikelet developed from RFT1-overexpressing transgenics. (A) Spikelet at Stage 2. (B) Outer parts of spikelet at Stage 2. (C) Meristem tissue located inside spikelet at Stage 2. (D) Spikelet at Stage 3. (E) Outer parts of spikelet at Stage 3. Scale bar: 2 ùóÜ ùóÜ (A, B, D, E, and F) or 1 ùóÜ ùóÜ (C). (F) Meristem tissue located inside spikelet at Stage 3. (G-R) Transcript levels of OsMADS14 (G), OsMADS15 (ùñß) , OsMADS18 (I), OsMADS34 (J), OsMADS1 (K), OsMADS5 (L), OsMADS7 (M), OsMADS8 (N), OsMADS50 (O), OsMADS2 (P), OsMADS3 (Q), and OsMADS4 (R) at 2 different stages. Expression levels are relative to OsUbi1. Error bars indicate standard deviation for 6 biological replicates.

Analyses of three FUL-clade MADS box genes (OsMADS14, OsMADS15, and OsMADS18) showed that all were strongly expressed at Stage 2 in meristem tissues that had been induced by RFT1 expression (Figs. 3G-3I). Expression of OsMADS14 was very low at Stages 1 and 3 (Fig. 3G). Levels of OsMADS15 transcripts were similar to those of OsMADS14, except that the former was expressed at a higher level than the latter in the floral organs at Stage 3 (Fig. 3H). Expression of OsMADS18 was similar to OsMADS15 but was detected at a moderate level at Stages 1 and 3 (Fig. 3I). Transcript levels for these three genes were low in WT samples at all three stages (Figs. 3G-3I). Our observations are consistent with the report that OsMADS14 and OsMADS15 expression is activated by RFT1 in the SAM (Komiya et al., 2008; 2009).

Transcript of OsMADS34 was present in Stage 2 RFT1 samples, but not at Stages 1 and 3 (Fig. 3J). This was consistent with previous reports that the SEP-clade gene functions in inflorescence meristems (Kobayashi et al., 2012). However, another SEP gene, OsMADS1, was specifically expressed at Stage 1 in the greening tissues of RFT1 plants (Fig. 3K). Transcripts of OsMADS5 were present in all three stages (Fig. 3L), while OsMADS7 transcripts were found mainly at Stages 1 and 2 (Fig. 3M), and OsMADS8 transcripts at Stages 2 and 3 (Fig. 3N). Greening tissue-specific expression was also observed for OsMADS50 when RFT1 was over-expressed (Fig. 3O).

Three FD-like genes ‚Äì OsFD1, OsFD2, and OsFD3 ‚Äì were expressed at all three stages in RFT1-expressing samples (Figs. 3P-3R), as was the RFL (Fig. 3S). In contrast, the floral organ identity genes OsMADS2, OsMADS3, and OsMADS4 were  expressed  in  the  Stage  3  samples  from RFT1- expressing calli, which confirming that the carpel- and stamen-like structures were indeed floral organs (Figs. 3T-3V).

# Leaf-like organs covering inflorescence meristems are palea/lemma

Results from anatomical analyses suggested that the leaf-like organs developed from undifferentiated green calli of RFT1- transgenics resembled palea and lemma. We separated the outer leaf-like green organs from the non-green organs located inside (Figs. 4A-4F) and determined that the FUL genes, which are involved in inflorescence meristem development, were preferentially expressed in those inner organs at Stage 2 (Figs. 4G-4I). Another inflorescence marker, OsMADS34, was also strongly expressed in the inner organs, especially at Stage 2 (Fig. 4J). These results confirmed that the inner organs are inflorescence meristems. In contrast, OsMADS1 was expressed at higher levels in the outer organs at both Stages 2 and 3 (Fig. 4K). Its relative expression was almost equivalent to that of the Ubi1 control. OsMADS1 was expressed at a lower level in the WT leaves and was further decreased in the RFT1-transgenic leaves (Fig. 1L).


Fig. 5. Transcript levels of florigen and FD genes in leaf blades at different developmental stages. Leaf blade samples were isolated at 3-d intervals starting from 70 DAS. Expression of RFT1 (A), Hd3a (B), OsFD1 (C), OsFD2 (D), and OsFD3 (E). Transcript levels are relative to OsUbi1. Error bars indicate standard deviation for 6 biological replicates.

This obser vation supported our proposal that the outer organs developed from RFT1-expressing calli were not leaves. Because the primary roles of OsMADS1 are in regulating lemma/ palea development (Jeon et al., 2000; Zhang et al., 2018), the outer organs were most likely palea/lemma in the spikelets. Expression of other SEP genes ‚Äì OsMADS5, OsMADS7, and OsMADS8 ‚Äì did not differ significantly between the inner and outer organs (Figs. 4L-4N), while that of OsMADS50 was higher in the outer organs (Fig. 4O). As we had expected, the floral organ identity genes OsMADS2, OsMADS3, and OsMADS4 were specifically expressed in the inner organs at Stage 3 (Figs. 4P-4R).

# Analysis of regulatory genes during plant development in the paddy

We investigated whether the differential expression patterns of the regulatory genes observed during inflorescence development from the calli are similar to those noted during normal meristem development in paddy-grown WT plants. Starting at 70 DAS, we collected leaf blades and shoot apexes at 3-d intervals. Analyses of RFT1 expression in the leaf samples showed low transcript levels until 76 DAS, which was followed by a rapid increase at 79 DAS, a peak at 85 DAS, and then a rapid decline (Fig. 5A). Hd3a expression presented a similar pattern, peaking at 85 DAS (Fig. 5B). These observations indicated that florigen molecules were starting to be produced at 79 DAS and reached a maximum level at 85 DAS.

We also studied expression levels of OsFD1 in the leaves because that gene appears to form a complex with the florigen proteins and induces flowering by increasing the expression of Ehd1, RFT1, and Hd3a (Brambilla et al., 2017). Transcript levels of OsFD1 were very low in the leaves at all stages, although they seemed to be slightly higher during the floral transition period when the florigen genes were actively expressed (Fig. 5C). In contrast, OsFD2 and OsFD3 were much more highly expressed, especially during those transitional stages (Figs. 5D and 5E).

We sampled vegetative SAMs and determined that Stage V1 occurred at 70 DAS while Stage V2 covered 73 to 76 DAS (Figs. 6A and 6B). Reproductive meristem regions were collected at R1, R2, R3, and R4 (Figs. 6D-6G). We also sampled panicles at In6, In7, and In8.

Transcript levels of OsMADS14 were very low in the vegetative SAM before the transition, but they increased during that stage and rose to higher levels at R2 through R4 before declining to moderate levels during inflorescence stages 6 to 8 (Fig. 6H). In mature leaves, this MADS box gene was expressed at a very low level. Very similar patterns were observed for OsMADS15 except that the gene was expressed at much lower levels in the vegetative SAMs (Fig. 6I). In contrast, OsMADS18 was highly expressed at almost the same levels in the meristems at all developmental stages as well as in the mature leaves (Fig. 6J).

The expression pattern of OsMADS34 was similar to that of OsMADS15, being highly expressed during the period of primary and secondary branch development (Fig. 6K). However, OsMADS1, OsMADS5, and OsMADS8 were highly expressed at later stages, during inflorescence formation (Figs. 6L, 6M and 6O), while OsMADS7, OsMADS50, and RFL did not show any specific patterns (Figs. 6N, 6P and 6S). As we had expected, the floral organ identity genes OsMADS2 and OsMADS4 were expressed strongly at the later stages of inflorescence development (Figs. 6Q and 6R). Similar to the pattern detected in the leaves, expression of OsFD1 was much lower than that of OsFD2 and OsFD3 in the meristems. Transcript levels for OsFD1 were higher during the transition and in the early reproductive stages but were low during inflorescence development (Fig. 6T). OsFD2 and OdFD3 were expressed during all stages, and at almost identical levels (Figs. 6U and 6V).

# DISCUSSION

# Overexpression of RFT1 induces extremely early flowering

We demonstrated that strong expression of RFT1 induced extremely early flowering in transgenic rice plants. A similar early flowering phenotype due to Hd3a overexpression has been reported (Izawa et al., 2002; Kobayashi et al., 2012; Kojima et al., 2002). Whereas most Hd3a transgenics produce plants that flower early and have only a few spikelets (Izawa et al., 2002; Kojima et al., 2002), we found here that a majority of our RFT1-transformed calli directly developed spikelets without the formation of any vegetative organs. Therefore, RFT1 appears to be a stronger promoter of reproductive development. It will be needed to investigate how RFT1 functions more strongly than Hd3a, especially because much of the study on floral transition in rice has been conducted with Hd3a. One possibility is that RFT1 and Hd3a may form a FAC complex with different partner. Another possibility is that their direct targets may be different to each other.

Studies with Arabidopsis, rice, and Triticum aestivum have indicated that florigen proteins do not have activation functions, and formation of FAC with FD is needed to stimulate downstream expression of the target genes (Abe et al., 2005; Taoka et al., 2011; Wigge et al., 2005). Because overexpression of the florigen gene alone induces early flowering, one might think that other components in the FAC are not limiting factors during the transition to the reproductive stage. However, we observed that expression of OsFD1 was highly induced in the RFT1-transgenic calli much before than OsMADS14 and OsMADS15 were induced (Fig. 3P), suggesting that RFT1 induces OsFD1 expression first in order to strongly promote target genes. This hypothesis is consistent with the report that OsMADS14 and OsMADS15 expression is strongly increased in the leaves of OsFD1-overexpression plants (Brambilla et al., 2017). It will be interesting to evaluate whether FAC directly enhances OsFD1. However, the latter was not enhanced in leaves from RFT1-overexpressing plants (Fig. 1R). Therefore, we propose that a repressor is possibly present in the leaf to suppress OsFD1 expression. Alternatively, a positive element residing in reproductive tissues may be absent in the leaves.

Transient overexpression of RFT1 and Hd3a downregulates endogenous Ehd1, Hd3a, and RFT1 in the leaf (Brambilla et al., 2017). However, we did not observe such a strong feedback suppression of endogenous RFT1 and Ehd1 in RFT1-overexpressing leaves, although Hd3a expression was somewhat reduced (Figs. 1U-1W). This discrepancy may be due to the nature of overexpression of the florigen genes. Whereas we constitutively expressed RFT1, Brambilla et al., (2017) transiently stimulated the gene.


Fig. 6. Expression analyses of regulatory genes in SAM of WT plants. Shoot apex regions in paddy-grown WT plants, harvested at vegetative stages V1 (A) and V2 (B), transition stage (C), and reproductive stages R1 (D), R2 (E), R3 (F), and R4 (G). Scale bars: 200 m . Transcript levels of OsMADS14 (H), OsMADS15 (I), OsMADS18 (J), OsMADS34 (K), OsMADS1 (L), OsMADS5 (M), OsMADS7 (N), OsMADS8 (O), OsMADS50 (P), OsMADS2 (Q), OsMADS4 (R), RFL (S), OsFD1 (T), OsFD2 (U), and OsFD3 (V). Expression levels are relative to OsUbi1. Error bars indicate standard deviation for 6 biological replicates.

# Overexpression of RFT1 stimulates regulatory genes that are associated with reproductive development

The FUL-clade MADS box genes are major regulatory elements responsible for initiating reproductive development (Kobayashi et al., 2012). In rice, OsMADS14, OsMADS15, and OsMADS18 function redundantly in that process. OsMADS34 is also required for the transition (Kobayashi et al., 2012). When spikelets were developing directly from greening RFT1 tissues, expression of those genes was strongly induced, preferentially in the reproductive meristematic tissues (Figs. 3G-3J). Therefore, our observations confirmed previous reports that these genes coordinately function in reproductive meristems.

The SEP-clade genes OsMADS1, OsMADS5, and OsMADS7 primarily mediate floral organ development (Jeon et al., 2000; Kobayash et al., 2012). They are expressed later than the FUL-clade genes in developing inflorescences from paddy-grown plants. However, we noted here that the SEP genes were highly expressed in greening RFT1 tissues before the spikelets formed (Figs. 3K-3M). This unexpected result suggested that the SEP genes have additional roles during the very early stages of reproductive development. Because they were induced before the FUL-clade genes, we must still examine whether the SEP-clade genes are also direct targets of RFT1-associate FAC. One possibility is that the SEP-clade genes are transiently required for the initiation of inflorescence development from greening tissues. OsFD1 and related genes OsFD2 and OsFD3 could possibly be involved in their induction.

Whereas L E A F Y(L F Y) in Arabidopsis is expressed consistently in FM and functions primarily in specifying them by directly activating A P7 (Weigel et al., 1992), the rice LFY homolog, RFL, is expressed mainly in incipient lateral branch primordia, where it is associated with maintenance of the inflorescence meristem (Rao et al., 2008). The rice protein suppresses the transition from inflorescence meristem to FM through  interactions  with  APO1,  the  rice  ortholog  of UNUSUAL FLORAL ORGANS in Arabidopsis (Ikeda-Kawakatsu et al., 2012). We observed that expression levels of RFL were induced in the RFT1-overexpressing calli at all stages (Fig. 3S), thereby supporting its role during reproductive development. OsMADS50 is an ortholog of Arabidopsis SOC1, which is up-regulated in the meristem and stimulates early fiowering (Lee et al., 2004; Ryu et al., 2009) This rice gene induces flowering preferentially under long-day conditions, but its role in the SAM during reproductive induction has been unknown. We observed that it was strongly induced in the RFT1 greening tissues (Fig. 3O). During spikelet development, the gene was expressed preferentially in the palea/lemma tissues (Fig. 4K). These observations suggested that OsMADS50 functions in green tissues during the formation of inflorescences. An interesting future study would be to examine whether suppression of OsMADS50 is necessary to develop those green tissues into reproductive tissues.

# RFT1 overexpression does not immediately stimulate reproductive development

Although inflorescences developed directly from our transgenic calli, that process required more than one month in the shoot induction medium for such initiation. In addition, such reproductive development occurred only from the green tissues. This indicated that RFT1 expression alone was insufficient. One possible explanation is that other components of the FAC were lacking in the undifferentiated calli, a hypothesis supported by our observation that OsFD1 was induced in the green calli. Further investigation is needed on whether the FAC components OsFD1 and G f/4c are expressed in undifferentiated tissues. In addition to O s F D 7. OsFD2 and OsFD3 were also strongly induced in the RFT1- expressing tissues (Figs. 3P-3R). Therefore, it will be interesting to investigate whether RFT1 also forms a complex with OsFD2 and OsFD3. If they interact to each other, further study would be needed to evaluate whether the different complexes induce the same target genes or different downstream genes.

Note: Supplementary information is available on the Molecules and Cells website (www.molcells.org).

# ACKNOWLEDGEMENTS

This work was supported in part by a grant from the Next Generation BioGreen 21 Program (Plant Molecular Breeding Center; No. PJ013210), Rural Development Administration, Republic of Korea and by the Republic of Korea Basic Research Promotion Fund (Grant No. NRF-2018R1A6A3A110 47894). We thank Priscilla Licht for her critical proofreading of the manuscript.
