# A Major QTL, Ghd8, Plays Pleiotropic Roles in Regulating Grain Productivity, Plant Height, and Heading Date in Rice

Wen-Hao Yan2, Peng Wang2, Hua-Xia Chen, Hong-Ju Zhou, Qiu-Ping Li, Chong-Rong Wang, Ze-Hong Ding, Yu-Shan Zhang, Si-Bin ğ˜ğ®^1 , Yong-Zhong Xing1 and Qi-Fa Zhang

National Key Laboratory of Crop Genetic Improvement and National Center of Plant Gene Research (Wuhan), Huazhong Agricultural University, Wuhan 430070, China

ABSTRACT Rice yield and heading date are two distinct traits controlled by quantitative trait loci (QTLs). The dissection of molecular mechanisms underlying rice yield traits is important for developing high-yielding rice varieties. Here, we report the cloning and characterization of Ghd8, a major QTL with pleiotropic effects on grain yield, heading date, and plant height. Two sets of near isogenic line populations were developed for the cloning of Ghd8. Ghd8 was narrowed down to a 20-kb region containing two putative genes, of which one encodes the OsHAP3 subunit of a CCAAT-box binding protein (HAP complex); this gene was regarded as the Ghd8 candidate. A complementary test confirmed the identity and pleiotropic effects of the gene; interestingly, the genetic effect of Ghd8 was dependent on its genetic background. By regulating Ehd1, RFT1, and Hd3a, Ghd8 delayed flowering under long-day conditions, but promoted flowering under short-day conditions. Ghd8 up-regulated MOC1, a key gene controlling tillering and branching; this increased the number of tillers, primary and secondary branches, thus producing 50% more grains per plant. The ectopic expression of Ghd8 in Arabidopsis caused early flowering by 10 dâ€”a situation similar to the one observed by its homolog AtHAP3b, when compared to wild-type under long-day conditions; these findings indicate the conserved function of Ghd8 and AtHAP3b in flowering in Arabidopsis. Our results demonstrated the important roles of Ghd8 in rice yield formation and flowering, as well as its opposite functions in flowering between rice and Arabidopsis under long-day conditions.

Key words: Near isogenic lines; CCAAT-box binding protein; ectopic expression; flowering pathway; yield formation.

# INTRODUCTION

Rice yield and heading date are two distinct traits that are controlled by quantitative trait loci (QTLs). The two traits are influenced both by the plantâ€™s genetic makeup and environmental factors. Many yield-related QTLs/genes have been isolated in rice in recent years. For example, Moc1 and IPA1 regulate tillers (Jiao et al., 2010; Li et al., 2003); GS3, GW2, and GW5/qSW5 control grain shape and grain weight (Fan et al., 2006; Shomura et al., 2008; Song et al., 2007; Weng et al., 2008), and Gn1a, APO1, and DEP1 regulate grain number (Ashikari et al., 2005; Huang et al., 2009; Ikeda et al., 2007). Ghd7, which encodes a CCT-domain-containing transcription factor, was reported to regulate rice grain yield, plant height, and heading date. The natural variation of Ghd7 is an important regulator of heading date and yield potential in rice (Xue et al., 2008).

The adaptation of flowering plants is largely determined by their flowering time, which is mainly controlled by photoperiod and temperature (Izawa, 2007). Arabidopsis is a typical long-day (LD) flowering plant whose flowering pathways have been well studied: the key flowering mechanism in Arabidopsis is mediated by CO (CONSTANS), which regulates the production of the florigen (FT) (Corbesier et al., 2007; Kardailsky et al., 1999; Kobayashi et al., 1999; Putterill et al., 1995; Samach et al., 2000). In contrast, rice is a typical short-day (SD) flowering plant whose flowering (heading) is greatly affected by day length. Recently, a series of rice homologs of the genes in the COâ€“FT pathway, namely Hd1 and Hd3a, have been identified (Kojima et al., 2002; Yano et al., 2000). The function of Hd1 is opposite to that of its Arabidopsis homolog, CO. Under long-day conditions, Hd1 delays heading in rice, whereas CO promotes flowering in Arabidopsis (Putterill et al., 1995; Yano et al., 2000). Ehd1, a rice-specific flowering regulator, plays a crucial role in the photoperiodic control pathway of rice heading by regulating the expression of the rice florigen genes, Hd3a and RFT1 (Doi et al., 2004; Komiya et al., 2008, 2009). Suppression of Ehd1 delays heading under SD conditions (Doi et al., 2004). Ehd1- and Hd1-mediated rice flowering pathways have been regarded as independent (Doi et al., 2004; Izawa, 2007; Komiya et al., 2009). Recently, the heterotrimeric heme activator protein (HAP) complex, composed of the three subunits, HAP2, HAP3, and HAP5, was reported to interact with CO to regulate flowering in Arabidopsis (Ben-Naim et al., 2006; Wenkel et al., 2006). The HAP complex binds to the CCAAT box, a cis-acting element present in approximately 25% of eukaryotic gene promoters (Bucher, 1990; Maity and de Crombrugghe, 1998; Mantovani, 1999). In yeast and mammals, each subunit of the HAP complex is encoded by a single gene, while in plants, it is encoded by a gene family (Gusmaroli et al., 2001; Thirumurugan et al., 2008). In rice, there are 11 homologous genes (OsHAP3Aâ€“OsHAP3K) encoding the OsHAP3 unit (Thirumurugan et al., 2008). Four HAP3 genes were functionally analyzed (Masiero et al., 2002; Miyoshi et al., 2003); the OsHAP3A, OsHAP3B, and OsHAP3C genes are involved in chloroplast biogenesis (Miyoshi et al., 2003).

Undoubtedly, the identification of more rice-yield-related and flowering genes/QTLs will provide us with more opportunities to breed diverse high-yield varieties (Zhang et al., 2008). Understanding the molecular mechanism of such genes could optimize the relationship between yield and heading date. In addition, the functional dissection of more gene-regulated yield traits with different approaches has provided us with the chance to further investigate the molecular mechanisms underlying rice yield (Hu et al., 2008; Luo et al., 2008; Wang et al., 2009). In our previous study (Zhang et al., 2006), we identified a major QTL on chromosome 8, Ghd8, which controls grain yield, heading date, and plant height. We report here the cloning and characterization of Ghd8. Ghd8 encodes the OsHAP3 subunit of the HAP complex, and it acts upstream of Ehd1, Hd3a, and RFT1. Ghd8 coordinates the development of rice panicles and heading.

# RESULTS

# Fine Mapping of Ghd8

In our previous study, the near isogenic line (NIL) of a rice QTL simultaneously controlling plant height, heading date, and the number of grains per panicle on chromosome 8 (referred to as Ghd8) was obtained within an ğ–¥_7 inbred line following a trait-performance-derived NIL strategy; the male parent HR5 allele contributed to delayed heading, tall culms, and large panicles (Zhang et al., 2006). Ghd8 greatly contributed to rice heading, plant height, and yield-related traits (Figure 1Aâ€“1D). The ğ–­ ğ–¨ ğ–«-ğ–¥_2 (HR5) population of 2256 individuals was investigated for grains per panicle, heading date, and plant height. In total, 326 plants with early heading, a small panicle, and a short culm were selected for Ghd8 fine mapping. Ghd8 was narrowed down to a 70-kb region flanked by the markers SEQ3-1 and SEQ5-1 (Figure 2A and Supplemental Table 1). According to the Rice Genome Annotation Project and the RiceGE (Rice Functional Genomic Express) Database, 11 genes were predicted within this region. The gene Loc_08g07740 was predicted to encode a histone-like transcription factor and archaeal histone family protein (CBFD_NF-YB, CCAAT box-binding factor) (http://pfam.janelia.org/search/sequence). In addition, another ğ–­ ğ–¨ ğ–«-ğ–¥_2(93-1 1) population containing 17 800 plants was used for fine mapping the QTL and finally narrowed Ghd8 down to a 20-kb region (Figure 2B). There were two predicted genes in the region containing Loc_08g07740; therefore, Loc_08g07740 was the likely candidate for Ghd8.

# Complementary Test of Ghd8

Comparative sequencing of the gene Loc_08g07740 among the four parental lines Zhenshan 97 (ZS97), 93â€“11, ZS, and HR5 revealed many nucleotide variations, which caused amino acid changes. HR5 shared the same allele as Nipponbare and had many polymorphisms in both the promoter and the Figure 1. Phenotypes of NILs and Transgenic Rice Transformed with Different Alleles.



(Aâ€“D) Phenotype of NILs. Image of whole plants (A), main stems (B), panicles (C), and grain yield per plant (D) for NILZS (left) and NILHR5 (right). (Eâ€“I) Phenotypes of NILs and transgenic rice with different alleles. The plants are ğ–­ ğ–¨ ğ–«^25 (E), ğ–­|ğ–«^Zğ–²- (F), NILHR5 (G), ğ–­ğ–¨ğ–«^ğ–­ ğ—‚ ğ—‰+ (H), ğ–­ğ–¨ ğ–«^93-11+( I), respectively. The photograph was taken when N l l^Z S reached maturity. Bar =20Â ğ–¼ ğ—† .

coding region of Loc_08g07740, compared to ZS and ZS97 (both carried nonfunctional alleles). In particular, the single nucleotide polymorphism (SNP) at the position +322 bp from the initiation codon ATG caused a frameshift mutation and led to a premature termination codon in ZS. In ZS97, a deletion of 1116 bp in the 3^' region resulted in a truncated protein that lacked seven amino acids in the C-terminus, compared to 93â€“11 (Figure 2C). This suggested that Loc_08g07740, which encodes the HAP3 subunit, may be the candidate for Ghd8.

To test this prediction, we introduced both Ghd8 alleles of 93â€“11 (Ghd8 (93â€“11)) and Nipponbare (Ghd8(Nip)) by Agrobacterium-mediated transformation into NILZS, which has a small panicle, short stature, and early heading compared to ğ–­ ğ–¨ ğ–«^ğ–§ ğ–±5 . The T_7 families of the T_0 transformants containing a single-copy of the Ghd8 (93â€“11) and (Ghd8 (Nip)) alleles were analyzed for their phenotypic effects under natural LD conditions (13â€“14-h day length). The positive T_1 plants showed significantly more grains, delayed heading and taller culms compared to the transgenic negative plants (Figure 1). The negative Ghd8 transgenic plants (ğ–­ğ–¨ğ–«^ZS-) showed a performance similar to their recipient line N l l^Z S in heading date, grain yield, grains per panicle, and plant height (Table 1). On average, a positive T1 plant (ğ–­|ğ–«^93-11+/ğ–­|ğ–«^ğ–­ ğ—‚ ğ—‰+) resulted in increases above 60% in grain yield per plant and grains per panicle compared to the recipient ğ–­ ğ–¨ ğ–«^Z S and the negative T1



Figure 2. High-Resolution Ghd8 Fine Mapping and Protein Alignment of the Four Mapping Alleles.
(A) Fine map of Ghd8 using the NIL- Â·ğ–¥_2 (HR5) population.
(B) Fine map of Ghd8 using the NIL- Â·ğ–¥_2 (93â€“11) population. The numbers below the markers represent the number of recombinants between the markers and Ghd8.
(C) Protein alignment of Ghd8 from the predicted protein of four parental lines (9311, HR5 or Nipponbare, ZS97 and ZS) using the EMBL software CLUSTAL 2.0.12 multiple sequence alignment tool.

Table 1. Trait Measurements of NILs and Transgenic Plants in the T_1 Generation derived from ğ–³_0 Plants with a Single-Copy under Natural Field Conditions in 2009.


HD, heading date; PH, plant height; GPP, grains per panicle in main culm; GY, grain yield. NILZS and NILHR5 mean the near isogenic line carrying homozygous alleles for ZS and HR5, respectively. ğ–­ğ–¨ ğ–«^+ and ğ–­ğ–¨ ğ–«^- mean the positive and negative transgenic plants carrying Nipponbare and 93â€“11 alleles with ğ–­ğ–¨ğ–«^Z S as the receipt.

plant (ğ–­ğ–¨ğ–«^ZS-) (Figure 1). The ğ–­ğ–¨ğ–«^ğ–­ ğ—‚ ğ—‰+ plant showed a performance similar to that of the NIL carrying the HR5 homozygotes (ğ–­ğ–¨ ğ–«^ğ–§ ğ–±5) in grain yield, grains per panicle, and plant height; however, the transformants with the Ghd8 (93â€“11) allele were delayed in flowering by 9 d, thus indicating the strong effect of the 93â€“11 allele on heading date (Table 1). In summary, the complementary test confirmed that the candidate gene was in fact Ghd8.

# Characterization of Ghd8

The expression of Ghd8 and other OsHAP3 genes has been detected in 37 RNA samples of all tissues from ZS97 at different developmental stages using Affymetrix rice gene chips (Wang et al., 2010). Ghd8 is expressed at a low level in all organs at different developmental stages, like OsHAP3H (namely Ghd8 in this study), whose expression was too low to be detected by RTâ€“PCR in the tissues investigated (Thirumurugan et al., 2008). However, three OsHAP3 genes were highly expressed in all tested organs at different developmental stages. One gene (Loc_OS02g49410) showed preferential expression in the panicle and endosperm (Supplemental Figure 1). These findings indicated that different members of the OsHAP3 family have different functions in the rice lifecycle. To check the spatio-temporal expression of Ghd8 under LD conditions, RNA in situ hybridization was performed with roots, young leaves, stems, and young panicles of NILHR5. Ghd8 was highly expressed in the phloem of leaves, the vascular system of the stem, and especially in cells of the inflorescence meristem. The inflorescence meristem formed bracts and inflorescence branches (Itoh et al., 2005), which probably determined panicle size (Supplemental Figure 2).

To assay the sub-cellular localization of GHD8, the coding sequence of GHD8 was fused to yellow fluorescent protein (YFP); in contrast, GHD7, which is a nuclear protein, was fused to cyan fluorescent protein (CFP). Both fluorescent proteins were individually driven by the constitutive 35S cauliflower mosaic virus promoter. The constructs were co-transfected into rice protoplasts of etiolated seedlings by polyethylene glycol. GHD8 co-localized to the nucleus with GHD7 (Supplemental Figure 3).

# Photoperiod Sensitivity of Ghd8

We tested the phenotypes of ğ–­ ğ–¨ ğ–«^Z S and ğ–­ ğ–¨ ğ–«^ğ–§ ğ–±5 under SD (9-h light, â†‘5Â·ğ— dark) and LD ( 13.5.6 light and 10.5.ğ— dark) conditions. There was no difference in grains per panicle, heading date, and plant height under SD conditions (Table 2). In contrast, NILHR5 was delayed in heading by 7.8 d and was accompanied by an increase of 17.3Â ğ–¼ ğ—† in plant height and by an average of 81.6 more spikelets per panicle under LD conditions (Table 2). Accordingly, the heading date of the transgenic plant ğ–­ğ–¨ ğ–«^Zğ–²+ was delayed by up to ^20Â d compared to the negative transgenic plant ğ–­ ğ–¨ ğ–«^Zğ–²- (Table 1). In contrast, Z S97^ğ–­ ğ—‚ ğ—‰ showed earlier heading by 3.6 d under SD conditions and later heading by 1.5 d under LD conditions, when compared to ZS97. A similar phenotypic variation pattern was also observed in transgenic plants Z S97^ğ–­ ğ—‚ ğ—‰+ (Table 2). Overall, these results showed that Ghd8 is sensitive to photoperiod.

# Ghd8 Action on Heading Date

To understand the role of Ghd8 in flowering in rice, we compared the expression patterns of Ghd8 and five other key genes (Hd1, Ehd1, OsMADS50, Hd3a, and RFT1) in the photoperiodic flowering pathway between ğ–­ ğ–¨ ğ–«^ğ–§ ğ–±5 and ğ–­ğ–¨ğ–«^Zğ–² (Doi et al., 2004; Kojima et al., 2002; Komiya et al., 2009; Ryu et al., 2009; Yano et al., 2000). Under SD conditions, the expression patterns of Ghd8, Ehd1, and Hd3a were almost identical between the two NILs (Supplemental Figure 4). This result agreed with the finding that there was no difference in heading date between the two NILs under SD conditions (Table 1). Under LD conditions, the expression levels of Ehd1, Hd3a, and RFT1 in ğ–­ ğ–¨ ğ–«^ğ–§ ğ–±5 were much lower than those in NILZS, but the expression levels of Hd1 and OsMADS50 were not different between ğ–­ ğ–¨ ğ–«^Z S and ğ–­ ğ–¨ ğ–«^ğ–§ ğ–±5 . This indicated that the expression of Hd1 and OsMADS50 is not affected by Ghd8 (Supplemental Figure 5). When compared to ZS97, Z S97^ğ–­ ğ—‚ ğ—‰ flowered earlier and later under SD and LD conditions, respectively (Table 2). That is to say, the Ghd8 allele of Nipponbare promoted flowering under SD conditions and inhibited flowering under LD conditions in ZS97. We also compared the diurnal expression of OsMADS51 (Kim et al., 2008) under SD conditions, that of OsMADS50 under LD conditions and that of OsGI, Hd1, Ehd1, Hd3a, and RFT1 under both conditions between positive and negative transgenic plants (Zğ–²97^ğ–­ ğ—‚ ğ—‰+ and Z S97^ğ–­ ğ—‚ ğ—‰-) . There was no difference in diurnal expression for OsGI, OsMADS51, OsMADS50, and Hd1, but a distinct difference was observed for Ehd1, Hd3a and RFT1 between Z S97^N i p+ and Z S97^ğ–­ ğ—‚ ğ—‰- (Figure 3A and 3B). These results indicated that Ghd8 acts upstream of Ehd1, Hd3a, and RFT1.

Table 2. Performance of Near Isogenic Lines for Ghd8 under Short-Day and Long-Day Conditions in 2010.


HD, heading date; PH, plant height; GPP, grains per panicle in main culm; GY, grain yield.,  mean the significant difference between the ZS97 back ground near isogenic lines and ZS97, and Ghd8 transgenic lines ZS97 Nip+ and ZS97 Nipâ€“ at the level of P<0.05 and 0.01 by t-test. No difference under SD conditions but significant difference (ğ–¯<0.01) ) in heading date, plant height, and grains per panicle in main culm under LD conditions between NILHR5 and ğ–­|ğ–«^Z S .

# Ghd8 Action on Panicle Development

We investigated the panicle architecture of ğ–­ ğ–¨ ğ–«^ğ–§ ğ–±5 and ğ–­ ğ–¨ ğ–«^Z S grown under natural LD conditions. Significant differences were observed in panicle length and the number of primary and secondary branches (Supplemental Figure 6). These results indicate that Ghd8 affected panicle size by acting on panicle branching development. To determine whether Ghd8 functioned in the shoot apex at the vegetative or reproductive stage, we detected its mRNA in the shoot apical meristem (SAM) of young seedlings, the SAM before phase transition, the inflorescence meristem of primary and secondary branch differentiation stages, and the floret meristem, using RNA in situ hybridization. Ghd8 was highly expressed in cells with meristem activity during almost all developmental stages under LD; however, low Ghd8 expression occurred in SD (Figure 4). To investigate which genes contribute to panicle phenotype, the expression of 10 genes (Supplemental Table 2) associated with panicle architecture was detected in young panicles at the secondary branch initiation stage (length of 1 ğ—† ğ—†, ) from the two genotypes of NIL and N l l^ Z S+ by quantitative PCR. The expression level of MOC1 was significantly higher in N l l^ Z S+ than in ğ–­ğ–¨ğ–«^Zğ–²- . Both quantitative PCR and RNA in situ hybridization showed that the expression of MOC1 in the panicle meristem was higher in NILHR5 than in ğ–­ğ–«^Z S (Figure 5).

# Ghd8 Function in Arabidopsis

Ghd8 encodes a CCAAT box-binding protein, which is present in animals and plants and which belongs to the HAP3 subfamily. The rice genome contains 12 HAP3 genes (http://drtf.cbi. pku.edu.cn/), while 13 genes encode HAP3 subunits in Arabidopsis. An amino acid sequence comparison of these 25 HAP3 genes in rice and Arabidopsis showed that Ghd8 was the homolog of AtHAP3b gene (Supplemental Figure 7), which was confirmed to control flowering in Arabidopsis. To test the function of Ghd8 in Arabidopsis, we transferred Ghd8 into the Arabidopsis thaliana Columbia ecotype. The transformants flowered about 10 d earlier than the wild-type under LD conditions, but no phenotypic change was observed under SD conditions (Figure 6A and 6B); these findings mirrored those of AtHap3b on flowering (Cai et al., 2007; Chen et al., 2007). In other words, flowering was delayed in a null mutant of HAP3b, and overexpression of HAP3b caused early flowering under LD conditions. No phenotypic variation was observed under SD within both mutant and overexpressing plants. We also compared the expression patterns of the five flowering genes GI, CO, HAP3a, HAP3b, and FT between transgenic plants and wild-type plants. There was no difference in the expression patterns of GI, CO, HAP3a, and H A P3b between the two genotypes, but a distinct increase in F T was detected in transgenic plants (Figure 6C); the latter scenario was similar to that caused by AtHAP3b. These results indicated that the function of Ghd8 in flowering is conserved between rice and Arabidopsis.

# DISCUSSION

# Ghd8 Coordinates Panicle Development and Heading Date

The cloned QTLs/genes that control the number of grains per panicle can be classified according to their functions, into the three following groups: genes (L A X7,F Z P, and MOC1) that underlie the process of panicle development, genes (G n 7a,A P O 1, ,and log) that regulate the rate of spikelet formation, and genes (Ghd7) that regulate panicle differentiation (Xing and Zhang, 2010). In the present study, NIL  had longer panicles and more primary and secondary branches, which led to more grains per panicle, when compared with NILZS; ğ–­ ğ–¨ ğ–«^ğ–§ ğ–±5 also showed an increase in yield under LD conditions. The Ghd8 transgenic plants showed delayed heading, and increased grains per panicle resulted from the increase in primary and secondary branches, when compared to the wild-type. The pleiotropic effects of Ghd8 were also confirmed in the study of DTH8 (Wei et al., 2010); DTH8 is the same gene as Ghd8. Ghd8 was highly expressed in cells with meristematic capacity, perhaps ensuring that more lateral organs develop. Ghd8 upregulated MOC1, which is involved in tiller branching (Li et al., 2003). Other than its effects on tillers, MOC1â€“3 (namely SPA) regulated axillary meristem formation during rice panicle development (Oikawa and Kyozuka, 2009). Therefore, Ghd8 likely functions in branch development during rice panicle development, to produce more lateral panicle organs, and ultimately more spikelets. Thus, Ghd8 coordinates panicle development and heading date. However, in Arabidopsis, the AtHAP3 family has not been reported to contribute to inflorescence development.


Figure 3. Diurnal Expression of Rice Flowering Genes between the transgenic positive and negative lines. Diurnal expression of OsGI, Hd1, OsMADS51, Ehd1, Hd3a, and RFT1 under SD (A) and OsGI, Hd1, OsMADS50, Ehd1, Hd3a, and RFT1 under LD (B) in Z S97^ğ—‡ ğ—‚ ğ—‰+ and Z S97^ğ—‡ ğ—‚ ğ—‰- , respectively. Each point represents three biological repeats (mean Â± standard error (SE)).


Figure 4. Expression Patterns of Ghd8 during Panicle Development under LD and SD Conditions.
(Aâ€“E) In situ hybridization of Ghd8 in the shoot apical meristem at the vegetative stage (A), shoot apical meristem before phase transition (B), primary branch initiation (C), secondary branch initiation (D), floret meristem (E) under LD conditions and primary branch meristem differentiation (G, H), secondary branch differentiation (I) under SD conditions, and (F) sense probe control; the sample was harvested at 15:00. Bar =50 ğ—† .
Figure 5. Comparison of MOC1 Expression Levels during Panicle Development between N l l^Z S and NILHr5. (A) Relative expression levels of MOC1 in ğ–­ ğ–¨ ğ–«^Z S and ğ–­ ğ–¨ ğ–«^ğ–§ ğ—‹5 (samples for RNA extraction were collected from young panicles at secondary branch initiation). (Bâ€“D) In situ hybridization of MOC1 during panicle development in ğ–­ ğ–¨ ğ–«^Z S (B) and ğ–­ ğ–¨ ğ–«^ğ–§ ğ–± ğ–² (C); (D) is the sense probe. Bar =50Â ğ—† .

In rice, two independent pathways control flowering; one is regulated by Hd1 and the other by rice-specific flowering regulator Ehd1 (Doi et al., 2004; Komiya et al., 2009; Yano et al., 2000). In the present study, the diurnal expression data showed that Ghd8 was located upstream of Ehd1, Hd3a, and RFT1 in the photoperiodic flowering pathway. The genetic interaction between Hd5 (namely Ghd8) and Hd1 was reported in a QTL study (Lin et al., 2003). Moreover, Ghd8 up-regulated the expression of Ehd1, Hd3a, and RFT1 under short-day conditions, but down-regulated their expression under long-day conditions; this pattern was similar to the regulating rhythm of Hd1. Such results may indicate that Ghd8 links the Hd1- and Ehd1-dominated pathways to control flowering, consequently changing the previous notion that Hd1 acts in parallel with Ehd1 to control flowering.


Figure 6. Ectopic Expression of Ghd8 in Arabidopsis and Expression of Related Flowering Genes. (A) Phenotype of transgenic plant (right) and wild-type (left). (B) Days to flowering of transgenic plant and wild-type plants. (C) Comparison of expression for flowering-related genes between transgenic plant and wild-type plant (mean Â± Sğ–¤) ).

Ourpreviouslyclonedgene,Ghd7,haspleiotropiceffectssimilar to those of Ghd8. Ghd7 encodes a CCT domain-containing protein, which is similar to the yeast HEME ACTIVATOR PROTEIN2 (HAP2). Ghd8 encodes an HAP3 subunit. HAP proteins always form a HAP2/HAP3/HAP5 trimeric complex with DNA binding activity in mammals (Goda et al., 2005; Tuncher et al., 2005). In rice, each HAP subunit is encoded by a gene family; therefore, various possibilities were available to form different HAP complexes, which regulate multi-developmental processes. This explanation may account for the pleiotropic effects of these Ghd7/Ghd8 genes. In addition, the relationship between Ghd7 and Ghd8 could be very interesting. However, like Zhenshan 97 (Xue et al., 2008), all the NILs of Ghd8 in this study werelackingGhd7alleles,becausenoampliconwasobtainedin the NILs with a pair of Ghd7-specific primers. NILs carrying functional alleles of both Ghd7 and Ghd8 are being developed for the dissection of their genetic relationship.

# Effects of Ghd8 Depend on the Genetic Background

We observed a delay of 6 d in heading in ğ–­ ğ–¨ ğ–«^ğ–§ ğ–±5 compared to the Z S97^N i p under LD conditions. While no difference in flowering was observed under SD conditions between NILHR5 and NILZS, Z S97^N i p promoted flowering, when compared to ZS97 (Table 2). Ehd1 was up-regulated in Z S97^N i p+ compared to Z S97^ğ–­ ğ—‚ ğ—‰- under SD conditions, whereas there was no difference in Ehd1 expression between NILHR5 and ğ–­ ğ–¨ ğ–«^Z S . However, HR5 and Nipponbare share the same Ghd8 allele. These results indicated that Ghd8 has different effects under different genetic backgrounds, and there may be another unknown factor that down-regulates Ehd1 in the ğ–­ ğ–¨ ğ–«^Z S background. Furthermore, the transgenic plants ğ–­ğ–¨ğ–«^93-11+ showed a delay in flowering of 9 d, while it had more grains and tillers than in NILNip+ transgenic plants, indicating that the different alleles have different effects. HAP proteins always function in the form of the HAP2/HAP3/HAP5 complex (Goda et al., 2005; Tuncher et al., 2005). Consequently, we speculated that the Ghd8 effects depend on the members of HAP2 and HAP5 forming the complex in the genetic background. Thus, for a given Ghd8 allele, the constitutions of the HAP complex were determined by other subunits, which resulted in varied functions in different genetic backgrounds. Identification of the interacting subunits would allow breeders to better manipulate the gene for adaptability and high-yielding improvement breeding.

# Opposite Function of Ghd8 in Flowering between Rice and Arabidopsis

Flowering was delayed in the mutant Arabidopsis gene, but flowering was promoted in plants overexpressing HAP3b under LD conditions (Cai et al., 2007). The same phenotype could be observed in the ectopic expression of Ghd8 in Arabidopsis. These results indicate that Ghd8 and HAP3b have conserved functions in Arabidopsis flowering.

In Arabidopsis, the CO-mediated photoperiod flowering pathway, whereby CO acts upstream of F T and downstream of _G I to promote flowering under LD conditions, has been well studied (Park et al., 1999; Putterill et al., 1995; Samach et al., 2000). Thus far, no clear result has shown that CO itself can bind DNA directly. It was reported that CO interacts independently with HAP3 and HAP5 (Wenkel et al., 2006). The CCT domain of the CO protein shares amino acid sequence similarities with the DNA-binding domain of the HAP2 protein (Ben-Naim et al., 2006; Wenkel et al., 2006). Overexpression of AtHAP2 caused late flowering in Arabidopsis. Taken together, these results indicated that CO may replace HAP2 to form the CO/ HAP3/HAP5 complex to promote flowering (Wenkel et al., 2006). Bioinformatic analysis revealed that Ghd8 was the closest homolog of AtHAP3b, and Hd1 was the CO homolog in rice (Yano et al., 2000). In addition, an epistatic interaction between Hd1 and Hd5 (Ghd8 here) was confirmed (Lin et al., 2003). The complex of GHD8/HD1/HAP5 was expected to be formed to control flowering in rice. In this study, Ghd8 acted upstream of Hd3a/RFT1, the rice homolog of F T, and the expression of Hd1 was identical between NILs of Ghd8, which may indicate that Hd1 acts upstream of Ghd8. More than that, a similar result was also obtained during the ectopic expression of Ghd8 in Arabidopsis. Thus, we propose that the HAP3b/Ghd8 genemediated flowering pathways are conserved between Arabidopsis and rice, but they have opposite functions in flowering under LD. Meanwhile, Ghd8 also acted on the rice-specific gene, Ehd1, to regulate flowering in rice (Figure 7).

Overall, the cloning of Ghd8 has provided a rare opportunity for studying the molecular mechanism of the association between grain yield and heading. Characterization of such mechanisms will lead to a series of discoveries of fundamental importance in plant biology.

# METHODS

# Plant Material

To clone Ghd8, the ğ–­ ğ–¨ ğ–«-ğ–¥_2 population (ğ–­|ğ–«-ğ–¥_2(ğ–§ğ–±ğ–²)) was obtained from a heterozygous ğ–¥_7 plant derived from a cross between an old version of Zhenshan 97 (ZS) and HR5 (Zhang et al., 2006). The plants in the population that carried HR5 and ZS homozygous alleles were named ğ–­ ğ–¨ ğ–«^ğ–§ ğ–±5 and ğ–­ ğ–¨ ğ–«^ZS , respectively. We also developed an ğ–­ ğ–¨ ğ–«-ğ–¥_2 population (ğ–­ ğ—… ğ—… - ğ–¥_2(93- 11)) from a backcross recombinant inbred plant (ğ™± ğ™²_1ğ–¥_8) , in which 93â€“11 was the recipient and new version of Zhenshan 97 (ZS97) was the donor. We developed two NILs that carried Ghd8 alleles of 93â€“11 and Nipponbare in the ZS97 background by four or five consecutive backcrosses. They were named Z S97^93-11 and ZS97 Nip.

# Transformation of Ghd8 to Rice and Arabidopsis thaliana

A clone that contained a candidate of Ghd8 was selected to test the function of the candidate from the 93â€“11 BAC library. A 10-kb SacI fragment containing the DNA sequence 3.6Â ğ—„ ğ–» upstream the transcription start site, the Ghd8 ORF, and the 5.7- kb downstream region from a BAC clone were constructed into the binary vector pCAMBIA1301. In addition, a 3.7-ğ—„ ğ–» fragment of Nipponbare containing 1.4Â ğ—„ ğ–» of DNA sequence upstream of the initiation codon, the Ghd8 ORF, and 1.2Â ğ—„ ğ–» of the 3^' UTR was constructed into binary vector pCAMBIA1301, respectively. Both fragments contained only one predicted gene, and they were transformed to ğ–­ ğ–¨ ğ–«^Z S and ZS97, respectively, by Agrobacterium-mediated transformation, using the EHA105 strain. The cauliflower mosaic virus 35S promoter was used to drive the expression of GHD8 (HR5).


Figure 7. Ghd8-Mediated Flowering Pathways Were Conserved in Arabidopsis and Rice CO/Hd1 Acts Upstream of HAP3b/Ghd8, and HAP3b/ Ghd8 Regulates FT/Hd3a/RFT1, which Affects Flowering. Either Ghd8 or HAP3b could promote flowering in Arabidopsis under LD conditions. In addition, Ghd8 could regulate Ehd1, which can act on Hd3a/RFT1, thus affecting flowering in rice.

The 35S::Ghd8 construct was introduced to Arabidopsis thaliana (Columbia ecotype) using the floral dipping method (Clough and Bent, 1998). Eight transgenic-positive individuals were selected on half MS medium supplemented with 25 ğ”¤ ğ—†|^-1 of hygromycin. Two transgenic lines, each with a single copy gene insertion, were used for further analysis. The seeds of homozygous transgenic and wild-type plants were sown directly in soil for 10Â d under day-neutral conditions (12 ğ—/12 ğ—) ; then, the seedlings were transferred for the long-day (LD) condition treatment (16-h light/8-h dark photoperiod) and short-day (SD) length treatment (16-h dark/8-h light photoperiod). Flowering time was recorded when the first flower appeared. Leaves were collected for expression analysis. Each leaf sample was composed of five individuals of the same genotype grown under the same conditions.

# Sub-Cellular Localization of Ghd8

The coding sequence of Ghd8 (HR5) was fused to PM999â€“YFP. The fusion protein with the insertion in the right direction was co-transfected into rice protoplasts with GHD7:CFP as described by Zhou et al. (2009), with minor modifications. The fluorescence image was obtained using a confocal microscope (Leica, Germany) after incubating the transformed cells in the dark at 28^âˆ˜C for ^20Â h .

USA) for RNA transcription. The respective sense and antisense probes were produced by SP6 and T7 transcriptase labeled with digoxigenin (Roche, Mannheim, Germany). Plant tissues were collected under LD or SD conditions and fixed in FAA solution 50% ethanol, 5% acetic acid, and 3.7% formaldehyde) at 4^âˆ˜ğ–¢ overnight after vacuum pumping. RNA in situ hybridization and immunological detection were carried out as described previously by De Block (1993).

# SUPPLEMENTARY DATA

Supplementary Data are available at Molecular Plant Online.

# FUNDING

This work was partly supported by grants from the National Natural Science Foundation of China (30830064, 30921091), the National Special Program for Research of Transgenic Plant of China (2008ZX08009â€“001), the National Key Program on Basic Research Projects, the Program for New Century Excellent Talents in University, and the Huazhong Agricultural University Scientific     Technological Self-innovation Foundation (2009YB006).

# ACKNOWLEDGMENTS

The authors kindly thank Dr Y. Zhao for her help with in situ hybridization, and the farm technicians Mrs J.B. Wang and J. Yu for their excellent field work. The authors also thank Dr M.Z. Luo for kindly providing the BAC clone and Dr J. Xu and Mr L. Wang for kindly providing the vectors. No conflict of interest declared.

# Diurnal Expression Analysis

After growing under neutral day-length conditions for 35 ğ–½, ,the rice plants were transferred to a growth chamber to receive light treatment. The conditions were set as follows: 15Â ğ”¥ dark and ^9Â h light for SD; 9 h dark and 15Â ğ”¥ light for LD, with a temperature of 30^âˆ˜ğ–¢ for the light period and 26^âˆ˜ğ–¢ for the dark period. The leaves of the main culm of each plant were harvested every 4 ğ”¥ , for 24 or 48 h . For each time point, leaves from two or three different individuals were collected as biological replicates. RNA was extracted using the TRIzol Reagent (Invitrogen, CA, USA) and treated with DNase I (Invitrogen, CA, USA). cDNA was synthesized from 2 ğ”¤ of RNA using SuperScript III Reverse Transcriptase (Invitrogen, CA,USA). The quantitative analysis of gene expression was performed with ğ–² ğ–¸ ğ–¡ ğ–±^(p) Premix Ex Taq' (TaKaRa, Otsu, JAPAN) on an Applied Biosystems 7500 Real-time PCR System. The data were analyzed with the relative quantification method.

# RNA In Situ Hybridization

Ghd8 was amplified by the of primer pair, C8dsF and C8dsR (Supplemental Table 1). The product contained 340Â ğ–» ğ—‰ that were located at the 3^' end of the mRNA; this product was then inserted into the pGEM-T vector (Promega, Madison,
