# EF8 is involved in photoperiodic flowering pathway and chlorophyll biogenesis in rice

Zhiming Feng âˆ™ Long Zhang âˆ™ Chunyan Yang âˆ™ Tao Wu âˆ™ Jia Lv âˆ™ Yilin Chen Â· Xi Liu Â· Shijia Liu Â· Ling Jiang âˆ™ Jianmin Wan

Received: 21 June 2014 / Revised: 29 July 2014 / Accepted: 4 August 2014 / Published online: 21 August 2014
Springer-Verlag Berlin Heidelberg 2014

# Abstract Key message A DTH8/Ghd8 allele suppresses flowering by altering the expression patterns of the â€˜florigenâ€™ genes, but also negatively regulates chlorophyll biogenesis.

Abstract Flowering time is a critical agronomic trait determining the growing season, regional adaptation and yield potential in rice (Oryza sativa L.). We characterized a mutant named early flowering 8 (ef8) selected from an ethyl methanesulfonate (EMS)-treated population of indica cultivar 93-11. It showed earlier flowering, less grains per main panicle and slightly darker green leaves than the wildtype 93-11 under natural long-day conditions, but was not significantly different from 93-11 under natural short-day conditions. We isolated the Early Flowering 8 (EF8) gene by map-based cloning. EF8 encodes a putative HAP3 subunit of the CCAAT-box-binding transcription factor, which is localized to the nucleus. EF8 was expressed in various tissues, especially in leaves, with a rhythmic expression pattern. Our data showed that EF8 delayed flowering time under long-day conditions by altering the rhythmic expression patterns of â€˜florigenâ€™ genes H d3a and RFT1. We also found that EF8 negatively regulates the expression of chlorophyll biosynthetic genes to reduce the chlorophyll content. Our data indicate that E F8 plays an important role in rice photoperiodic flowering pathway as well as yield potential and chlorophyll biogenesis and will be an important target for rice breeding programs.

Keywords Rice (Oryza sativa L.) Â· Flowering time Â· Photoperiod Â· Chlorophyll biogenesis

# Abbreviations

Chl a Chlorophyll a
Chl b Chlorophyll b
cv Cultivar
EMS Ethyl methanesulfonate
GFP Green fluorescent protein
HAP Heterotrimeric heme activator protein
LDs Long-day conditions
NLDs Natural long-day conditions
NSDs Natural short-day conditions
ORF Open reading frame
QRT-PCR Quantitative real-time PCR
SDs Short-day conditions

# Introduction

In flowering plants, the transition from vegetative to reproductive development is regulated by multiple pathways (Simpson and Dean 2002). The appropriate timing of this transition is important in maximizing reproductive fitness and is controlled by adaptation of plants to their natural environment, in response to photoperiod and temperature (Izawa 2007). Flowering time is a major agronomic trait and determines the growing season and region to which a plant genotype is adapted. Given its importance, flowering time has attracted much attention from plant breeders and scientists (Srikanth and Schmid 2011).

Arabidopsis thaliana is a model long-day plant for which flowering mechanisms controlled by photoperiod have been well documented (Corbesier et al. 2007). In long-day conditions (LDs), the circadian clock gene GIGANTEA (G I) regulates a zinc finger transcription factor gene CONSTANS (C O) (Huq et al. 2000; Kobayashi and Weigel 2007; Park et al. 1999), and CO promotes the expression of FLOWERING LOCUS T (FT) by binding to its promoter (Corbesier et al. 2007; Kardailsky et al. 1999; Kobayashi et al. 1999; Putterill et al. 1995; Samach et al. 2000; Tiwari et al. 2010), which is regulated by phytochrome A (phy A), phytochrome B (phy B) and cryptochrome 2 (cry 2) through post-translational modification of the CO protein (Valverde et al. 2004; Yanovsky and Kay 2002). Furthermore, the FLAVIN-binding Kelch Domain F-box protein 1 (FKF1) forms a protein complex with GI to regulate the expression of CO in a blue-light-dependent manner (Sawa et al. 2007).

Rice (Oryza sativa L.), one of the main food crops of humanity, is a model short-day plant. Breeding highyielding cultivars combined with moderate maturity is a major target for rice breeders. Many factors influence yield, including grain number and flowering time. Some genes regulating grain number have already been identified, such as Gn1a, APO1 and DEP1 (Ashikari et al. 2005; Huang et al. 2009; Ikeda et al. 2007). A counterpart of the G I/-C O_- FT pathway in Arabidopsis, namely the OsGIGANTEA (OsGI)â€“Heading date 1 (Hd1)â€“Heading date 3a (Hd3a) pathway, is present in rice (Hayama et al. 2003; Kojima et al. 2002; Yano et al. 2000). The function of Hd1 is dual: it promotes flowering by regulating the expression of Hd3a in short-day conditions (SDs), but suppresses flowering in LDs, whereas CO promotes flowering in LDs in Arabidopsis (Komiya et al. 2008; Putterill et al. 1995; Yano et al. 2000). Early heading date 1 (Ehd1), encoding a rice-specific B-type response protein, positively regulates Hd3a and RICE FLOWERING LOCUS T 1 (RFT1) gene expression, and then promotes flowering under both LDs and SDs (Doi et al. 2004; Komiya et al. 2009). OsMADS51, encoding a type I MADS-box protein, promotes Ehd1 expression under SDs (Kim et al. 2007), whereas the role of promotion of OsMADS50 to Ehd1 is in LDs (Lee et al. 2004). Ghd7 (grain number, plant height and heading date 7), encoding a CCT domain protein, suppresses expression of Ehd1 and Hd3a, and thereby delays flowering under LDs (Xue et al. 2008). DTH8/Ghd8, encoding a putative HAP3 subunit of the CCAAT-box-binding transcription factor, regulates the expression of Ehd1, Hd3a and RFT1, suppressing flowering and increasing plant height and grain number under LDs (Wei et al. 2010; Yan et al. 2011). Ehd4 (Early heading date 4), encoding a highly conserved rice-specific CCCH-type zinc finger protein, up-regulates the expression of Hd3a and RFT1 through Ehd1, promoting flowering (Gao et al. 2013). DTH2 (Days to heading on chromosome 2), encoding a CONSTANS-like protein, induces Hd3a and RFT1 independently of Hd1 and Ehd1 to promote flowering (Wu et al. 2013). However, the regulatory networks of photoperiodinduced flowering in rice are still unclear.

Some features of the HAP (heterotrimeric heme activator protein) complex, binding to the CCAAT-box and consisting of the three subunits, HAP2 (NF-YA/CBF-B), HAP3 (NF-YB/CBF-A) and HAP5 (NF-YC/CBF-C) (Maity and de Crombrugghe 1998; Mantovani 1999), were identified. In yeast and mammals, a single gene encodes each subunit of the HAP complex, whereas, in plants, all subunits are encoded by gene families (Gusmaroli et al. 2001, 2002; Mantovani 1999). In Arabidopsis, the numbers of members in gene families encoding the AtNF-YA, AtNF-YB and AtNF-YC subunits are 10, 13 and 13, respectively (Gusmaroli et al. 2002). In rice, the numbers of member in gene families encoding OsHAP2, OsHAP3 and OsHAP5 subunits are 10, 11 and 7, respectively (Thirumurugan et al. 2008). In Arabidopsis, CO interacts with the HAP complex to regulate flowering (Ben-Naim et al. 2006). Also, the HAP3 genes, LEAFY COTYLEDON ^ l and LEC1-LIKE, play important roles in regulating embryogenesis (Kwong et al. 2003; Lotan et al. 1998). In rice, the OsHAP3A, OsHAP3B and OsHAP3C genes were reported to influence the amount of chlorophyll and chloroplast biogenesis (Miyoshi et al. 2003). However, the roles of other members of these rice HAP protein families are still not well understood.

Here, we characterized an early-flowering mutant ef8 with pleiotropic phenotypes under natural long-day conditions (NLDs) and isolated the EF8 gene by map-based cloning. The EF8 gene, which was an allele of DTH8/ GHD8 (Wei et al. 2010) from cultivar (cv) 93-11, was found to encode a putative HAP3 subunit of the CCAATbox-binding transcription factor. Our data showed that EF8 is sensitive to photoperiod and delayed flowering time by influencing the rhythmic expression patterns of the â€˜florigenâ€™ genes under LDs. EF8 was also found to play an important role in the regulation of grain number per panicle and chlorophyll biogenesis.

# Materials and methods

# Plant material and growth conditions

A stably inherited ef8 mutant, which exhibits earlier flowering, less grain number and slightly darker green compared to wild type, was identified from an ethyl methanesulfonate (EMS)-treated population of rice cv 93-11 (indica) under NLDs in Nanjing (day length >14Â h ), China in 2010. To analyze the mutant phenotype, the mutant and its wild type were grown under natural shortday conditions (NSDs) (about 11.6 h light/12.4 h darkness) in Lingshui County, Hainan Province. The pre-germinated seeds were sown in nursery beds in natural fields and then the 1-month seedlings were transplanted into rice fields. All plants were grown under the same environmental and artificial conditions. Days to flowering of each plant were identified and recorded when the fist panicle emerged. In addition, 93-11 and e f8 mutant were grown in LDs ^16Â h light at 28Â ^âˆ˜C/8Â h darkness at 24Â ^âˆ˜C) ) and SDs ( ^8h light at 28 ^âˆ˜C/16 h darkness at 24 ^âˆ˜C, in the growth chamber for expression analysis of E F8 and other genes. For diurnal expression analysis, leaves from three different individuals were collected every 4Â h , for 24Â h , as biological replicates.

# Map-based cloning of EF8

For genetic analysis, a population of 413 F_2 was obtained from a cross between e f8 mutant and wild-type 93-11 and the flowering time of every plant was recorded. For primary mapping of EF8, 2,000 Â F_2 plants were generated from a cross between e f8 mutant and japonica cv 02428 and 97 individuals with clear e f8 phenotypes were selected for mapping this locus using the method described by Zhang et al. (1994). To further fine map this locus, a population of about 4,400 BC_1F_2 plants was planted and 452 recessive early-flowering individuals were selected. Additional molecular markers were designed by comparing the sequences of indica cv 93-11 and japonica cv Nipponbare. The molecular markers chosen for mapping are listed in Supplemental Table S1. PCR was performed following the procedure: 95Â ^âˆ˜C for 5Â min , followed by 33 cycles of 95Â ^âˆ˜C for 30Â s , annealing for 30 s, 72 ^âˆ˜ğ‚ for 40Â s and a final elongation step at 72Â ^âˆ˜ğ‚ for 5Â min .

# RNA isolation and quantitative real-time PCR analysis

Total RNA samples were extracted from young leaves and young roots of 2-week-old seedlings from the shoot apices of 1-month-old seedlings and from mature leaves, mature roots, leaf sheaths, culms and young panicles at the booting stage using an RNA Prep Pure Plant kit (Tiangen Co., Beijing, China) and were reverse transcribed using a SuperScript II kit (TaKaRa). Quantitative real-time PCR (QRT-PCR) was performed using an SYBR Premix Ex TaqTM kit (TaKaRa) on an ABI prism 7500 real-time PCR System and the Actin gene was used as an internal control. The 2-DDCT method was used to analyze relative changes in gene expression (Livak 2001). QRT-PCR primers used for other photoperiodic genes (OsGI, Hd1, Ehd1, Hd3a, OsMADS51, OsMADS50, RFT1) were obtained as reported previously (Wei et al. 2010; Yan et al. 2011). The QRTPCR primers for Actin and chlorophyll biosynthetic genes are listed in Supplemental Table S2.

# Subcellular localization of EF8

To determine the cellular localization of EF8, GFP was fused to the C-terminus of EF8 and ef8 (the protein encoded by the e f8 mutated allele) under the control of the 35S promoter in the PAN580 vector. In addition, the nuclear marker Ghd7â€“ mCherry was constructed. The EF8â€“GFP fusion, ef8â€“GFP fusion or GFP was transiently co-transferred into rice protoplasts with the Ghd7â€“mCherry constructs as described by Bart et al.(2006). Fluorescenceimages were observed usinga Zeiss LSM510 confocal laser microscope.

Determination of pigment content Equal weights of fresh leaves of e f8 mutant and wild-type plants from 1- and 2-month-old seedlings were marinated in 95Â % ethanol for 48Â h in darkness and used to determine chlorophyll contents according to a described method (Arnon 1949). Supernatant absorbances at 665, 649 and 470 nm were measured using a Hitachi U-1800 Spectrophotometer (Tokyo, Japan). Three biological repeats were measured for each sample.


Fig. 1 Phenotypes of wild type (WT) and mutant ef8. a Wild type (WT) and mutant e f8 grown under NSDs in Hainan. e Days to (WT) and mutant e f8 , when mutant e f8 was heading. b Wild type flowering of wild type (WT) and mutant e f8 in NLDs and NSDs. (WT) and mutant e f8 when wild type was mature. c Grains from the f Grain numbers of main panicles of wild type (WT) and mutant ef8 main panicles of wild type (WT) and mutant ef8. aâ€“c Wild type (WT) under NLDs and NSDs and mutant e f8 were grown under NLDs in Nanjing. d Wild type

# Results

Phenotypes of the ef8 mutant We screened our EMS-induced population in NLDs and identified an early-flowering mutant named e f8 which flowered about 7.9 days earlier (98.9Â±1.2 days) than the wild type (WT) ( [106.8Â±1.2 days) under NLDs, but there was no significant difference in flowering time between the ef8 mutant (112.0Â±1.3$ days) and wild type (112.8Â±1.7days) under NSDs (Fig. 1a, b, d, e; Table 1).

We compared other agronomic traits and found that thee f8mutant had about 45.3 less grains per main panicle compared to wild type under NSDs, but there was no significant difference in grain numbers under NSDs (Fig. 1c, f; Table 1). The ef8 mutant also displayed a slightly darker green leaf phenotype in the developmental stages before maturation under LDs, but there was no significant difference under SDs compared to wild type (Fig. 2a, b; Supplemental Fig. S1A, B). We therefore measured chlorophyll a (Chl a) and chlorophyll b (Chl b) contents in 30- and 60-day-old seedlings, under both LDs (16 hlight at28Â ^âˆ˜C/8h darkness at24Â ^âˆ˜C_') and SDs (^8hlight at28Â ^âˆ˜ğ‚/16Â hdarkness at24Â ^âˆ˜C,in the growth chamber. Compared with WT, the ef8 mutant had significantly more Chl a and Chl b under LDs, but not under SDs (Fig. 2c; Supplemental Fig. S1C).

Fine mapping of EF8

For genetic analysis,e f8mutant was crossed to wild-type 93-11. TheF_1generation showed a similar heading date to 93-11. In the F2 population, the ratio of normal heading date to early heading date was 318:95, fitting a 3:1 ratio(Ï‡2=0.88<Ï‡2_0.05, 1=3.84)and suggesting that a single recessive gene controlled the ef8 mutant phenotype. For

Table 1 Comparison of agronomic traits between the wild type (WT) and mutant ef8 under NLDs and NSDs



Data are the average of 20
samplesÂ± SD^**Indicates significant
differences between WT and ef8
atP=0.01preliminary mapping, an F2 mapping population of about 2,000 plants was generated from a cross between ef8 mutant and 02428 (a japonica variety). We selected 97 individuals with cleare f8phenotypes and initially mapped the ef8 locus to the short arm of chromosome 8 (Fig. 3A).

To further fine mapE F8and reduce the effect of the indicaÃ—japonica difference, theF_1, derived from ef8 mutantÃ—Â cvÂ  02428, backcrossed to 02428 and aB C_1F_2population containing 4,400 plants, was generated. Using 452 recessive early-flowering individuals selected from this population, the EF8 gene was mapped to a38Â kbgenomic region between markersFm45and_Fm31and co-segregated withFm40(Fig. 3A). This region contains six open reading frames (ORFs; http://www.gramene.org/; Fig. 2a). All of these genes were sequenced in thee f8mutant and wild type, and by comparison we found that only the sixth ORF(L o c_O s O8g O7740)was changed, with a one base deletion at position 323 bp from the start codon in the ef8 mutant; this caused a frame-shift and premature termination of translation (Fig. 3b,c; Supplemental Fig. S2). In addition, ef8 mutation included 9 bp deletion and 6 bp insertion (Fig. 3b; Supplemental Fig. S2).

Loc_Os08g07740 encodes a putative HAP3 subunit of the CCAAT-box-binding transcription factor and has been named OsHAP3H (Thirumurugan et al. 2008). This protein was reported as a flowering suppressor named DTH8/ GHD8, which has several alleles (Wei et al. 2010; Yan et al. 2011). The full-length CDS of the EF8 allele in cv 93-11 is 891 nucleotides, encoding a protein of 296 amino acids (Supplemental Fig. S2).

Expression pattern and localization of EF8 The expression profile of EF8 was studied using QRTPCR. As shown in Fig. 4a, EF8 was expressed in various tissues, including young leaves, young roots of 2-week-old seedlings, shoot apices at 1-month-old stage and mature leaves, mature roots, leaf sheaths, culms and young panicles at the booting stage. The highest level of expression was in young leaves and the lowest in culms. To investigate whether EF8 expression was regulated by the circadian clock, we checked the diurnal expression pattern of EF8. Our date showed thatE F8expression was rhythmic in leaves under both SDs and LDs (Fig. 4b, c); it started to accumulate4 hafter dawn under SDs and after dawn under LDs, reaching a peak4 hafter dusk under both SDs and LDs, and thereafter falling sharply to reach the lowest points under both SDs and LDs (Fig. 4b, c). In addition, the expression level and pattern of ef8 mutated allele were not changed compared to EF8 (Fig. 4b, c), which indicated that this mutation of EF8 did not affect its expression profile in thee f8mutant.

To investigate subcellular localization of EF8, an EF8â€“ GFP (green fluorescent protein) fusion protein and the GHD7â€“mCherry fusion protein (a nuclear marker) were constructed and co-transfected into rice leaf protoplasts. EF8 co-localized to the nucleus with GHD7 and thus functions in the nucleus (Fig. 4d). What is more, we examined the subcellular localization of the ef8 and found that ef8 also localized the nucleus (Fig. 4d). The ef8 lost 189 amino acid residues in the C-terminal region compared to EF8 (Supplemental Fig. S2), so the nuclear localization signal of EF8 was located in the N-terminal region.


Fig. 2 Phenotype of seedlings of wild type and ef8 mutant under LDs in the growth chamber. a Phenotypes 30-dayold wild-type ande f8seedlings. b Phenotypes 30-day-old wildtype ande f8leaves. c Chlorophyll contents of leaves of wild type ande f8mutants at two developmental stages: 30 and 60 days. Data are meanÂ± SDâŒˆn=3âŒ‰). Error bars represent SD of three independent experiments. Chla chlorophyll a, Chlb chlorophyll b, fr.wt fresh weight
Fig. 3 Map-based cloning ofE F8. a Fine mapping ofE F8.TheE F8locus was mapped to the short arm of chromosome 8 and was narrowed to a38Â kbgenomic DNA region between indel markersFm45andFm31located on bacterial artificial chromosome clone OSJNBa0054L03. b Structure of EF8 in WT and ef8. Arrowheads and small triangular boxes represent deletions and insertions, respectively. The red line represents a frameshift mutation caused by a 1 bp deletion and creates a premature stop codon. c Molecular identification of the 1 bp deletion by a dCAPS marker, d1. The left and rightpanels show the dCAPS analysis associated with wild type (WT) ande f8mutant


Fig. 4 Expression pattern and subcellular localization ofE F8. a EF8 The black and white bars represent dark and light periods, respectranscript levels in different tissues (meanÂ± Sğƒ,n=3).Y Lyoung tively. Each point represents three biological repeats [meanÂ±stanleaves, YR young roots, SA shoot apices,M Lmature leaves,M Rdard error (SE)]. d Subcellular localization of EF8â€“GFP fusion mature roots,L Sleaf sheaths,Cculms,Y Pyoung panicle. Rhythmic protein and ef8â€“GFP fusion protein expression of EF8 ande f8mutated allele under LDs(b)and SDs (c).

# EF8 alters the rhythmic expression patterns of the â€˜florigenâ€™ genes under LDs

Activating the two â€˜florigenâ€™ genes, Hd3a and RFT1, in leaves is required for the floral transition induced by photoperiod in rice (Kojima et al. 2002; Komiya et al. 2008). The rhythmic expression pattern of EF8 suggested that it may be involved in the regulation of flowering pathway genes controlled by photoperiod. To test this, we compared the expression levels of Hd3a and RFT1 between wild type and thee f8mutant by QRT-PCR under LDs and SDs. Under LDs, the expression levels and rhythmic expression patterns of Hd3a and RFT1 were changed in the ef8 mutant compared to wild type (Fig. 5a). The peaks of their expression levels were changed from4 hafter dusk in wild type to4Â hafter dawn in the ef8 mutant, and their expression levels in the ef8 mutant were higher than those in wild type during the entire light period under LDs (Fig. 5a). However, under SDs, there were no significant differences in the rhythmic expression patterns and levels of the two â€˜florigenâ€™ genes between the ef8 mutant and wild type (Fig. 5b).

Two independent floral regulators, Hd1 and Ehd1, control flowering through regulating Hd3a and RFT1 in rice (Izawa 2007; Tsuji et al. 2011). To investigate whether the alteration of expression patterns of the two â€˜florigenâ€™ genes by EF8 is mediated by Hd1, Ehd1 or other regulators of Hd1 and Ehd1, such as OsMADS51, OsMADS50 and OsGI (Hayama et al. 2003; Kim et al. 2007; Lee et al. 2010), we examined their mRNA levels between wild type and the ef8 mutant under LDs and SDs. Under LDs, to our surprise, no significant differences in the expression of any of these genes were detected between ef8 mutant and wild type (Fig. 5a). Under SDs, the expression pattern of Ehd1 was slightly changed in ef8 mutant compared to wild type, but the expression patterns of other genes were not changed (Fig. 5b).

Together, these results indicated that EF8 delayed flowering through altering the rhythmic expression patterns of the â€˜florigenâ€™ genes Hd3a and RFT1 independently of Hd1 and Ehd1 under LDs.


Fig. 5 Rhythmic expression patterns of rice flowering genes comparing ef8 mutant and wild type. The rhythmic expression pattern of OsGI, Hd1, Ehd1, Hd3a, OsMADS50 and RFT1 under LDs (a) and OsGI, Hd1, Ehd1, Hd3a, OsMADS51 and RFT1 under SDs(b)in the ef8 mutant and wild type. Each point represents three biological repeats [meanÂ±standard error (SE)]

EF8 negatively regulates chlorophyll biogenesis under LDs

The changes of expression level of chlorophyll biosynthetic genes result in the changes of leaf color and chlorophyll content in rice (Jung et al. 2003; Lee et al. 2005; Wang et al. 2010; Wu et al. 2007; Zhang et al. 2006a). The ef8 mutant displayed a slightly darker green leaf phenotype and more chlorophyll content compared to wild type under LDs (Fig. 2; Supplemental Fig. S1), which suggested that EF8 may have effect on chlorophyll biogenesis. To test this, we extracted total RNAs from 30-dayold leaves of wild-type ande f8mutant seedlings under both LDs and SDs, and performed QRT-PCR to examine the expression of genes encoding the enzymes in the chlorophyll biosynthetic pathway in rice. Most of these genes were up-regulated to different degrees except chlorophyll synthetase (CHLG) in ef8 mutant compared to wild type under LDs (Fig. 6a). For example, Mg-protoporphyrin IX methyltransferase (CHLM) and NADPH:protochlorophyllide oxidoreductase A (PORA) were up-regulated more than threefold, and Mg chelatase H subunit (CHLH) and divinyl reductase (DVR) about twofold. However, there were no significant differences in the transcription level of these genes under SDs (Fig. 6b). This result indicated that the higher chlorophyll content was due to higher expression levels of chlorophyll biosynthetic genes in the ef8 mutant compared to wild type and EF8 negatively regulated chlorophyll biogenesis under LDs.


Fig. 6 Expression of chlorophyll biosynthetic genes in wild type and ef8 mutant under LDs (a) and SDs (b). These genes were glutamyltRNA reductase (HEMA), glutamate 1-semialdehyde aminotransferase (HEML), porphobilinogen synthase (5-aminolevulinate dehydratase) (HEMB), hydroxymethylbilane synthase (porphobilinogen deaminase) (HEMC), uroporphyrinogen III decarboxylase (HEME), coproporphyrinogen oxidative decarboxylase (URO-D), coproporphyrinogen oxidative decarboxylase (HEMF),M gchelataseDsubunit (CHLD), Mg chelataseHsubunit (CHLH), Mg chelatase I subunit (CHLI), Mg-protoporphyrinI Xmethyltransferase (CHLM),M g-protoporphyrinogenI Xmonomethylester cyclase(C R D), divinyl reductase (DVR), NADPH:protochlorophyllide oxidoreductase A (PORA), chlorophyll synthase (CHLG) and chlorophyllide oxygenase(C A O). Data are meanÂ± SD(n=3 $ ). ^** and indicate significant differences between WT and e f8 at P=0.05 and P=0.01 by Studentâ€™s t test

# Discussion

In this study, we identified and characterized EF8 which simultaneously controlled flowering time, grain number and chlorophyll content. EF8 is sensitive to photoperiod and functions only under LDs (Fig. 1; Table 1). The phenotypes of the ef8 mutant are early flowering, decreased grain number and increased amount of chlorophyll under LDs (Fig. 1; Table 1). Previously, the QTL DTH8/Ghd8 with several allelic types was reported to control pleiotropic effects including flowering time, grain number and plant height (Wei et al. 2010; Yan et al. 2011). The functional DTH8 allele from japonica cv. Asominori delays flowering time by about 13 days and increases grain number per main panicle by about 50 % and plant height by about 20Â % compared to CSSL61 (a chromosome segment substitution line that carries the nonfunctional DTH8 allele) under LDs (Wei et al. 2010). NILHR5 (a homozygous, near isogenic line carrying another DTH8 functional allele from indica cv HR5) is 7.8 days later in flowering than NIL^ZS (a near isogenic line carrying a nonfunctional allele from indica cv Zhenshan 97) and is 22 % taller with a 43 % increase in grain number per main panicle compared to NIL^ZS under LDs (Yan et al. 2011; Zhang et al. 2006b). In our study, E F8 , an allele of DTH8/Ghd8 from indica cv 93-11, delays flowering time by about 7.9 days and increases grain number per main panicle by about 24 % , but decreases the amount of chlorophyll and does not increase plant height compared with ef8 mutant under LDs (Table 1; Fig. 2a). What is more, the functional DTH8 allele from japonica cv. Asominori could down-regulate the transcriptions of Ehd1 and Hd3a under LDs as described by Wei et al. (2010), and another DTH8 functional allele from cv HR5 delayed flowering by regulating Ehd1, RFT1 and Hd3a under LDs as described by Yan et al. (2011). However, our results indicated that EF8 could alter the rhythmic expression patterns of the â€˜florigenâ€™ genes, Hd3a and RFT1, independently of Hd1 and Ehd1 under LDs (Fig. 5). Thus, different alleles of one gene may have different effects and roles, and may lead to different phenotypes due to unknown mechanisms. On the one hand, Asominori shares the same protein sequence of L o c_O s O8g O7740 as HR5, but has many polymorphisms in the promoter of L o c_O s O8g O7740 compared to HR5(Yan et al. 2011), indicating that the different effects of these two alleles may be due to the polymorphism of the promoter of Loc_Os08g07740 and/or the differences of genetic background. On the other hand, compared to the allele from cv Asominori, there are seven single amino acids substitutions and a single amino acid deletion in the protein sequence of the allele in cv 93-11 (Supplemental Fig. S3). Whether the difference of protein sequence and/or the differences of genetic background are responsible for the different allelic effects is an interesting question for further study.

EF8 encoded a putative HAP3 subunit of the CCAATbox-binding HAP complex composed of subunits HAP2, HAP3 and HAP5. Each subunit functions in DNA binding and proteinâ€“protein interactions (Maity and de Crombrugghe 1998; Mantovani 1999). It was reported that the OsHAP3A, OsHAP3B and OsHAP3C genes in rice influence chloroplast biogenesis, and the HAP complex was involved in chloroplast development through gene regulation (Miyoshi et al. 2003). Thus, EF8 may interact with several other unknown HAP2/HAP5 subunits to form HAP complexes that bind to the promoter regions of downstream target genes in the nucleus to thereby regulate gene expression. Here, EF8 was involved in regulation of pleiotropic traits including flowering time, grain number and chlorophyll biogenesis, so its target genes may be related to these pathways. Therefore, it is likely that EF8 forms HAP complexes to directly or indirectly regulate the expression of the â€˜florigenâ€™ and chlorophyll biosynthetic genes. This needs to be confirmed in further studies.

Flowering time is often associated with yield-related traits including grain number, because early-flowering plants often have insufficient vegetative growth periods, which results in low yields (Kawano and Tanaka 1968). Ghd7, a QTL controlling heading date, delays flowering and simultaneously has increased panicle size and grain number under LDs (Xue et al. 2008). Ehd4, an important regulator promoting flowering, was shown to promote flowering and decreased grain number under both LDs and SDs (Gao et al. 2013). In our study, the flowering time of the ef8 mutant was significantly earlier than that of its wild type, 93-11, and early-flowering plants showed a lower grain number per main panicle under LDs (Table 1). Thus, EF8 may have an important role in improving yields on rice breeding.

The appropriate flowering time is responsible for successful sexual reproduction in flowering plants. Flowering time is a complex quantitative trait controlled by numerous minor-effect QTLs and major genes (Buckler et al. 2009; Huang et al. 2012). The indica cultivar 93-11 is often used as an elite restorer for two- and three-line hybrids in Chinese rice breeding due to its outstanding agronomic properties, such as high yield, good grain and eating quality, and multiple disease resistances. In our study, E F8 , which delayed flowering time by about 7.9 days (Table 1), was a minor-effect allele of DTH8 in cv 93-11 under LDs. It could be introduced into rice breeding programs via MAS as previously reported (Young 1999). Furthermore, since the ef8 mutant shows early flowering it could prove beneficial for flower synchronization on rice breeding and be an important breeding material for achieving practical goals.

Acknowledgments This research was supported by grants from the National Basic Research Program of China (2011CB100102), the 863 Program (2011AA10A101), High Technology Program from NDRC ([2012]1961), National Science and Technology Support Program (2013BAD01B02-16, 2011BAD35B02-02), Jiangsu Science and Technology Development Program (BE2013301) and Qing Lan Project.

Conflict of interest The authors declare that they have no conflict of interest.
