Mukesh Jain ‚àô Navneet Kaur ‚àô Akhilesh K. Tyagi Jitendra P. Khurana

# The auxin-responsive GH3 gene family in rice (Oryza sativa)

Received: 9 January 2005 / Revised: 25 March 2005 / Accepted: 25 March 2005 / Published online: 22 April 2005
Springer-Verlag 2005

Abstract Auxin regulates plant growth and development by altering the expression of diverse genes. Among these, the genes of Aux/IAA, SAUR, and GH3 classes have been extensively studied in dicots, but little information is available on monocots. We have identified 12 members of GH3 gene family in rice using sequences of full-length cDNA clones available from KOME and analysis of the whole genome sequence of rice. The genomic organization as well as chromosomal location of all the OsGH3 genes is reported. The rice GH3 proteins can be classified in two groups (groups I and II) on the basis of their phylogenetic relationship with Arabidopsis GH3 proteins. Based upon the sequences available in the database, not a single group III GH3 protein could be identified in rice. An extensive survey of EST sequences of other monocots led to the conclusion that although GH3 gene family is highly conserved in both dicots and monocots but the group III is conspicuous by its absence in monocots. The in silico analysis has been complemented with experimental data to quantify transcript levels of all GH3 gene family members. Using real-time polymerase chain reaction, the organ-specific expression of individual OsGH3 genes in light- and dark-grown seedlings/plants has been examined. The transcript abundance of nearly all OsGH3 genes is enhanced on auxin treatment, with the effect more pronounced on OsGH3-1, -2, and -4. The functional validation of these genes in transgenics or analysis of gene-specific insertional mutants will help in elucidating their precise role in auxin signal transduction.

Keywords Auxin ¬∑ GH3 gene family ¬∑ Phylogenetic analysis ¬∑ Rice

# Introduction

The phytohormone auxin plays a critical role in regulating a number of plant responses including cell elongation, cell division, phototropism, gravitropism, root initiation, and apical dominance. In an attempt to understand the molecular mechanism of auxin action, a number of early or primary auxin-responsive genes have been identified and characterized from different plant species including pea, soybean, Arabidopsis, tobacco, mung bean, and cucumber (Hagen and Guilfoyle 2002). These auxin-responsive genes have been broadly grouped into three major classes: Aux/ IAA, SAUR, and GH3 (Guilfoyle 1999). A number of putative auxin-responsive elements (AuxREs) have been defined within upstream promoter regions of auxin-responsive genes, such as one or more copies of a conserved motif, TGTCTC, or some variation of this motif (e.g., TGTCCC, TGTCAC), which confer auxin responsiveness (Hagen et al. 1991; Li et al. 1994; Liu et al. 1994; Ulmasov et al. 1995; Guilfoyle 1999). The DNA-binding domain of auxin-responsive factors (ARFs) binds to AuxREs and regulates the expression of auxin-responsive genes (Kim et al. 1997; Ulmasov et al. 1997; Tiwari et al. 2003).

Initially, early auxin-responsive genes of Aux/IAA class were isolated from soybean (Walker and Key 1982; Ainley et al. 1988) and later from pea (Theologis et al. 1985), mung bean (Yamamoto et al. 1992), Arabidopsis (Conner et al. 1990; Abel et al. 1995), and rice (Thakur et al. 2001). The SAUR genes have also been isolated and studied in plants such as soybean (Hagen et al. 1984), mung bean (Yamamoto et al. 1992), and Arabidopsis (Gil et al. 1994). The first GH3 gene was isolated from soybean by differential screening as an early auxin-responsive gene (Hagen et al. 1984). The soybean GH3 transcription was shown to be specifically induced by biologically active auxins within 5 min of exogenous auxin application. Also, unlike SAURs and Aux/IAAs, GH3 mRNAs do not accumulate in response

to protein synthesis inhibitor cycloheximide (Franco et al.
1990).

The GH3 homologues have been found in other plants too (Guilfoyle et al. 1993; Roux and Perrot-Rechenmann 1997). The GH3 genes are represented as a multigene family in the Arabidopsis genome comprising of 20 members (Hagen and Guilfoyle 2002). Many of the AtGH3 genes were induced by exogenous application of auxin (Hagen and Guilfoyle 2002; Tanaka et al. 2002). Light also regulates the transcript levels of Arabidopsis GH3 genes through phytochromes A and B (Nakazawa et al. 2001; Tepperman et al. 2001; Tanaka et al. 2002; Takase et al. 2003, 2004). The analysis of an Arabidopsis mutant identified FIN219, a GH3 protein, as a phytochrome A signaling component having a crucial role in photomorphogenesis (Hsieh et al. 2000); it negatively regulates COP1, a key repressor of photomorphogenic development. Two other Arabidopsis GH3 proteins, DFL1 and YDK1, also identified by mutant analysis, were shown to negatively regulate shoot and hypocotyl cell elongation and lateral root formation (Nakazawa et al. 2001; Takase et al. 2004). Arabidopsis GH3 proteins have been reported to adenylate plant hormones such as indole-3-acetic acid (IAA), jasmonic acid (JA), and salicylic acid (SA) (Staswick et al. 2002). Recently, some of the Arabidopsis GH3 genes have been shown to encode IAA-amido synthetases, which synthesize a diversity of IAA‚Äìamino acid (aa) conjugates to help maintain auxin homeostasis (Staswick et al. 2005).

To our knowledge, there is no report on the characterization of any GH3 gene from a monocot species, although several EST/cDNA sequences are available in the database. Here, we report the identification and a comprehensive analysis of the GH3 gene family from rice (Oryza sativa), the model monocot plant. The work involved the identification of GH3 gene family from rice, analysis of their genomic organization, chromosomal distribution, sequence homology, and phylogenetic relationship. Systematic identification of ESTs representing different members of the GH3 gene family, along with real-time PCR analysis, indicated their differential expression in various organs/tissues and in response to light and auxin treatment. Extensive survey of ESTs of this family has also been carried out in other monocots including wheat, maize, sorghum, sugarcane, and barley.

# Materials and methods

Identification of GH3 gene family sequences The predicted longest open-reading frames (ORFs) from the Knowledge-Based Oryza Molecular Biological Encyclopedia (KOME, http://www.cdna01.dna.affrc.go.jp/cDNA)(Kikuchi et al. 2003), National Centre for Biotechnology Information (NCBI, http://www.ncbi.nlm.nih.gov/BLAST), and TIGR genomic and annotated database resources (http:// www.tigrblast.tigr.org/euk-blast) were used for the identification of GH3 homologues in rice. The Arabidopsis GH3 protein and cDNA sequences were downloaded from The Arabidopsis Information Resource (TAIR) (Garcia-Hernandez et al. 2002). The protein sequences of 20 GH3 proteins from Arabidopsis were used to search for their homologues in rice in KOME [28,469 full-length cDNA (FL-cDNA) clones] using TBLASTN program (Altschul et al. 1997). This led to the identification of 12 predicted GH3 proteins, which included nine nonredundant clones having high-sequence similarity with AtGH3 proteins. Additional members of rice GH3 gene family were identified by TBLASTN search in TIGR database. The predicted GH3 genes in O . sativa were named OsGH3 (Table 1).

# Sequence analysis

The number and positions of exons and introns for individual OsGH3 genes were determined by comparison of the cDNAs with their corresponding genomic DNA sequences. The position of each gene on rice chromosomes was found by BLASTN search in genomic sequences of rice chromosome pseudomolecules available at TIGR (Release 3). Multiple-sequence alignments were done using the ClustalX (version 1.83) program (Thompson et al. 1997) and the phylogenetic analysis carried out by neighbor-joining method (Saitou and Nei 1987). The unrooted phylogenetic tree was displayed using the NJPLOT program (Perriere and Gouy 1996). The DNA and protein sequence analysis were performed using Gene Runner program version 3.04. Pairwise comparisons were done with the DNASTAR MegAlign 4.03 package to determine the sequence identities. Conserved residues were identified by alignment using ClustalX.

# Plant material and growth conditions

Rice (O. sativa L. spp. indica var. Pusa Basmati 1) seeds were obtained from Regional Station of the Indian Agricultural Research Institute, Karnal. After disinfection with 0.1% HgCl_2 for 1 ¬†h and thorough washing with reverseosmosis (RO) water, seeds were soaked overnight in RO water. Etiolated shoots, roots, and green shoots were harvested from 6-day-old seedlings grown on cotton saturated with RO water, at 28¬±1^‚àòC_: either in complete darkness or in a culture room with a daily photoperiodic cycle of 14 h light and 10 h dark. Flowers were collected from field-grown rice plants. Calli were raised as previously described (Jain et al. 2004). For auxin treatment, coleoptile segments (10¬†mm) from the apical portion of 3-day-old etiolated rice seedlings were incubated in KPSC buffer ( 10¬†mM potassium phosphate, ùóâ ùñß 6.0 ,2% sucrose, 50 M chloramphenicol) for 16 h to deplete endogenous auxins, with the buffer changed every 2¬†h , and then transferred to fresh buffer with 30¬†M concentration of 2,4-dichlorophenoxyacetic acid (2,4-D) for 3 h .

Table 1 GH3 gene family in rice


Systematic designation given to rice GH3 genes bAccession no. of full-length cDNA sequence from KOME (http://cdna.01.dna.affrc.go.jp/cDNA/) or TIGR locus ID (marked with asterisk) in release 3 Length [number of amino acids (aa)] of the deduced polypeptide Length of open-reading frame in base pairs eNumber of introns present within ORF Name, accession number, and approximate cM position of the BAC/PAC clone in which OsGH3 gene is present Nearest marker to the OsGH3 gene Representative EST accession no. and number of ESTs present in Genbank release 111204.NF (not found)

# RNA isolation and quantitative real-time PCR expression analysis

Total RNA was extracted using the RNeasy Plant mini kit (Qiagen, Germany) according to the manufacturer‚Äôs instructions, followed by DNase I treatment to remove any genomic DNA contamination. For each RNA sample, absorption at 260¬†nm was measured and RNA concentration calculated as A_260√ó40 (g/ml)√ó dilution factor.

First strand cDNA was synthesized by reverse transcribing 1.5 g of total RNA (reaction volume 50 l) using highcapacity cDNA Archive kit (Applied Biosystems, USA) according to the manufacturer‚Äôs instructions. Primers were designed by using the primer design software Primer Express 2.0 (PE Applied Biosystems, USA). To ensure that the primers amplify a unique and desired cDNA segment, each pair of primers was checked in the BLAST program in rice genomic sequence available in TIGR database. The primer sequences are listed in Table S1 of Supplementary material. The diluted cDNA samples were used as template and mixed with 200 nM of each primer and SYBR Green PCR Master Mix (Applied Biosystems, USA) for real-time PCR analysis, using ABI Prism 7000 Sequence Detection System and Software (PE Applied Biosystems, USA). PCR reactions were performed using the following parameters: 2 min at 50^‚àòC ,10 min at 95^ , 40 cycles of 15 s at 95^ , and 1 min at 60^‚àòC in 96-well optical reaction plates (Applied Biosystems, USA). The identities of the amplicons and the specificity of the reaction were verified by agarose gel electrophoresis and melting curve analysis, respectively. The relative mRNA levels of the individual OsGH3 genes in different RNA samples were computed with respect to the internal standard, UBQ5, to normalize for variance in the quality of RNA and the amount of input cDNA. At least two different RNA isolations and cDNA syntheses were used for quantification, and each cDNA was measured in triplicate.

# EST analysis

The search for ESTs in rice and other monocots was performed by comparing the rice GH3 cDNA sequences against EST sequences available at NCBI in BLASTN searches. The OsGH3 cDNA sequences were searched using MEGABLAST tool against the EST databases of rice (o . sativa), wheat (Triticum aestivum), corn (Zea mays), Sorghum bicolor, sugarcane (Saccharum officinarum), and barley (Hordeum vulgare) at NCBI (http://www.ncbi.nlm. nih.gov/dbEST/index.html). The searches were limited to one species at a time to detect all the existing ESTs that showed significant similarity. Details of only top hits of the BLASTN search results for each OsGH3 gene showing a bit score of 500 or more were extracted into a separate file. An optimized value of a word size of 11 and expect value of 10, without the low-complexity filter, were used for analysis in order to record the optimum number of matches between rice genes and ESTs from other monocots.

# Results and discussion

# GH3 gene family in rice

The information on rice genomic sequence (Goff et al. 2002; Yu et al. 2002; http://www.tigr.org/tdb/e2k1/osa1/; http:// www.rgp.dna.affrc.go.jp/) provides a powerful tool to identify putative homologous proteins by database searches with genes of known function from other organisms. In an attempt to identify GH3 protein-coding genes in rice, a TBLASTN search of FL-cDNA clones of rice available at KOME was performed using 20 AtGH3 proteins as query, which identified nine nonredundant clones (among a total of 12) and were designated as O . sativa GH3 genes ( O s G H3-I to -ùí¢ ;Table 1). To identify additional members of this family not represented in KOME cDNA collection, the whole rice genome, dynamically translated in all reading frames, was analyzed employing TBLASTN search, using the 20 GH3 proteins of Arabidopsis and nine nonredundant rice GH3 proteins identified through KOME. Three additional members, O s G H3-I O to -12, showing significant similarity with other OsGH3 proteins could be identified in this search. The overall analysis of the complete genome of rice revealed that auxin-inducible GH3 gene family is comprised of 12 members. However, in A. thaliana, 19 G H3 genes, along with an additional partial gene (coding only for aminoterminal third of the protein), have been reported (Hagen and Guilfoyle 2002).

A comparative study was done for all the OsGH3 proteins found in KOME with the corresponding annotated proteins present in the NCBI and TIGR databases (Supplementary material, Table S2). The amino acid sequences of all were found to be essentially identical in three databases, except for OsGH3-2, -3, and -6. Therefore, the amino acid and cDNA sequences of OsGH3-2, -3, and -6 were examined in detail. The C-terminal amino acid sequence of OsGH3-2 protein from KOME did not show any homology with other known GH3 proteins. Its cDNA sequence was aligned with annotated cDNA sequence from NCBI and TIGR and the corresponding genomic sequence [phage artificial chromosome (PAC) clone P0403C05]. Two nucleotides (GC) were found to be missing in KOME cDNA sequence within ORF leading to the change in coding frame. However, the corresponding protein from TIGR and NCBI were identical and showed significant similarity with other GH3 proteins and included in Table 1 for further analysis. The OsGH3-3 protein predicted by KOME and TIGR was identical (462 aa). However, NCBI predicted it to be 623 aa long, the C-terminal region of which did not show identity with the sequence available in KOME or TIGR database. The cDNA sequence of OsGH3-3 was supported by two full-length cDNAs from KOME (AK072125 and AK120515) and the two ESTs (CB660383 and CB660384); thus, it is most likely that OsGH3 protein comprises 462 aa (Table 1). The OsGH3-6 protein was predicted to be 488, 486, and 908 amino acid long in KOME, NCBI, and TIGR databases, respectively;theNCBI-annotated sequenceshowed 100% identity with KOME OsGH3-6 sequence except that two amino acids were found to be missing in NCBI protein. The C-terminal sequence of OsGH3-6 from TIGR did not show any homology to GH3 proteins. After alignment of all the three OsGH3-6 cDNA sequences with corresponding genomic sequence [bacterial artificial chromosome (BAC) clone OJ1607_F09], it was concluded that KOME sequence is in order and was thus included in Table 1 for further analysis.


Fig. 1 Exon‚Äìintron organization of rice GH3 genes. The coding and untranslated regions are represented by black and open boxes, respectively. Lines represent the introns. The numbers 0, 1, and 2 represent phase 0, 1, and 2 introns, respectively

# Genomic organization and chromosomal distribution

A comparison of the full-length cDNA sequences with the corresponding genomic DNA sequences showed that the coding sequences of OsGH3 genes are disrupted by one to four introns except for OsGH3-6 and -ùí¢_- , which do not harbor any intron (Table 1). Although the exon‚Äìintron organization of OsGH3 genes in terms of their numbers (Fig. 1) does not seem to be very similar, their intron phasing is highly conserved, which is indicative of exon shuffling (Kolkman and Stemmer 2001). The bacterial artificial chromosome (BAC) or phage artificial chromosome (PAC) clones carrying the OsGH3 genes were identified (Table 1). The chromosome map positions of BAC/PACs given in centiMorgans (cM) from top of the chromosome and the nearest marker to each OsGH3 gene are indicated in Table 1. Also, the position (in megabases) and direction of transcription (arrows) of each gene was determined on rice chromosome pseudomolecules as shown in Fig. 2.


Fig. 2 Genomic distribution of GH3 genes on rice chromosomes. White ovals on the chromosomes (vertical bars) indicate the position of centromeres. The arrows next to gene names show the direction of transcription. The numbers in parentheses designate the position of the first exon of corresponding GH3 gene in megabases (Mb) on rice chromosome pseudomolecules available at TIGR (release 3). The chromosome numbers are indicated at the top of each bar

The 12 rice GH3 genes are distributed on only five of the 12 rice chromosomes. Four of the OsGH3 genes are present on chromosome 7, three on chromosomes 5 and 7 each, whereas only one GH3 gene is present each on chromosomes 6 and 11, respectively (Fig. 2). The distribution of 12 members of this multigene family did not reveal evident clusters. However, in Arabidopsis, two clusters of GH3 genes were found, one on chromosome V containing five genes (AtGH3-12 to -16) and the other on chromosome I containing three genes (AtGH3-18 to -20) organized in tandem in the same orientation (Hagen and Guilfoyle 2002). The amino acid sequences of AtGH3-13 to -16 ( 74% to 85% identity) and AtGH3-18 and -19 ( 74% identity) are highly homologous. These results suggested that they arose relatively recently by gene duplication and may have similar or overlapping functions and also account for an enlarged GH3 gene family in Arabidopsis in comparison to rice.

# Sequence analysis and phylogenetic relationship

Arabidopsis GH3 proteins did not show the presence of any known conserved motif or domain even within the highly conserved regions. However, they were found to be distantly related ( 13% sequence identity) to the acyl adenylateforming firefly luciferase super family (Staswick et al. 2002). OsGH3 proteins also did not show any match in the currently recognized motifs in the databases. The multiplesequence alignment of the full-length OsGH3 proteins was done using ClustalX (Fig. 3). The core region showed significant conservation (Fig. 3a; marked with arrows) as compared to the N- and C-terminal regions of predicted GH3 proteins. Pairwise analyses with the full-length protein sequences indicate that the overall identities range from 17% to 78% (Table 2). Relatively high identities were found between OsGH3-1 and ^-4 (ca 78% ), OsGH3-8 and -9 (ca 75% ). Such high homologies may indicate that they may perform essentially similar functions. However, other family members show a variation of 17% to 61% identity among them, which may indicate their distinct origin and function. To examine the phylogenetic relationship among the OsGH3 proteins, an unrooted tree was constructed from alignment of their full-length protein sequences (Fig. 4). Two major groups (groups I and II), with very strong bootClustalX. Fully and partially conserved (present in more than 50% of aligned sequences) residues are highlighted in black and gray boxes, respectively. Gaps (marked with dashes) have been introduced to maximize the alignments




Fig. 3 (A) Alignment profile of rice GH3 proteins obtained with the ClustalX program. The height of the bars indicates the number of identical residues per position. The region between the arrows indicates the core region with high-sequence similarity. (B) Multiple alignments of the full-length rice GH3 proteins obtained with

Table 2 Percentage identities among full-length OsGH3 proteins


Highest and lowest identities are presented in bold Tripple asterisk (***) refers to 100 percent identity between the same protein

strap support, were observed, which included four and eight OsGH3 proteins, respectively. Twelve OsGH3 proteins formed only three sister pairs. The sister pair OsGH3-1 and ^-4 was found to be present on duplicated chromosomal block 1 present between chromosomes 1 and 5 as described by Paterson et al (2004). However, the sister pair OsGH3-9 and -10 may represent a local duplication event. The third sister pair (OsGH3-3 and -12) may also be present on unidentified duplicated chromosomal block. Taken together, these observations throw some light on the diversification of rice GH3 genes during evolution.

The Arabidopsis GH3 proteins have been classified into three groups on the basis of their protein structure and specificity to adenylate plant hormones (Staswick et al 2002). To examine protein relationships of rice and Arabidopsis GH3 proteins, an unrooted tree was constructed from alignments of their full-length GH3 protein sequences (Fig. 5). Based upon the sequence homology, all the rice and Arabidopsis GH3 proteins clustered distinctly into three groups (I, II, and III). Group I consisted of only six members, four from rice cluster, i.e., OsGH3-3, -5, -6, and -12, along with AtGH-10 and -11. AtGH3-11 (JAR1/FIN219) has been shown to adenylate jasmonic acid (JA) (Staswick et al. 2002). Seven of the OsGH3 proteins belonged to group II AtGH3 proteins, which are involved in IAA adenylation (Staswick et al. 2002). The group II Arabidopsis GH3 genes encode IAA-amido synthetases that conjugate amino acids to IAA and regulate plant growth and development (Staswick et al. 2005). However, group III included only Arabidopsis GH3 proteins with unknown function, which do not adenylate IAA, JA, or SA. Surprisingly, no group III GH3 protein could be identified in rice sequence databases searched. It can be speculated that group III GH3 proteins were lost in rice after divergence of monocots and dicots or evolved in dicots after the divergence from monocots and may perform dicot-specific functions. However, OsGH3-7 grouped with group II rice GH3 proteins (Fig. 4) but did not group with Arabidopsis group II GH3 proteins (Fig. 5).


Fig. 4 Phylogenetic relationship among the rice GH3 proteins. The unrooted tree was generated using ClustalX program by neighborjoining method. Bootstrap values from 1000 replicates are indicated at each node

# Differential expression of OsGH3 genes

The frequency of ESTs or cDNAs available in different databases has been considered as a useful tool for preliminary analysis of gene expression (Adams et al. 1995). Searching different databases, ESTs were identified for most of the OsGH3 genes, but the frequency of ESTs for individual genes varies greatly (Table 1; Supplementary material, Table S3). For example, eight ESTs are deposited for OsGH3-2 and 28 for OsGH3-5, whereas no ESTs could be identified for OsGH3-1, -6, -9, -10, and -12 (Table 1). The full-length cDNA clones have been identified for OsGH3-1 through OsGH3-9. The transcription of OsGH3-


Fig. 5 Phylogenetic relationship of rice and Arabidopsis GH3 At1g23160; AtGH3-8, At5g51470; AtGH3-9, At2g47750; AtGH3- proteins. The unrooted tree was generated using ClustalX program by 10, At4g03400; AtGH3-11/FIN219/JAR1, At2g46370 ; AtGH3-12, neighbor-joining method. Bootstrap values (above 50% ) from 1000 At5g13320; AtGH3-13, At5g13350; AtGH3-14, At5g13360; AtGH3- replicates are indicated at each node. The AGI gene names of 15, At5g13370; AtGH3-16, At5g13380; AtGH3-17, At1g28130; Arabidopsis GH3 proteins are: AtGH3-1, At2g14960 ; AtGH3-2/ AtGH3-18, At1g48660; AtGH3-19, At1g48670. AtGH3-20 has not YDK1, At4g37390; AtGH3-3, At2g23170 ; AtGH3-4, At1g59500; been included as it codes for a partial GH3 protein AtGH3-5, At4g27260; AtGH3-6/DFL1, At5g54510; AtGH3-7,

10 and -12 genes, however, remained unclear, as no corresponding full-length cDNA or EST could be identified from the databases. The expression of all the 12 OsGH3 genes was thus verified experimentally and quantitated by real-time PCR analysis in the present study.

Many of the GH3 genes in Arabidopsis, soybean, and tobacco were found to be differentially expressed in different tissues or in response to exogenous auxin and light stimuli (Tepperman et al. 2001; Hagen and Guilfoyle 2002; Tanaka et al. 2002; Takase et al. 2004). To determine the organ-specific expression pattern of each OsGH3 gene, real-time PCR was performed with total RNA isolated from etiolated shoots, green shoots, roots, flowers, and callus. This analysis revealed that OsGH3 genes are differentially expressed in various tissues/organs (Fig. 6a). The transcription of OsGH3-12 was found to be root-specific and was hardly detectable in other tissues examined (Fig. 6a). However, OsGH3-10 was found to be expressed at very low levels in all the tissues. In contrast, other OsGH3 genes were found to be expressed in almost all the tissues examined but at different levels. Also, significant differences were found in the transcript abundance of OsGH3 genes in etiolated and green shoots (Fig. 6a), indicating their light regulation. The transcript levels of most of the OsGH3 genes were upregulated by auxin treatment as is evident from the data presented in Fig. 6b, although to varying degree. The effect is more pronounced on OsGH3-1, -2, -4 and, to some extent, on OsGH3-8 too. The difference in the inducibility of individual GH3 genes is likely due to a variety of factors such as kinetics of induction, tissue-specific auxin reception, cell type-dependent and differential regulation of free auxin concentrations, or different modes of auxin-dependent transcriptional and posttranscriptional regulation.

# EST analysis in other monocots

A search of Genbank database showed that GH3-related genes are present throughout the plant kingdom. ESTs have been found for GH3 proteins in blue-green alga, Synechocystis, and dicots (Arabidopsis, tobacco, soybean, Capsicum). An extensive analysis of the EST sequences in Genbank database revealed the presence of a large number of ESTs in other monocots like T. aestivum, Z. mays, S. bicolor, S. officinarum, and H. vulgare with high-sequence similarity with OsGH3 genes (Supplementary material, Table S3). These results suggest that the GH3 proteins may be present in diverse plant species and are likely to play important roles in plant cellular processes. However, like in case of OsGH3 gene family, no ESTs corresponding to group III GH3 genes of dicots could be identified in other monocots too. What precisely is the evolutionary significance of this observation will only be unraveled through functional validation of this group of genes in dicots like Arabidopsis.


Fig. 6 Real-time PCR expression profiles of individual OsGH3 callus). (B) The relative mRNA levels of individual O s G H3 genes in genes. (A) The relative mRNA levels of individual O s G H3 genes control ( ^16 h auxin-depleted) and 2,4-D (30¬†m) -treated coleoptile normalized with respect to housekeeping gene UBQ5 in different segments of 3-day-old etiolated rice seedlings. Asterisks indicate tissues E S etiolated shoots, G S green shoots, R roots, F flowers, C that the expression was close to the detection limit

# What are the functions of OsGH3 proteins?

The GH3 proteins in Arabidopsis have been shown to be involved in plant growth and development, photomorphogenesis, light- and auxin-signaling, and auxin homeostasis (Hsieh et al. 2000; Nakazawa et al. 2001; Tepperman et al. 2001; Tanaka et al. 2002; Takase et al. 2004; Staswick et al. 2005). The results of structural analyses of rice GH3 proteins will be helpful in their functional analysis. Our analysis showed that GH3 proteins from Arabidopsis and rice are divided into three and two groups, respectively. GH3 proteins classified in the same groups may have the similar functions in events common to both monocot and dicot plants. Group III GH3 proteins may have dicot-specific functions as they could not be identified from rice or other monocots. Organ-specific differential expression suggests diverse roles of these proteins during plant growth and development. The effect of light and auxin treatment on the transcript levels of OsGH3 genes implies their role in light and auxin signal transduction.

To further elucidate the functions of these genes, we investigated the phenotypes of rice Tos17 retrotransposon insertion mutants (Miyao et al. 2003) of these genes with the aid of Tos17 mutant panel database (http://www.tos.nias.affrc. go.jp/) using the BLAST program. We could enlist eight insertion mutants corresponding to two (OsGH3-5 and -7) of the 12 OsGH3 genes (Supplementary material, Table S4). The phenotypes of the insertion mutants of these genes showed dwarfism, sterility, vivipary, and leaf withering. From these phenotypes, it can be speculated that these genes may play a role in different metabolic pathways and cellular processes influenced by light and auxin, although the assumption that these phenotype changes are specific to insertional mutagenesis of OsGH3 gene remains to be proven experimentally. A detailed analysis of these insertional mutants already available and RNAi strategy for the remaining O s G H3 genes will greatly help in elucidation of the role of these genes and their functional validation.

Acknowledgements MJ acknowledges the award of Senior Research Fellowship from the Council of Scientific and Industrial Research, New Delhi. This research work was financially supported by the Department of Biotechnology, Government of India, and the University Grants Commission, New Delhi. We gratefully acknowledge KOME, TAIR, TIGR, and IRGSP database resources for availability of detailed sequence information on rice and Arabidopsis.
