# Fine mapping and identification of a novel albino gene OsAL50 that is required for chlorophyll biosynthesis and chloroplast development in rice (Oryza sativa L.)

Yuehui Zeng1,2 · Xinyu Wei1,2 · Changchun Xiao1,2 · Rui Zhang1,2 · Jianhong Huang1,2 · Xuming ×𝐮^2,3⊕

Received: 18 October 2023 / Accepted: 26 December 2023 / Published online: 23 January 2024
⊚ The Author(s), under exclusive licence to Springer Nature B.V. 2024

# Abstract

Leaf color is a highly important agronomic trait, and mutants with altered leaf coloration can serve as excellent models for studies on chloroplast development and chlorophyll biosynthesis, enabling the cloning of genes involved in these processes in rice (Oryza sativa L.). In this study, we isolated a stable genetic rice mutant, oryza sativa albino leaf 50 (osal50), from a breeding population of the japonica cultivar GP50. This mutant exhibited a distinctive albino phenotype, with white-striped leaves in seedlings and white panicles at the heading stage. Compared with wild-type GP50, the osal50 mutant showed lower chlorophyll and carotenoid accumulation, together with abnormal chloroplast ultrastructure. Genetic analysis demonstrated that a recessive nuclear gene was responsible for the albino phenotype of osal50, and a map-based cloning strategy delimited O s A L5O to a 160-kb physical interval on chromosome 1, flanked by two single nucleotide polymorphism (SNP) markers, CAPS-08 and CAPS-37, that included 26 putative open reading frames. Sequence and expression analyses revealed L O C_-O s O I g20I I O as the candidate OsAL50 gene, which was confirmed by knockout using CRISPR/Cas9. Subcellular localization and protein sequence analyses suggested that OsAL50 likely encodes an endoribonuclease E-like protein localized to the chloroplasts. Further investigation indicated that O s A L5O plays a vital role in the regulation of photosynthetic pigment metabolism, photosynthesis, and chloroplast biogenesis. In summary, we identified a novel albino mutant that will serve as useful genetic material for studies of chlorophyll biosynthesis and chloroplast development in rice.

Keywords  Rice · OsAL50 · Albino · Leaf color · Chlorophyll biosynthesis · Chloroplast development

# Introduction

Rice (Oryza sativa L.) is one of the most important food crops globally and an ideal model for monocotyledonous plants in both genetic and functional genomic research. It is especially useful as it has a relatively small genome, exhibits good co-linearity with other grasses, and is amenable to manipulation using established transformation systems (Han et al. 2007). The primary photosynthetic organs in rice are the leaves. Changes in leaf color are common and tend to occur routinely over the course of growth and development (Su et al. 2017; Chen et al. 2018). Rice leaf-color mutants exhibit various leaf colors and patterns, such as bright green, green-revertible albino, zebra, striped, spotted, albino, yel-lowing, and light green (Li et al. 2018b). These different phenotypes are caused by variations in the levels and types of pigments, including carotene and chlorophyll, present in the leaves (Wang et al. 2018). The green chlorophyll pigment is present at high levels in plants and algae; it is localized to chloroplasts where it plays a critical role in the capture of light energy and its photosynthetic conversion into chemical energy, thus both controlling plant growth and development and determining the color of the leaves (Yang et al. 2016; Chen et al. 2009). Chlorophyll deficiencies inevitably contribute to reduced photosynthetic efficiency and reduced crop productivity, and may even cause the death of affected plants in severe instances (Wang et al. 2018). Albino mutants are especially associated with chlorophyll deficiency and have been identified in several species, including rice (Oryza sativa L.), soybean (Glycine max), tobacco (Nicotiana tabacum L.), wheat (Triticum aestivum L.), Arabidopsis (Arabidopsis thaliana), barley (Hordeum vulgare L.), and maize (Zea mays L.) (Bae et al. 2001; Dunford and Walden 1991; Hsieh and Goodman 2005; Long et al. 1993). Albino mutants exhibiting distinct coloration can be readily selected and removed, providing an easily recognized phenotypic marker; this has been used to monitor self-pollination during hybrid rice production to ensure the purity of the genetic stock (Chen et al. 2018). While mutants with albino leaves or panicles have been the focus of extensive research owing to their distinctive and easily detectable phenotype and clear relevance to the effective breeding and production of superior rice varieties, the underlying genetic mutations and molecular mechanisms associated with the albino phenotype in rice are not yet fully understood.

At present, over 200 distinctive leaf color mutants in rice have been found, and most of which are under the control of a single recessive nuclear gene. Approximately 175 of these mutants have been characterized, leading to identification of 150 genes associated with leaf color mapped to 12 rice chromosomes. However, at least 53 of these genes, including 14 with more than one allele, have been successfully isolated from different rice cultivars, and their locations on chromosomes have been determined using map-based cloning strategies (Du et al. 2020; Deng et al. 2014). Several genes, such as VAL1, albg, PAPST1, YSA, cisc(t), Gra(t), St1, V3, V2, and V1, have been identified as responsible for the mutant phenotype of albino leaves (Zhang et al. 2018; Jian et al. 2017; Xu et al. 2013; Su et al. 2012; Lan et al. 2010; Chen et al. 2009; Yoo et al. 2009; Sugimoto et al. 2007; Kusumi et al. 2011). The successful cloning of these genes has facilitated substantial progress in clarifying the molecular mechanisms underlying leaf pigmentation alterations. The V1 gene produces an NUS1 protein that localizes to the plastid and contains the bacterial antitermination factor NusB. NUS1 plays a vital role in regulating the transcription and metabolism of ribosomal RNA in the chloroplast. It is imperative during the initial phases of leaf growth and development and is essential for proper chloroplast differentiation when exposed to cold conditions (Kusumi et al. 2011). V2, by contrast, codes for a guanosine kinase that localizes to the mitochondria and plastids, catalyzing (d)

GMP phosphorylation to (d)GDP in the guanine nucleotide metabolism pathway and facilitating plastid-to-nucleus signaling that is essential for proper chloroplast differentiation (Sugimoto et al. 2007). St1 and V3 respectively encode RNRS1 and RNRL1, which are the small and large ribonucleotide reductase (RNR) subunits, enabling RNR-mediated regulation of deoxyribonucleotide production rates in the context of DNA repair and biosynthesis while also helping to maintain an appropriate balance between cell division and plastid biogenesis during growth and development of the leaves (Yoo et al. 2009). The chloroplast-localized elongation factor Tu is encoded by G r a(t) and may be functionally related to chloroplast development (Chen et al. 2009). The cisc(t), YSA, and Albg genes code for a pentatricopeptide repeat (PPR) protein, with cisc(t) seedlings exhibiting a cold-sensitive albino phenotype (Lan et al. 2010). Abnormal PPR protein biosynthesis in cisc(t), ysa, and albg mutant plants results in dramatic reductions in total chlorophyll content and impaired chloroplast development (Su et al. 2012; Lan et al. 2010; Jian et al. 2017). The 3^ · -phosphoadenosine 5' -phosphosulfate transporter encoded by PAPST1 facilitates retrograde signaling between chloroplasts and the nucleus. OsPAPST1 localizes to the chloroplast envelope and outer mitochondrial membrane, and papst1 mutants exhibit defective thylakoid development that results in leaf chlorosis in the initial phases of leaf growth (Xu et al. 2013). VAL1 produced a phosphoribosylamine-glycine ligase that localizes to the chloroplast. It functions as the second enzyme in the de novo purine biosynthesis pathway and plays a crucial role in cell division throughout leaf development, chlorophyll biosynthesis, and in regulating chloroplast biogenesis (Zhang et al. 2018).

In rice, white panicle mutants have rarely been reported. Map-based cloning strategies have identified just eight genes related to this mutant phenotype to date: WLP1 (Song et al. 2014), WLP2 (Lv et al. 2017), St-wp (Chen et al. 2015), SLWP (Zhou et al. 2018), WP1 (Wang et al. 2016), WP2 (Wang et al. 2021), WP3 (Li et al. 2018a), and WSP1 (Zhang et al. 2017). WLP1 codes for a chloroplast 50S ribosome L13 protein, a primary plastidial ribosome component necessary for chloroplast development in rice, particularly at lower temperatures (Song et al. 2014). WLP2 encodes a plastidencoded RNA polymerase (PEP)-associated protein that localizes to chloroplasts and is primarily expressed within green tissues. Both WLP2 and its paralog OsFLN2 are capable of directly interacting with the thioredoxin OsTRXz to form a TRX-FLN regulatory module that facilitates appropriate chloroplast development under conditions of heat stress (Lv et al. 2017). Various mutations in the S t l allelic genes S t-w p and S L W P are responsible for different phenotypic characteristics, with S t l encoding RNRS1 as discussed above (Zhou et al. 2018; Chen et al. 2015). The Val-tRNA synthetase OsvalRS2 encoded by WP1 is an essential regulator of the biogenesis of chloroplast ribosomes and overall chloroplast development, with a single base change in the WP1 gene resulting in rice seedlings exhibiting virescent to albino phenotypes with white panicles at the heading stage (Wang et al. 2016). WP2 encodes OsTRXz, which regulates RNA editing by controlling the redox status of multiple organellar RNA editing factors, serving as an essential regulator of chloroplast development under hightemperature conditions (Wang et al. 2021). Appropriate interactions and coordination between chloroplasts and mitochondria are crucial for the growth and development of plants. The nuclear WP3 gene is expressed specifically in rice panicles and codes for a product that is located in the mitochondria. WP3 is crucial in maintaining proper mitochondrial function, which regulates chloroplast development in rice panicles. Mutations in WP3 lead to significant developmental abnormalities in both mitochondria and chloroplasts (Li et al. 2018a). WSP1 encodes a chloroplast-localized RNA editing factor, and a point mutation in this gene affects the editing of multiple organellar RNA sites such that wsp1 mutants exhibit impaired biogenesis of chloroplast ribosomes, impacting the development of chloroplasts in panicles and leaf tissues during the flowering stage (Zhang et al. 2017).

While a growing number of genes controlling leaf and panicle coloration in rice have been identified, the specific mechanisms underlying these changes in coloration remain an area of active research interest. In the present study, a naturally occurring rice albino mutant designated osal50 (oryza sativa albino leaf 50), derived from a breeding population of the japonica cultivar GP50 was identified and characterized. The mutant plants exhibit white-striped leaves as seedlings and white panicles at the heading stage owing to severe defects in both chloroplast development and chlorophyll biosynthesis. Moreover, this mutant has numerous white-striped leaves during the tillering stage, which is the period of most active growth. Notably, the albino phenotype exhibited by the osal50 mutant is distinct from that of any other variegated mutants reported to date in rice. We determined that the osal50 mutant phenotype is controlled by a recessive nuclear gene tentatively named 𝑂 𝑠 𝐴 𝐿50 and confined to a 160­kb physical interval on chromosome 1 between two single nucleotide polymorphism (SNP) markers, CAPS-08 and CAPS-37. Sequence and expression analyses revealed L O C_-O s O I g20I I O as the candidate OsAL50 gene, which was confirmed by knockout using CRISPR/Cas9. Subcellular localization and protein sequence analyses indicated that the OsAL50 gene likely encodes an endoribonuclease E (RNase E)-like protein localized to the chloroplasts, where it is involved in RNA processing within the plastids and is an essential regulator of the biogenesis and development of chloroplasts. Our results offer valuable insights into the function of OsAL50 and provide genetic resources for studying the molecular determinants of changes in leaf and panicle coloration.

# Materials and methods

# Plant materials, mapping populations, and growth conditions

The osal50 is a spontaneous rice albino mutant derived from an advanced-generation breeding population of the japonica cultivar GP50 grown under normal field conditions and originating from the Sanming Academy of Agricultural Sciences (SAAS). This osal50 mutant and its progeny exhibited distinctive albinism affecting their leaves and panicles, and the mutant phenotype was stably inherited over several generations. In this study, segregating populations used for genetic analysis and gene mapping were developed by crossing the osal50 mutant with the japonica cultivars GP50 and Ningjing44 (NJ44) as well as the indica cultivar 9311. In total, 526 and 685 recessive F_2 individuals exhibiting the albino phenotype were obtained from the respective 𝐍𝐉44×o s a l5O and 9311×o s a l50 crosses and used for genetic mapping of O s A L5O . The rice plants utilized in the present study were cultivated using standard management practices in a paddy field at the SAAS in Shaxian, Fujian Province, China.

# Analysis of agronomic traits

During the summer of 2022, the identical crop fields at the SAAS were planted with wild-type GP50 and osal50 mutant rice plants. The plants were cultivated under the same environmental conditions and management techniques. Three biological replicates were used to investigate the following key agronomic traits after the plants reached the mature stage: plant height, seed setting rate, 1000-grain weight, panicle length, number of grains per panicle, effective panicle number, grain length, grain width, flag leaf length, and flag leaf width.

# Analysis of pigment levels

Fresh leaves, glumes, and branches from wild-type GP50 and osal50 mutant plants were sampled (approximately 0.2 g) during different periods of growth. The samples were cut into segments and incubated for 24 h in 10 mL of 100% (v/v) alcohol at 26 ^∘𝐂 in the dark until the segments were fully discolored. Using a multifunctional enzyme-labeling instrument (TECAN infinite M200PRO), absorbances at 470, 649, and 665 nm were determined after the supernatants were collected. For every sample, three biological duplicates were examined. The levels of carotenoid (Car)

and chlorophyll (Chl a and Chl b) in the samples were determined as described by Lichtenthaler (1987).

# Transmission electron microscopic (TEM) analysis of chloroplast ultrastructure

White sections of fresh leaves harvested from osal50 plants at the four-leaf stage and white sections of fresh branches and glumes obtained separately from young panicles at the flowering stage were collected for analysis, as were the corresponding green tissue samples from wild-type plants. The samples were separated into small pieces, and fixed with 2.5% (v/v) glutaraldehyde (diluted with phosphate buffer, pH 7.2) under vacuum until the specimens were completely submerged. Standard protocols were used to fix, dehydrate, embed, section, and stain the specimens, and the processed specimens were then examined under a TEM (HITACHI HT-7800, Tokyo, Japan) to analyze chloroplast ultrastructure (Wu et al. 2016; Rao et al. 2015).

# DNA extraction, gene pool construction, and development of molecular markers

The genomic DNA was extracted from fresh leaves using a slightly modified version of the cetyltrimethyl ammonium bromide-based (CTAB) approach (Murray and Thompson 1980). The multifunctional enzyme-labeling equipment was used to quantify the genomic DNA's OD value. DNA samples from each plant were diluted to a final concentration of 50 ng/ , and diluted DNA of randomly selected wildtype or mutant individuals from the F_2 population _n=20 each) were mixed to produce two extreme DNA bulk-pools (Michelmore et al. 1991). Simple sequence repeat (SSR) primers uniformly distributed over the 12 rice chromosomes were collected from the International Rice Microsatellite Initiative (http://​archi​ve.​grame​ne.​org/​marke​rs/​micro​sat). Genome sequences of the japonica and indica cultivars Nipponbare and 9311 were obtained by referring to the National Center for Biotechnology Information (NCBI) GenBank database (https://​www.​ncbi.​nlm.​nih.​gov/​genba​nk/) and the Rice Genome Annotation Project (RGAP; http://​rice.​plant biolo​gy.​msu.​edu/). Sequences were searched for SSR repeats using the SSRIT program (http://​archi​ve.​grame​ne.​org/​db/ marke​rs/​ssrto​ol). Furthermore, Clustal Omega (https://​www. ebi.​ac.​uk/​Tools/​msa/​clust​alo/) and DNAMAN (http://​www. lynnon.​com/) were used to align the genomic sequences of Nipponbare and 9311 to find any loci for insertion/deletion (InDels) or single nucleotide polymorphisms (SNPs). Using Primer Premier 5.0 (Premier Biosoft International, CA, USA), site-specific primer pairs flanking SSR, SNP, and InDel sequences were developed. These were subsequently used to examine the polymorphism in both the parental lines and the extreme pools and to genotype F_2 individuals with mutant phenotypes using 8% nondenaturing polyacrylamide gel electrophoresis. This analysis uncovered SSR, InDel, and SNP molecular markers closely associated with the OsAL50 locus, facilitating fine mapping to be carried out (Wei et al. 2019). Online Resource 1 lists the enzymes utilized, PCR product sizes, and primer sequences.

# Genetic analysis and fine mapping of OsAL50

Reciprocal crosses were made between osal50 and indica cultivar 9311 as well as osal50 and the japonica cultivars GP50 and NJ44 to determine whether the osal50 mutant phenotype was under the control of a single recessive nuclear gene. After that, the χ^2 test was used to examine and assess the phenotypes of the F_1 and F_2 plants that resulted from these crosses.

For the genetic mapping of O s A L5O , recessive class analysis (RCA) and bulk segregation analysis (BSA) were employed (Michelmore et al. 1991). F_2 populations developed from crossings between the osal50 mutant and the japonica cultivar NJ44 or the indica cultivar 9311 were used for fine mapping. A standard paddy field was used for growing parental, F_1 , and F_2 plants. Employing Mapmaker/ Exp 3.0, a linkage analysis for molecular markers and the OsAL50 locus was carried out based on genotyping data for 1,211 recessive F_2 individuals displaying the mutant phenotype (Lander et al. 1987). After converting recombination rates into genetic distances using the Kosambi operating function, a genetic linkage map was produced using the MapDraw program (Liu and Meng 2003). Based on the contig map, a physical map spanning the O s A L5O locus was generated using chromosomal walking (International Rice Genome Sequencing Project 2005).

# Sequencing and phylogenetic analysis

Gene prediction and structural analysis were performed using the RAP-DB (http://rapdb.dan.affrc.go.jp/) and RGAP (http://rice.plantbiology.msu.edu/) databases. Candidate genes were amplified using genomic DNA from the wildtype GP50 and the osal50 mutant as templates. The products amplified from these DNA sequences were separated using 1% (w/v) agarose gel electrophoresis, purified using a DNA Gel Extraction Kit (Takara, Dalian, China), and sequenced by Takara Biotech (Dalian, China). Clustal Omega and DNAMAN software were used to assemble and align the sequences.

The BLASTP Search Program of the NCBI database (https://​www.​ncbi.​nlm.​nih.​gov/) was utilized to find homologous sequences of O s A L50 , and the DNAMAN program was then utilized for multiple sequence alignments. Employing MEGA 7.0, a phylogenetic tree was constructed using the neighbor-joining method with 1000 replicates. PlantCARE (http://​bioin​forma​tics.​psb.​ugent.​be/​webto​ols/​plant​care/ html/) was used to search for cis-elements in the approximately 1300 bp of DNA sequence that came before the ATG start codon in the OsAL50 promoter, which was received via RGAP.

# RNA extraction and gene expression analysis

Using a MiniBEST Universal RNA Extraction Kit (Takara, Dalian, China), total RNA was isolated from the fourth leaves (four-leaf stage), seventh leaves (seven-leaf stage), and immature panicles (comprising young branches and glumes; heading stage) of wild-type and osal50 plants. After undergoing RNase-free DNase treatment, the purity of the RNA was evaluated using 1% (w/v) agarose gel electrophoresis. Following this, 2 g of RNA per sample was used to prepare cDNA with HiScript Reverse Transcriptase SuperMix (Vazyme, Nanjing, China). ChamQ SYBR Green qPCR Master Mix (Vazyme, Nanjing, China) and an ABI QuantStudio 3 instrument were used for real-time fluorogenic quantitative PCR analysis, with three biological replicates per sample. Relative gene expression was calculated using the comparative 2^- CT method. Sixteen target candidate genes were analyzed, with OsActin serving as a reference control. Online Resources 2 and 3 contain the primers utilized in sequencing analysis and for determining the expression levels of candidate genes, respectively. The transcript levels of genes associated with photosynthetic pigment metabolism (OsCAO1, OsCHLI, OsCHLM, OsCHLD, OsCHLa/b, OsDVR, OsHEMA1, OsPSY1, OsPSY2, and OsPSY3), chloroplast development (OsPOLP, Ftsz, OsRPOTP, OsRPOB, V2, and V3), and photosynthesis (OsCAB1R, OsCAB2R, OsPETA, OsPETB, OsPSAA, OsPSBA, OsRBCL, and OsRBCS) were further examined in various tissues and at different growth stages of wild-type and osal50 mutant plants. Specific primers used for analyzing the expression levels of these genes were designed according to previous reports (Zhang et al. 2019; Shang et al. 2019) and are listed in Online Resource 4.

# Vector construction for the CRIPSR/Cas9 system

OsAL50 was knocked out in wild-type GP50 using a modified tRNA-processing method based on the CRISPR/ Cas9 system. The methods employed have been described previously (Wang et al. 2018). Two sgRNA target sites (CAA​TTG​ATG​CAT​GCT​GTT​AGAGG, TCT​GGT​GGA​CAT CAA​TGG​CTTGG) in exon 1 were designed to construct the OsAL50-Cas9 vector. This vector was then transformed into wild-type calli by an Agrobacterium tumefaciens-mediated process (Lv et al. 2020). Positive transgenic plants from gene-edited lines of the T_0 generation were identified using the specific functional markers FMHPT-F/R 5^' -ATT​TGT GTA​CGC​CCG​ACA​GT- .3^' and 5^'. -GTG​CTT​GAC​ATT​GGG GAG​TT-3′) and FMCas9-F/R ( E^' -AGA​ACC​TCT​CCG​ATG CTA​TCC- .3^' and 5^' -AGC​AAG​AGG​ACA​ACG​TAG​-3′) (Wei et al. 2023).

# Subcellular localization of OsAL50

The pOsAL50-GFP vector was constructed by fusing the full-length coding sequence of O s A L5O to the N-terminus of GFP in the pAN580 vector. As previously described, the pOsAL50-GFP vector and the empty vector pAN580 were transformed into rice protoplasts (Lv et al. 2020). Employing a confocal laser scanning microscope (LSM700, Jena, Germany), images of the GFP signals were acquired.

# Statistical analysis

The data is presented as the average value ± standard deviation (SD) derived from three biological replicates. Comparison of results was conducted using unpaired Student’s t -tests. A significance level of P<0.05 was used to determine statistical significance. The ^*P<0.05 and ∗∗ P<0.01 indicated significance levels. The statistical analyses in this study followed the techniques mentioned above.

# Results

# Phenotypic and agronomic characteristics of the osal50 mutant

We initially subjected osal50 mutant rice plants to comprehensive phenotypic characterization. As shown in Fig. 1, both leaves and panicles of the mutant plants displayed significant albino characteristics in a growth stage-dependent manner when cultivated under normal field conditions. Specifically, in seedlings, it was observed that the leaves of this mutant developed white-green stripes running parallel to the mid-veins, with some leaves appearing almost fully white. However, we noticed that the branches of young panicles were completely white, while the glumes were white with some yellow and green coloration at the flowering stage. To fully explore the development of the albino character in the osal50 mutant, we next investigated osal50 mutant plants during different stages of growth, finding that the seedlings of this mutant produced a white-striped leaf phenotype starting from the three-leaf stage, which was particularly obvious at the four-leaf stage, whereafter they gradually became increasingly green from the five-leaf stage onward (Fig. 1a–e). When reaching the seven-leaf stage, the osal50 mutant seedlings exhibited an almost completely normal green leaf phenotype consistent with that of the wild type (Fig. 1f). Moreover, it is interesting that these mutant plants also had numerous white-striped or white leaves intermixed with normal leaves at the tillering stage, which is the period of most active growth (Fig. 1g). The osal50 mutant plants thus showed inconsistent leaf coloration at the tillering stage, with varying patterns and degrees of variegation even within a single plant, ranging from leaves that were completely white to those that were green, heavily striped, and slightly striped (Fig. 1h). In contrast to the light green glumes and branches exhibited by immature panicles of wild-type plants at the heading stage, young panicles of osal50 showed distinctive albino features, as described above (Fig. 1i–k). While osal50 mutant branches remained white when reaching the mature stage, the glumes of the panicles gradually turned black-brown in color (Fig. 1l). We were then cultivated wild-type and osal50 plants at a range of temperatures (20, 25, 30, and 35 ^∘C_, ) to determine whether the albino phenotype was temperature-sensitive. Based our examination, no pronounced phenotypic differences were observed in the osal50 mutant plants compared with the wild type at any of the tested temperatures, suggesting that generation of the albino phenotype in osal50 is not influenced by temperature changes (data not shown). To evaluate the link between the mutant phenotype and yields, we next investigated the following agronomic traits: plant height, 1000-grain weight, panicle length, number of grains per panicle, effective panicle number, seed setting rate, grain length, grain width, flag leaf length, and flag leaf width. Key yield-related traits, including 1000-grain weight, seed setting rate, and effective panicle number were lower in osal50 mutant plants than in the wild type (Fig. 2). This suggests that changes in leaf and panicle color may lead to a reduction in photosynthetic efficiency, thereby decreasing the accumulation of substances required for effective growth in rice.


Fig. 1   Phenotypic characterization of wild-type GP50 and osal50 panicle (j) phenotypes at the heading stage. k–l Wild-type and osal50 mutant plants. a–d, f Phenotypic comparisons of wild-type and  grain phenotypes at the heading ( k) and mature (l) stages. osal50-1, osal50 seedlings at the one- (a), two- (b), three- (c), four- (d), and  -2, and -3 correspond to three different types of white-striped leaves seven-leaf (f) stages. e White-striped leaves from osal50 mutant from osal50 mutant plants at the four-leaf stage (e) and tillering stage plants at the four-leaf stage. g-h Wild-type and osal50 leaf pheno- (𝐡) , respectively. Bars: 1 cm (a–f, k, l); 5 cm (g, h, j); 10 cm (i) types at the tillering stage. i–j Wild-type and osal50 plant (i) and

# Photosynthetic pigment levels and chloroplast ultrastructural characteristics of osal50 mutant plants

The white-striped leaf phenotypes exhibited by osal50 mutant plants were most pronounced at the four-leaf and tillering stages, and marked differences in the extent of leaf albinism were seen even within a given plant during these growth stages. Therefore, to determine the levels of photosynthetic pigments in osal50 mutant plants over the course of growth and development, we analyzed chlorophyll a, chlorophyll b, and carotenoid levels in three different types of white-striped leaves, osal50-1 (slightly striped), osal50- 2 (heavily striped), and osal50-3 (almost fully white), collected from osal50 mutant plants at the four-leaf and tillering stages, respectively (Fig. 1e, h), together with green leaves harvested from wild-type plants at these two stages. As expected, osal50 leaves had significantly lower pigment levels relative to those in wild-type leaves, with 26.01-94.91% ,33.56-96.87% , and 40.67-98.19% less chlorophyll a, chlorophyll b, and carotenoid in these white-striped leaves at the four-leaf stage, respectively, and 29.96-89.03% ,35.43-91.89% , and 42.93-96.33% less chlorophyll a, chlorophyll b, and carotenoid, respectively, at the tillering stage (Fig. 3a, b). Similarly, the chlorophyll a, chlorophyll b, and carotenoid levels in glumes and branches harvested at the heading stage from young panicles of osal50 were 54.06% ,49.09% , and 61.79% lower and 95.81% ,97.4% , and 92.07% lower, respectively, than those of the wild type (Fig. 3c, d).


Fig. 2   Comparisons of agronomic traits in wild-type GP50 and  grain weight (g); i Flag leaf length (cm); j Flag leaf width (cm). Data osal50 mutant rice plants. a Plant height (cm); b Effective panicle  represent the mean ± SD of three biological replicates. Student’s t -test number; c Panicle length (cm) ; d Grain length (cm); e Grain width  was used for statistical analysis (cm); f Seed setting rate (%) ;𝐠 Number of grains per panicle; h 1000-

The  underlying  physiological  basis  for  lower photosynthetic pigment levels in osal50 mutant plants was explored by investigating differences in chloroplast morphology and development in various tissues and at different growth stages of wild-type and osal50 mutant plants via TEM. Leaves collected at the four-leaf stage, as well as glumes and branches harvested at the heading stage, were examined. As expected, organelles in the green sections of various tissue samples from wild-type plants were intact and evenly distributed within appropriately developed cells, with fully developed chloroplasts showing tightly stacked grana lamellae (Fig. 3e–j). By contrast, most cells in the white sections of leaves, glumes, and branches from osal50 mutant plants showed extremely simple internal structures; the chloroplasts were severely degraded and lacked any organized thylakoid structures. In addition to overall reductions in chloroplast numbers and volumes, there was a clear absence of lamellae stacking within the disordered grana (Fig. 3k–p). The findings demonstrate that the mutant phenotype and decreased chlorophyll and carotenoid levels of osal50 result from abnormal chloroplast development under natural conditions.


Fig. 3   Comparisons of photosynthetic pigment levels and chloroplast ultrastructures in wild-type GP50 and osal50 mutant plants. a, b Photosynthetic pigment levels in leaves harvested at the four-leaf stage (a) and tillering stage ( b) of wild-type and osal50 mutant plants. c-d Photosynthetic pigment levels in glumes (c) and branches (d) from young panicles of wild-type and osal50 mutant plants at the heading stage. e-j Chloroplast ultrastructural characteristics in green sections of leaves (e, f), branches (𝐠,𝐡) , and glumes (𝐢,𝐣) from wild-type plants. 𝐤-𝐩 Chloroplast ultrastructural characteristics in white sec
tions of leaves (𝐤,𝐥) , branches (𝐦,𝐧) , and glumes (Φ,  p) from osal50 mutant plants. Bars: 1 m (𝐟,𝐡,𝐣,𝐥,𝐧,𝐩) ;5 m (e, g, i, k, m, o). FW fresh weight; osal50-1, osal50-2, and osal50-3 three different types of white-striped leaves from osal50 mutant plants as shown in e and 𝐡 ;C h chloroplast; N nucleus; G grana stacks; A amyloplast; o G osmophilic plastoglobuli; C W cell wall; M mitochondria. Data represent the mean ± SD of three biological replicates. Student’s t -test was used for statistical analysis, ^*P<0.05 ,**P<0.01

# Genetic analysis and preliminary mapping of OsAL50

The F_1 and F_2 populations derived from reciprocal crosses between the osal50 and the japonica cultivars GP50 or NJ44 or the indica cultivar 9311 were used for genetic analysis to determine whether a single recessive nuclear gene was responsible for the osal50 mutant phenotype. Leaves from all F_1 plants exhibited normal phenotypes consistent with those of wild-type GP50 plants, confirming that the osal50 mutant phenotype is recessive. Moreover, no intermediate phenotypes were identified, and the mutant phenotype was segregated in the F_2 populations. The segregation ratio of the wild-type to the mutant phenotype in these four populations was approximately 3:1 as determined through a χ^2 test (χ^2<χ^2_0.05=3.84 ;P>0.05 ; Table 1), suggesting that the osal50 mutant albino phenotype is under the control of a single recessive nuclear gene.

Table 1   Segregation analysis of osal50 crosses



We next performed O s A L5O mapping using the F_2 population generated from the cross between osal50 and NJ44. Both parental lines and two extreme DNA bulk-pools were initially screened for polymorphisms using 336 pairs of SSR markers released by the International Rice Microsatellite Initiative, distributed evenly across all 12 rice chromosomes. Polymorphic SSR markers associated with the albino phenotype in both the parental lines and the extreme pools were confirmed by analysis of F_2 mutant individuals. An SSR marker pair, Rm562 , on the short arm of chromosome 1 that was linked to the O s A L5O locus was successfully identified (Fig. 4a). Analysis of 115 pairs of SSR markers densely distributed in regions adjacent to Rm562 revealed four additional pairs linked to O s A L5O (Rm6642, Rm10701, Rm10710 , and Rm10747_. ) (Fig. 4b). PCR genotyping of 526 F_2 individuals exhibiting the mutant phenotype derived from the cross between osal50 and NJ44 parental lines was next performed, leading to the identification of 35, 5, 4, and 15 recombinant events, respectively. The recombinant positions of the Rm6642 , Rm10701, and Rm10710 markers were identical and were located on the same side of the OsAL50 locus, whereas Rm10747 was on the other side of the locus along with Rm562 . These results were used to preliminarily map O s A L5O to chromosome 1 between the Rm10710 and Rm10747 markers at respective genetic distances of 0.4 cM and 1.8 cM (Fig. 4b). Further review of the chromosomal location of O s A L50 suggested that it was distinct from other genes previously associated with changes in leaf or panicle coloration, so we speculate that O s A L5O is a newly identified gene.

# Fine mapping of OsAL50

The preliminary mapping results above were next used to perform additional fine mapping of OsAL50 using a new mapping population developed by crossing the osal50 mutant with the indica rice cultivar 9311, which showed significant polymorphism in the target mapping region. To facilitate the localization of O s A L5O to a smaller region on chromosome 1 and develop tightly linked molecular markers, the publicly available genome sequences of the preliminary mapping regions in Nipponbare and 9311 were obtained from the GRAMENE and NCBI databases. These sequences were then analyzed using the SSRIT program, which allowed for the development of 80 pairs of new SSR markers within the preliminary mapping region. Alignment of the Nipponbare and 9311 sequences led to the development of 95 pairs of de novo InDel and CAPS markers that were used to narrow the genomic region encompassing the OsAL50 locus between the flanking Rm10710 and Rm10747 markers. Following screening of polymorphic markers in the two parental lines and the two extreme DNA bulk-pools, and confirmation by analysis of F_2 mutant individuals, Mapmaker/Exp 3.0 was employed to conduct a linkage analysis, leading to the identification of 14 molecular markers closely associated with the O s A L5O locus (InDel-01, CAPS-08, CAPS-40, CAPS-37, InDel-07, InDel-08, ZZY-01, ZZY-04, InDel11, InDel-13, InDel-15, InDel-18, InDel-20, and InDel-29) (Fig. 4b). These markers were used for genotyping 685 F_2 individuals with the mutant phenotype derived from the cross between osal50 and 9311. Three recombinant events were detected with the InDel-01 marker, four with the ZZY01 or ZZY-04 markers, five with the InDel-11, InDel-13, InDel-15, InDel-18, InDel-20, or InDel-29 markers, and just one with the CAPS-08 or CAPS-37 markers; the CAPS-40 marker co-segregated with OsAL50. We thus determined that the OsAL50 locus is delimited by the closely flanking CAPS08 and CAPS-37 SNP markers, with a genetic distance of 0.1 cM from each of them. Finally, a high-resolution genetic linkage map of the O s A L5O locus was constructed using a combination of Mapmaker/Exp 3.0 and MapDraw software (Fig. 4b). Moreover, DNA sequences of bacterial artificial chromosome (BAC) clones and their corresponding physical locations within the Nipponbare (japonica) genome allowed anchorage of the molecular markers linked to OsAL50 to appropriate positions of the BAC clones, enabling the construction of a physical contig map spanning the O s A L50 locus based on the genotyping results of 1211 F_2 mutant individuals (Fig. 4c). This demonstrated that the O s A L5O locus is located within the AP002873 and AP003567 regions, and that the CAPS-40 marker co-segregating with O s A L5O is located on the AP002873 BAC clone (Fig. 4c).


Fig. 4   Genetic and physical maps of the region including the O s A L5O bers of recombinants identified from 526 (𝐍𝐉44×o s a l5O) and 685 locus. a Molecular marker-based linkage maps of rice chromosome (9311× o s a l50) F_2 mutant individuals. Physical locations on chromo1, with the short arm of the chromosome at the top. Marker names some 1 of the corresponding linkage markers are denoted by numbers and corresponding map distances (centimorgan, cM) are shown, below the map that were determined using the GRAMENE database with the centromere (CEN) and approximate O s A L5O locus loca- (http://​www.​garme​ne.​org).  Relative  positions  of  linkage  markers tions represented by black and red rectangles, respectively. b Fine in Nipponbare BAC clones are designated with vertical and dashed genetic linkage map of the O s A L5O locus on chromosome 1. Markers lines. d The locus spanning O s A L5O was narrowed down to a 160­kb tightly linked to O s A L5O are named on the right of the map, while region of the genome between the CAPS-08 and CAPS-37 markers. genetic distances are shown to the left in cM. c Physical contig map e Twenty-six predicted open reading frames identified as candidate of the region spanning the O s A L5O locus, generated using 1,211 F_2 O s A L5O genes according to genomic information of the japonica rice mutant individuals. Chromosomal orientation is as noted, and num- cultivar Nipponbare (http://​rice.​plant​biolo​gy.​mus.​edu/) bers in parentheses beneath each molecular marker denote the num

# Candidate genes for OsAL50

Fine mapping of the OsAL50 locus successfully delimited it to a 160­kb region flanked by the CAPS-08 and CAPS-37 markers, covering portions of the AP002873 and AP003567 BAC clones (Fig. 4d). Using the RAP-DB and RGAP genomic annotation databases, 26 predicted open reading frames were identified within this 160-kb region as candidate OsAL50 genes (Fig. 4e). These included L O C_- Os01g19970, LOC_Os0120140, and L O C_-OsO I g20I6O ,which, respectively, encode a member of the MYB transcription factor family, a latency-associated nuclear antigen, and an OsHKT1; 5-Na^+ transporter. Of these genes, 13 (LOC_Os01g19910, LOC_Os01g19940, LOC_Os01g19950, LOC_Os01g19980, LOC_Os01g19990, LOC_Os01g20000, LOC_Os01g20010, LOC_Os01g20020, LOC_Os01g20030, LOC_Os01g20060, LOC_Os01g20100, LOC_Os01g20110, and LOC_Os01g20120) encode hypothetical or expressed proteins, whereas 10 (LOC_Os01g19920, LOC_ Os01g19930, LOC_Os01g19960, LOC_Os01g20040, LOC_ Os01g20050, LOC_Os01g20070, LOC_Os01g20080, LOC_ Os01g20090, LOC_Os01g20130, and L O C_-O s O I g20I5O) encode transposons or retrotransposons (Online Resource 5). These latter 10 genes were excluded, considering the potential relationships between their predicted functions and the mutant phenotypes observed in osal50 mutant plants. Next, we performed a series of semi-quantitative reversetranscription PCR (RT-PCR) analyses to investigate the transcript levels of the remaining 16 genes in wild-type GP50 and osal50 mutant leaves at the four-leaf stage, as this was the stage at which the albino phenotype was most pronounced. The transcript levels of L O C_-O s O I g20I I O were markedly lower in the osal50 mutant compared with those in the wild type (Fig. 5a, b). Of the remaining 15 candidate genes, L O C_-O s O I g I9980 and L O C_-O s O I g20I60 exhibited extremely poor expression in both the wild type and the osal50 mutant, while the other 13 genes were expressed at comparable levels in both lines (Fig. 5a, b). Considering these findings, we further examined the accumulation of L O C_O s O I g20I I O mRNA transcripts in young panicles of wild type and osal50, including both glumes and branches, at the heading stage using a combination of semi-quantitative RT-PCR and quantitative RT-PCR (qRTPCR), with additional qRT-PCR analysis used to confirm the previously detected differences in L O C_-O s O I g20I I O expression in leaves at the four-leaf stage between the two parental lines. Consistent with earlier results, expression levels of L O C_-O s O I g20I I O were markedly lower in osal50 mutant glumes and branches at the heading stage and leaves at the four-leaf stage, with 51.9% ,58.5% , and 71.5% lower expression levels, respectively, relative to wild-type GP50 plant tissues (Fig. 5b, c). Overall, these results indicated that L O C_-O s O I g20I I O is the most likely candidate for the OsAL50 gene.

# Sequence analysis and functional confirmation of OsAL50

To confirm the identity of L O C_O s O I g20I I O as the OsAL50 gene, the coding sequences (CDS) of L O C_-O s O I g20I I O were amplified using genomic DNA from the wild-type GP50 and the osal50 mutant as templates, followed by sequencing. The DNA sequences were then aligned to compare differences in the L O C_-O s O I g20I I O sequences between the two parental lines. The RGAP database (http://​rice.​plant Fig. 5   OsAL50  candidate  gene  analysis.  a  Expression  levels  of 15 candidate genes in wild-type GP50 and osal50 mutant leaves at the four-leaf stage, determined by semi-quantitative RT-PCR. b–c Expression levels of L O C_O s O I g20I I O in wild-type GP50 and osal50 mutant plants, determined by semi-quantitative RT-PCR (b) and qRT-PCR (c). Leaves were collected at the four-leaf stage (left) and glumes (middle) and branches (right) were harvested separately from young panicles at the heading stage for cDNA preparation. d biolo​gy.​msu.​edu/) indicates that L O C_-O s O I g20I I O is a single-copy gene consisting of seven exons and six introns, with a 1338-bp CDS encoding a 445-aa polypeptide. However, we did not identify any differences in the coding regions between the allelic L O C_-O s O I g20I I O gene sequences from the wild type and the osal50 mutant. Accordingly, we amplified the respective promoter regions upstream of the gene sequences from the two parental genomes and identified 20 mutation sites within the promoter of L O C_-O s O I g20I I O in the osal50 mutant comprising mostly single-base substitution and deletion mutations. Two small fragments, namely, ‘TAT​TTA​CTT​GTA​’ and ‘ATG​TAC​CG’, were inserted into adjacent positions 2813 and 2808 bp upstream of the ATG start codon, respectively (Fig. 5d, e). Further investigation of the promoter and coding sequences of L O C_-O s O I g20I I O in Nipponbare, 9311, and NJ44 revealed that these sequences were identical to those of the wild-type GP50 (Fig. 5d, e). Moreover, to confirm that these mutations were exclusive to the osal50 mutant, we examined another 20 rice varieties Mutation analysis in the L O C_O s O I g20I I O promoter region, where “–” indicates base deletion. e Mutation types and locations in the defined promoter region of L O C_O s O I g20I I O , with specific positional information being provided in brackets. Relative gene expression was calculated using the comparative 2^-CT method, OsActin served as a normalization control. Error bars indicate SD .n=3] . Student’s t -test was used for statistical analysis, ^*P<0.05 ,∗∗ P<0.01



and found that only the osal50 mutant, and not even one of the wild-type rice varieties, contained these specific mutations in the promoter region of L O C_-O s O I g20I I O (Online Resource 6). These results further supported the tentative identification of L O C_O s O I g20I I O as the candidate O s A L5O gene. To confirm this conclusion, we used the CRISPR/ Cas9 system to knock out L O C_-O s O I g20I I O in the wildtype GP50 (Fig. 6a). Two homozygous positive transgenic plants were identified among gene-edited lines of the T_0 generation, and seedlings of these independent transgenic lines displayed an obvious white-striped leaf phenotype consistent with that of the osal50 mutant (Fig. 6b, c). Specifically, 3-bp and 11-bp deletions were detected at the target site of L O C_-O s O I g20I I O in the two transgenic lines, respectively (Fig. 6d). These deletions resulted in frameshift mutations. Thus, we verified that L O C_O s O I g20I I O was the candidate O s A L5O gene, with mutations in the associated promoter region potentially contributing to the observed albino phenotype in osal50 mutant plants.


Fig. 6   CRISPR/Cas9 verification of the OsAL50 gene. a Vector map of Cas9/gRNA . LB, UBI, Cas9, gRNA, rU6, 35S, Hygro, and RB represent the vector left border, UBI promoter, Cas9 gene, guide RNA, rice U6 promoter, 35S promoter, hygromycin, and vector right border, respectively. b PCR identification of positive transgenic plants in gene-edited lines of the T_0 generation using the functional markers FMHPT-F/R (left) and FMCas9-F/R (right), respectively, where

The non-coding promoter region contains many different sites for transcription-factor binding, including the basal core promoter (BCP) as well as regions involved in transcriptional regulation. The BCP facilitates the basal transcriptional level and serves as the minimum genomic unit necessary for proper transcription. Other transcriptional regulatory regions are associated with adjustment of the expression levels of specific genes in response to changes in environmental conditions (Du et al. 2020). Given that O s A L5O mutations were only detected in the promoter region in the osal50 mutant, a 1300-bp region located 2100‒3400 bp upstream of the O s A L5O start codon and including the primary locations of these mutation sites was selected for in-depth analysis to understand the regulatory mechanism of O s A L5O expression. Sequences of this region were analyzed using PlantCARE, which revealed the presence of a cluster of light-responsive G-box cis-acting elements, meristem expression-related CAT-box elements, MeJA-responsive CGTCA- and TGACG-motifs, light-responsive TCT-motifs and Sp1 elements, salicylic acid-responsive TCA-elements, and auxin-responsive TGA-elements, in addition to abundant core promoter elements including TATA- and CAAT-boxes (Online Resource 7). Thus, it appears likely that O s A L50 serves as a regulator of photosynthesis in rice, given the abundant light-responsive elements in its promoter region, ω̇+ω̇ and “−” indicate the positive control and negative control, respectively; M represents a 2000-bp DNA marker. c Phenotype of T_0 mutant seedlings generated using the CRISPR/Cas9 system. Bar: 5 cm . d Mutation analysis at the target site of L O C_O s O I g20I I O in transgenic lines cas9- 1^# and -2^# , where “-” indicates base deletion and PAM represents the protospacer adjacent motif

with the promoter sequence covering these mutations being essential for OsAL50 function.

Multiple sequence alignment revealed that OsAL50 shares high sequence similarity with homologous proteins from Zea mays, Brachypodium distachyon, Sorghum bicolor, Triticum aestivum, Hordeum vulgare, and Setaria italica, with identities of 74.45% ,85% ,86.35% ,86.57% ,87.41% , and 88.32% , respectively (Fig.  7a). Notably, genes homologous to O s A L5O in these plant species were predicted to encode an RNase E-like protein according to annotations in the NCBI and Phytomozome databases (https://​phyto​zome-​next.​jgi.​doe.​gov/). Moreover, this protein is localized to the thylakoids, where it plays a crucial role in the processing of plastid RNA and is required for chloroplast development and biogenesis (Schein et al. 2008; Mudd et al. 2008; Walter et al. 2010). Furthermore, a phylogenetic tree was constructed to investigate the possible relationships between OsAL50 homologs in evolutionary history. As shown in Fig. 7b, they could be divided into two subgroups: the proteins from monocotyledonous plants were placed clearly into one subgroup, while the proteins from dicotyledonous plants formed another subgroup (Fig. 7b). These data indicate that OsAL50 is highly conserved in higher plants, particularly among the Gramineae. To determine whether OsAL50 was localized to the chloroplasts, the pOsAL50-GFP vector was introduced into rice protoplasts. The green fluorescent signals from the OsAL50-GFP fusion proteins overlapped with the chloroplast autofluorescence (Online Resource 8). Considering these findings, we inferred that OsAL50 most likely encodes an RNase E-like protein localized to the chloroplasts that is essential for chlorophyll biosynthesis and chloroplast development in rice.


Fig. 7   Sequence analysis of the OsAL50 protein. a Alignment of by MEGA 7.0 using the neighbor-joining method with 1000 replicaderived amino acid sequences of OsAL50 and its homologous pro- tions. All protein sequences were obtained from the NCBI database, teins. Dark blue highlights identical residues, while similar residues  with corresponding GenBank accession numbers shown in parentheare marked in pink or light blue. b Phylogenetic tree representing the ses. The scale indicates the percentage substitution rate per site alignment of OsAL50 and homologous protein sequences constructed

# Expression patterns of genes associated with photosynthetic pigment metabolism, chloroplast development, and photosynthesis in osal50

Both the leaves and panicles of osal50 mutant plants exhibited distinctive growth stage-dependent albino characteristics during the regular growing season. Changes in the coloration of these tissues and organs are associated with photosynthetic pigment metabolism, chloroplast development, and photosynthesis. We therefore investigated the transcript levels of 25 genes involved in these processes in various tissues of wild-type and osal50 mutant plants during different stages of development using qRT-PCR analysis. These included 11 genes associated with chlorophyll and carotenoid biosynthesis, encoding glutamyl-tRNA reductase (OsHEMA1; Schmied et al. 2011), chlorophyllide a oxygenase 1 (OsCAO1; Yang et al. 2016), chlorophyll synthase (OsCHLa/b; Karlin-Neumann 1988), magnesium chelatase subunits I and D (OsCHLI and OsCHLD; Zhang et al. 2006), 𝐌g -protoporphyrin IX methyltransferase (OsCHLM; Tomiyama et al. 2014), uroporphyrinogen-III synthase (OsHEMD), 3,8-divinyl reductase (OsDVR; Wang et al. 2010), and phytoene synthases 1, 2, and 3 (OsPSY1, OsPSY2, and OsPSY3; Yang et al. 2021). Further analysis was conducted on six chloroplast development-related genes encoding guanylate kinase (V2 and V3; Sugimoto et al. 2007; Yoo et al. 2009), plastid division protein (Ftsz; Gong et al. 2014), RNA polymerase  subunit (OsRPOB; Inada et al. 1996), plastid RNA polymerase (OsRPOTP; Kusumi et al. 2004), and plastid polI-like DNA polymerase (OsPOLP; Takeuchi et al. 2007), as well as eight genes related to photosynthesis, encoding the light-harvesting chlorophyll a/b-binding protein 1/2R of PSII (OsCAB1R and O s C A B2R ; Matsuoka 1990), photosystem I/II reaction-center protein A/B (OsPSAA and OsPSBA; Kapoor et al. 1994), cytochrome subunits f and B6 (OsPETA and OsPETB; Shang et al. 2019; Zhang et al. 2019), and the small and large subunits of soluble Rubisco proteins (OsRBCS and OsRBCL; Kyozuka et al. 1993). All of these genes were significantly down-regulated in osal50 mutant leaves compared with wild-type leaves at the fourleaf stage (Fig. 8a), with consistent expression patterns of transcriptional suppression in young panicles, including glumes and branches, of osal50 at the heading stage, except for the chloroplast development-related FtsZ, OsRPOB, and V3 genes (Fig. 8c, d). Moreover, the expression levels of all of these genes were markedly increased in osal50 mutant leaves at the seven-leaf stage (when the leaves turn green) compared with those at the four-leaf stage (Fig. 8b), with OsCHLI, OsCHLD, OsDVR, OsPSAA, OsPSBA, OsRBCL, OsPOLP, and V2 expression restored to wild-type levels and OsHEMA1, OsCAB1R, OsPETA, OsPETB, and OsRBCS expression levels in the osal50 mutant even higher than those in the wild type at the seven-leaf stage (Fig. 8b). These findings indicate that OsAL50 plays a vital role in the regulation of gene expression associated with chlorophyll biosynthesis, chloroplast development, and photosynthesis in rice.


Fig. 8   Expression patterns of genes associated with photosynthetic  type and osal50 glumes (c) and branches (d) from young panicles at pigment metabolism, photosynthesis, and chloroplast development in  the heading stage. Relative gene expression was calculated using the wild-type GP50 and osal50 mutant plants, determined by qRT-PCR.  comparative 2^-CT method, OsActin served as a normalization cona, b Relative expression in wild-type and osal50 mutant leaves at the  trol. Error bars indicate SD (n=3) . Student’s t -test was used for stafour- (a) and seven-leaf (b) stages. c, d Relative expression in wild- tistical analysis, ^*P<0.05 ,**P<0.01

# Discussion

In addition to being a staple food crop throughout the world, rice also serves as a model monocotyledon plant in research settings. Variegated mutants with altered leaf or panicle coloration have long been the focus of intensive scientific interest in rice, particularly in the case of albino mutants (Peng et al. 2012). Efforts to characterize the genetic basis of these albino mutants have provided new insights into the molecular mechanisms governing leaf and panicle coloration and the roles that nuclear genes play in shaping chloroplast development and chlorophyll biosynthesis (Chen et al. 2009). Here, we identified and characterized a variegated rice mutant, osal50, which exhibits a relatively normal green phenotype through the L2 stage (Fig. 1a, b), during which the second leaf blade is wellexpanded, but develops longitudinal white-striped leaves through the four-leaf stage (Fig. 1c, d); these gradually turn green such that plants exhibit normal green coloration at the seven-leaf stage (Fig. 1f). Additionally, osal50 plants exhibit mixed phenotypes at the tillering stage, with many white-striped leaves and fewer green leaves on a given plant (Fig. 1g), together with white panicles that are evident at the flowering stage and distinct from those exhibited by wild-type GP50 plants (Fig. 1i, j). The present findings imply that, in contrast to the majority of variegated mutants in rice that have been documented to date, the osal50 mutant phenotype is highly growth stage-dependent. Previous reports have described many mutants associated with albinism before the three-leaf stage that become more noticeable during the two-leaf stage, but the leaves return normal green coloration following the four-leaf stage, including gra(t) (Chen et al. 2009), hw-1(t) (Guo et al. 2012), graS (Du et al. 2020), albg (Jian et al. 2017), and ysa (Su et al. 2012). The osal50 mutant exhibits an albino leaf phenotype in a very different growth stage-dependent manner from these mutants under natural field conditions. Moreover, the albino phenotype exhibited by osal50 plants is unaffected by temperature, instead changing only over the course of development; this is in contrast to other previously reported green-revertible albino mutants, where leaf color is sensitive to temperature, including cisc (Lan et al. 2010), gra(t) (Chen et al. 2009), st1 (Yoo et al. 2009), ν3 (Yoo et al. 2009), ν2 (Sugimoto et al. 2007), and ν l (Kusumi et al. 2011). The phenotypic characteristics of osal50 mutant plants are also distinct from those of most other mutants with white panicles described to date, including wp1 (Wang et al. 2016), wp2 (Wang et al. 2021), wp3 (Li et al. 2018a), wsp1 (Zhang et al. 2017), wlp1 (Song et al. 2014), and wlp2 (Lv et al. 2017). Of these, the wp2 and wlp2 mutants exhibit more pronounced albinism under higher temperatures, while wsp1 and wlp1 mutant phenotypes are particularly sensitive to cold exposure. The w p3 mutant exhibits white panicles and white-striped leaves, and the generation of white-striped leaves in wp3 mutant plants is independent of changes in environmental conditions, such as temperature and light levels. Unusually, the white panicles in the wp3 mutant plants exhibit typical Mendelian inheritance, whereas the white-striped leaves instead show a pattern of maternal inheritance.

In the present study, map-based cloning confined the O s A L5O locus to a 160-kb physical interval on chromosome 1, flanked by the CAPS-08 and CAPS-37 SNP markers (Fig. 4). This region contained 26 annotated genes, of which 10 encoded transposons or retrotransposons, 13 encoded hypothetical or expressed proteins of unknown function, and 3 (LOC_Os01g19970, LOC_Os01g20140, and LOC_Os01g20160) encoded proteins with predicted functions (Online Resource 5). L O C_-O s O I g I9970 is predicted to encode a member of the MYB transcription factor family, which is among the largest transcription factor families in plants, with members serving as key regulators of diverse processes related to development, secondary metabolism, hormone signaling, organ morphogenesis, cell differentiation, disease resistance, stress tolerance, and cell cycle progression (Katiyar et al. 2012). Several MYB transcription factors have recently been found to be regulators of anthocyanin and flavonoid biosynthesis in plants, thereby influencing the coloring of leaves, fruits, flowers, and other tissues (Li et al. 2019). L O C_O s O I g20I40 and L O C_-O s O I g20I6O are respectively predicted to encode a latency-associated nuclear antigen (LANA) and an OsHKT1;5- ·Na^+ transporter. LANAs are among the most widely expressed nuclear proteins during latency periods, serving as essential regulators of a range of nuclear processes including the recruitment of DNA replication machinery and proper chromatid segregation into daughter cells. The OsHKT1; ^5-Na^+ transporter is found in the plasma membrane and the phloem of diffuse vascular bundles in basal nodes. It has crucial functions in preventing the transmission of Na^+ to young leaf blades by excluding Na^+ in the phloem. Furthermore, it performs a crucial role over different rice growth stages by safeguarding the subsequent generation of seeds and leaf blades against the detrimental effects of salt stress (Kobayashi et al. 2017). BLAST searches of available genomic and protein sequences indicated that multiple copies of L O C_-O s O I g20I40 are present within the rice genome and that it does not exhibit any clear association with leaf coloration. As such, we focused on the two other functionally annotated genes, L O C_- O s O I g I9970 and L O C_O s O I g20I60. No variations in allelic gene sequences between the wild type and the osal50 mutant were found when we amplified and sequenced the promoter and coding regions of these potential genes in both parental lines (data not shown). Sequence and expression analyses ultimately led to the identification of L O C_-O s O I g20I I O as the O s A L5O candidate gene (Fig. 5), which was confirmed by knockout using the CRISPR/Cas9 system (Fig. 6). In total, 20 mutation sites were identified within the L O C_- O s O I g2O I I O promoter region in the osal50 mutant (Fig. 5d, e). Furthermore, a combination of phylogenetic analysis and multiple sequence alignment suggested that O s A L5O likely encodes an RNase E-like protein (Fig. 7). RNase functions by catalyzing the cleavage of phosphodiester bonds within substrate RNA molecules, with Endo-RNase generating RNA fragments by cleaving internal sequences. In contrast, Exo-RNase removes individual nucleotides from the 5' or 3^ · ends of RNA substrates (Ait-Bara and Carpousis 2015). RNase E is an Endo-RNase widely distributed throughout chloroplasts and bacterial species. In Escherichia coli, where it has been the most thoroughly studied. RNase E functions as a comprehensive regulator of gene expression, regulating various aspects of RNA metabolism. This includes the processing, maturation, and quality control of stable RNA molecules, the degradation of non-coding regulatory RNA sequences and RNA primers involved in DNA replication, as well as the turnover of mRNA molecules (Ait-Bara and Carpousis 2015; Schein et al. 2008). Higher plants include homologs of bacterial RNase E that are encoded in the nuclear genome. Their responsibility lies in the breakdown of RNA within the chloroplast, an organelle originating from a prokaryote resembling cyanobacteria, via a degradation pathway facilitated by polyadenylation (Schein et al. 2008). In the research they performed, Mudd et al. (2008) provided evidence that Arabidopsis contains a single nuclear gene, A t2g O4270 , which produces a protein with a conserved catalytic domain similar to that of the RNase E/G-like protein found in chloroplasts. This gene, A t2g O4270 , is essential for the appropriate development of chloroplasts and autotrophic growth. Additionally, it is expressed together with other nuclear genes involved in regulating RNA metabolism in plastids. The protein displays ribonucleolytic activity and influences the accumulation of plastid RNA. Plants with a homozygous lack of A t2g O4270 function display a pale green in color. The plastids in the leaves of these mutant plants are smaller, with fewer thylakoids and shorter grana stacks, compared to the chloroplasts of wild-type plants. The albino phenotype exhibited by the osal50 mutant, derived from a breeding population of the japonica cultivar GP50, is also associated with abnormal chloroplast morphology and development. While some chloroplast-like organelles are observed within cells from the white portions of osal50 plant tissues, they lack well-developed grana lamellar stacks and organized thylakoid structures, contributing to the whitestriped leaves and white panicles that characterize these mutant plants (Fig. 3𝐤-p. ). Given these findings, we speculate that impaired expression of O s A L5O , which is predicted to encode an RNase E-like protein necessary for proper chloroplast development, leads to the albino phenotype observed in osal50 mutant rice plants. This impaired expression likely results from an abnormal promoter structure.

Plant chloroplasts retain their own genomic and translational systems, and chloroplast development is regulated by both nuclear and plastid genomes (Zheng et al. 2019). Moreover, chloroplasts harbor both nuclear- and plastid-encoded RNA polymerases (NEP and PEP, respectively), with coordination between the two being essential for normal plastid development and the biosynthesis of chlorophyll (Kusumi and Iba 2014). Loss of function of proteins associated with the PEP and NEP could lead to an albino phenotype (Zheng et al. 2019). Glutamyl-tRNA facilitates switching between NEP and PEP utilization during chloroplast development in Arabidopsis (Hanaoka et al. 2005). In the present study, we investigated expression levels of chlorophyll biosynthesis and chloroplast development-related genes in various tissues of wild type and osal50 during different stages of growth and development. The qRT-PCR analysis revealed down-regulation of genes associated with glutamyl-tRNA synthetase, involved in the chlorophyll biosynthetic pathway, in osal50 mutant leaves at the fourth-leaf stage and young panicles at the heading stage; these included genes such as HEMA1 and HEMD, involved in the processing of glutamyl-tRNA to aminolevulinic acid (ALA) and ALA to protoporphyrin IX, as well as genes involved in the processing of protoporphyrin to chlorophyll a and b (OsCHLI, OsCHLH, OsCHLM, OsCHLD, OsCAO1, and OsDVR) (Fig. 8a, c, d). In addition, nuclear-encoded genes (e.g., OsCAB1R, OsCAB2R, OsRPOTP, and RbcS), as well as plastid-encoded genes (e.g., OsPSAA, OsPSBA, OsPOLP, OsRPOB, and RbcL) related to photosynthesis and chloroplast development, exhibited comparable patterns of transcriptional suppression in osal50 mutant leaves and young panicles, the two albino stages (Fig. 8a, c, d); however, at the seven-leaf stage (when the leaves turn green), the transcript levels of these genes in osal50 mutant leaves were largely comparable to those in the wild type (Fig. 8b). These expression patterns closely matched the growth stage-dependent albino phenotype observed in osal50 mutant plants. Our findings demonstrate that O s A L5O is essential for regulating chlorophyll biosynthesis and chloroplast development in rice and that impaired O s A L5O expression disrupts the mRNA transcript levels of genes involved in these processes.

# Conclusion

We identified and characterized a naturally occurring rice albino mutant designated osal50 (oryza sativa albino leaf 50), derived from a breeding population of the japonica cultivar GP50. Leaves and panicles of mutant plants displayed distinctive albino characteristics in a growth stagedependent manner under natural field conditions. Moreover, the osal50 mutant exhibited decreased chlorophyll and carotenoid accumulation and abnormal chloroplast ultrastructure. Map-based cloning delimited O s A L5O to a region of 160­kb on chromosome 1 flanked by two SNP markers, CAPS-08 and CAPS-37; this region contained 26 putative open reading frames. Sequence and expression analyses revealed L O C_-O s O I g20I I O as the candidate gene of OsAL50, which was confirmed by knockout using CRISPR/Cas9. In total, 20 mutation sites were detected in the L O C_-O s O I g20I I O promoter region, resulting in severe suppression of mRNA expression. Subcellular localization and protein sequence analyses suggested that OsAL50 likely encodes a chloroplast-localized endoribonuclease E-like protein. Further investigation indicated that O s A L5O plays a vital role in the regulation of chlorophyll metabolism, chloroplast development, and photosynthesis. Together, these findings provide new molecular insights into the genetic basis for changes in leaf and panicle coloration in rice.

Supplementary Information  The online version contains supplementary material available at https://​doi.​org/​10.​1007/​s10725-​023-​01116-8.

Acknowledgements  This study was supported by the China Agriculture Research System of MOF and MARA (CARS-01), the Fujian Provincial Natural Science Foundation (2021J01535, 2021J01536), and the Sanming Municipal Science and Technology Project (2022-N-7).

Author contributions  YZ and XW contributed equally to this work. YZ carried out molecular-marker development, genetic analysis, molecular mapping, candidate gene analysis, physical-map and phylogenetic-tree construction, and wrote the manuscript. XW carried out molecularmarker development, genetic analysis, molecular mapping and expression analysis. CX and RZ participated in genetic analysis, molecular mapping, and physical-map construction. JH carried out mapping-population construction and participated in genetic analysis and molecular mapping. XX designed the research and wrote the manuscript. All authors read and approved the final manuscript.

Funding  Funding was supported by Agriculture Research System of China,CARS-01 ,Natural Science Foundation of Fujian Province,2021J01536, 2021J01535, Sanming Municipal Science and Technology Project,2022-N-7.

# Declarations

Conflict of interest  The authors declare that they have no conflicts of interest.
