# A G-protein pathway determines grain size in rice

Shengyuan Sun, Lei Wang, Hailiang Mao, Lin Shao, Xianghua Li, Jinghua Xiao, Yidan Ouyang     Qifa Zhang Manipulating grain size is an effective strategy for increasing cereal yields. Here we identify a pathway composed of five subunits of the heterotrimeric G proteins that regulate grain length in rice. The _ùñ¶Œ≤ protein is essential for plant survival and growth. GŒ± provides a foundation for grain size expansion. Three GŒ≥ proteins, DEP1, GGC2 and GS3, antagonistically regulate grain size. DEP1 and GGC2, individually or in combination, increase grain length when in complex with GŒ≤. GS3, having no effect on grain size by itself, reduces grain length by competitively interacting with _ùñ¶Œ≤ . By combining different G-protein variants, we can decrease grain length by up to 35% or increase it by up to 19% , which leads to over 40% decreasing to 28% increasing of grain weight. The wide existence of such a conserved system among angiosperms suggests a possible general predictable approach to manipulating grain/organ sizes.

rain size is a major determinant of grain weight, which is a Gyield component trait for cereals. Grain size is also a trait for grain quality focused by rice breeders, as long and slender grains are preferred by rice consumers in many countries. Recent advances in rice functional genomics facilitated the cloning of a series of loci controlling grain size, including genes for grain length such as G S3^1 ,GƒΩ3Ãå.1^2,3 ,A n-I^4 ,G L W7Ãá^5 and GS26,7; genes for grain width such as GW28, GW59,10, GS511, GW812 and G W7^13,14 ; and genes for grain weight such as G I F I^15 G E^16 ,T G W6^17 ,G W6a^18 ,BƒûI^19 and XIAO20.

G proteins are guanine nucleotide-binding trimeric proteins consisting of GŒ±, G and _GŒ≥ subunits, and are involved in transmitting signals from a variety of stimuli outside a cell to its interior, thereby regulating various biological processes both in animals and plants. In animals, there are 16 GÃÖa, 5 G, 14 G proteins and a large number of agonist-bound G-protein coupled receptors (GPCRs)^21 . In mammalian cells, G protein signaling is initiated by GPCRs, such that the GŒ± subunit is activated through the exchange of a GDP for GTP, which causes its dissociation with the G dimer. The disassociated _Ga and G activate their own downstream effectors respectively thus performing various biological functions22‚Äì24. Most plant genomes contain much fewer G proteins , and plant G proteins are self-activating without involving GPCR^24 . However, biological functions of plant G proteins have been much less studied.

Rice encodes one each of GŒ± and G , and five _GŒ≥ proteins25,26. Both GŒ± and G proteins are positive regulators of cell proliferation and grain size growth27‚Äì30. _G_ proteins are divided into three distinct groups according to their C-terminal structures31. Two typical _G_ subunits RGG1 (Group I) and RGG2 (Group II) are involved in regulation of abiotic stresses32‚Äì34. Natural variants of an atypical _G_ subunit DEP1 (Group III) showed functions in panicle architecture and nitrogen-use efficiency . GS3, encoding a _GŒ≥ subunit (Group III) of the heterotrimeric G proteins, is a major QTL for grain size1,38. The wild-type GS3 protein produces medium grain, a loss-of-function allele results in long grain, while a truncated form lacking the C-terminus produces very short grain39. Numerous studies have established that GS3 is the most important regulator of grain length in both natural and breeding populations of cultivated rice40‚Äì42 and its natural variants have contributed greatly to the improved productivity and grain quality of rice in a global scale. The GŒ±, G and _G_ proteins were also found to influence organ size and shape in Arabidopsis43‚Äì46 , suggesting a general function of G protein signaling in organ size regulation in plants.

In this study, we identify a pathway made up of five subunits of the heterotrimeric G proteins that regulates grain size in rice. We show that manipulating the three _GŒ≥ proteins, DEP1, GGC2 and GS3, can achieve designed grain size, demonstrating a predictable approach to improve grain yield and quality.

# Results

GS3 and DEP1 conferred divergent functions on grain size. The GS3 protein shares significant similarity with DEP1 in the Nterminal region, which was named OSR (organ size regulation) domain39 and later recognized as _GŒ≥ -like domain26. The OSR domain of GS3 showed 68.7% identity to DEP1 at the DNA level and 50% identity by protein sequence. A number of natural variants of DEP1 and GS3 were found showing differences mostly in the length and composition of the C-terminal cysteine-rich domain36,39,47,4 8 (referred to as ‚Äútail‚Äù hereafter) (Fig. 1). DEP1 has a long tail of 305 amino acids in length; a 625-bp deletion in DEP1 causes a premature stop codon35,36 , producing a truncated protein dep1 with a tail of 75 amino acids. The wild-type GS3 allele from the rice variety Zhenshan 97 (GS3-1) contains a relative short tail (115 amino acids). The GS3-4 allele from a rice variety Chuan 7 encodes a truncated GS3 protein lacking the entire tail and containing the OSR domain only39.

To compare the effects of these alleles on grain size, we transformed Zhonghua 11 (ZH11), a rice variety harboring functional alleles of both DEP1 and GS3-1 with medium grain size, with constructs overexpressing DEP1 (DEP1OE), dep1 (dep1OE), GS3-1 (GS3-1OE) and GS3-4 (GS3-4OE) respectively, and RNA-interference of DEP1 (DEP1Ri) and GS3-1 (GS3-1Ri). Transcript levels of all genes were checked in the young panicles of the transformants (Supplementary Fig. 1a, b).

We field-tested 2 or 3 T_1 families randomly selected from T_0 plants. Compared with the negative segregants, elevated DEP1 accumulation increased the grain length by 6.85-9.58% with a normal plant stature. However, plants overexpressing dep1 showed very similar phenotype to DEP1Ri plants, both of which reduced grain length by ‚àº4.5% , together with dwarf stature and erect panicles (Fig. 1a and Supplementary Table 1). The expression level of endogenous DEP1 in dep1OE was not reduced (Supplementary Fig. 1c). Thus dep1 showed a dominant-negative effect over DEP1 in regulating grain size rather than cosuppression of the two genes. In contrast to DEP1OE, GS3-1OE showed an average 9.07% reduction in grain length together with reduced plant stature, very similar to the phenotype of dep1OE (Fig. 1b and Supplementary Table 1). GS3-4OE produced even smaller plants and grain size with an average 19.10% reduction of grain length relative to the negative segregant, while GS3-1Ri resulted in an average 5.78% increase of grain length. These results were similar to those of Mao et al .

Because the major difference of DEP1, dep1 and GS3 lies in the C-terminal domain, the phenotype differences of the transgenic plants suggested that the tail length and composition of these genes are important for their functions. DEP1 with a long tail had a positive effect on grain length, while dep1 and GS3-1 with short tails played negative roles in grain length. Complete loss of the tail in GS3-4 further enhanced the negative effect of GS3 in grain length regulation.

Interaction between DEP1 and GS3 in grain size regulation. We crossed GS3-1Ri with dep1OE plants, all GS3-1Ri/dep1OE F_1 plants exhibited reduced grain length, similar to dep1OE transgenic plants. In the F_2 generation, all the GS3-1Ri/dep1OE plants also showed the dep1OE phenotype with respect to grain size, plant height and panicle length (Supplementary Fig. 2a).

Next, we introduced GS3-1 and dep1 driven by their own native promoters into a T-DNA mutant of GS3 with increased grain size (Supplementary Fig. 2b-c). Both transformants produced signi cant shorter grains, again supporting the conclusion that GS3- 1 and dep1 had similar role in grain size regulation.

Using CRISPR/Cas9 editing, we obtained two independent T_1 families for each of GS3-1 (ƒûS3‚Äìl^k o) and DEP1 (D EPÃÇI^k o) single and double ( G S3-I^k oD E P I^k o) mutants in the background of ZH11 (Fig. 2a and Supplementary Fig. 3a‚Äìd). Consistent with the observation in RNAi plants, homozygous G S3-I^k o mutant increased grain size, whereas homozygous D E P I^k o mutant reduced grain size. The grain length of G S3-1^k oD E P1^k o double mutant was intermediate between that of the single mutants of G S3-I^k o and D E P I^k o . To confirm the result, we crossed a GS3- I^k oD E P I^k o double mutant with ZH11. In the F_2 segregants, GS3- I^k o homozygous plants also increased grain length, the D E P I^k o homozygote reduced the grain length, while the grain length of G S3-1^k o‚å£DE P1^k o double mutant was intermediate between that of the G S3-I^k o and D E P I^k o single mutants.

We further crossed GS3-1OE and GS3-4OE with D E P I^k o plants. It was shown that knocking out DEP1 in the GS3-1OE or GS3-4OE background did not further reduce the gain size of GS3-1OE or GS3-4OE (Supplementary Fig. 2d). We also crossed DEP1OE with GS3-1Ri and GS3-1OE transgenic plants. Further increased grain length was observed when overexpressing DEP1 in GS3-1Ri background, whereas the DEP1OE/GS3-1OE hybrid showed the GS3-1OE phenotype of short grain (Supplementary Fig. 2e).


Fig. 1 Genetic effects of three GŒ≥ proteins on grain size and plant morphology. The structures of the proteins are shown on top of each panel, with the numbers of residues of each protein indicated. Plant, bar =20 ùñº ùóÜ ; Panicle, bar =5 ùñº ùóÜ ; grains, bar =5 ùóÜ ùóÜ . a Top: the protein structures of DEP1 and dep1. Bottom: whole plants, panicles, and grains of WT (wild type ZH11), DEP1OE, dep1OE, and DEP1Ri plants. b Top: the protein structures of GS3-1 and GS3-4. Bottom: whole plants, panicles, and grains of WT, GS3-1OE, GS3-4OE, and GS3-1Ri plants. c Top: the protein structure of GGC2. Bottom: whole plants, panicles, and grains of WT, GGC2OE, and G G C2^k o plants

An atypical ùêÜ subunit GGC2 functions additively with DEP1. Both DEP1 (chromosome 9) and GS3 (chromosome 3) are atypical _G_ subunits. By searching the reference genome sequences of rice49,50 , we found another atypical _GŒ≥ protein (GGC2) on chromosome 8, showing 66% and 48% identities to DEP1 and GS3, respectively. Transcripts of the three genes accumulated in the panicles, whereas DEP1 and GGC2 were also highly expressed in shoot apex relative to GS3 (Supplementary Fig. 1e). Overexpression of GGC2 (GGC2OE) in ZH11 increased grain length significantly, whereas knock-out mutant of G G C2^kÃÉo in ZH11 reduced the grain length (Fig. 1c and Supplementary Fig. 1d, 3a, e), suggesting that like DEP1, GGC2 also acted as a positive regulator of grain length in rice.

To validate the relationship of GGC2 with DEP1 and GS3, we generated D E P1^k oG G C2^k o double mutant and D E P I^k oG G C2^k- ‚ñµo_G S3-lk o triple mutant (Supplementary Fig. 3f, g). Two independent T_1 families for each mutant were used for further investigation. Compared to the single mutant of D E P I^k o or G G C2^k o , knocking out both of them resulted in much smaller grains than either of the single knock-out mutants, suggesting that DEP1 and GGC2 worked additively in positive regulation of grain length. Moreover, the D E P1^k oG Gƒä2^k oƒ†S3-1^k o triple mutant produced similar phenotype to that of D E P I^k oG G C2^k o , thus GS3- I^k o mutant could not increase grain length when both DEP1 and GGC2 were knocked-out (Fig. 2a).

Putting together the data from the above genetic analyses, an interesting outcome emerged concerning the genetic effects and functions of these genes. Taking the double mutants DEP1- ^k oG G C2^k o that produced the smallest grain as the base line, adding either DEP1 (as shown by G G C2ÃÜ^k o‚ü© or GGC2 (D E P I^k o) could greatly increase grain size, and the effect on grain size increase of GGC2 was larger than that of DEP1. Having both DEP1 and GGC2 could further increase grain size (as shown by the wide-type ZH11). Although G S3-I^k o increased grain size in the presence either or both of DEP1 and GGC2, it could not increase grain size in the double mutant (D E P I^k oG G C2^k o) .Therefore it became clear that DEP1 and GGC2 functioned positively in regulating grain size in an additive manner, while the role of GS3 in grain size regulation was to repress the effects of DEP1 and GGC2 on increasing grain size.

Functional dependence of the ùêÜ subunits on RGB1 and RGA1. The G -mediated process requires a  subunit to form a G dimer45,51,52. To investigate the genetic relationship between the three _GŒ≥ proteins with ùêÜÃÖ subunit RGB1 in grain size regulation, we tried to knock out RGB1 in ZH11 by CRISPR/Cas9, but failed to obtain a homozygous mutant in the T_1 generation, likely because of lethality of the null mutant (Supplementary Fig. 4a). Therefore, we knocked down RGB1 by RNAi (RGB1Ri), and the transgenic plants with suppressed expression of RGB1 showed reduced grain size and plant height, together with brown lamina joint and internodes (Supplementary Fig. 4b), which was similar to the previously observed phenotype of suppressing R G B I^30 . We crossed DEP1OE and GS3-1Ri with RGB1Ri transgenic plants, respectively, and overexpressed GGC2 in RGB1Ri.


Fig. 2 Genetic interactions of three _ùñ¶Œ≥ proteins and _ùñ¶Œ≤ or _GŒ± proteins. a Genetic interactions of DEP1, GGC2, and GS3. Grains and grain length of WT (wild type ZH11) h=10) ), single mutants of G S3-7^k o .n=73; ,D E P1^k o .n=13) , and G G C2^k o ùî´=73; , double mutants of G S3-7^k oD E P1^k o ‚ü® n=10‚ü© and D E P‚Üë^k oG G C2^k o (n=5) , and triple mutant of D E P1^k oG G C2^k oG S3-7^k o I n=5, . The data shown are mean ¬±ùñ≤ ùñ§ ùñ¨ . Different letters indicate significant differences ranked by the Fisher‚Äôs Least Significant Difference (LSD) test ^'P<0.05) . Bar =5 ùóÜ ùóÜ . b Genetic interactions of the _‚äôŒ≥ proteins with RGA1 and RGB1. Grains and grain length of WT, GS3-1Ri, DEP1OE, GGC2OE, and their hybrids with RGB1Ri and R G A7^k o , respectively. Values are given as mean ¬± SEM. Different letters indicate significant differences ranked by the LSD test ^P<0.05) . Bar =5 ùóÜ ùóÜ

The RGB1Ri/DEP1OE and RGB1Ri/GGC2OE plants showed reduced grain length compared to the wild-type. Similar result was also obtained in the RGB1Ri/GS3-1Ri hybrid (Fig. 2b). Thus the effects of grain length increase by DEP1OE, GGC2OE and GS3-1Ri were dependent on RGB1.

We also knocked out RGA1 in ZH11. A homozygous R G A I^k o. mutant with a 1269-bp deletion produced very small grains and plant size relative to the wild-type (Supplementary Fig. 4c), which was similar to the previously observed phenotype of rga1 (d l) mutant28,29. Three double mutants involving RGA1 were generated including: G S3-I R i/R G A I^k o produced by editing RGA1 (created an 804-bp deletion) in GS3-1Ri, DEP1OE/RGA1 generated by editing RGA1 (1269-bp deletion) in DEP1OE, and «¶G C2O E/Rƒ†A l^k o that overexpressed GGC2 in R G A I^k o background. All three double mutants produced very small grains, similar to R G A I^k o single mutant (Fig. 2b). Therefore, the effects of DEP1OE, GGC2OE and GS3-1Ri on grain size increase were also dependent on RGA1.

Thus the results of the genetic analyses showed that while RGB1 appears to play essential role for overall growth, RGA1 provided the baseline for the grain size regulation pathway composed of the three _GŒ≥ proteins, which may be summarized in Fig. 3a. In the presence of RGA1, DEP1 and GCC2, singly or together, function to increase grain size, while GS3 suppresses the effects of DEP1 and GCC2 to reduce grain size. Thus, relative to the wild type, knocking out RGA1 produces extremely short grains showing ‚àº35% reduction in grain length, similar phenotype can be obtained when both DEP1 and GCC2 are knocked out (Fig. 3b). Knocking out either DEP1 or GCC2 results in moderate short grains (7.47 to 11.18% reduction), and overexpressing GS3 produces similar phenotype. Conversely, overexpressing either DEP1 or GCC2, or knocking out GS3 results in moderate increase (6.85 to 13.03% in grain length and grain weight. Further increase in grain length .‚àº20% increase) is obtained when DEP1 is overexpressed in the knock-down mutant of GS3, which also leads to large increase (28.45%) in grain weight. Based on the above results, we predict that overexpressing both DEP1 and GCC2 in the knock-out mutant of GS3 has the potential to further increase grain size. Thus grain size could be manipulated with combinations of the alleles of _G_ proteins, providing a highly flexible and predictable approach to modifying grain size and increasing grain yield.

The role of tail and competing interactions of GŒ≥s with RGB1. To investigate the relative abundance of the proteins, we generated transgenic plants expressing Flag tag-fused GS3-4 (no tail), GS3-1 (short tail), dep1 (short tail) and DEP1 (long tail) respectively in ZH11 driven by the ubiquitin promoter. These transgenic plants phenocopied the corresponding ones with elevated expression of these genes without the Flag tag (Fig. 4a, b). Western blot analysis showed that GS3-4:Flag protein accumulated in much higher amount than did GS3-1:Flag (Fig. 4c). After treating with proteasome inhibitor MG132, GS3-1: Flag protein accumulation increased to a comparable level to GS3-4:Flag (Fig. 4d), indicating different rates of degradation of the GS3-1 and GS3-4 proteins in vivo. These results suggested that the C-terminal tail of GS3 is necessary for degradation of the protein and such tail-mediated proteolysis is critical for their function in grain size regulation. However, for unknown reason, the protein could not be detected in DEP1:Flag and dep1:Flag (Fig. 4c).


Fig. 3 Schematic representation of the functions of the G proteins in grain size regulation. a A genetic model depicting the pathway of the G proteins in grain size regulation. The black arrow indicates positive effect, and the red bars represent negative effect. b Manipulating grain size by combinations of G proteins. KO indicates knocking out of the gene, Ri for RNAi plants, and OE for overexpression of the gene. The empty space indicates wild type. c A model explaining how these _‚äôŒ≥ proteins work

Using yeast two-hybrid (Y2H), BiFC, and luciferase activity assays, we found that GS3-1, GGC2, and DEP1 interacted with RGB1 through the OSR domain (amino acid 1-94 in GS3-1) (Fig. 5a‚Äìc). In assays using the protoplast of GS3-1OE, GS3-4OE and ZH11, the interaction of RGB1 with DEP1 or GGC2 was suppressed in the GS3-1OE background compared with that in ZH11, suggesting that there existed a competition between GS3 and DEP1 or GGC2 in interacting with RGB1 (Fig. 5d). In addition, the interaction of RGB1 with DEP1 or GGC2 was even weaker in GS3-4OE background, presumably due to the over-accumulation of GS3-4 protein in transgenic plants. Such competition of GS3, DEP1 and GGC2 in interaction with RGB1 was also supported by an in vitro yeast three-hybrid assay, showing that the interaction between DEP1/GGC2 and RGB1 was disrupted by the expression of GS3, and conversely the RGB1‚ÄìGS3 interaction could also be influenced by DEP1 (Fig. 5e). These results suggested that RGB1-DEP1 and RGB1-GGC2 dimers positively regulate grain size, while occupation of RGB1 by the GS3 protein disrupts such function, and over-abundant GS3 results in short grain. Such competitive interactions provide an explanation for the antagonistic activities of these proteins in grain size regulation, allowing for the possibility that other subunits of G-proteins may also have roles in this pathway.

Application of the ùêÜ system in organ size manipulation. We searched protein sequences with signi cant similarity to DEP1, GS3 or GGC2 from a range of angiosperm plants, using reciprocal BLASTP (query cover >30% ,E<e^-4 , identity >30% ) in the NCBI database. A total of 96 proteins were found from 55 species, including both monocot and dicot plants (Supplementary Data 1). A maximum-likelihood phylogenetic tree was constructed to depict the relatedness of these proteins, which revealed two distinct clusters (Supplementary Fig. 5). The first cluster comprised 63 proteins from dicots, and the Arabidopsis homolog AGG3 was in this cluster showing the highest similarity to DEP1 (E=2e^-24) ). DEP1, GGC2 and GS3 belonged to the second cluster with 32 proteins from the monocots. Two maize _GŒ≥ proteins, DAA61661 (designated as ZmDEP1a, E=4e^-48 ) and DAA40167 (designated as ZmDEP1b, E=e^-72 ), were identified showing higher similarity to DEP1 than to GGC2 and GS3. A maize _GŒ≥ protein of ACZ02400 (designated as ZmGS3 ,E=2e^-40 )was identified as the orthologous copy of GS3. Reciprocal blast also showed that a maize _G_ protein DAA45508 (designated as ZmGGC2, E=5e^-50 ) has the highest similarity (73%) to rice GGC2. This suggested that the DEP1-GGC2-GS3 system is conserved in monocots. In addition, homologs of DEP1 also exist widely in dicots.

To investigate the effects of the various homologs on grain size regulation, we overexpressed DEP1, GS3-1, the Arabidopsis ortholog AGG3, the maize homolog ZmGS3 and the soybean homolog GmDEP1 in Daohuaxiang2 (DHX), a variety lacking functional GS3. AGG3, ZmGS3 and GmDEP1 contained short tail sequence, with 50% ,49% and 48% of protein identities to DEP1, and 45% ,74% , and 33% identities to GS3-1. The transgenic plants overexpressing DEP1 produced expected longer grains. And overexpressing AGG3, ZÃÖmG S3 and GmDEP1 in DHX resulted in reduced plant size and grain length, similar to the overexpression of GS3-1, also as expected on the basis of tail length. Furthermore, overexpressing the predicted OSR domains of AGG3 and ZmGS3 in DHX both resulted in reduced grain length and plant size (Fig. 6a and Supplementary Table 2). Thus differences in sequences of the N-terminal OSR domain or the entire proteins did not make difference in grain length regulation, but the tail length is critical. The long-tailed protein produced long grain, while the short-tailed and tailless proteins or the OSR domains alone produced short grain.


Fig. 4 Immunoblot analysis of the proteins in transgenic plants with Flag tag. a Whole plants and grains in WT (wild type ZH11), and transgenic plants of Ubi::GS3-1:Flag (GS3-1F), Ubi::GS3-4:Flag (GS3-4F), Ubi::dep1:Flag (dep1F), and Ubi::DEP1:Flag (DEP1F). Plant, bar =20 ùñº ùóÜ ; grains, bar =5 mm . b Grain length and relative expression levels of the genes in transgenic plants. Values for grain length are given as mean ¬± SEM ( .n=10; .P values are given on the top of the bars, which are based on two-tailed t-tests. Values for expression analysis are given as mean ¬±ùñ≤ ùñ§ ùñ¨ (nÃà=3! ). c Immunoblot analysis of the transgenic plants. Total proteins are extracted from the shoots of 15-day-old seeding, and 20 g amount was used for western blotting. d Immunoblot analysis of Flag fused GS3-1 and GS3-4 proteins after the treatments with 30 ùñ¨ CHX and 50 M MG132. C K, the GS3- ¬∑‚ÑêFÃÖ and GS3-4F transgenic plants without treatment. CHX, Actidione. The two bands of GS3-1 protein are likely due to protein modification by ubiquitination. The numbers (6, 12, 20) indicate hours after the treatments

But the situation is different in Arabidopsis transformation. It was previously shown that overexpression of AGG3 in Arabidopsis increased organ size43. Introduction of DEP1, GGC2, and GS3-1 driven by the 35S promoter into agg3-2, a knock-out mutant of AGG3 with decreased organ size43, recovered the phenotype showing increased flower organ, silique length, and seed size compared with the agg3-2 mutant (Fig. 6b, c). Therefore, Arabidopsis did not distinguish the tail length difference of _G_ proteins, suggesting that Arabidopsis may not have evolved such a mechanism to be impacted by the tail-length difference in organ size regulation.

# Discussion

Based on the results of this study, we proposed a model to explain how these _GŒ≥ proteins work in grain size regulation (Fig. 3c). In this model, DEP1 and GGC2, when coupled with RGB1, promote grain size by tail-mediated signaling. GS3, though having no function in promoting or inhibiting grain size, reduces grain size by blocking the interaction of DEP1 and GGC2 with RGB1. The tail-mediated self-degradation of GS3 in the RGB1-GS3 complex provides a dynamic balance between blocking and the DEP1/GGC2‚ÄìRGB1 interaction, thus a plant carrying GS3-1 produces medium grain. Without the tail (GS3-4), accumulated GS3-4 would largely occupy RGB1, resulting in short grain. The net outcome of the functional interactions among these genes thus determines grain size, allowing for the possibility that other subunits of G-proteins may also be involved in such a signaling pathway.



It should also be mentioned that similar behavior of DEP1 and GGC2 in grain size regulation does not mean that they have redundant functions at the whole plant level. In fact, these two genes show different expression profiles; constitutively overexpressing GGC2 (GGC2OE) resulted in shorter plants with fewer tillers and wider tillering angles than both wild-type and DEP1OE (Fig. 1c), suggesting that these two genes have different roles in growth and development.

A question thus arose: why did GS3 arise and how has it been maintained in the population at a predominantly high frequency? To answer this, we investigated the transgenic plants overexpressing DEP1 and GGC2, as well as RNAi of GS3 for yield related traits (Supplementary Table 1). Although these transgenes significantly increased grain length and weight, without exception, they reduced number of grains per panicle to the extents similar to their effects on grain length increase, indicating that the increased grain size is at the cost of fitness. Thus from fitness viewpoint, medium grain is probably the optimal for rice reproduction; increasing grain size to make it larger than medium would compromise fitness of the rice plant. As a strategy, the rice plant installed GS3 specifically to serve as a check to prevent the grain from becoming too large, as suggested by its specific expression in the developing panicle. Thus, while DEP1 and GGC2 function to promote grain size, which is essential for yield increase in breeding, GS3 plays a role to keep balance for maintaining fitness in response to natural selection.

Based on the understanding advanced in this study, the interactions of three _G proteins can be used for a predictable design of grain size in rice, by manipulating these genes, individually or in combination, to improve rice grain yield and quality. Moreover, as the _GŒ≥ proteins are highly conserved in a very wide range of plants, manipulating these proteins may provide a general strategy for modifying organ size and yield in crop breeding.

# Methods

BLAST search and phylogenetic analyses. Reciprocal BLASTP searches were done against the NCBI (http://blast.ncbi.nlm.nih.gov) and MSU (http://rice. plantbiology.msu.edu) databases, using the protein sequences of DEP1, GS3, and GGC2 (query cover >30% ,E< e^-4 , identity >30% ). The phylogenetic tree was built based on the alignment of protein sequences, using MEGA 6.¬∑0.6^53 . The maximumlikelihood method was used with the parameters of Poisson model and bootstrap 1000 replicates.

Field planting and trait measurement. The rice plants were grown in the field in the normal rice growing seasons in Wuhan, China, and in Hainan in winter seasons. The planting density was 16.5 cm between plants in a row, and 26.5 cm between rows. Field management including irrigation, fertilizer application, and pest control followed essentially normal agricultural practices. Harvested grains were air-dried and stored at room temperature. Fully filled grains were used to measure the grain size using YTS system .

Arabidopsis plants were grown in soil at 22 ^‚àòC under long-day conditions (16 h light/8 h dark). Petals and siliques were scanned to produce a digital image, and seeds were photographed under a Leica microscope. The petal length, silique length, and seed size were measured using image J software (https://imagej.nih.gov/ ij/).

and GS3-4 were amplified from full-length cDNA of Guangluai 4 (Osigcea013f09t3) in pBluescript SK2 (Stratagene). The coding sequences of DEP1 and dep1 were amplified from rice cultivar ZH11 by RT-PCR. The four sequences were inserted into pCAMBIA1301u to generate the constructs of GS3-1OE, GS3-4OE,

DEP1OE, and dep1OE respectively, under an ubiquitin gene promoter. The coding sequence of GGC2 was also amplified from the rice cultivar ZH11 by RT-PCR, and was inserted to pCAMBIA1301s to generate the construct GGC2OE.
To construct the RNAi vectors for GS3, DEP1, and RGB1, ‚àº500 -bp fragment of the cDNA sequence was inserted into a pDS1301 vector modified from
ùöô ùô≤ ùô∞ ùôº ùô± ùô∏ ùô∞ ùöï301^55 .
The coding sequences of GS3-1 and dep1 were cloned into a pCAMBIA2300 vector under a 1.9-kb promoter of GS3 and a 2.9‚Äìkb promoter of DEP1 to generate the constructs GG, Gd, DG, and Dd, respectively. These vectors were transformed into gs3 (PFG_3A-03580), a homozygous mutuant with a T-DNA insertion in the second intron of GS3, using Agrobacterium tumefaciens EHA105.
Genome sequences of AÃÖ ÃÖGÃÖ ÃÖGÃÖ3ÃÖ^ÃÖ4ÃÖ3ÃÖ and G m D E P I (from soybean variety Zhongdou 32), and cDNA sequence of Z m G S3^56 were amplified and inserted into
pCAMBIA1301s to generate 35 S::AGG3, 35 S::GmDEP1, 35 S::ZmGS3, 35 S::»¶G G3^O S R , and 35 SÃÖ√∑Z m G S3^O S R vectors. All the constructs confirmed by
sequencing were introduced into Agrobacterium tumefaciens EHA105 and transformed into ZH11 or DHX by Agrobacterium-mediated transformation as described in Lin and Zhang (2005) with minor modifications57.
The DEP1, GGC2, and GS3 sequences were amplified and inserted into pCAMBIA1301s to construct 35 S::DEP1, 35 S::GGC2, and 35 S::GS3 vectors. They were introduced into an Arabidopsis agg3-2 mutant using Agrobacterium
tumefaciens EHA105. The medium supplemented with hygromycin (30 mg ml^-1) )was used to select the transformants. All primers for vector construction are listed in Supplementary Data 2.

# Generation of knock-out mutants using CRISPR/Cas9 technology. The

CRISPR/Cas9 system was used to generate the knock-out mutants58. The single
mutants of D E·πñI^k o ( designed in exon1) and G S3^k o (sgRNA designed in
exon1) were generated under the rice promoters of OsU3, OsU6b and OsU6a,
OsU6c respectively. The double mutant G S3^k oD E P I^k o was generated using the four
sgRNAs in single mutants (Supplementary Fig. 3a‚Äìd). The double mutant D E P I^koÃÖG G C2^k o was generated using the sgRNAs targeting
both GGC2 and DEP1 in exon1, and the single mutant G G C2ÃÜ^k o was generated using
the sgRNAs targeting exon2. The triple mutant D E P1^k oG G C2^k oG S3-1^k o was
obtained using the sgRNAs targeting DEP1, GGC2, and GS3 in exon1
(Supplementary Fig. _3e-g) The R G B I^k»Ø and R GƒÇI^k o mutants were generated using the three sgRNAs
under the rice O s U3 , OsU6a, and O s U6b promoters (Supplementary Fig. 4). Mutations were confirmed by PCR sequencing in T_0 and T_1 generations.

RNA extraction and qRT-PCR. Tissues used for expression analysis included root, shoot, leaf sheath, flag leaf, panicles of 0.5 cm ,1 cm ,2 cm ,3.5 cm ,6.5 cm ,8 cm ,11 cm ,16.5 cm ,19 cm ,22 cm in length, and shoot apex at 10, 22, 34, 40, and 44 days after germination. Total RNA was isolated using an RNA extraction kit (TRIzol reagent; Invitrogen), and quantified using Nanodrop (Thermo). For qRT-PCR, approximately 3 g of total RNA was reverse-transcribed using SuperScript reverse transcriptase (Invitrogen) in a volume of 20 l to obtain the cDNA. qRT-PCR was carried out in a total volume of 25 l containing 2 l of the above reverse-transcribed product, 0.25 mM gene-specific primers,

Fig. 5 Competition of the GŒ≥ proteins in interacting with RGB1. a Interactions of GS3, DEP1 and GGC2 with RGB1 using yeast two-hybrid assay. GS3 interacts with RGB1 through the OSR domain but not the C-terminal cysteine-rich domain (TV). OSR: OSR domain of GS3, amino acid 1-94; TV: cysteinerich domain of GS3, amino acid 95-231. AD: GAL4 activation domain; BD: GAL4 binding domain. -LW: selective medium lacking Trp and Leu; -LWHA: selective medium lacking Trp, Leu, His and Ade. b Interactions of GS3, DEP1 and GGC2 with RGB1 using BiFC assay. ùñ° ùñ∫ ùóã=20 ùóÜ . c Interactions of GS3, GGC2 and DEP1 (fused to C-terminal fragment of firefly luciferase) with RGB1 (fused to N-terminal fragment of firefly luciferase) using luciferase activity assay. Rice protoplasts with transient expression of RGB1-nLuc plus cLuc-DEP1, cLuc-GGC2, cLuc-GS3-1, and cLuc-GS3-4, but not the C-terminal cysteinerich domain (TV), show high luciferase activity relative to the control. Empty vectors of cLuc plus RGB1-nLuc, and nLuc plus cLuc-DEP1, cLuc-GGC2, cLucGS3-1, and cLuc-GS3-4 were used as the negative control. Data are normalized to the internal control 35 S::REN. Values are given as mean ¬±ùñ≤ ùñ§ ùñ¨ (n=3) .Different letters indicate significant differences ranked by the LSD test (P<0.05) ). d Interactions of DEP1 and GGC2 with RGB1 in the background of ZH11, GS3-1OE, and GS3-4OE using luciferase activity assay. RGB1-nLuc plus cLuc-DEP1, and RGB1-nLuc plus cLuc-GGC2 are transiently expressed in the protoplasts of ZH11, GS3-1OE, and GS3-4OE, respectively. Empty vector of cLuc plus RGB1-nLuc is used as the negative control. Data are normalized to the internal control 35 S::REN. Values are given as mean ¬± SEM (n=3) . Different letters indicate significant differences ranked by the LSD test ^'P<0.05) . e Yeast three-hybrid assay for protein interactions of DEP1/GGC2 and GS3 with RGB1. The interaction of DEP1/GGC2 and GS3 with RGB1 is analyzed using fusions with AD (AD-DEP1, GGC2 or GS3) and BD (BD-RGB1). Empty vectors are used as a negative control. Quantitative analysis of interactions by  -galactosidase assay is shown. Data for all the assays are shown as mean ¬± SD .n=3. )


Fig. 6 The functions of three GŒ≥ proteins and their homologs in the genetic backgrounds of rice and Arabidopsis. a Whole plants and grains of DHX (wild type plants), DEP1OE, AGG3OE, A G G3^O S RO E, GmDEP1OE, ZmGS3OE, Z m G S3^O S RO E, , GS3-1OE, and GS3-4OE rice plants in T_1 generation. Plant, bar =20 ùñº ùóÜ ;grains, bar =5 ùóÜ ùóÜ . b Flowers and siliques of the wild type Arabidopsis strain Col-0, agg3-2, and GS3OE, DEP1OE and GGC2OE transgenes in agg3-2 (a TDNA mutant of AGG3) background. Bar =2 ùóÜ ùóÜ . c Petal length, silique length, and seed area of the wild type Col-0, agg3-2, and GS3OE, DEP1OE and GGC2OE transgenes in agg3-2 background. The fully opened flowers were measured for the sizes of petals. Values for petal length (n=9) are given as mean ¬±ùñ≤ ùñ§ ùñ¨ relative to the respective wild type values setting at 100% . Values for silique length of Col-0 m=12. ), agg3-2 .n=70; ), GS3OE-1 ‚ü® n=16‚ü© ,GS3OE-2 .n=13. , DEP1OE-1 .n = 74) , DEP1OE-2 .n = 77) , GGC2OE-1 .n=70; and GGC2OE-2 m=10; are given as mean ¬± SEM relative to the respective wild type values setting at 100% . P values are given based on two-tailed t -tests. For seed size, the seeds are classified into three groups Œ∂<0.08 , 0.08 to 0.09, and >0.09 ùóÜ ùóÜ^2; ). Values of seed areas of Col-0 .n=61) , agg3-2 ( .n=147; , GS3OE-1 m=78. ), GS3OE-2 m=74 ), DEP1OE-1 .n=96, , DEP1OE-2 ùî´= 90), GGC2OE-1 .n=68. ) and GGC2OE-2 .n=71; are expressed as a percentage of the total seed number analyzed

and 12.5 l SYBR GreenMasterMix (Applied Biosystems) on an Applied Biosystems 7500 Real-time PCR system according to the manufacturer‚Äôs instructions. Data were normalized with rice ubiquitin gene (LOC_Os03g13170). Primers used for qRT-PCR analysis are listed in Supplementary Data 2. Themeasurements were obtained using the relative quantification method .

Yeast two-hybrid and yeast three-hybrid assays. Yeast two-hybrid experiment was performed using the Matchmaker Two-Hybrid System (Clontech). The coding sequences of RGB1, GS3, DEP1, GGC2 and their truncated forms were amplified using primers listed in Supplementary Data 2. The obtained fragments were cloned into PGADT7 and PGBKT7 separately. The yeast twohybrid assay was performed through cotransformation of the respective prey and bait vectors in the yeast (Saccharomyces cerevisiae) strain AH109 according to the lithium acetate transformation method . The transformed yeast cells were selected by plating them onto synthetic dropout selection medium lacking Leu and Trp (SD/-LW). Interactions were assayed on synthetic dropout interaction medium lacking Leu, Trp, His and Ade (SD/- LWHA).

For the yeast three-hybrid assay61, the prey vectors pGADT7-GS3, pGADT7- DEP1, and pGADT7-GGC2 were cotransformed with the different pBRIDGE (Clontech) bait vectors in the yeast strain AH109, respectively. Yeast strain AH109 was streaked three times to single colony on agar plates lacking Methionine (SD/- Met) since they tended to lose tolerance to this medium if not maintained on it routinely. Transformed cells were selected on synthetic dropout selection medium lacking Leu, Trp, Met (SD/-LWM) and interactions on SD/-Leu-Trp-Met-His (SD/-LWMH). The specificity of the stringency of the assay was tested by adding 5 mM 3-aminotriazole. For liquid  -galactosidase assay with CPRG as substrate, liquid cultures in SD-LWM were inoculated with three yeast colonies and incubated overnight. The calorimetric  -galactosidase assay of the supernatant and the following activity calculation were done, as described in the Clontech Yeast Protocols Handbook.

BiFC assay. The coding sequences of GS3, DEP1, GGC2 and their variants were amplified using primers listed in Supplementary Data 2. The obtained fragments were cloned into pSPYNE(R)173 and pSPYCE(MR), respectively . The plasmid mixtures were introduced into Arabidopsis protoplasts. After incubation in the dark overnight, the fluorescence was observed with Olympus FluoView FV1000.

Luc activity assay. The coding sequences of GS3-1, GS3-4, DEP1 and GGC2 were cloned to the C-terminal Luc fusion vector PUC19-cLUC, whereas RGB1 was cloned to the N-terminal fusion Luc vector PUC19-nLUC63. The rice protoplast was isolated from rice shoot of 15-day-old seeding under dark conditions. For protein‚Äìprotein interaction, N- and C-terminal Luc fusion genes were cotransformed with 35 S::REN by the ratio 10:10:1, with the latter as the internal control. Luc and REN reporter activity was detected with a Dual-Luciferase Reporter Assay System (Promega E1910). Relative activity of the Luc reporter was expressed as the ratio of Luc to REN.

Western blot. Rice shoot (100 mg) ) was collected and lysed in 800 l lysis buffer (50 mM Tris, pH 7.5, 150 mM NaCl, 0.1% Triton X-100, 0.2% Nonidet P-40, protease inhibitor cocktail (Roche, one tablet for 10 ml) ) before centrifugation at 13,500√ó g. . for the supernatant. Plant extracts were loaded onto SDS‚ÄìPAGE gel, and run until bromophenol blue was approximately touched the bottom of the gel. The proteins were transferred to a PVDF membrane. The membrane was blocked in 5% (wt/vol) nonfat milk for 1 h at room temperature and with anti-Flag antibody (Sigma Catalog F3165, dilution 1:10000) overnight at 4 ^‚àòC . The membrane was washed in TBST 20 mM Tris, 150 mM NaCl, pH=8.0 plus 0.05% Tween 20) four times, incubated with Goat anti-mouse secondary antibody (SouthernBiotech Catalog 1031-05, dilution 1:10000) for 1 h , and washed in TBST four times at room temperature. The uncropped scans with marker position are shown in Supplementary Fig. 6.

Statistical analysis. Statistical analysis was conducted using SPSS statistics 23.0 for Windows (IBM, Armonk, NY, USA). The two-tailed t test was used for comparing agronomic traits of each transgenic line with the control. The Fisher‚Äôs Least Significant Difference (LSD) test was used for multiple mean comparisons.

Data availability. All the vectors and the seeds of the plant materials generated in this study are available from the corresponding author upon request.

Received: 2 August 2017 Accepted: 18 January 2018
Published online: 27 February 2018
