# RESEARCH ARTICLE SUMMARY

PLANT SCIENCE

# A transcriptional regulator that boosts grain yields and shortens the growth duration of rice

Shaobo Wei†, Xia Li†, Zefu Lu, Hui Zhang, Xiangyuan Ye, Yujie Zhou, Jing Li, Yanyan Yan, Hongcui Pei, Fengying Duan, Danying Wang, Song Chen, Peng Wang, Chao Zhang, Lianguang Shang, Yue Zhou, Peng Yan, Ming Zhao, Jirong Huang, Ralph Bock, Qian Qian, Wenbin Zhou

INTRODUCTION: Rapid population growth, rising meat consumption, and the expanding use of crops for nonfood and nonfeed purposes increase the pressure on global food production. At the same time, the excessive use of nitrogen fertilizer to enhance agricultural production poses serious threats to both human health and the environment. To achieve the required yield increases and make agriculture more sustainable, intensified breeding and genetic engineering efforts are needed to obtain new crop varieties with higher photosynthetic capacity and improved nitrogen use efficiency (NUE). However, progress has been slow, largely due to the limited knowledge about regulator genes that potentially can coordinately optimize carbon assimilation and nitrogen utilization.

RATIONALE: Transcription factors control diverse biological processes by binding to the promoters (or intragenic regions) of target genes, and a number of transcription factors have been identified that control carbon fixation and nitrogen assimilation. A previous comparative analysis of maize and rice leaf transcriptomes and metabolomes revealed a set of 118 candidate transcription factors that may act as regulators of C_4 photosynthesis. We screened these transcription factors for their responsiveness to light and nitrogen supply in rice, and found that the gene Dehydration-Responsive Element-Binding Protein 1C (OsDREB1C), a member of the APETALA2/ethylene-responsive element binding factor (AP2/ERF) family, exhibits properties expected of a regulator that can simultaneously modulate photosynthesis and nitrogen utilization.


OsDREB1C coordinates yield and growth duration. OsDREB1C was identified by its responsiveness to light and low nitrogen in a screen of 118 transcription factors related to ∁_4 photosynthesis. Transcriptional activation of multiple downstream target genes by OsDREB1C confers enhanced photosynthesis, improved nitrogen utilization, and early flowering. Together, the activated genes cause substantial yield increases in rice and wheat.

RESULTS: OsDREB1C expression is induced in rice by both light and low-nitrogen status. We generated overexpression lines (OsDREB1COE) and knockout mutants (OsDREB1C-KO) in rice, and conducted field trials in northern, southeastern, and southern China from 2018 to 2021. OsDREB1C-OE plants exhibited 41.3 to 68.3% higher yield than wild-type (WT) plants due to increased grain number per panicle, elevated grain weight, and enhanced harvest index. We observed that light-induced growth promotion of OsDREB1C-OE plants was accompanied by enhanced photosynthetic capacity and concomitant increases in photosynthetic assimilates. In addition, ^15N feeding experiments and field studies with different nitrogen fertilization regimes revealed that NUE was improved in OsDREB1C-OE plants due to elevated nitrogen uptake and transport activity. Moreover, OsDREB1C overexpression led to more efficient carbon and nitrogen allocation from source to sink, thus boosting grain yield, particularly under low-nitrogen conditions. Additionally, the OsDREB1C-OE plants flowered 13 to 19 days earlier and accumulated higher biomass at the heading stage than WT plants under longday conditions.

OsDREB1C is localized in the nucleus and the cytosol and functions as a transcriptional activator that directly binds to cis elements in the DNA, including dehydration-responsive element (DRE)/C repeat (CRT), GCC, and G boxes. Chromatin immunoprecipitation sequencing (ChIP-seq) and transcriptomic analyses identified a total of 9735 putative OsDREB1C-binding sites at the genome-wide level. We discovered that five genes targeted by OsDREB1C [ribulosel,5-bisphosphate carboxylase/oxygenase small subunit 3 (OsRBCS3), nitrate reductase 2 (OsNR2), nitrate transporter 2.4 (OsNRT2.4), nitrate transporter 1.1B (OsNRT1.1B), and flowering locus T-like 1 (OsFTL1)] are closely associated with photosynthesis, nitrogen utilization, and flowering, the key traits altered by OsDREB1C overexpression. ChIP-quantitative polymerase chain reaction (ChIP-qPCR) and DNA affinity purification sequencing (DAP-seq) assays confirmed that OsDREB1C activates the transcription of these genes by binding to the promoter of OsRBCS3 and to exons of OsNR2, OsNRT2.4, OsNRT1.1B, and OsFTL1. By showing that biomass and yield increases can also be achieved by OsDREB1C overexpression in wheat and Arabidopsis, we have demonstrated that the mode of action and the biological function of the transcription factor are evolutionarily conserved.

CONCLUSION: Overexpression of OsDREB1C not only boosts grain yields but also confers higher NUE and early flowering. Our work demonstrates that by genetically modulating the expression of a single transcriptional regulator gene, substantial yield increases can be achieved while the growth duration of the crop is shortened. The existing natural allelic variation in OsDREB1C, the highly conserved function of the transcription factor in seed plants, and the ease with which its expression can be altered by genetic engineering suggest that this gene could be the target of future crop improvement strategies toward more efficient and more sustainable food production.▪

# READ THE FULL ARTICLE AT

https://doi.org/10.1126/science.abi8455

# RESEARCH ARTICLE

PLANT SCIENCE

# A transcriptional regulator that boosts grain yields and shortens the growth duration of rice

Shaobo Wei1†, Xia Li1†, Zefu Lu1, Hui Zhang2, Xiangyuan Ye1, Yujie Zhou1, Jing Li1, Yanyan Yan1, Hongcui Pei1, Fengying Duan1, Danying Wang3, Song Chen3, Peng Wang4, Chao Zhang5, Lianguang Shang5, Yue Zhou6, Peng Yan6, Ming Zhao1, Jirong Huang2, Ralph Bock7, Qian Qian1,3, Wenbin Zhou1

Complex biological processes such as plant growth and development are often under the control of transcription factors that regulate the expression of large sets of genes and activate subordinate transcription factors in a cascade-like fashion. Here, by screening candidate photosynthesis-related transcription factors in rice, we identified a DREB (Dehydration Responsive Element Binding) family member, OsDREB1C, in which expression is induced by both light and low nitrogen status. We show that OsDREB1C drives functionally diverse transcriptional programs determining photosynthetic capacity, nitrogen utilization, and flowering time. Field trials with OsDREB1C-overexpressing rice revealed yield increases of 41.3 to 68.3% and, in addition, shortened growth duration, improved nitrogen use efficiency, and promoted efficient resource allocation, thus providing a strategy toward achieving much-needed increases in agricultural productivity.

lobally, >800 million people are suffering from hunger and food insecurity (I) .By 2050, crop production needs to be increased by 50 to 70% to feed nearly 10 billion people despite the reduced availability of arable land on the planet (2, 3). Meeting this challenge will likely require the development of new breeding and genetic engineering strategies that optimize photosynthetic capacity as well as water and nutrient use efficiency (4). Growth and crop yield depend on carbon and nitrogen assimilation and photosynthate translocation from vegetative source organs to sink tissues (5, 6) . For example, nitrogen uptake and transport must be coordinated with carbon fixation and the production of carbohydrates by photosynthesis. Therefore, research efforts have been devoted to identifying transcriptional regulators that control the coordination between carbon assimilation and nitrogen utilization (7, 8).

As the regulators of biological processes, transcription factors control plant metabolism, growth, and development by binding to the promoters (or intragenic regions) of target genes (9, 10). An example of a transcription factor in plant architecture is TB1 (Teosinte Branched 1) of maize, which limits branch outgrowth and initiates the formation of female inflorescences (11, 12). The transcription factor IPA1 (Ideal Plant Architecture 1) promotes rice yield by reducing unproductive tillers and increasing grain number per panicle. Elevated IPA1 levels also enhance pathogen immunity (13, 14). Another transcription factor, HYR (HIGHER YIELD RICE), enhances the expression of photosynthesis genes and can increase rice yield under multiple stress conditions (1̅5̅) . The rice transcription factor GRF4 (GROWTH-REGULATING FACTOR4) coordinates nitrogen assimilation, carbon fixation, and growth (7). Often, binding motifs and functions of transcription factors are conserved in monocot and dicot species (7, 16).

Previous work has been directed at identifying key transcription factors that regulate photosynthesis, and nitrogen and carbon metabolism, using comparative analysis of maize and rice leaf transcriptomes and metabolomes (Iτ) . A set of 118 transcription factors were considered as candidate regulators of photosynthesis and, especially, of favorable properties related to C_4 photosynthesis (Iτ) . Here, we screened these transcription factors for their responsiveness to light and nitrogen supply in rice. We report the identification of a transcription factor from the DREB (Dehydration Responsive Element Binding) family, OsDREB1C, that modulates both photosynthesis and nitrogen utilization.

Overexpression of OsDREB1C not only increases rice yields, but also confers early grain maturation because of higher rates of photosynthesis, improved nitrogen utilization, and early flowering. The versatile functions of OsDREB1C are likely conferred by the transcription factor acting near the top of the hierarchy and coordinately targeting multiple genes and pathways. Recognition of the conserved dehydration-responsive element (DRE)/ C repeat (CRT) motif present in these loci facilitates fine-tuning of the intricate networks of carbon assimilation, nitrogen utilization, resource allocation, and induction of flowering. Our results uncover OsDREB1C as a transcriptional regulator that promotes the expression of key genes in carbon and nitrogen metabolism and controls flowering pathways in crops, thus providing a target for future crop improvement strategies.

# RESULTS

# OsDREB1C boosts grain yield and harvest index

To investigate how the coordination of photosynthesis and nitrogen utilization affects grain yield in cereals, we conducted RNA-sequencing (RNA-seq) analyses in rice plants grown under low- versus high-nitrogen conditions (18). We examined the differential expression of the subset of transcription factors associated with photosynthesis gene expression in maize (17). We detected nitrogen-regulated expression for 13 of these transcription factors, with five genes showing a greater than fourfold induction under low-nitrogen conditions (Fig. 1A). Investigation of light-regulated mRNA accumulation showed that one of the five genes, Os06g0127100 (encoding a DREB-type transcriptionfactor previously shown to be inducible by abiotic stress), displayed a diurnal rather than a circadian expression profile, with expression increasing with the duration of light exposure (Fig. 1B and fig. S1).

To facilitate the functional analysis of this apparently nitrogen-regulated and light-induced transcription factor, we generated a series of OsDREB1C-overexpressing lines (OsDREB1C-OE) and OsDREB1C-knockout mutants (OsDREB1CKO) in the Oryza sativa cv. Nipponbare genetic background (fig. S2). Field tests of these plant lines in Beijing in 2018 revealed that OsDREB1C overexpression led to increases in grain yield per plant of 45.1 to 67.6% and in yield per plot of 41.3 to 68.3% compared with wild-type (WT) plants (Fig. 1C). Conversely, OsDREB1C KO resulted in yield decreases (from 16.1 to 29.1% in yield per plant and 13.8 to 27.8% in yield per plot) compared with the WT (Fig. 1, D and E, and table S1). A detailed phenotypic analysis showed that the higher yield of the OsDREB1C-OE lines was mainly attributable to an enhanced grain number per panicle and an increased 1000-grain weight (Fig. 1F and fig. S3C), traits apparently resulting from increased (A) List of the top 13 genes up-regulated in response to nitrogen deprivation (adjusted P<0.05 ). The genes represent the overlap of previously reported RNAseq datasets (I̅7̅) and an expression analysis of a subset of 118 rice transcription factors (16), and were sorted by the fold change in low versus normal nitrogen supply. The color scale represents the 10g_2 -fold change of the FPKM (fragments per kilobase of transcript per million mapped reads) ratio under low- versus highnitrogen conditions, with the FPKM value of each gene under high-nitrogen conditions set to 1.00. (B) qRT-PCR analysis of O s O6 g O27 O O expression in 10-day-old O. sativa cv. Nipponbare seedlings grown in soil in a growth chamber under long-day photoperiod (16 hours light/8 hours dark, 28^∘C ). The white bar below secondary branch number and grain length, width, thickness, and density (fig. S3, A and B and D to K, respectively). The OsDREB1C-OE plants exhibited higher grain yield but reduced straw weight compared with WT plants (Fig. 1G), thus leading to an increased harvest index (the ratio of grain yield to aboveground biomass; Fig. 1H) and raising the possibility that OsDREB1C controls resource allocation between vegetative and reproductive tissues. The harvest index of OsDREB1C-OE plants was increased by 40.3 to 55.7% , whereas it was decreased by 22.4 to 33.7% in OsDREB1C-KO plants (table S1). In addition, key grain quality traits were enhanced in OsDREB1C-OE plants, suggesting that yield improvement does not entail a quality penalty (table S2).


Fig. 1. OsDREB1C overexpression in transgenic plants boosts grain yield.
the χ. -axis indicates the light period, and the black bar indicates the dark period. Data are presented as means ± S D ⌈ n=3 biological replicates). ^*P<0.05 ,^**P<0.01 compared with the first time point (11:00 p.m.), Student’s t test. (C) Phenotypes of WT and transgenic rice plants grown in Beijing in 2018. (D to H) Yield-related parameters including grain yield per plant (D), grain yield per plot (E), grain number per panicle (F), straw weight (G), and harvest index (H). The data were obtained from the field experiment shown in (C). Box plots in (D) and (F) to (H) show median (horizontal lines) and 10th to 90th percentiles, and outliers are plotted as dots n=138 biological replicates). Data in (E) are presented as means ± S D ln=3 plots, 44 plants within a plot). ^*P<0.05 ,^**P<0.01 compared with WT, Student’s t tests.

To assess the stability of the yield enhancement conferred by OsDREB1C, we conducted field trials over several years and at three different sites that represent very different environmental conditions (tables S1 and S3 to S5). Data for the Beijing field trial in 2019 showed an even larger increase in grain yield for OsDREB1C-OE plants than in 2018 and similar yield reductions in OsDREB1C-KO plants (table S3). Having tested in temperate (Beijing; tables S1 and S3 and fig. S3M), tropical (Hainan Province; table S4 and fig. S3N), and subtropical (Zhejiang Province; table S5) locations, we noted that the most pronounced yield increases were detected in the long-day photoperiod and temperate climate conditions of Beijing. Nevertheless, there was still a strong yield improvement for OsDREB1C-OE plants in the short-day and tropical conditions of Hainan, with yield increases in the range of 7.8 to 16% yield per plant and 12.0 to 37.0% yield per plot (table S4 and fig. S3N). Also, the harvest index of OsDREB1C-OE plants (∼0.62) was higher than in WT _(∼0.54) and OsDREB1C-KO (0̃.32) plants in Hainan (table S4).

# OsDREB1C improves photosynthetic capacity and nitrogen utilization

To further explore the molecular basis of the yield enhancement conferred by OsDREB1C overexpression, a series of physiological measurements were conducted with hydroponically grown WT and transgenic plants. In these experiments, we observed faster growth of OsDREB1C-OE plants already at the seedling stage, whereas the growth of OsDREB1C-KO seedlings was retarded compared with WT plants (fig. S4, A and B). In addition, we noticed that OsDREB1C-OE plants displayed longer roots, probably related to their increased auxin content (fig. S5). We detected no growth differences among the WT, overexpression, and KO plants when the seedlings were cultivated in the dark (figs. S4, C to F, and S6), although the OsDREB1C-OE lines showed taller shoots during the first 10 days, likely because of the larger grain size, which provides more reserves for initial growth (fig. S6). Overall, these data suggested a light-induced mechanism of growth improvement. We next investigated whether photosynthetic capacity is improved by the overexpression of OsDREB1C. Leaves of OsDREB1C-OE plants contained higher levels of photosynthetic pigments (chlorophylls and carotenoids) compared with WT plants, whereas pigment levels were reduced in OsDREB1CKO plants (fig. S7A). Analysis of leaf mesophyll cells revealed that both chloroplast number and size were increased in OsDREB1C-OE plants (fig. S7, B and C). Biochemical analysis of photosynthetic protein complexes by blue-native polyacrylamidegel electrophoresisrevealed elevated levels of photosystem I (PSI) and PSII dimers, PSII-CP43monomers, andlight-harvestingcomplex II (LHCII) trimers in OsDREB1C-OE plants (fig. S7D). Immunoblotting confirmed an increased abundance of PSI, PSII, cytochrome b_6/f, ATP synthase, and LHC proteins in leaves of the overexpression lines (fig. S7E). OsDREB1C overexpression also led to enhanced amounts of the large and small subunits of ribulose bisphosphate carboxylase/oxygenase (RbcL and RbcS,respectively)andribulose-1,5-bisphosphate Fig. 2. OsDREB1C overexpression promotes photosynthetic capacity. (A and B) RuBisCO content (A) and RuBisCO activity (B) in the fourth leaf of 3-week-old rice seedlings grown hydroponically. Data are presented as means ± S D ··Π= 6 biological replicates). (C) Light-response curve of net photosynthesis fitted by the Farquhar–von Caemmerer–Berry (FvCB) model and generated at 30^∘C and 400 ppm ∁0_2 concentration in the field in Hainan in 2019. Data are presented as means ± S D ln=3 biological replicates). ^*P<0.05 for OE1, OE2, and OE5 compared with WT, Student’s t test. (D) ∁0_2 -response curve of net photosynthesis (A- C_i curve) fitted by the FvCB model and generated at 1200 𝗆 𝗈 𝗅 𝗆^-2 𝗌^-1 photosynthetic photon flux density and 30^∘0 . Data are presented as means ± S D ln=3 biological replicates). ^*P<0.05 for OE1, OE2, and OE5 compared with WT, Student’s t test. (E) Diurnal changes in the photosynthesis rate of flag leaves measured with a LICOR-6400 XT instrument at the heading stage (from 8:00 a.m. to 4:00 p.m.) in the field in Hainan. ^**P<0.01 for OE1, OE2, and OE5 compared with WT, ^*P<0.05 for OE1 at 8:00 a.m., all Student’s t test. (F) Stomatal conductance of flag leaves at the heading stage in Hainan. Measurements were performed with a LICOR-6400 XT instrument at 30^∘0 in ambient air between 10:00 a.m. and 11:00 a.m. Data in (E) and (F) are presented as presented as means ± S D Π̂=6 biological replicates). (G and H) Maximum rates of carboxylation (V_𝖼 𝗆 𝖺 𝗑) (G) and electron transport (J_max) (H) in WT, OsDREB1C-OE, and OsDREB1C-KO plants grown in the field in Hainan. Values were generated from the A- ·C_i curve and fitted by the FvCB model. Data are presented as means ± S D ln=3 biological replicates). All measurements in (C) to (H) were performed with flag leaves of field-grown rice plants at the heading stage. ^*P<0.05 ,^**P<0.01 compared with WT, Student’s t test.



carboxylase-oxygenase (RuBisCO) activase (fig. S7E). Accordingly, both RuBisCO content and activity were increased in OsDREB1C-OE plants (Fig. 2, A and B).

Next, we evaluated the role of OsDREB1C in regulating photosynthetic capacity by investigating rice plants grown in paddy fields. Consistent with the above-described results, key photosynthetic parameters, including diurnal changes, light-response curves, and CO_2 -response curves of net photosynthesis, were improved in OsDREB1C-OE plants and compromised in OsDREB1C-KO plants (Fig. 2, C to E). In addition, OsDREB1C-OE plants displayed higher stomatal conductance while maintaining a similar intercellular CO_2 concentration (Fig. 2F and fig. S8), suggesting that the higher carboxylation rates supported by higher RuBisCO contentandactivitypreventthebuildupofhigher intercellular CO_2 levels despite the opened stomata. This is in agreement with the higher maximum rate of RuBisCO carboxylation and the higher maximum rate of electron transport derived from modeling of A– 𝒞_i curves (Fig. 2, G and H). Analysis of key products of photosynthetic metabolism showed that OsDREB1COE leaves accumulated higher amounts of starch, sucrose, and fructose, thus potentially explaining the improved grain filling (fig. S9). Together, these results suggest that the observed growth and yield increases in OsDREB1C-OE rice plants result, at least in part, from enhancement of photosynthetic capacity.

To investigate whether OsDREB1C also influences nitrogen utilization, we conducted a ^15N feeding experiment and monitored nitrogen uptake and transport activity in hydroponically grown seedlings. After 3 hours of incubation in a ^15N· -nitrate solution, OsDREB1C-OE seedlings had higher ^15N contents in shoots and roots compared with WT and OsDREB1C-KO plants (Fig. 3, A and B). Additionally, OsDREB1C overexpression increased both nitrogen uptake by roots and transport activity from roots to shoots (Fig. 3, C and D), thus resulting in elevated nitrogen content, nitrogen use efficiency (NUE), and protein abundance in leaves of field-grown plants (Fig. 3, E and F, and fig. S10). Consistent with these findings, mature field-grown OsDREB1C-OE plants had increased total nitrogen contents in above-ground organs (Fig. 3G and fig. S11A), along with improved photosynthetic NUE in a range of nitrogen supply conditions and a higher intrinsic water use efficiency upon low-level nitrogen application (fig. S12), indicating higher efficiency of carbon gain at lower nitrogen cost. Analysis of carbon and nitrogen distribution showed that the OsDREB1C-OE plants accumulated more carbon and nitrogen in the grains, but less in their mature leaves, without substantial alterations in the carbon-to-nitrogen ratio (Fig. 3H and fig. S11). These findings indicate more efficient resource allocation from source (leaves) to sink (grains) in the overexpression plants compared with WT plants (fig. S11), leading to a further boost in grain yield, especially under conditions of low nitrogen supply (fig. S13). In conclusion, OsDREB1C appears to (i) stimulate nitrogen uptake by roots, (ii) enhance nitrogen transport to aerial organs, and (iii) promote resource allocation from leaves and shoots to grains.


Fig. 3. OsDREB1C overexpression increases nitrogen uptake and transport. (A and B) ^15N content in shoots (A) and roots (B) of 3-week-old WT, OsDREB1C-OE, and OsDREB1C-KO seedlings incubated with 0.5 𝗆 𝖬 𝖪^15𝖭 𝖮_3 for 3 hours. (C) ^15𝖭 -nitrate uptake activity of roots. (D) ^15𝖭 transport activity from roots to shoots. Data in (A) to (D) are presented as means ± S D ⌈ n=5 biological replicates). (E) Nitrogen content of flag leaves of WT, OsDREB1C-OE, and OsDREB1C- ·𝖪 𝖮 plants at the heading stage grown in the field in Beijing in 2021. Data are presented as means ± S D ṅ>5 biological replicates). (F) NUE of WT, OsDREB1C-OE, and OsDREB1C-KO plants grown with 100 or 200 𝗄 𝗀 𝗁 𝖺^-1 nitrogen supply in the field in Beijing in 2021. Box plot shows median (line) and individual values (black dots) ⌈ n>5 biological replicates). ^*P<0.05 ,^**P<0.01 compared with WT, Student’s t test. (G and H) Nitrogen distribution (G) and nitrogen distribution ratio (H) in seeds, straw, and leaves of mature plants grown in the field in Beijing in 2019. Data are presented as means ± S D ℸ=4 biological replicates). ^*P<0.05 ,^**P<0.01 compared with WT, Student’s t test.

# OsDREB1C promotes flowering and shortens growth duration

OsDREB1C-OE plants also exhibited a pronounced early flowering phenotype in our field trials (Fig. 4, A and B). Early flowering was accompanying with higher biomass accumulation at the heading stage (Fig. 4C). OsDREB1COE plants flowered 13 to 19 days earlier than the WT, whereas OsDREB1C-KO plants flowered 3 to7 dayslaterunder thelong-dayconditions in the field in Beijing (Fig. 4B). Overexpression of OsDREB1C also had an effect on leaf maturation and duration of the growth period (Fig. 4D). However, these differences were less evident undertheshortphotoperiodinHainan(tableS4). To further explore the influence of OsDREB1C on flowering, the expression of key genes involved in the induction of flowering was analyzed, including the two F T/⋯ -like genes, Hd3a (OsFTL2) and R F T I(O s F T L3)(l9,2O) , and the downstream transcription factor gene OsMADS14. Expression of all three genes was up-regulated in OsDREB1COE plants and down-regulated in OsDREB1C-KO plants (Fig. 4, E to G). By contrast, expression of Hd1 and Ehd1, two genes upstream of florigen, was not affected in OsDREB1C-OE and OsDREB1C-KO plants (Fig. 4, H and I), indicating that OsDREB1C affects flowering time by controlling a specific subset of flowering regulators.

# OsDREB1C enhances yield in an elite cultivar

To further test whether OsDREB1C overexpression can increase the yield of elite rice varieties, we transformed the p35S::OsDREB1C construct into Xiushui 134 (XS134), a high-yielding elite temperate japonica cultivar that is widely cultivated in southern China. Transgenic XS134 rice plants (OsDREB1C-XSOE) exhibited increased height, longer panicles, higher grain numbers per panicle, and higher grain yields over 2 consecutive years, similar to those seen in OsDREB1C-OE plants in the Nipponbare background (Fig. 5, A to G, and figs. S14 and S15). The grain yield per plot was increased by 10.3 to 12.7% in 2020 and 30.1 to 41.6% in 2021 in Hangzhou (fig. S14G and Fig. 5H), accompanied by an increased harvest index of up to 10.5 and 15.7% , respectively (fig. S14H and Fig. 5I). Moreover, OsDREB1C-XSOE lines flowered 2 days earlier than the WT in Hangzhou (fig. S14I). Accordingly, OsDREB1C-XSOE lines displayed 26.2 to 42.4% higher grain yields per plant in Hainan (fig. S15), whereas no difference in flowering time was observed because of the short-day growth conditions.

Next, we analyzed the OsDREB1C sequences in 709 rice accessions, including 299 indica, 355 temperate japonica, 14 tropical japonica, and 41 intermediate varieties (21). Three distinct haplotypes were identified (Hap. 1 to Hap. 3) on the basis of nucleotide polymorphisms, most of which reside in the promoter regions (fig. S16).


Fig. 4. OsDREB1C overexpression leads to early flowering and shortens the overall growth period. (A) Growth of WT, OsDREB1C-OE, and OsDREB1C
KO plants in natural long-day conditions in Beijing in 2019. Scale bar, 50 cm. (B) Flowering time of field-grown plants. DAS, days after sowing. (C) Biomass of rice plants at the heading stage of OsDREB1C-OE plants (105 DAS) grown in the field in Beijing in 2021. (D) Soil plant analysis development (SPAD) value of flag leaves at different development stages (112, 129, 139, and 152 DAS) of plants grown in the field in Beijing in 2019. ^**P<0.01 for OE1,

OE2, and OE5 at four time points and for KO1, KO2, and KO3 at 112 and 129 DAS compared with WT, all Student’s t test. (E to I) Relative gene expression levels of the flowering regulators Hd3a (E), RFT1 (F), OsMADS14 (G), Hd1 (H), and Ehd1 (I) in WT, OsDREB1C-OE, and OsDREB1C-KO plants. RNAs were extracted from leaves of field-grown plants before heading (9̃0 DAS) in the Beijing field in 2019. Data in (B) to (I) are presented as means ± SD [n=3 biological replicates, except for n=𝚕0 for (C)]. ^*P<0.05 ,^**P<0.01 compared with WT, Student’s t test.

# Identification of target genes of OsDREB1C

We next wanted to characterize the molecular functions of OsDREB1C in more detail. Multiple amino acid sequence alignment revealed a conserved AP2 domain among all OsDREB1C homologs (figs. S17 and S18). Expression analysis showed that OsDREB1C was expressed ubiquitously in all rice tissues examined (root, stem, leaf, and panicle), but particularly strongly in the root (fig. S19A). During the growth period, OsDREB1C transcript levels peaked at the tillering stage (fig. S19B). Transient expression assays in rice protoplasts revealed that OsDREB1C-GFP and YFP-OsDREB1C fusion proteins mainly localize to the nucleus, but a substantially weaker signal in the cytoplasm was also discernable (fig. S19C). Very similar patterns of OsDREB1C-GFP subcellular localization were observed in Arabidopsis and Nicotiana benthamiana (fig. S19, D and E).

Sequence analysis with PlantPan3.0 suggested that the OsDREB1C protein may bind to the DRE/CRT (GCCGAC) motif that had been identified as a core cis-acting element regulating gene expression in response to drought, salt, and cold stresses (22). Yeast one-hybrid assays verified the direct binding of OsDREB1C to DRE/CRT (GCCGAC) as well as the GCC box (GCCGCC) and G box (CACGTG) cis elements in vitro (fig. S20A). Measurement of the transcription-stimulating activity in rice protoplasts demonstrated that OsDREB1C was able to activate transcription of the GUS reporter gene (fig. S20, B and C). Taken together, these data suggest that OsDREB1C functions as a transcriptional activator.

To identify genome-wide binding sites of OsDREB1C in vivo, we performed chromatin immunoprecipitation sequencing (ChIP-seq) experiments with rice protoplasts transiently expressing the OsDREB1C-GFP fusion protein. These analyses identified a total of 9735 putative OsDREB1C-binding sites, of which 68% localized to genic regions and 32% to intergenic regions (Fig. 6A). The core motif found to be enriched in the OsDREB1C-binding regions was DRE/CRT (GCCGAC) (Fig. 6B). We next analyzed the differentially expressed genes (DEGs) between OsDREB1C-OE and WT plants (as determined by RNA-seq), and extracted the DEGs associated with OsDREB1C-binding peaks. In this way, 345 up-regulated genes were identified as putative OsDREB1C targets (Fig. 6C). Gene ontology (GO) enrichment analysis was then conducted to associate biological processes with those DEGs. Transmembrane transport–related genes were found to be most strongly enriched and included the two nitrogen transporter genes, OsNRT2.4 and OsNRT1.1B (Fig. 6D). Other genes with functions in the nitrogen metabolic process were also present in the DEG set, including the nitrate reductase gene OsNR2. When searching for floweringrelated DEGs that could potentially explain the pronounced early-flowering phenotype, the gene OsFTL1 (FT-Like 1) was found. OsFTL1 is a homolog of the Arabidopsis FT gene that plays a central role in integrating signals from the different flowering pathways (23, 24). Moreover, a key photosynthesis-related gene, OsRBCS3, encoding the RuBisCO small subunit and known to be transcriptionally activated by light (25), was also up-regulated in OsDREB1C-OE plants.

# OsDREB1C directly activates key pathway genes

To verify whether these five candidate genes (OsRBCS3, OsNR2, OsNRT2.4, OsNRT1.1B, and OsFTL1) were targets of OsDREB1C, we performed ChIP–quantitative polymerase chain reaction (qPCR) experiments using transgenic plants expressing an OsDREB1C-GFP fusion protein and DNA affinity purification sequencing (DAP-seq) assays in vitro. The results revealed that OsDREB1C directly binds to the promoter of OsRBCS3 and to exons of OsNR2, OsNRT2.4, OsNRT1.1B, and OsFTL1 (Fig. 6E and fig. S21A). Electrophoretic mobility shift assay (EMSA) confirmed that the DRE/CRT elements are necessary for OsDREB1C binding (Fig. 6F and fig. S22). Moreover, luciferasebased transient transactivation assays verified that OsDREB1C activates the expression of OsRBCS3, OsNR2, OsNRT2.4, OsNRT1.1B, and OsFTL1 (Fig. 6G). Gene expression analyses revealed that the mRNA levels of OsRBCS3, OsNR2, OsNRT2.4, OsNRT1.1B, and OsFTL1 correlated with OsDREB1C levels, in that the expression levels were increased in OsDREB1COE plants and decreased in OsDREB1C-KO plants (fig. S21B). Taken together, these results suggest that OsDREB1C can activate gene expression directly by binding to the promoter of OsRBCS3 and to the exons of OsNR2, OsNRT2.4, OsNRT1.1B, and OsFTL1.


Fig. 5. OsDREB1C confers yield gains in an elite rice germplasm. (A) Whole-plant phenotypes of mature Xiushui134 (XS134) and XS134-OsDREB1C-OE plants (XSOE-8/9/12) grown in Hangzhou in 2020. Scale bar, 20 𝖼 𝗆 . (B to I) Yield parameters of XS134 and XS-OE plants grown in Hangzhou in 2021, including plant height (B), panicle number (C), grain number per panicle (D), seed setting rate (E), 1000-grain weight (F), straw weight (G), grain yield per plot (H), and harvest index (I). The box plots in panels (B), (C), and (G) show the median (horizontal line) and individual values (black dots) ⌈ n=100 biological replicates). Data in F(D) to (I) except (G) are presented as means ± S D ⌈ n=6 plots). ^*P<0.05 ,^**P<0.01 compared with XS134, Student’s t test.

To clarify whether OsFTL1 induction is responsible for the early-flowering phenotype of OsDREB1C-OE plants, we generated OsFTL1 overexpression lines in the rice cultivar Nipponbare (fig. S23, A and E). The heading time of OsFTL1-OE plants was drastically shortened, ranging from 45 to 47 days, whereas that of WT plants was 116 to 118 days under the long-day photoperiod in Beijing (fig. S23, B and D). Under the short-day photoperiod in Hainan, OsFTL1-OE plants flowered 10 to 13 days earlier (fig. S23C). These results are consistent with a previous study on floral induction in transgenic plants grown in culture vessels (26). OsFTL1-OE plants were dwarfed and exhibited reduced grain yields (fig. S23F), probably because of their Fig. 6. OsDREB1C induces transcription of photosynthesis, nitrogen utilization, and floweringrelated genes. (A) Distribution of candidate OsDREB1C-binding regions across the rice genome as determined by ChIP-seq. TSS, transcription start site; TTS, transcription termination site. (B) Motif analysis using HOMER to identify core motifs enriched within the experimentally determined (by ChIP-seq) OsDREB1C-binding regions. (C) Venn diagram showing the overlap between putative OsDREB1C target genes identified by ChIP-seq and differentially up-regulated genes in OsDREB1C-OE1 relative to the WT as identified by RNA-seq. (D) GO enrichment analysis of the overlapping gene set in (C). (E) OsDREB1C preferentially binds to the OsRBCS3 promoter and to exons of OsNR2, OsNRT2.4, OsNRT1.1B, and OsFTL1, as validated by both ChIP-qPCR and DAP-seq. The diagrams in (E) depict the putative promoter region of OsRBCS3 and exons of the OsNR2, OsNRT2.4, OsNRT1.1B, and OsFTL1 genes. P1, P2, and P3 indicate primers used in the ChIP-qPCR experiments for the five examined loci shown in fig. S21. There are DRE/CRT elements within P3 of OsRBCS3, P1 of OsNR2 and OsNRT1.1B, and P2 of OsNRT2.4 and OsFTL1. (F) EMSA data confirming that the GST-OsDREB1C protein binds to promoters and exons containing DRE/CRT elements. (G) OsDREB1C activates transcription from the OsRBCS3 promoter and from OsNR2, OsNRT1.1B, OsNRT2.4, and OsFTL1 exon-luciferase fusion constructs in transient transactivation assays. Shown are relative ratios of the transcriptional activities conferred by OsDREB1C expression to the empty vector control. Asterisks indicate significant differences between the control and OsDREB1C. LUC/REN, ratio of firefly luciferase to Renilla luciferase activity. Data are presented as means ± S D ln≥3 biological replicates). ^*P<0.05 ,^∗∗P<0.01 compared with WT, Student’s t test.



shortened vegetative phase (and the insufficient buildup of resources for allocation to seeds).

# OsDREB1C effects in wheat and Arabidopsis

To determine whether the function of DREB1C is conserved in other plant species, we generated transgenic wheat and Arabidopsis plants overexpressingOsDREB1C.Analysisofthetransgenic lines showed that OsDREB1C overexpression in the wheat cultivar Fielder also improved photosynthetic capacity, reduced the time to flowering by 3 to 6 days, and conferred increased grain yield per plant by 17.2 to 22.6% in the field and by 18.6 to 23.5% in the greenhouse (Fig. 7, A to F, and fig. S24). Likewise, transgenic Arabidopsis lines overexpressing OsDREB1C had more and bigger leaves, and flowered up to 4 days earlier thantheWT(Fig.7,GtoJ).Thebiomassyieldwas increased by 14.2 to 35.8% in the OsDREB1C-OE Arabidopsis plants (Fig. 7, K and L).

# DISCUSSION

Transcription factors of the DREB subfamily belong to the AP2/ERF family and have been and OsDREB1C-OE Arabidopsis (p35S::OsDREB1C-GFP, AtOE-10/11/12) plants at the seedling and flowering stages. Note the early flowering of all three overexpression lines. Plants were grown in short-day conditions (8 hours light/16 hours dark) in a growth chamber for 2 weeks and then transferred to long-day conditions (16 hours light/8 hours dark). Scale bar, 4 cm .(H) OsDREB1C expression levels in OsDREB1C-OE Arabidopsis plants. Data are presented as means ± S D ⌈ n=3 biological replicates). (I to L) Flowering time (I), rosette leaf number (J), fresh weight (K), and dry weight (L) of WT (Col-0) and OsDREB1C-OE Arabidopsis plants. Data are presented as means ± S D (⌈ n=10 biological replicates). ^*P<0.05 ,^∗∗P<0.01 compared with Col-0, Student’s t test.


Fig. 7. OsDREB1C increases flowering, photosynthetic capacity, and yields in wheat and Arabidopsis. (A) Early flowering phenotype of OsDREB1COE field-grown wheat plants (pUBI::OsDREB1C, TaOE-5/8/9) compared with WT plants (cv. Fielder) at the booting stages. Scale bar, 10 cm. (B and C) Flowering time (B) and photosynthesis rate (C) of Fielder and OsDREB1C-OE wheat plants at the heading stage grown in the field in Beijing in 2021. Data are presented as means ± S D ṅ>5 biological replicates). (D to F) Grain number per spike (D), 1000-seed weight (E), and grain yield per plant (F) of Fielder and OsDREB1C-OE wheat plants grown in the field in Beijing in 2021. Data are presented as means ± S D ṅ>30 biological replicates). ^*P<0.05 , P < 0.01 compared with Fielder, Student’s t test. (G) Phenotype of WT (Col-0)

demonstrated to activate multiple downstream genes in response to abiotic stresses such as drought, salinity, and freezing in various seed plants (27, 28). The DREB-binding DRE/CRT cis-element (GCCGAC motif) is present in the promoter of many stress-inducible genes (29). However, the previous work has also revealed a trade-off between growth and stress tolerance, in that constitutive overexpression of DREB1 genes in Arabidopsis and rice, although conferring improved stress tolerance, often leads to growth retardation and yield penalties (27, 29–31). In the course of this work, we have shown that the inverse relationship between stress tolerance and yield can be uncoupled and evenreversed. Overexpression of OsDREB1C in rice plants resulted in substantial yield increases of 41.3 to 68.3% , and these yield gains were accompanied by a shortened vegetative phase, in that the overexpression plants flowered much earlier than the WT. Although “highyielding” and “early-maturing” have long been seen as conflicting traits in crop breeding, a report has shown that the long noncoding RNA Ef-cd shortens maturity duration without incurring a yield penalty (32). Similarly, overexpression of the nitrate transporter gene OsNRT1.1A confers both high yield and early maturation (33).

An unsolved problem pertinent to both agricultural productivity and the environmental footprint of current agricultural practices is the requirement for a high level of nitrogen fertilizer to attain high yields. This is mainly because of the low NUE of most major crop varieties (34). In addition to its negative environmental impact, excessive application of nitrogen fertilizer has other undesired effects, including delayed flowering, extended growth duration, and reduced yield potential (35). In this study, we have shown that by engineering the expression of a transcription factor that controls and coordinates photosynthesis, nitrogen utilization, and flowering time without affecting known genes involved in high yield and early flowering (fig. S25), it is possible to achieve enhanced growth and increased yields while at the same time improving the efficiency of nitrogen utilization. Thus, our work demonstrates that three key agricultural traits can be improved simultaneously by OsDREB1C overexpression: yield, NUE, and flowering time. We provide several lines of physiological and molecular evidence in support of the assumption that OsDREB1C confers accelerated vegetative growth and biomass accumulation before heading by (i) enhancing photosynthetic capacity through OsRBCS3; (ii) enhancing nitrogen uptake and transport through expression of OsNRT1.1B, OsNRT2.4, and OsNR2; and (iii) promoting early flowering through OsFTL1. We propose that efficient subsequent allocation of assimilated carbohydrates and nitrogen from leaves to the panicle further contributes to the elevated yield, presumably through coordinated regulation of several target genes of OsDREB1C, including amino acid and ammonium transporters (figs. S26 and S27). OsDREB1C binds to the exon regions rather than the promoters of four of the five verified target genes. However, the preferential binding of transcription factors to intragenic regions (exons and/or introns) of target genes has been demonstrated in a number of previous studies (10, 36–38).

Currently, the relative rates of yield increase achieved by plant breeding are declining and have fallen below 1% per year for most cereal crops (39). In view of this trend and the need to double the world’s food production by 2050 despite reduced availability of arable land and the challenges of climate change, the very large yield increases achieved in the field by engineering the expression of a single transcriptional regulator gene are unprecedented. Our findings suggest that after centuries of breeding for yield, there is still potential for substantial leaps in the yields of the world’s main staple crops.

In the present study, overexpression of OsDREB1C was achieved using transgenic technologies. Alternatively, genome-editing technologies could be used to achieve OsDREB1C overexpression, forexample, byusingbaseeditors to introduce expression-enhancing point mutations into the promoter region of the OsDREB1C gene (40, 41), thus creating transgene-free, highyielding varieties. Also, the existing natural variation of OsDREB1C in rice provides a genetic resource that can be readily tapped. Finally, the conserved function of OsDREB1C in seed plants offers the potential to substantially improve biomass and yield in other crops. In summary, our data suggest OsDREB1C engineering as a generally applicable strategy to increase crop yields, with the added benefits of shortening the growth duration and lowering the environmental footprint of agriculture.

# Methods summary

The candidate gene OsDREB1C was identified by a screen of 118 transcription factors that are related to C_4 photosynthesis, induced by light, and highly expressed under low-nitrogen conditions. The function of OsDREB1C was investigated by transgenic overexpression and CRISPR-Cas9–based gene-editing approaches. Agrobacterium-mediated transformation was used to introduce the constructed vectors into rice (Nipponbare and Xiushui134) and wheat (Fielder), and the floral dip method was used to transform Arabidopsis (Col-0). Gene expression levels in the overexpression and KO lines were evaluated by quantitative reverse transcription PCR (qRT-PCR).

The growth rates of hydroponically grown rice seedlings were monitored under standard growth conditions. Etiolated growth was assessed in constant darkness. Photosynthesis rates were measured with a LICOR-6400XT gas exchange system in field-grown plants. Accumulation levels of photosynthesis-related proteins were assessed by immunoblotting. RuBisCO content was quantified by immunoblot assays, and RuBisCO activity was determined as the rate of CO_2 fixation on RuBP using spectrophotometry. Sugar and starch contents were measured using enzymatic digestion methods. Nitrogen uptake and transport activity were evaluated with the ^15N -labeling assay. Carbon and nitrogen contents were assessed with an IsoPrime 100 analyzer. NUE was calculated as the ratio of grain yield to applied nitrogen fertilizer. Phytohormone contents were quantified using ultraperformance liquid chromatography quadropule ion trap mass spectrometry. Agronomic and yield traits were assessed in field experiments with randomized block design and performing three replicates for each experiment in Beijing, Hangzhou, and Hainan in four successive years (May 2018 to May 2022).

A sequence alignment of OsDREB1C orthologs was constructed and phylogenetic analysis was performed using ESPript 3.0 and MEGA 7, respectively. Genetic variation and haplotype association analyses were performed by variant cell format tools using a general linear model implemented in TASSEL 5.0. Subcellular localization of OsDREB1C was assessed by transient expression assays in rice protoplasts and tobacco leaves and in stable transgenic Arabidopsis plants expressing OsDREB1C-GFP.ThebindingabilityofOsDREB1C to the predicted core promoter motifs was tested with yeast one-hybrid assays. The transactivation capacity of OsDREB1C was evaluated by transient transactivation assays.

To identify the genome-wide binding sites of OsDREB1C, ChIP-seq experiments were performed with rice protoplasts transiently expressing the OsDREB1C-GFP fusion protein, and RNA-seq analysis with WT and OsDREB1C-OE plants grown in the field. Up-regulated genes in OsDREB1C-OE plants associated with OsDREB1C-binding peaks were extracted as putative OsDREB1C targets, and GO enrichment analysis was conducted to identify the associated biological processes. ChIP-qPCR using transgenic plants expressing an OsDREB1C-GFP fusion protein and DAP-seq using expressed HALOTag-OsDREB1C fusion proteins in vitrowere performed to verify the binding of OsDREB1C to target genes and test for target gene activation. Direct binding of OsDREB1C to target sequences was examined by EMSA. The transcriptional activation of targets genes by OsDREB1C was tested by dual luciferase reporter assays and qRT-PCR.

Details for experimental procedures are provided in the supplementary materials.
