# Functional divergence of two duplicated Fertilization Independent Endosperm genes in rice with respect to seed development

Xiaojun Cheng1,â€ , Meiyao Pan1,â€ , Zhiguo E2, Yong Zhou1, Baixiao Niu1 and Chen Chen1, Jiangsu Key Laboratory of Crop Genetics and Physiology/Key Laboratory of Plant Functional Genomics of the Ministry of Education/Jiangsu Key Laboratory of Crop Genomics and Molecular Breeding/Jiangsu Co-Innovation Center for Modern Production Technology of Grain Crops, College of Agriculture, Yangzhou University, Yangzhou, China, and 2Key Laboratory of Rice Biology, China National Rice Research Institute, Hangzhou, China

Received 4 September 2019; revised 14 May 2020; accepted 26 June 2020; published online 6 July 2020. For correspondence (e-mail chenchen@yzu.edu.cn)
â€ These authors contributed equally to this work

# SUMMARY

Fertilization Independent Endosperm (FIE) is an essential member of Polycomb Repressive Complex 2 (PRC2) that plays important roles in the developmental regulation of plants. OsFIE1 and OsFIE2 are two FIE homologs in the rice genome. Here, we showed that OsFIE1 probably duplicated from OsFIE2 after the origin of the tribe Oryzeae, but has a specific expression pattern and methylation landscape. During evolution, OsFIE1 underwent a less intensive purifying selection than did OsFIE2. The mutant osfie1 produced smaller seeds and displayed reduced dormancy, indicating that OsFIE1 predominantly functions in late seed development. Ectopic expression of OsFIE1, but not OsFIE2, was deleterious to vegetative growth in a dose-dependent manner. The newly evolved N-terminal tail of OsFIE1 was probably not the cause of the adverse effects on vegetative growth. The CRISPR/Cas9-derived mutant osfie2 exhibited impaired cellularization of the endosperm, which suggested that OsFIE2 is indispensable for early seed development as a positive regulator of cellularization. Autonomous endosperm was observed in both Î´o s F I E2^+- and osfie1/ âˆ‡o s F I E2^+- but at a very low frequency. Although OsFIE1-PRC2 exhibited H3K27me3 methyltransferase ability in plants, OsFIE1-PRC2 is likely to be less important for development in rice than is OsFIE2-PRC2. Our findings revealed the functional divergence of OsFIE1 and OsFIE2 and shed light on their distinct evolution following duplication.

Keywords: cellularization, dormancy, endosperm development, function divergence, polycomb repressive complex 2, rice.

# INTRODUCTION

The endosperm is a product of double fertilization in higher plants (Olsen, 2001). As a triploid tissue, the endosperm is initiated by fusion between a central cell (2n) and a sperm cell (n). The primary endosperm cell usually undergoes several rounds of mitotic division uncoupled from cytokinesis, which results in a multinucleate cell with many free nuclei, termed a syncytium (Olsen, 2001; Wu et al., 2016). Then, the syncytium starts to cellularize and generate the first layer of endosperm cells (Olsen, 2001; Wu et al., 2016). Subsequent cell divisions allow the endosperm cells to fill the seed. The fate of the endosperm is different in dicots and monocots; the endosperm is usually consumed in dicots during seed development, while it is retained in mature seeds of monocots. However, the developmental process of endosperm is quite highly conserved in many plants (Olsen, 2004; Agarwal et al., 2011). As a nutritional supply tissue, the endosperm is indispensable for embryo development. Either delayed or accelerated cellularization can lead to seed failure in interploidy and interspecific hybrids (Walia et al., 2009; Ishikawa et al., 2011; Kradolfer et al., 2013; Tonosaki et al., 2018). Developmental defects in the endosperm will cause embryo abortion, which has been found to occur in many plant species, where it acts as an important reproductive barrier (LafonPlacette and Kâ‚¬ohler, 2016; Chen et al., 2016b).

Polycomb repressive complex 2 (PRC2) plays an important role in developmental regulation by controlling epigenetic modification of the genome (Mozgova and Hennig, 2015; Mozgova et al., 2015). PRC2 has methyltransferase activity for the methylation of Lys27 in histone H3 (H3K27) (Cao et al., 2002; Nallamilli et al., 2013). The major components of PRC2 are highly conserved in animals and plants (Tonosaki and Kinoshita, 2015), both of which include four group members. For example, the PRC2 of Drosophila is composed of WD40 protein p55 (p55), Suppressor of Zeste 12 [Su(z)12], Enhancer of Zeste [E(z)] and extra sex combs (ESC). Fertilization Independent Seed (FIS)-PRC2 of Arabidopsis, composed of FIS2 [Su(z)12 member], Fertilization Independent Endosperm [FIE, (ESC) member], MEDEA [MEA, E(z) member] and Multicopy Suppressors of IRA 1 (MSI1, p55 member), acts as a key regulator of seed development (Mozgova et al., 2015). Mutation of these genes results in autonomous endosperm, developing without fertilization (Ohad et al., 1996; Chaudhury et al., 1997; Kiyosue et al., 1999). In addition, FIS-PRC2 is indispensable for the transition from the syncytium to cellularized cells in Arabidopsis (Grossniklaus et al., 1998; Luo et al., 2000; Vinkenoog et al., 2000; Hennig, 2003).

Different plant species have evolved distinct members of each PRC2 group (Tonosaki and Kinoshita, 2015). For example, rice lacks MEA that belongs to the E(z) group and lacks FIS2 that belongs to the ð–² ð—Ž(ð—“)Ï„2 group (Luo et al., 2009). FIE is the only ESC member that encoded by the Arabidopsis genome. However, species of the Poaceae usually have multiple FIE homologs (Danilevskaya et al., 2003a; Luo et al., 2009; Kapazoglou et al., 2010). OsFIE1 and OsFIE2, which show marked similarities, are two FIE homologs encoded by the rice genome (Luo et al., 2009). To date, the mechanism by which PRC2 regulates endosperm development in species other than Arabidopsis is largely unknown. Overexpression of OsFIE1 led to a phenotype of short stature, low seed-setting rate and small seeds (Zhang et al., 2012; Folsom et al., 2014), but no altered phenotype or a very mild phenotype change was exhibited by the RNA interference (RNAi)-mediated knockdown plant or T-DNA mutant of OsFIE1 according to previous reports (Yang et al., 2013; Huang et al., 2016). On the other hand, OsFIE2 RNAi plants displayed phenotypes similar to those of the OsFIE1-overexpressors, such as dwarfism and reduced seed-setting rate (Nallamilli et al., 2013; Li et al., 2014; Liu et al., 2016). Some inconsistent findings have been reported according to previous studies. For example, Li et al. (2014) found that knock-down of OsFIE2 led to an autonomous endosperm but Nallamilli et al. (2013) did not observe such an effect. Luo et al. (2009) found that a T-DNA mutant line of OsFIE1 did not cause any morphological changes, whereas Huang et al. (2016) reported that the RNAi of OsFIE1 reduced seed size and delayed embryo development.

MEA and FIS2 are recently evolved PRC2 members because of gene duplication in Arabidopsis (Spillane et al., 2007). Evolutionary analysis revealed that MEA had undergone natural selection in the promoter region (Kawabe et al., 2007; Spillane et al., 2007; Miyake et al., 2009). Currently, our understanding of how the PRC2 members evolved and how they contribute to adaptation in plants other than Arabidopsis is limited (Furihata et al., 2016). Given the strong similarities between OsFIE1 and OsFIE2, we believed that it would not be a good strategy to use RNAi lines for functional analysis, owing to the spatiotemporal overlaps of expression of the two genes in seeds (Li et al., 2014). Therefore, systemic analysis of OsFIEs using null mutants is preferred to achieve a better understanding of the PRC2-regulated endosperm development and to clarify the controversies that have been reported. In the present study, we combined the use of evolutionary and genetic approaches to dissect the functional divergence of OsFIE1 and OsFIE2. Our results suggested that OsFIE1 evolved from OsFIE2 by gene duplication after the tribe Oryzeae had evolved. Our findings showed that OsFIE1 and OsFIE2 have evolved distinct functions in seed and endosperm development, and that they experienced different natural selection pressures.

# RESULTS

# Divergence of the FIEs in rice

OsFIE1 (Os08g0137100) and OsFIE2 (Os08g0137250) are closely arrayed on chromosome 8 of rice (Oryza sativa), and are separated by a putative actin gene (Figure S1a). Such an arrangement is highly conserved among different Oryza species, such as O. meridionalis (A genome), O. punctata (B genome) and O. brachyantha (F genome) (Figure S1a), but differs from that in maize, where the two FIE homologs are located on different chromosomes (Danilevskaya et al., 2003b). Next, we used the amino acid sequences of the OsFIEs as queries to search against the genomes of 20 diploid monocot species that had been deposited in the EnsemblPlants database, namely 11 members of the Oryzeae, four members of the Pooideae subfamily, three species of the Panicoideae subfamily and two non-grass species (Dioscorea rotundata and Musa acuminata). Most of the monocots (16 of 21) we studied have two FIE genes, while Dioscorea rotundata, Musa acuminate, Setaria italica and Hordeum vulgare have one and Brachypodium distachyon has four (Figure S1b). A phylogenetic analysis indicated that the duplication events of the FIEs probably occurred independently in different monocot genera. For example, two FIEs of maize were grouped together, but OsFIE1 and OsFIE2 belonged to two different clades (Figure S1b). The OsFIE1 homologs were exclusively found in the Oryzeae tribe and were relatively distant from the other FIE homologs (Figure S1b). The FIE2 homologs of the Oryzeae were closer to the FIEs of non-Oryzeae members of the other monocot species (Figure S1b). The findings strongly suggested that FIE1 of the Oryzeae evolved from FIE2, and that the duplication event probably occurred after the origin of the tribe Oryzeae.

Compared with OsFIE2, OsFIE1 had an additional 82 amino acid residues at its N-terminal end (Figure S1c). This tail was conserved among Oryza species, but the FIE1 homolog of Leersia perrieri, which belongs to a different genus of the tribe Oryzeae, did not carry this tail (Figure S2). This indicated that this extra N-terminal tail of OsFIE1 had probably evolved after the genera Oryza and Leersia had diverged, which occurred 14.2 million years ago (Kellogg, 2009). Using L. perrieri as an outlier group, we calculated the ratio of the non-synonymous (dN) to synonymous (dS) substitutions of the FIE1 and FIE2 coding sequences of different Oryza species. The results showed that the dN/dS ratio of FIE2 (0.03â€“0.11) was significantly lower than that of FIE1 (0.21â€“0.27) and other polycomb genes including EMF2a, EMF2b, iEZ and CLF, whereas MSI1 showed a resemble dN/dS ratio to that of FIE2 (Figure S3). The average dN/dS value in AA genome rice species is about 0.4 (Huang et al., 2012; Zhang et al., 2014), which is much higher than that of the rice FIE2 (0.058). The results implied that FIE2 had experienced the more intense purifying selection than the majority of the other polycomb genes in rice.

In line with previous findings (Li et al., 2014), we found that OsFIE1 and OsFIE2 displayed quite different spatiotemporal expression patterns. Expression of OsFIE1 was specifically activated in the caryopsis, whereas OsFIE2 is ubiquitously expressed (Figure S4a,b). OsFIE1 and OsFIE2 showed similar expression profiles during caryopsis development, being highly upregulated from 3 to 10 days after fertilization (DAF) (Figure S4c). However, OsFIE2 transcripts were more abundant than those of OsFIE1 were in the developing caryopsis. The findings suggested that OsFIE1 had probably been subfunctionalized because of functional divergence following duplication. Divergence of expression usually reflects a change in gene regulation. We had previously found that application of a DNA methylation inhibitor, 5-aza- .2^' -deoxycytidine, could activate the expression of OsFIE1 ectopically in seedlings, whereas OsFIE2 show limited response to the chemical (Chen et al., 2018a). Therefore, we proposed that OsFIE1 and OsFIE2 might have distinct DNA methylation patterns in the genome. To test this hypothesis, we performed bisulfite polymerase chain reaction (PCR) sequencing to detect DNA methylation of the two genes. Generally, OsFIE2 showed higher levels of methylation than OsFIE1 in the promoter region as well as in the coding sequence (Figure S4d). Overall, the methylation level of OsFIE2 was stable across leaf and endosperm tissues. However, OsFIE1 was generally hypomethyalted in the endosperm at 6 DAF in comparison with the leaf (Figure S4d). In agreement, the data deposited in the Plant Methylome database showed that overall, OsFIE2 has a higher DNA methylation level than OsFIE1 in leaves, particularly on the promoter regions (Figure S4e).

# Deleterious effects of ectopically expressed OsFIE1

Using the ubiquitin promoter of maize to drive the ectopic expression of OsFIE1 (Ubi::OsFIE1) resulted in substantially decreased plant height and increased tiller numbers (Figure 1a,b), phenotypic effects that were consistent with previous findings (Zhang et al., 2012; Folsom et al., 2014). Seven independent transgenic events showed very similar phenotype. We then analyzed histone modifications of the Ubi::OsFIE1 lines using antibodies that react exclusively against tri-methylated histone H3 at lysine 27 (H3K27me3). The results showed that H3K27me3 was significantly elevated in Ubi::OsFIE1 (Figure 1c). This finding indicated that, similar to OsFIE2 (Nallamilli et al., 2013), the OsFIE1- PRC2 complex has histone methyltransferase activity for H3K27me3 in vivo. We also used the OsFIE2 promoter to drive the expression of OsFIE1 (pOsFIE2::OsFIE1) in rice (Figure 1d). The transgenic plants showed moderate (approximately 20-fold) upregulation of OsFIE1 expression in leaves (Figure 1e). However, the OsFIE2::OsFIE1 lines did not show the dwarf phenotype as was displayed by Ubi:: OsFIE1 (Figure 1a,d). These findings suggested that the adverse effects of OsFIE1 expressed ectopically occurred when OsFIE1 products accumulated to a high concentration in vegetative tissues. However, moderate upregulation, resembling that of OsFIE2, appeared not to induce these defects. Therefore, the adverse effects are dose dependent.

OsFIE1 has an extra tail segment at the N-terminus (Figure S1c). To discuss whether the tail contributes to the deleterious phenotypic effects on vegetative growth, we generated two chimeric OsFIEs, namely Chi-OsFIEa and Chi-OsFIEb, by swapping the N-terminus of OsFIE1 to OsFIE2 as illustrated in Figure 1f. Overexpression of the chimeric OsFIEs driven by the ubiquitin promoter of maize (Figure 1g) did not cause growth defects as had been displayed in Ubi::OsFIE1 (Figure 1a,h,i). Although the chimeric OsFIEs were ectopically expressed in the leaves of ChiOsFIEa and Chi-OsFIEb at a resemble abundance to OsFIE1 that expressed in Ubi::OsFIE1 (Figure S5a), the plant height and tiller number of Chi-OsFIEa and Chi-OsFIEb were comparable with the corresponding values for the wild type (WT) but were significantly different from those in Ubi:: OsFIE1 (Figure S5b,c). The results suggested that the extra N-terminal segment of OsFIE1 alone was not the cause of the vegetative defects observed.

# Upregulation of OsFIE1 expression in endosperm reduced the seed size of rice

Previous studies had suggested that overexpression of OsFIE1 led to reduced seed size (Figure 2a; Zhang et al., 2012; Folsom et al., 2014). However, it was unclear whether



Figure 1. Phenotype of OsFIE1- and chimeric OsFIE overexpressors.
(a) Morphology of a representative wild type (WT) and a representative OsFIE1-overexpressed plant (Ubi::OsFIE1) at the heading stage.
(b) Relative expression of OsFIE1 in leaves of Ubi::OsFIE1 plants. Ubi::OsFIE1-1 and -2 are two independent transgenic events. Three biological replicates were used for analysis. Error bars indicated standard deviations.
(c) Elevation of H3K27me3 in the leaves of Ubi::OsFIE1.
(d) Morphology at the heading stage of a representative WT, and of a representative pOsFIE2::OsFIE1 plant, in which the OsFIE1 expression was driven by the OsFIE2 promoter.
(e) Relative expression of OsFIE1 in leaves of p O s F I E2::O s F I E1 plants. pOsFIE2::OsFIE1-1 and -2 are two independent transgenic events. Three biological replicates were used for analysis. Error bars indicated standard deviations.
(f) Scheme of the chimeric OsFIEs, achieved by swapping the extra N-terminal tail of OsFIE1 to OsFIE2. White and gray bars indicated OsFIE1 and OsFIE2 proteins, respectively.
(g) Confirmation of ectopic expression of the chimeric OsFIEa (Chi-OsFIEa) and Chi-OsFIEb in transgenic plants using quantitative real-time polymerase chain reaction (qRT-PCR) assay. 1-qRT and 2-qRT indicate the segments used to distinguish the OsFIE1- and OsFIE2-origin transcripts. Corresponding positions of 1- qRT and 2-qRT are indicated in (f). Three biological replicates were used; the error bars indicated standard deviations.
(h, i) Phenotypes of representative Chi-OsFIEa-1 (h) line and Chi-OsFIEb-1 (i) plants at the heading stage.

the phenotype was directly caused by the overexpression of OsFIE1 in seeds, or whether it was an indirect effect of ectopic expression of OsFIE1, reflecting the reduced stature and impaired vegetative growth (Figure 1a). To clarify the effects of OsFIE1 overexpression on seed development, we specifically activated its expression in the endosperm, using the promoter of the rice glutelin 1 gene (Russell and Fromm, 1997) to drive OsFIE1 (GT1::OsFIE1). The expression of OsFIE1 in the endosperm in independent transgenic lines was significantly increased, whereas the expression of OsFIE2 was not altered in the transgenic plants (Figure 2b). Unlike the Ubi::OsFIE1 plants, the GT1::OsFIE1 lines showed vegetative development similar to that of the WT (Figure 2c). However, all three independent GT1:: OsFIE1 transgenic lines analyzed produced seeds signifi- cantly smaller than the WT seeds (Figure 2a and Figure S6). The 1000-grain weight of GT1::OsFIE1 was reduced to 86â€“88% of the WT value (Figure S6d). The results unambiguously indicated that overexpression of OsFIE1 could directly affect seed development.

# Mutant osfie1 showed smaller seeds and increased preharvest sprouting

To investigate the function of OsFIE1, we generated independent CRISPR/Cas9 osfie1 mutant lines using different targets (1-T1 and 1-T2) (Figure 3a). osfie1-1 produces truncated proteins that caused by a 7-bp deletion at 1-T1; osfie1-2 also encoded a premature protein due to the frameshift caused by a 4-bp deletion at 1-T2. The vegetative growth of the osfie1 mutants was not distinguishable from that of the WT (Figure 3b). In line with the findings by Huang et al. (2016), our results showed that the seed size and seed weight of osfie1 were greatly reduced relative to the WT (Figure 3c), these effects being mainly associated with reduced seed width and thickness, rather than shorter seed length (Figure S7).


Figure 2. Overexpression of OsFIE1 resulted in reduced seed size. (a) Morphology of the seeds produced by the wild-type (WT), Ubi::OsFIE1 and G TÂ·Â·â‹¯O s F I E| plants. (b) Upregulation of OsFIE1, but not OsFIE2, in the caryopses of three G T7âˆµO s F I E7 transgenic lines at 6 days after fertilization. Three biological replicates for each line were used for the quantitative reverse transcriptionâ€“polymerase chain reaction analysis. Error bars indicate standard deviations. (c) Morphology of a representative WT and a representative G T::O s F I E: plant at the heading stage.


Figure 3. Phenotypes of the osfie1 mutants. (a) Illustration of the targets (1-T1 and 1-T2) used for generation of CRISPR/Cas9 mutants and the mutations of the mutants used in the present study. (b) Morphology of a representative wild-type (WT) plant and two osfie1 mutants at the heading stage. (c) Seed phenotype of WT and osfie1 mutants. Images from top to bottom are WT, osfie1-1 and osfie1-2, respectively. (d) Reduced dormancy of the osfie1 mutants in comparison with the WT. (e) Dynamic germination curves of WT, osfie1-1 and osfie1-2. Near-mature seeds (approximately 25â€“30 days after fertilization) produced by the main stem of each genotype were directly harvested for the assay. (f) Germination rate of different-aged seeds of WT, osfie1-1 and osfie1-2. Different-aged seeds (14â€“30 days after fertilization) were harvested for the germination assay. Germination rate was investigated at 7 days after imbibition.

Notably, we observed that osfie1 exhibited a signifi- cantly higher pre-harvest sprouting phenotype in the paddy field in each of two seasons (Figure 3d), which suggested that OsFIE1 might positively regulate dormancy in rice. To confirm this, we carried out germination tests on near-mature seeds (approximately 30 DAF) that had been directly collected from field without drying. More than 15% of the seeds of osfie1-1 and osfie1-2 geminated, but less than 7% WT seeds geminated after 7 days after imbibition (Figure 3e). Next, we used differently aged seeds of different genotypes in the germination test (Figure 3f). The results showed that osife1 had a significantly higher germination potential than WT in terms of the immature seeds, with more than 20% of the 23 DAF seeds germinating at

7 days after imbibition, which was significantly higher than that of the WT (Figure 3f).

# OsFIE2, but not OsFIE1, is required for cellularization of the endosperm

Previous studies had shown that OsFIE2 RNAi plants had a lower seed-set rate, with some of the seeds produced being abnormal (Nallamilli et al., 2013; Li et al., 2014), but the mechanism by which OsFIE2 affects seed development is still largely unknown. Because of the strong similarities between OsFIE1 and OsFIE2 (Figure S1c), we believed that it would not be a good strategy to use RNAi lines for the study of seed or endosperm development, owing to the overlap in terms of spatiotemporal expression of these two

Figure 4. osfie2 and osfie1,2 mutants were lethal. (a) Schematic drawing of the three targets (2-T1, 2- T2, and 2-T3) that were used for generation of CRISPR/Cas9 mutants of osfie2, and the genomic sequence of OsFIE2 in wild type (WT), osfie2 and osfie1,2.
(b) Cumulative percentage of the not-edited (WT), monoallele-edited (Hetero) and diallele-edited homozygous (Homo) individuals that were regenerated from Agrobacterium-mediated transformation at T_0 generation. The number of individuals of each genotype are indicated on the bars. (b) Caryopses collected from a single panicle of an O s F I E2^+- plant at 15 days after fertilization (DAF). Two classes of caryopsis were observed: Class 1 that produced starchy endosperm, and Class 2 that produced watery endosperm.
(c) Cumulative percentage of well-filled, unfilled and unfertilized seeds produced by WT (n=985) ,osfie1-1 ;m=1768) , osfie1-2 [n=1 1 15] ,O s F I E2^+- (n=4772) and osfie1 /O s F I E2^+- .n=3626) .



genes in seeds (Figure S4c). Therefore, we made efforts to generate osfie2 mutants using a CRISPR/Cas9 approach (Figure 4a). Unfortunately, despite the use of three different targets (2-T1, 2-T2 and 2-T3), our attempts to obtain callus-regenerated homozygous mutants of OsFIE2 at the T_0 generation failed, although heterozygous O s F I E2^+- plants were readily obtained (Figure 4b). In contrast, we could obtain homozygous mutants of O s F/Eâ†‘ at a high frequency (approximately 30% ) at the T_0 generation (Figure 4b). In addition, we used a single construct with a cassette consisting of both 1-T2 and 2-T2, attempting to knockout O s F/Eâ†‘ and OsFIE2 simultaneously. We succeeded in obtaining the osfie1 mutant in the homozygous condition at the OsFIE1 locus, but again only heterozygous (osfie 1/O s F/E2^+-) were obtained at the OsFIE2 locus (Figure 4b). According to these findings, we inferred that OsFIE2, but not O s F I E1, is indispensable for rice regeneration. Alternatively, OsFIE2 is more essential for development, because the transformants lacking functional OsFIE2 were not viable. In the subsequent experiments, we used an O s F I E2^+- line with a mutant allele caused by the insertion of an additional "âŠº" at 2-T1 (Figure 4a). The osfie1/ O s F I E2^+- line used for analyses has a âˆ‡^'Î¦^'' inserted at 2-T2, while there has a 4-bp deletion at 1-T2 (Figures 3a and 4a). All these mutations resulted in truncated proteins due to frameshift.

In a segregating population from self-pollinated O s F I E2^+- , only O s F I E2^+ and O s F I E2^+- plants were obtained (Figure S8a). After 5 DAF, the seeds produced by O s F I E2^+- could readily be categorized into two seed classes (Figure 4c). Seeds of Class 1 showed an appearance similar to that of the WT, while seeds of Class 2 showed growth cessation frequently, illustrated with a swollen belly consisted watery endosperm (Figure 4c). At maturity, the Class 2 seeds were completely empty (Figure S8b). In agreement with this observation, the seed-set rate of O s F I E2^+- and osfie1 /O s F I E2^+ was significantly lower than that of WT or osfie1 (Figure 4d). In particular, the rate of unfertilized seeds, which failed to show enlargement of the caryopses, was significantly increased in O s F I E2^+- (16.5%) and osfie â„/O s F I E2^+- (5.1%) in comparison with that of the WT (1.7%) . This was likely not caused by lower viability of the male or female gametes. Because the development of pollens and embryo sacs seemed normal in O s F I E2^+- and osfie 1/O s F I E2^+- (Figure S9aâ€“h).

The Class 1 seeds produced a starchy endosperm at 15 DAF, whereas the Class 2 seeds produced a watery endosperm instead at this time point (Figure 4c). Because we had failed to obtain the osfie2 mutant, we believed that the Class 2 seeds were the homozygotes. The Class 1 seeds were morphologically different from the Class 2 seeds after 4â€“5 DAF. Early endosperm development of Class 2 seeds was impaired (Figures 5aâ€“l and S10). In caryopses of WT and osfie1 at 4 DAF, cellularized endosperm cells completely filled the embryo sac (Figure 5i,j), but only a few layers of endosperm cells were observed in osfie2 or osfie1,2 caryopses at this time point (Figure 5k,l), which indicated that cellularization of the Class 2 seeds was delayed or impaired. At 7 DAF, most of the osfie2 seeds still had an empty vacuole. A few seeds had multiple layers of loosely packed endosperm cells (Figure S5m,n), but as revealed by l_2 -KI staining assay, much less starch had accumulated in these cells compared with WT seeds (Figure 5mâ€“p). In line with this observation, expression of some key genes involving starch biosynthesis, including OsISA2, OsAGPL2, OsPUL, OsGBSSI, RPBF and RISBZ1, were substantially downregulated in osfie2 and osfie1,2 seeds at 6 DAF (Figure 5qâ€“v). On the other hand, expression of OsAPGL1 and a negative regulator gene of starch biosynthesis, RSR1, were upregulated (Figure 5w,x). The |_2 -KI staining experiment showed that starch accumulation was not affected in the seed coat of osfie2 (Figure 5o,p), suggesting that the development of the seed coat and integument were probably not strongly impaired. Notably, early endosperm development of osfie1 did not seem to be affected (Figure 5b,f,j). Together, the results suggested that OsFIE2, but not OsFIE1, is essential for early endosperm development in rice.


type (WT), osfie1, osfie2 and osfie1,2. (aâ€“l) Cross-sections of 2 days after fertilization (DAF) (aâ€“d), 3 DAF (eâ€“h), and 4 DAF (iâ€“l) endosperm of WT (a,e,i), osfie1-1 (b,f,j), osfie2 (c,g,k) and osfie1,2 (d,h,l). Sections were stained with 0.1% Toluidine Blue. Scale bars: 200Â m . (mâ€“p) |_2^-ð–ª ð–¨ staining of 7 DAF endosperm of WT (ð—†,ð—ˆ) and osfie2 (n,p). (o) and (p) Magnified images of the boxed regions in (ð—†) and (n), respectively. Scale bars: 100Â m in (ð—†,ð—‡) ,200Â m in (o,p). 'ð–¯ ð–¾^' and â€™enâ€™ indicate pericarp and endosperm, respectively, in (m,n). (e,f) Relative expression of key genes involved in starch biosynthesis in the 10 DAF caryopses of WT, osfie1, osfie2 and osfie1,2. Corresponding gene names are given under the bars of each figure. (ð—Š_-ð–·) Expression of some key genes involved in starch biosynthesis. Three biological replicates were used; error bars indicated standard deviations.

The embryo development of neither osfie2 nor osfie1,2 was significantly affected at 4 DAF, relative to the WT. As with the WT, globular embryos could be observed in the mutants at 4 DAF (Figure S11aâ€“c). However, embryos were not still observable for many osfie2 seeds at 15 DAF (Figure S11d,e). The embryos had probably degenerated by this point. We found embryos in two of the five osfie1,2 seeds we studied, but the development of these had already ceased (Figure S11f). The results suggested that the embryogenesis of osfie2 and osfie1,2 was arrested, as was endosperm development.

# Autonomous endosperm production was occasionally observed in osfie2 mutants

Mutation of FIE in Arabidopsis can lead to autonomous endosperm production without fertilization (Ohad et al., 1996; Chaudhury et al., 1997; Kiyosue et al., 1999). However, whether OsFIE1 and OsFIE2 play similar roles in the repression of ovary proliferation in rice is not clear. In the present study, WT, osfie1, O s F I E2^+- and osfie 1/O s F I E2^+- panicles were emasculated before flowering and bagged to prevent cross-pollination. No autonomous seeds were produced by the 162 emasculated spikelets of WT and the 76 emasculated spikelets of osfie1 (Figure 6a). However, O s F I E2^+- and osfie1 /O s F I E2^+- were able to produce enlarged seeds at a very low frequency (2.6% of O s F I E2^+- and 4.9% of osfie 1/O s F/E2^+-) (Figure 6a). The seeds were completely empty when dried due to defects of starch accumulation (Figure 6b,c). Furthermore, to confirm the enlarged seeds are autonomous, we performed confocal laser scanning microscopy observations. In contrast to the selfing control, the embryo was not observable in the enlarged seeds of emasculated osfie2 at 4 days after emasculation (Figure 6d,e). However, we could find nuclear division of the central cell in these seeds forming a syncytium (Figure 6e,f). Autonomous endosperm cells were more apparent in most of the enlarged but embryoless seeds produced by the emasculated osfie2 at 15 days after emasculation (Figure 6g).


Figure 6. Autonomous fertilization of O s F I E2^+- and osfie1/OsFIE2 . (a) Cumulative percentages of the non-fertilized and enlarged seeds in wild type (WT), osfie1-1, osfie1-2, O s F I E2^+- and osfie1 1/O s F I E2^+- . Number of each type of seed is indicated on the bars. (b,c) Morphology of dried autonomous seeds produced by O s F I E2^+- (b) and osfie1 /O s F I E2^+- (c). (d) Confocal laser scanning microscopic observation of an artificially fertilized osfie2 caryopsis at 4 days after pollination. Red arrowhead shows the embryo. Scale bar: 100Â m (eâ€“g) Confocal laser scanning microscopic observation of the autonomous endosperm of emasculated osfie2 caryopses at 4 days after emasculation (DAE) (e,f) and 10 DAE (g). Yellow arrows in (e) indicate the divided nuclei of a proliferated central cell. No embryo was observable in an autonomous seed. Red arrowhead indicate the corresponding region of a fertilized seed. (f) Magnified image of the boxed region in (e) Î¨^Î¼ðšðž^'' in (g) indicate the autonomous endosperm cells at 15 DAE. No embryo was observable in the embryo sac. Scale bars: 100Â m .

# Additive effects of OsFIE1 and OsFIE2 on gene expression

To dissect how OsFIE1 and OsFIE2 function in seed development, we carried out RNA-sequencing (RNA-Seq) to discover differentially expressed genes (DEGs) from osfie1, osfie2 and the osfie1,2 double mutant. In caryopses at 6 DAF, using the cutoff thresholds of fold-change ^>2 and false discovery rate (FDR) <0.05_ 112 and 701 genes were down- and upregulated, respectively, in osfie1 when compared with WT (Figure 7a,b). The number of DEGs was much higher in osfie2 and osfie1,2. In total, 1110 downregulated DEGs and 6175 upregulated DEGs were identified from osfie2, whereas 1152 downregulated DEGs and 5826 upregulated DEGs were identified from osfie1,2 (Figure 7a,b). Most of the DEGs were upregulated relative to the WT (Figure 7a,b), a finding that was consistent with the functions of OsFIE1 and OsFIE2, which are involved in the H3K27me3 modification of chromatin.

Most of the DEGs identified from osfie2 and osfie1,2 overlapped (Figure 7a,b). MapMan analysis suggested that genes related to storage protein biosynthesis, lipid metabolism, cell wall synthesis and DNA synthesis were enriched (Benjaminiâ€“Hochberg corrected FDR <0.05âŸ© in osfie2 and osfie1,2 (Figure 7c). This result was in line with observations that cellularization of osfie2 was delayed and starch was not accumulated in osfie2 endosperm (Figure 5n,p). Notably, several groups of transcription factor, such as homeobox, MADS box, WRKY and C2H2 CO-like zinc finger genes, were also enriched among the DEGs (Figure 7c), indicating that these transcription factors play essential roles in early seed development of rice. Several type I MADS-box genes of Arabidopsis were found to act as negative regulators of cellularization (Portereiko et al., 2006; Kang et al., 2008). Notably, many MADS-box genes in rice, including MIKC-type and type I genes were signifi- cantly upregulated in osfie1 and osfie1,2 seeds (Figure S12). These genes may also be involved in the early endosperm development of rice.

Compared with osfie2 and the osfie1,2 double mutant, osfie1 showed fewer effects on expression in seeds early in development (Figure 7a,b), which further supported our hypothesis that O s F/Eâ†‘ is less important than OsFIE2 for early seed development. Gene Ontology analysis revealed that genes involved in photosynthesis (ð–¥ ð–£ ð–±=9.4ð–¾-05) , cell wall organization or biogenesis (ð–¥ ð–£ ð–±=9.4ð–¾-05) and response to stimuli (ð–¥ ð–£ ð–±=0.0032) were significantly enriched among the DEGs identified from osfie1. About 85.7% of the downregulated DEGs and 63.5% of the upregulated DEGs identified from osfie1 also showed differential Figure 7. Transcriptome analysis of caryopses 6 days after fertilization (DAF) of wild type (WT), osfie1, osfie2 and osfie1,2.



(a,b) Venn diagrams of the upregulated (a) and downregulated (b) genes identified from osfie1, osfie2 and osfie1,2 in comparison with WT. (c) MapMan pathway enrichment analysis of the differentially expressed genes. Circle size and colors indicate the log scale of the enrichment. (d) Heatmap of the expression of differentially expressed genes common to osfie1, osfie2 and osfie1,2. Gene expression was indicated by log2 (FPKM). (e,f) Additive effects of OsFIE1 and OsFIE2 on expression of storage protein biosynthesis-related genes (e) and photosynthesis-related genes (f). Expression of genes are indicated by FPKM values. CoA, coenzyme A; FA, fatty acid; FPKM, fragments per kilobase of transcript per million fragments mapped.

expression in osfie2 or osfie1,2 (Figure 7a,b), suggesting that these genes are probably common targets regulated by OsFIE1 and OsFIE2.

Intriguingly, we found that the DEGs common to osfie1, osfie2 and osfie1,2 were additively regulated by OsFIE1 and OsFIE2. Overall, mutation of OsFIE2 led to a larger effect than mutation of OsFIE1, whereas the osfie1,2 double mutant showed the greatest change in expression (Figure 7d). For example, among the common downregulated DEGs (68 in total), 23 were storage-protein-precursor encoding genes (Table S2). These genes were highly expressed in the WT, but were greatly suppressed in the mutants, with the degree of suppression being in the order osfie1<osfie2<osfie1,2 (Figure 7e and Table S2). In contrast, 17 genes related to photosynthesis were upregulated in the mutant lines, which overall showed greatest transcript abundance in osfie1,2 and moderate upregulation in osfie2, while osfie1 showed the least upregulation (Figure 7f and Table S2). Taken together, these findings suggested that OsFIE1 plays only a limited function in early seed development. However, OsFIE1 can coordinate with OSFIE2 in an additive manner to regulate most of the DEGs that were identified from osfie1.

# DISCUSSION

The components of PRC2 are conserved between animals and plants. For animals, there are usually only one or two members of each of the PRC2 groups, p55, E(z), ð–² ð—Ž(ð—“)12 and ESC. However, plants usually have multiples of each group (Furihata et al., 2016). For example, most of the monocots we analyzed in the present study had two or more FIE genes (Figure S1b). The FIEs of the different graminaceous genera probably evolved because of independent duplication events. A duplication that occurred after the origin of the tribe Oryzeae generated two FIE orthologs, OsFIE1 and OsFIE2, in rice. The phylogenetic and alignment analyses suggested that OsFIE2 is the more ancient gene (Figure S1b) because it is more similar to the FIEs of other grass species. OsFIE2 could promote cellularization such as AtFIE acts in Arabidopsis (Vinkenoog et al., 2000), but OsFIE1 showed only limited effects in early endosperm development in rice (Figure 5aâ€“l). The expression pattern and the methylation landscape of OsFIE1 and OsFIE2 were quite distinct (Figure S4), which suggested that the genes had probably diverged since they evolved. OsFIE2 is universally expressed, but it is highly methylated in the promoter region when compared with OsFIE1 (Figure S4b,d,e). Usually, methylation acts as a repression mark for gene expression (Deng et al., 2016), whereas there have been findings suggesting that hypermethylation may also be involved in upregulation of certain genes in crops (Lang et al., 2017). Expression of OsFIE1 is restricted to the endosperm (Figure S4a), possibly due to its deleterious effects when ectopically expressed in vegetative tissues (Figure 1a). Previous studies had found that overexpression of OsFIE1, but not OsFIE2, could lead to dwarfism (Zhang et al., 2012; Folsom et al., 2014). In the current study, we found that the adverse effects caused by OsFIE1 were dose dependent, with moderate upregulation of OsFIE1 in the seedling not resulting in developmental defects (Figure S1d,e). So far, we still do not know why OsFIE1, but not OsFIE2, is harmful to vegetative growth. The â€œgene swappingâ€ experiments suggested that the extra N-terminal tail of OsFIE1 only was not able to induce the deleterious effects (Figure 1fâ€“i). The alignment analysis indicated that, in addition to the N-terminal difference, OsFIE1 has an extra 8-amino acid segment at the C-terminal end (Figure S1c). This difference, as well as the amino acid substitutions found in OsFIE1, may contribute to the deleterious effects.

The osfie1 mutant produced smaller seeds and showed increased frequency of pre-harvest sprouting relative to the WT (Figure 3), suggesting that OsFIE1 plays roles in late seed development (Figure S13). In support of this hypothesis, many DEGs identified in osfie1 were responsible for storage protein biosynthesis (Figure 7c and Table S1). Interestingly, OsEMF2b, a member of the ð–² ð—Ž(ð—“) 12 group in rice, has recently been found to function in seed dormancy by modulating the expression of O s V P7 (Chen et al., 2017). Therefore, OsFIE1 and OsEMF2b are possibly recruited into the same PRC2 that regulates seed dormancy and germination. In agreement, many regulators of late seed development and germination of Arabidopsis, such as LEAFY COTYLEDON 2, ABSCISIC ACID INSENSITIVE 3, FUSCA 3, ABSCISIC ACID INSENSITIVE 4 and DELAY OF GERMINATION 1, were upregulated in fie mutant seedlings, possibly due to depletion of H3K27me3 of these genes (Bouyer et al., 2011). The mechanism by which OsFIE1 regulates dormancy in rice need to be further investigated. Because we failed to obtain an osfie2 mutant, it is unclear whether OsFIE2 functions on dormancy as well. However, PRC2-regulated seed dormancy and germination have been reported in monocots and dicots, suggesting that this function of PRC2 is conserved in plants. Taking into account that OsFIE2 is probably the more ancient ortholog (Figure S1b), we believe that OsFIE2 functions in late seed development as well (Figure S13).

Evidence from the present study suggested that OsFIE2 is essential for early endosperm development via the promotion of cellularization (Figure 5 and Figure S10). In Arabidopsis, FIS2-PRC2 can regulate a group of type I MADSbox genes in early endosperm development (Koâ‚¬hler et al., 2003; Figueiredo et al., 2015; Zhang et al., 2018). Seeds with dysfunctional MADS genes, such as AGAMOUS-LIKE 62 (AGL62) and AGL80, may be aborted due to cellularization defects (Portereiko et al., 2006; Kang et al., 2008). Type I MADS-box genes probably act as negative regulators of cellularization (Pires, 2014), which is opposite to the role that FIE2-PRC2 plays. We previously found that the type I MADS-box genes in rice were associated with syncytium development; their expression being substantially suppressed during cellularization (Chen et al., 2016a). In the present study, we found that expression of many MADSbox genes was disrupted in osfie2, whereas osfie1 displayed limited impacts on these genes (Figure 7c and Figure S12). The results suggested that OsFIE2-PRC2 might suppress the expression of these MADS-box genes, to promote cellularization in rice.

Plants that overexpressed OsFIE1 showed more H3K27me3 marks on the chromatin (Figure 1c), indicating that, as well as OsFIE2 (Nallamilli et al., 2013), OsFIE1 exhibits methyltransferase activity in vivo. However, several lines of evidence revealed in the current study suggested that OsFIE1 is probably not as essential for rice development as is OsFIE2. First, in an evolutionary scenario, OsFIE2 had undergone a purifying selection that was more intensive than OsFIE1, as indicated by the lower dN/dS of OsFIE2 in comparison with OsFIE1 (Figure S3). Secondly, a mutation in OsFIE1 had only mild effects on seed development. Cellularization in osfie1 was not affected, whereas it was significantly delayed in osfie2 (Figure 5 and Figure S10). Thirdly, transcriptome analysis revealed that, for the common DEGs identified from osfie1, osfie2 and the double mutant, additive effects were observed for the expression of the genes (Figure 7dâ€“f). The osfie1,2 double mutant usually showed the greatest effects, whereas osfie2 showed only a moderate interference. Expression of these DEGs was affected to a lesser extent when compared with osfie2 or osfie1,2. Finally, yet importantly, we could readily obtain homozygous mutants of osfie1 from the callus-regenerated plants at the T_0 generation (Figure 4b), but osfie2 homozygotes were not available by the same approach, probably due to its lethality. The results suggested that OsFIE2 are indispensable for regeneration and vegetative growth of rice. Previous studies had found that OsFIE2 RNAi plants showed phenotypes such as short stature and low seed setting (Nallamilli et al., 2013; Liu et al., 2016), which are similar to the phenotypes exhibited by OsFIE1 overexpressors (Zhang et al., 2012; Folsom et al., 2014). Therefore, we assumed that OsFIE1 and OsFIE2 might compete with each other for members of PRC2. Overexpression of OsFIE1 may increase the malfunctional OsFIE1-PRC2 but decrease the functional FIE2-PRC2 in plants.

Autonomous endosperm has been observed in FIS2- PRC2 mutants (Ohad et al., 1996; Chaudhury et al., 1997; Kiyosue et al., 1999). However, inconsistent findings have been reported, regarding whether rice PRC2 mutants are able to induce autonomous endosperm (Nallamilli et al., 2013; Li et al., 2014). Our findings suggested that osfie2 may cause autonomous endosperm, but at a low frequency (Figure 6). About 2.6% of the emasculated O s F I E2^+- produced autonomous endosperm. This rate was slightly higher in osfie 1/O s F I E2^+ (approximately 4.9% ). If the frequency of the autonomous endosperm was all contributed by the osfie2 haploid (representing half of the female gametes produced by a heterozygote), the frequency of autonomous endosperm by osfie2 homozygotes would be no more than 10% in rice. This is much lower than that observed in AtFIE of Arabidopsis. Nearly 50% of female gametes produced by an FIE/fie heterozygote had multinucleate central cells at 6 days after the anthers were removed (Ohad et al., 1996), but in a genetic backgrounddependent manner (Roszak and Kohler, 2011). The findings suggested that, in addition to the PRC2 complex, suppression of proliferation of the central cell before fertilization requires other regulators in rice.

# EXPERIMENTAL PROCEDURES

# Plant materials and growth conditions

The rice (Kitaake, O. sativa ssp. japonica) plants were grown in paddy fields in Yangzhou, Jiangsu Province, China, during the summer season, and in paddy fields in Lingshui, Hainan Province, during the winter season. The plants were managed with the addition of regular amounts of water and nutrients, according to local farming practices. The seeds within the spikelets were labeled with marker pens with respect to the date of anthesis. Differentaged caryopses, along with other tissues, were collected for subsequent experiments.

# Phylogenetic analysis and calculation of dN/dS

The FIE homologs of monocots were identified through BlastP searching, using OsFIE1 and OsFIE2 protein sequences as the queries to search against the protein database of EnsemblPlants (http://plants.ensembl.org/index.html). The homologs obtained were aligned by Clustal Omega (https://www.ebi.ac.uk/Tools/ msa/clustalo/). The aligned sequences were submitted to MEGA7 for phylogenetic analysis using the maximum likelihood method (Kumar et al., 2016). The ð–ª ð–º/ð–ª ð—Œ program of the TBtools package (Chen et al., 2018b) was used to calculate dN/dS. Leersia perrieri was used as an outgroup. Coding sequences were used for the calculations.

# Vector constructions and transformation

A gene SOEing (Splicing by Overlap Extention) approach (Horton et al., 2013) was applied to amplify ChiFIE1, ChiFIEb and pOsFIE2:: OsFIE1. The coding sequence of OsFIE1 and the interchanged OsFIEs were cloned into pENTR/D-TOPO entry vector (Invitrogen, Carlsbad, CA, USA) and then recombined into the destination vector, pANIC6A, for the Ubi::OsFIE1 and Chi-OsFIEs constructs. The reactions were carried out following the instructions of the manufacturer of the LR Clonase Kit (Invitrogen, Carlsbad, CA, USA). The coding sequence of OsFIE1 was cloned into pCAMBIA1300-GT1 to obtain the OsGT1::OsFIE1 construct using the ClonExpress II One Step Cloning Kit (Vazyme, Nanjing, China). The promoter region of OsFIE2 and the coding sequence of OsFIE1 were simultaneously cloned into pCAMBIA1300 for the pOsFIE2::OsFIE1 construct using the ClonExpress II Multi step Cloning Kit (Vazyme, Nanjing, China). The targets for CRISPR/Cas9 constructs were cloned into BGK032 using a CRISPR/Cas9 vector constructing kit (Biogle, Changzhou, China). The primers used for vector constructions are listed in Table S3. The constructs were then transformed into the embryo-induced calli of rice. The methodologies were as described in our previous report (Chen et al., 2016a).

# DNA extraction and bisulfite sequencing

Genomic DNA was isolated from seedling leaves and 6 DAF endosperm of WT rice using the EasyPure Plant Genomic DNA Kit (Transgen, Beijing, China). Bisulfite sequencing was performed as described in our previous report (Chen et al., 2016a). Briefly, 1 ð”¤ of genomic DNA was treated with sodium bisulfite using an EZ DNA Methylation Kit (Zymo, Irvine CA, USA). The treated DNA was used as the templates to amplify the OsFIE1 and OsFIE2 segments (150â€“300 bp; primers are listed in Table S3). We then cloned the PCR products into a pEAZY-T3 cloning vector (Transgen). For each amplicon, at least 10 colonies were sequenced. The sequencing data were submitted to Kismeth for bisulfite analysis (Gruntman et al., 2008).

# RNA extraction and real-time PCR assay

Total RNA was extracted from different tissues of rice â€œKitaakeâ€ using Plant RNA Kit (Omega). First-strand complementary DNA was synthesized by HiScript Q-RT SuperMix for quantitative PCR (qPCR; Vazyme) using the oligo-dT primer. Quantitative real-time qPCR was performed according to the manufacturerâ€™s instructions of the AceQ qPCR SYBR Green Master Mix (Vazyme), using the CFX Connect Real-Time PCR Detection System (Bio-Rad, Hercules, CA, USA). Three biological replicates were set up for each experiment. The primers used in the experiment are listed in Table S3.

# RNA-Seq and differential expression analysis

RNA isolated from caryopses of the WT, osfie1, osfie2 and osfie1,2 at 6 DAF was used for RNA-Seq. We set up three biological replicates for WT and osfie1, and two biological replicates for osfie2 and osfie1,2. The samples were submitted to Novogene Co. Ltd (Tianjin, China) for library preparation and sequencing. Briefly, 1.5 g high-quality RNA per sample was used for library preparation using NEBNext Ultra  RNA Library Prep Kit, and index codes were added to each sample. Library quality was assessed on the Agilent Bioanalyzer 2100 system (Santa Clara, CA, USA). The clustering of the index-coded samples was performed on a cBot Cluster Generation System using the TruSeq PE Cluster Kit v3-cBot-HS (Illumina, San Diego, CA, USA). The libraries were sequenced using the Illumina Nova 6000 platform.

The raw fastq reads were processed through in-house perl scripts to remove adapters and low-quality reads. The clean reads were imported into the CLC Genomics Workbench 12.0 and mapped to the reference genome using global alignment mode with the default mapping parameters. The expression values, including total counts, unique counts, transcripts per million and reads per kilobase of transcript per million reads mapped, were calculated for the identification of DEGs, using the threshold foldchange ^>2 and FDR <0.05 .

# Nuclear protein extraction and Western blotting

The leaves from WT and Ubi:OsFIE1 were finely ground into a powder with liquid nitrogen. The nuclear protein extraction buffer 120Â ð—† ð—† Tris-HCl [pH 8.0], 300Â ð—† ð—† NaCl, 0.4Â ð–¬ sucrose, 10Â ð—† ð–¬ ð–¬ ð—€ ð–¢ ð—…_2, 1 mM dithiothreitol, 1 mM phenylmethylsulfonyl fluoride and 5 mM 2-mercaptoethanol) was added with the ratio leaves (g)/buffer (ð—†|) of 1:20 for histone protein extraction. The extracted mixture was sequentially precipitated by sulfuric acid solution (0.25Â ð–¬) , trichloroacetic acid (90% 0.25 M sulfuric acid, 10% trichloroacetic acid) and precooled acetone. After removing the supernatant and drying the pellet, an appropriate amount of phosphate-buffered saline Â·(ð—‰ ð–§  7.0) was added to resuspend the pellet.

The nuclear proteins were used for Western blot analysis. Antibodies used in this study were anti-H3K27me3 (Merck Millipore, Temecula, CA, USA) and anti-H3 (Abcam, Cambridge, MA, USA). Two independent experiments were performed with two biological replicates for each sample.

# Sectioning and microscopy

The caryopses were fixed and infiltrated in formaldehyde acetic acid solution, and stored at 4^âˆ˜âˆ for use. Samples were dehydrated through a graded ethanol series, infiltrated with xylene, embedded in resin, sectioned at 2.5Â M and then stained with 0.1% Toluidine Blue. The embryo sac and embryogenesis observations were performed by a previously described method with some modifications (Huang et al., 2017). Briefly, the caryopses were fixed in formaldehyde acetic acid overnight. After replacing the solution with 70% ethanol, the samples were incubated overnight, subjected to an ethanol series and washed with 2% aluminum potassium sulfate for 30 min . The samples were then stained with 10Â ð—† ð—€/ð–« eosin B (Sigma, Saint Louis, MO, USA) in 4% sucrose for 16Â ð”¥ at room temperature. Next, the samples were washed with 2% aluminum potassium sulfate for 30 m i n, ,dehydrated by an ethanol series and then washed twice with absolute ethanol for 60 ð—† ð—‚ ð—‡ . After dehydration, the samples were treated with a methanolâ€“methyl salicylate series (4:1, 3:1, 2:1, 1:1, 1:2, 1:3 and 1:4  [ð—/ð—] each for 1 h) , washed three times with methyl salicylate for â†‘ð—_ð— , kept at 4^âˆ˜âˆ overnight in methyl salicylate and observed by a confocal laser microscope (Zeiss, Jena, Germany; LSM710).

# Germination assay

To test the germination of near-mature seeds, the seeds (approximately 25â€“30 DAF) produced by the main stem were collected. The panicles were put into a horizontally placed germination bag in a growth chamber (14-h day and 10-h night settings, 25^âˆ˜ð–¢/20^âˆ˜ð–¢ day/night temperature). The germination rate was calculated every day. To test the germination ability of different-aged seeds, spikelets were marked at the day anthesis, different-aged seeds were harvested and then put into a germination bag in a growth chamber. The germination rate was investigated at 7 days after imbibition.

# Assessment of the frequency of autonomous endosperm

The spikelets were emasculated 1 day before flowering. A pollination bag was used to cover the emasculated panicle in case of cross-pollination. If a caryopsis was enlarged without pollination at 25 days after the anthesis removed, we regard it as an autonomous endosperm.

# ACKNOWLEDGEMENTS

This work was supported by National Key R   D Program of China (2016YFD0100902), National Natural Science Foundation of China (31771744, 31571623 and 31701392), Science Fund for Distinguished Young Scholars of Jiangsu Province (BK20180047) and the Priority Academic Development of Jiangsu Higher Education Institutions. The authors thank Dr. Dongping Zhang for critical reading of the manuscript.

# CONFLICT OF INTERESTS

The authors declare that they have no competing interests.

# AUTHOR CONTRIBUTIONS

CC conceived the project; CC, BN, XC and MP designed the research; XC, MP, YZ and BN performed the experiments; CC, XC, MP and ZE analyzed the data; CC wrote the manuscript.

# DATA AVAILABILITY STATEMENT

All relevant data can be found within the manuscript and its supporting materials.

# SUPPORTING INFORMATION

Additional Supporting Information may be found in the online version of this article.

Figure S1. Evolutionary analyses of FIE homologs of rice.
Figure S2. Alignment of the N-terminus of OsFIE1 and OsFIE2 homologs.
Figure S3. Whisker plot showing dN/dS of different FIE1 and FIE2 homologs in the genus Oryza.
Figure S4. Expression profiles and DNA methylation landscapes of OsFIE1 and OsFIE2.
Figure S5. Relative expression, tiller numbers and plant height of the Ubi::OsFIE1 and chimeric OsFIE lines.
Figure S6. Brown seed size of the GT1::OsFIE1 lines.
Figure S7. Brown seed size of the osfie1 mutants.
Figure S8. Lethality of osfie2.
Figure S9. Viability of pollen grains and embryo sac of WT, osfie1, osfie2 and osfie1,2.
Figure S10. Early endosperm development of the caryopses of WT, osfie1, osfie2 and osfie1,2.
Figure S11. CLSM observation of the embryo development of WT, osfie2 and osfie1,2.
Figure S12. Relative expression of some MADS-box genes in the endosperm of WT, osfie1, osfie2 and osfie1,2.
Figure S13. Functional divergence between OsFIE1 and OsFIE2 with respect to seed development of rice.
Table S1. Differentially expressed genes identified from osfie1, osfie2 and osfie1,2.

Table S2. Additive effects of OsFIE1 and OsFIE2 on storage protein and photosynthesis-related gene expression.

Table S3. Primes used in the study.
